<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/header-attrs-2.12/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.21/datatables.js"></script>
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.5.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-12
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>MM_44/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220412code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220412)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220412code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220412)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongef7d0d7">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> ef7d0d7
</a>
</p>
</div>
<div id="strongRepositoryversionstrongef7d0d7"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version ef7d0d7.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/results_table_OOPS_CDDP_vs_OOPS_NT.xlsx
    Untracked:  output/results_table_OOPS_CDDP_vs_OOPS_NTno_imp.xlsx
    Untracked:  output/results_table_WCE_CDDP_vs_WCE_NT.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/ANOVA.Rmd</code>) and HTML
(<code>docs/ANOVA.html</code>) files. If you’ve configured a remote Git
repository (see <code>?wflow_git_remote</code>), click on the hyperlinks
in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
ef7d0d7
</td>
<td>
FabioBedin
</td>
<td>
2022-04-12
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)
library(&quot;GGally&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- expdesign %&gt;% filter(!str_detect(condition, &quot;LPS&quot;))
# expdesign &lt;- expdesign %&gt;% filter(!str_detect(condition, &quot;LPS&quot;)) 
# 
# expdesign &lt;- expdesign %&gt;% filter(!label == &quot;BCR4ALPS4h&quot; &amp; !label == &quot;BCR4ALPS2h&quot; )
# 
# expdesign &lt;- expdesign %&gt;% filter(!label == &quot;BCR4ALPS1h&quot; &amp; !label == &quot;BCR1AUT&quot; &amp; !label == &quot;BCR4ALPS30min&quot;)

#expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<hr />
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the
<strong>proteinGroups</strong> obtained by <a
href="https://www.maxquant.org/">MaxQuant</a> software, I remove
proteins that are marked form the software as potential contaminant,
only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins
identified by less than 0 peptides.<br />
Now, we generate some quality control plots to investigate the structure
of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “VSN normalization is used”</p>
<pre class="r"><code>vettore &lt;- c(&quot;ADAMTS1&quot;, &quot;AGRN&quot;, &quot;AHCTF1&quot;, &quot;AHCY&quot;, &quot;ANKRD17&quot;, &quot;ANXA3&quot;, &quot;ANXA5&quot;, &quot;ANXA7&quot;, &quot;AP3S1&quot;, &quot;APLP2&quot;, &quot;APP&quot;, &quot;ARHGDIA&quot;, &quot;ATL3&quot;, &quot;ATP6V1H&quot;, &quot;AXL&quot;, &quot;BMS1&quot;, &quot;BPTF&quot;, &quot;BYSL&quot;, &quot;C15orf52&quot;, &quot;C5orf24&quot;, &quot;C9orf114&quot;, &quot;CAPRIN1&quot;, &quot;CCAR2&quot;, &quot;CCDC59&quot;, &quot;CENPF&quot;, &quot;CEP170&quot;, &quot;CFL2&quot;, &quot;CHAMP1&quot;, &quot;CHERP&quot;, &quot;CHMP1A&quot;, &quot;CHST3&quot;, &quot;CLIC4&quot;, &quot;CLUH&quot;, &quot;CNN2&quot;, &quot;COIL&quot;, &quot;COL18A1&quot;, &quot;COL1A1&quot;, &quot;CPSF4&quot;, &quot;CSTF2&quot;, &quot;CSTF2T&quot;, &quot;CYR61&quot;, &quot;DCAF13&quot;, &quot;DDX10&quot;, &quot;DDX17&quot;, &quot;DDX24&quot;, &quot;DDX27&quot;, &quot;DDX47&quot;, &quot;DDX5&quot;, &quot;DDX52&quot;, &quot;DDX54&quot;, &quot;DDX56&quot;, &quot;DDX6&quot;, &quot;DGCR8&quot;, &quot;DNAJC10&quot;, &quot;DNAJC5&quot;, &quot;DRG1&quot;, &quot;DYNLL2&quot;, &quot;EBNA1BP2&quot;, &quot;EEF1B2&quot;, &quot;EIF4E&quot;, &quot;EIF4ENIF1&quot;, &quot;EIF5A&quot;, &quot;EPS15&quot;, &quot;FAM208A&quot;, &quot;FAM98A&quot;, &quot;FRAS1&quot;, &quot;FSTL1&quot;, &quot;FTSJ3&quot;, &quot;FXR2&quot;, &quot;GLG1&quot;, &quot;GNL2&quot;, &quot;GPC6&quot;, &quot;GRPEL1&quot;, &quot;GSTP1&quot;, &quot;HCCS&quot;, &quot;HDLBP&quot;, &quot;HIRIP3&quot;, &quot;HMGB1&quot;, &quot;HNRNPLL&quot;, &quot;HNRNPM&quot;, &quot;HSPG2&quot;, &quot;IGF2BP3&quot;, &quot;IGFBP3&quot;, &quot;IGFBP6&quot;, &quot;IGFBP7&quot;, &quot;IL6ST&quot;, &quot;INO80E&quot;, &quot;ITGB4&quot;, &quot;KIAA0020&quot;, &quot;KIF23&quot;, &quot;KPNA4&quot;, &quot;KRR1&quot;, &quot;LAMA3&quot;, &quot;LAMA5&quot;, &quot;LAMB3&quot;, &quot;LAMC1&quot;, &quot;LAMC2&quot;, &quot;LARP4&quot;, &quot;LGMN&quot;, &quot;LOXL2&quot;, &quot;LRP8&quot;, &quot;MAP7D1&quot;, &quot;MCM7&quot;, &quot;MDN1&quot;, &quot;MEPCE&quot;, &quot;MET&quot;, &quot;MEX3C&quot;, &quot;MEX3D&quot;, &quot;MICA&quot;, &quot;MRPL51&quot;, &quot;MRPS35&quot;, &quot;MRPS9&quot;, &quot;MTHFSD&quot;, &quot;MYBBP1A&quot;, &quot;MYEF2&quot;, &quot;MYO1B&quot;, &quot;NDC80&quot;, &quot;NGDN&quot;, &quot;NID2&quot;, &quot;NIFK&quot;, &quot;NOC3L&quot;, &quot;NOL11&quot;, &quot;NONO&quot;, &quot;NOP14&quot;, &quot;NOP9&quot;, &quot;NRP1&quot;, &quot;NSA2&quot;, &quot;NSUN5&quot;, &quot;NUFIP2&quot;, &quot;NUP133&quot;, &quot;NUP153&quot;, &quot;NUP50&quot;, &quot;NUSAP1&quot;, &quot;NUTF2&quot;, &quot;OTUD4&quot;, &quot;PALLD&quot;, &quot;PAWR&quot;, &quot;PAX8&quot;, &quot;PCBD1&quot;, &quot;PDCD11&quot;, &quot;PHB2&quot;, &quot;PHLDB1&quot;, &quot;PLAU&quot;, &quot;PNO1&quot;, &quot;POLR1E&quot;, &quot;POP1&quot;, &quot;PPAN&quot;, &quot;PPIA&quot;, &quot;PPIE&quot;, &quot;PPP1R10&quot;, &quot;PRKDC&quot;, &quot;PRPF8&quot;, &quot;PRRC2B&quot;, &quot;PRRC2C&quot;, &quot;PSMA2&quot;, &quot;PSMD3&quot;, &quot;PSMD6&quot;, &quot;PSPC1&quot;, &quot;PTBP3&quot;, &quot;PTGES3&quot;, &quot;PTPRA&quot;, &quot;PTPRF&quot;, &quot;PUM1&quot;, &quot;PUM1.1&quot;, &quot;PURB&quot;, &quot;PWP2&quot;, &quot;QKI&quot;, &quot;RAB8A&quot;, &quot;RACGAP1&quot;, &quot;RANBP1&quot;, &quot;RBM10&quot;, &quot;RBM15B&quot;, &quot;RBM19&quot;, &quot;RBM22&quot;, &quot;RBM28&quot;, &quot;RBM47&quot;, &quot;RBM5&quot;, &quot;RBM6&quot;, &quot;RBM7&quot;, &quot;RBMXL1&quot;, &quot;RFC1&quot;, &quot;RHOC&quot;, &quot;RPL10A&quot;, &quot;RPL7A&quot;, &quot;RPL7L1&quot;, &quot;RPS27&quot;, &quot;RPS5&quot;, &quot;RRP1&quot;, &quot;RRP36&quot;, &quot;RRS1&quot;, &quot;RSL1D1&quot;, &quot;S100A6&quot;, &quot;SAP30BP&quot;, &quot;SART3&quot;, &quot;SELH&quot;, &quot;SET&quot;, &quot;SETD2&quot;, &quot;SFPQ&quot;, &quot;SLC25A24&quot;, &quot;SLC38A2&quot;, &quot;SND1&quot;, &quot;SNRPB2&quot;, &quot;SOD1&quot;, &quot;SP100&quot;, &quot;SP3&quot;, &quot;SPATS2L&quot;, &quot;SPEN&quot;, &quot;SRP72&quot;, &quot;STAM&quot;, &quot;STRBP&quot;, &quot;SUGP2&quot;, &quot;SURF6&quot;, &quot;SYNE1&quot;, &quot;TAGLN2&quot;, &quot;TBCA&quot;, &quot;TCERG1&quot;, &quot;TFB2M&quot;, &quot;TIMP1&quot;, &quot;TIMP2&quot;, &quot;TMEM43&quot;, &quot;TMSB10&quot;, &quot;TMSB4X&quot;, &quot;TNFRSF10D&quot;, &quot;TNRC6B&quot;, &quot;TPM3.1&quot;, &quot;TPT1&quot;, &quot;TRIP13&quot;, &quot;TRMT1L&quot;, &quot;UBAP2&quot;, &quot;UBAP2L&quot;, &quot;UBE2V2&quot;, &quot;UBXN1&quot;, &quot;UHRF1&quot;, &quot;UPF1&quot;, &quot;URB1&quot;, &quot;UTP11L&quot;, &quot;UTP14A&quot;, &quot;UTP15&quot;, &quot;UTP20&quot;, &quot;UTP3&quot;, &quot;UTP6&quot;, &quot;VCAN&quot;, &quot;VWA1&quot;, &quot;WDR46&quot;, &quot;WDR75&quot;, &quot;WWC2&quot;, &quot;XRCC5&quot;, &quot;XRN2&quot;, &quot;YBX2&quot;, &quot;YLPM1&quot;, &quot;YTHDC2&quot;, &quot;YTHDF2&quot;, &quot;ZC3H14&quot;, &quot;ZC3H4&quot;, &quot;ZC3H7A&quot;, &quot;ZC3H7B&quot;, &quot;ZCCHC7&quot;, &quot;ZCCHC8&quot;, &quot;ZFC3H1&quot;, &quot;ZFR&quot;, &quot;ZNF622&quot;, &quot;ZNF638&quot;, &quot;AARS&quot;, &quot;AATF&quot;, &quot;ABCC1&quot;, &quot;ABCE1&quot;, &quot;ABCF1&quot;, &quot;ABT1&quot;, &quot;ACADVL&quot;, &quot;ACAT1&quot;, &quot;ACIN1&quot;, &quot;ACLY&quot;, &quot;ACO2&quot;, &quot;ACP1&quot;, &quot;ACTA1&quot;, &quot;ACTG1&quot;, &quot;ACTN1&quot;, &quot;ACTN4&quot;, &quot;ACTR1A&quot;, &quot;ACTR2&quot;, &quot;ADAM10&quot;, &quot;ADAR&quot;, &quot;ADH5&quot;, &quot;ADK&quot;, &quot;ADNP&quot;, &quot;AFG3L2&quot;, &quot;AGPS&quot;, &quot;AHNAK&quot;, &quot;AHNAK2&quot;, &quot;AIFM1&quot;, &quot;AIMP1&quot;, &quot;AK2&quot;, &quot;AKAP12&quot;, &quot;AKAP8&quot;, &quot;AKAP8L&quot;, &quot;AKAP9&quot;, &quot;ALCAM&quot;, &quot;ALDH18A1&quot;, &quot;ALDH2&quot;, &quot;ALDH6A1&quot;, &quot;ALDH7A1&quot;, &quot;ALDH9A1&quot;, &quot;ALDOA&quot;, &quot;ALDOC&quot;, &quot;ALYREF&quot;, &quot;ANKRD20A3&quot;, &quot;ANLN&quot;, &quot;ANP32A&quot;, &quot;ANP32B&quot;, &quot;ANP32E.1&quot;, &quot;ANXA1&quot;, &quot;ANXA11&quot;, &quot;ANXA2&quot;, &quot;ANXA4&quot;, &quot;ANXA6&quot;, &quot;AP2B1&quot;, &quot;AP3B1&quot;, &quot;AP3D1&quot;, &quot;APEX1&quot;, &quot;API5&quot;, &quot;APMAP&quot;, &quot;AQR&quot;, &quot;ARCN1&quot;, &quot;ARF3&quot;, &quot;ARF4&quot;, &quot;ARFGEF3&quot;, &quot;ARGLU1&quot;, &quot;ARHGEF2&quot;, &quot;ARL6IP4&quot;, &quot;ARPC2&quot;, &quot;ARPP19&quot;, &quot;ASPH&quot;, &quot;ATAD2&quot;, &quot;ATAD3A&quot;, &quot;ATG9A&quot;, &quot;ATP1A1&quot;, &quot;ATP1B1&quot;, &quot;ATP1B3&quot;, &quot;ATP2A2&quot;, &quot;ATP5A1&quot;, &quot;ATP5B&quot;, &quot;ATP5C1&quot;, &quot;ATP5D&quot;, &quot;ATP5F1&quot;, &quot;ATP5H&quot;, &quot;ATP6AP1&quot;, &quot;ATRX&quot;, &quot;ATXN2&quot;, &quot;ATXN2L&quot;, &quot;B2M&quot;, &quot;BAG2&quot;, &quot;BAG6&quot;, &quot;BAIAP2&quot;, &quot;BASP1&quot;, &quot;BAZ2A&quot;, &quot;BCAM&quot;, &quot;BCAP31&quot;, &quot;BCLAF1&quot;, &quot;BCLAF1.1&quot;, &quot;BIN1&quot;, &quot;BOD1L1&quot;, &quot;BOP1&quot;, &quot;BRD2&quot;, &quot;BRD4&quot;, &quot;BRIX1&quot;, &quot;BSG&quot;, &quot;BST2&quot;, &quot;BTF3&quot;, &quot;BTF3L4&quot;, &quot;BUB3&quot;, &quot;BZW1&quot;, &quot;C11orf68&quot;, &quot;C11orf98&quot;, &quot;C14orf166&quot;, &quot;C17orf85&quot;, &quot;C1QBP&quot;, &quot;C8orf59&quot;, &quot;CACYBP&quot;, &quot;CAD&quot;, &quot;CADM1&quot;, &quot;CADM4&quot;, &quot;CALD1&quot;, &quot;CALR&quot;, &quot;CALU&quot;, &quot;CAND1&quot;, &quot;CANX&quot;, &quot;CAP1&quot;, &quot;CAPN2&quot;, &quot;CAPZA1&quot;, &quot;CAPZB&quot;, &quot;CASC4&quot;, &quot;CAST&quot;, &quot;CAV1&quot;, &quot;CBX1&quot;, &quot;CBX3&quot;, &quot;CBX5&quot;, &quot;CCAR1&quot;, &quot;CCDC115&quot;, &quot;CCDC12&quot;, &quot;CCDC124&quot;, &quot;CCDC137&quot;, &quot;CCDC47&quot;, &quot;CCNT1&quot;, &quot;CCT2&quot;, &quot;CCT3&quot;, &quot;CCT4&quot;, &quot;CCT5&quot;, &quot;CCT6A&quot;, &quot;CCT7&quot;, &quot;CCT8&quot;, &quot;CD109&quot;, &quot;CD276&quot;, &quot;CD3EAP&quot;, &quot;CD44&quot;, &quot;CD46&quot;, &quot;CD46.1&quot;, &quot;CD47&quot;, &quot;CD55&quot;, &quot;CD58&quot;, &quot;CD63&quot;, &quot;CD70&quot;, &quot;CD81&quot;, &quot;CD99L2&quot;, &quot;CDC37&quot;, &quot;CDC42&quot;, &quot;CDC5L&quot;, &quot;CDH2&quot;, &quot;CDK1&quot;, &quot;CDK11A&quot;, &quot;CDK9&quot;, &quot;CDKN2AIP&quot;, &quot;CDV3&quot;, &quot;CEBPZ&quot;, &quot;CELF1&quot;, &quot;CEP131&quot;, &quot;CEP290&quot;, &quot;CES2&quot;, &quot;CFL1&quot;, &quot;CHCHD3&quot;, &quot;CHD1&quot;, &quot;CHD4&quot;, &quot;CHORDC1&quot;, &quot;CIRBP&quot;, &quot;CISD1&quot;, &quot;CKAP4&quot;, &quot;CKAP5&quot;, &quot;CLIC1&quot;, &quot;CLINT1&quot;, &quot;CLTA&quot;, &quot;CLTB&quot;, &quot;CLTC&quot;, &quot;CMPK1&quot;, &quot;CNOT1&quot;, &quot;CNP&quot;, &quot;CNPY2&quot;, &quot;CNTNAP3B&quot;, &quot;COL12A1&quot;, &quot;COL4A2&quot;, &quot;COPA&quot;, &quot;COPB1&quot;, &quot;COPG1&quot;, &quot;CORO1C&quot;, &quot;COTL1&quot;, &quot;COX4I1&quot;, &quot;COX5A&quot;, &quot;CPD&quot;, &quot;CPNE1&quot;, &quot;CPSF6&quot;, &quot;CPSF7&quot;, &quot;CRKL&quot;, &quot;CROCC&quot;, &quot;CS&quot;, &quot;CSDE1&quot;, &quot;CSE1L&quot;, &quot;CSTB&quot;, &quot;CSTF1&quot;, &quot;CSTF3&quot;, &quot;CTGF&quot;, &quot;CTNNA1&quot;, &quot;CTNND1&quot;, &quot;CTR9&quot;, &quot;CTSB&quot;, &quot;CTTN&quot;, &quot;CWC15&quot;, &quot;CWC22&quot;, &quot;CYB5B&quot;, &quot;CYB5R3&quot;, &quot;CYC1&quot;, &quot;DAG1&quot;, &quot;DAP&quot;, &quot;DARS&quot;, &quot;DAZAP1&quot;, &quot;DBI&quot;, &quot;DBN1&quot;, &quot;DBT&quot;, &quot;DCD&quot;, &quot;DCTN1&quot;, &quot;DCTN2&quot;, &quot;DDB1&quot;, &quot;DDOST&quot;, &quot;DDX1&quot;, &quot;DDX18&quot;, &quot;DDX21&quot;, &quot;DDX23&quot;, &quot;DDX39B&quot;, &quot;DDX3X&quot;, &quot;DDX42&quot;, &quot;DDX46&quot;, &quot;DECR1&quot;, &quot;DEK&quot;, &quot;DENR&quot;, &quot;DHRS2&quot;, &quot;DHX15&quot;, &quot;DHX30&quot;, &quot;DHX36&quot;, &quot;DHX37&quot;, &quot;DHX9&quot;, &quot;DIABLO&quot;, &quot;DIDO1&quot;, &quot;DIMT1&quot;, &quot;DKC1&quot;, &quot;DLD&quot;, &quot;DNAJA1&quot;, &quot;DNAJA2&quot;, &quot;DNAJB1&quot;, &quot;DNAJB11&quot;, &quot;DNAJC1&quot;, &quot;DNAJC8&quot;, &quot;DNAJC9&quot;, &quot;DNM1L&quot;, &quot;DNMT1&quot;, &quot;DNTTIP2&quot;, &quot;DOHH&quot;, &quot;DPP4&quot;, &quot;DPYSL2&quot;, &quot;DPYSL3&quot;, &quot;DSG2&quot;, &quot;DSP&quot;, &quot;DSTN&quot;, &quot;DTYMK&quot;, &quot;DYNC1H1&quot;, &quot;DYNC1I2&quot;, &quot;DYNC1LI2&quot;, &quot;ECE1&quot;, &quot;ECHS1&quot;, &quot;EDC4&quot;, &quot;EDF1&quot;, &quot;EEA1&quot;, &quot;EEF1A1&quot;, &quot;EEF1D&quot;, &quot;EEF1G&quot;, &quot;EEF2&quot;, &quot;EFHD2&quot;, &quot;EFTUD2&quot;, &quot;EGFR&quot;, &quot;EHD4&quot;, &quot;EIF1&quot;, &quot;EIF1AX&quot;, &quot;EIF2AK2&quot;, &quot;EIF2S1&quot;, &quot;EIF2S2&quot;, &quot;EIF2S3&quot;, &quot;EIF3A&quot;, &quot;EIF3B&quot;, &quot;EIF3C&quot;, &quot;EIF3D&quot;, &quot;EIF3F&quot;, &quot;EIF3G&quot;, &quot;EIF3I&quot;, &quot;EIF3J&quot;, &quot;EIF4A1&quot;, &quot;EIF4A3&quot;, &quot;EIF4B&quot;, &quot;EIF4G1&quot;, &quot;EIF4G2&quot;, &quot;EIF4H&quot;, &quot;EIF5&quot;, &quot;EIF5B&quot;, &quot;EIF6&quot;, &quot;ELAVL1&quot;, &quot;ENO1&quot;, &quot;ENSA&quot;, &quot;EPB41L2&quot;, &quot;EPHA2&quot;, &quot;EPRS&quot;, &quot;EPS8L2&quot;, &quot;ERAL1&quot;, &quot;ERBB2&quot;, &quot;ERH&quot;, &quot;ERI1&quot;, &quot;ERO1L&quot;, &quot;ERP29&quot;, &quot;ERP44&quot;, &quot;ESF1&quot;, &quot;ESYT1&quot;, &quot;ETFB&quot;, &quot;EWSR1&quot;, &quot;EXOSC10&quot;, &quot;EZR&quot;, &quot;F3&quot;, &quot;FABP5&quot;, &quot;FAM120A&quot;, &quot;FAM129B&quot;, &quot;FAM98B&quot;, &quot;FARSB&quot;, &quot;FASN&quot;, &quot;FASTKD2&quot;, &quot;FBL&quot;, &quot;FBLL1&quot;, &quot;FEN1&quot;, &quot;FH&quot;, &quot;FHL2&quot;, &quot;FKBP1A&quot;, &quot;FKBP2&quot;, &quot;FKBP3&quot;, &quot;FKBP9&quot;, &quot;FLNA&quot;, &quot;FLNB.1&quot;, &quot;FMR1&quot;, &quot;FN1&quot;, &quot;FOLR1&quot;, &quot;FSCN1&quot;, &quot;FTH1&quot;, &quot;FUBP1&quot;, &quot;FUBP3&quot;, &quot;FUS&quot;, &quot;FXR1&quot;, &quot;FXYD5&quot;, &quot;G3BP1&quot;, &quot;G3BP2&quot;, &quot;G6PD&quot;, &quot;GADD45GIP1&quot;, &quot;GALNT2&quot;, &quot;GANAB&quot;, &quot;GAPDH&quot;, &quot;GAR1&quot;, &quot;GARS&quot;, &quot;GART&quot;, &quot;GCN1L1&quot;, &quot;GDI2&quot;, &quot;GGH&quot;, &quot;GIGYF2&quot;, &quot;GLRX3&quot;, &quot;GLS&quot;, &quot;GLS.1&quot;, &quot;GLTSCR2&quot;, &quot;GLUD1&quot;, &quot;GMPS&quot;, &quot;GNAI2&quot;, &quot;GNAI3&quot;, &quot;GNAS&quot;, &quot;GNB2&quot;, &quot;GNB2L1&quot;, &quot;GNL3&quot;, &quot;GNL3L&quot;, &quot;GNS&quot;, &quot;GOLGA2&quot;, &quot;GOLGB1&quot;, &quot;GOLIM4&quot;, &quot;GOLM1&quot;, &quot;GOT2&quot;, &quot;GPATCH4&quot;, &quot;GPATCH8&quot;, &quot;GPC1&quot;, &quot;GPC4&quot;, &quot;GPI&quot;, &quot;GPRC5A&quot;, &quot;GPS1&quot;, &quot;GRSF1&quot;, &quot;GRWD1&quot;, &quot;GSN&quot;, &quot;GTPBP4&quot;, &quot;GYG1&quot;, &quot;H1FX&quot;, &quot;H2AFV&quot;, &quot;H2AFX&quot;, &quot;H2AFY&quot;, &quot;HADH&quot;, &quot;HADHA&quot;, &quot;HADHB&quot;, &quot;HDAC2&quot;, &quot;HDGF&quot;, &quot;HDGFRP2&quot;, &quot;HDGFRP3&quot;, &quot;HEATR1&quot;, &quot;HERC5&quot;, &quot;HEXIM1&quot;, &quot;HIST1H1B&quot;, &quot;HIST1H1C&quot;, &quot;HIST1H2AJ&quot;, &quot;HIST1H2BL&quot;, &quot;HIST1H4A&quot;, &quot;HIST2H2AC&quot;, &quot;HIST2H3A&quot;, &quot;HIST2H3PS2&quot;, &quot;HLA-A&quot;, &quot;HLA-B&quot;, &quot;HM13&quot;, &quot;HMGA1&quot;, &quot;HMGB2&quot;, &quot;HMGB3&quot;, &quot;HMGCS1&quot;, &quot;HMGN5&quot;, &quot;HMOX2&quot;, &quot;HNRNPA0&quot;, &quot;HNRNPA1&quot;, &quot;HNRNPA2B1&quot;, &quot;HNRNPA3&quot;, &quot;HNRNPAB&quot;, &quot;HNRNPC&quot;, &quot;HNRNPC.1&quot;, &quot;HNRNPD&quot;, &quot;HNRNPDL&quot;, &quot;HNRNPF&quot;, &quot;HNRNPH1&quot;, &quot;HNRNPH2&quot;, &quot;HNRNPH3&quot;, &quot;HNRNPK&quot;, &quot;HNRNPK.1&quot;, &quot;HNRNPL&quot;, &quot;HNRNPR&quot;, &quot;HNRNPR.1&quot;, &quot;HNRNPU&quot;, &quot;HNRNPUL1&quot;, &quot;HNRNPUL2&quot;, &quot;HOXB6&quot;, &quot;HP1BP3&quot;, &quot;HSD17B10&quot;, &quot;HSD17B4&quot;, &quot;HSP90AA1&quot;, &quot;HSP90AB1&quot;, &quot;HSP90B1&quot;, &quot;HSPA1A&quot;, &quot;HSPA4&quot;, &quot;HSPA5&quot;, &quot;HSPA6&quot;, &quot;HSPA8&quot;, &quot;HSPA9&quot;, &quot;HSPB1&quot;, &quot;HSPBP1&quot;, &quot;HSPD1&quot;, &quot;HSPE1&quot;, &quot;HSPH1&quot;, &quot;HTATSF1&quot;, &quot;HUWE1&quot;, &quot;HYOU1&quot;, &quot;IARS&quot;, &quot;IDH1&quot;, &quot;IDH2&quot;, &quot;IGF2BP2&quot;, &quot;IGF2R&quot;, &quot;IK&quot;, &quot;IKBIP.1&quot;, &quot;ILF2&quot;, &quot;ILF3&quot;, &quot;ILF3.1&quot;, &quot;IMPDH2&quot;, &quot;IMUP&quot;, &quot;INF2&quot;, &quot;INPP5F&quot;, &quot;IPO5&quot;, &quot;IPO7&quot;, &quot;IQGAP1&quot;, &quot;ISCU&quot;, &quot;ITFG3&quot;, &quot;ITGA3&quot;, &quot;ITGA5&quot;, &quot;ITGA6&quot;, &quot;ITGAV&quot;, &quot;ITGB1&quot;, &quot;IWS1&quot;, &quot;JUP&quot;, &quot;KANK2&quot;, &quot;KARS&quot;, &quot;KDM2A&quot;, &quot;KDM4B&quot;, &quot;KHDRBS1&quot;, &quot;KHDRBS3&quot;, &quot;KHSRP&quot;, &quot;KIAA0319L&quot;, &quot;KIAA1217&quot;, &quot;KIAA1524&quot;, &quot;KIAA1598&quot;, &quot;KIF2A&quot;, &quot;KIF2C&quot;, &quot;KIF4A&quot;, &quot;KIF5B&quot;, &quot;KLC1&quot;, &quot;KMT2A&quot;, &quot;KPNA2&quot;, &quot;KPNA3&quot;, &quot;KPNB1&quot;, &quot;KRI1&quot;, &quot;KRT18&quot;, &quot;KTN1&quot;, &quot;L1CAM&quot;, &quot;L1RE1&quot;, &quot;LAMB1&quot;, &quot;LAMB2&quot;, &quot;LAMP1&quot;, &quot;LAMP2&quot;, &quot;LAP3&quot;, &quot;LARP1&quot;, &quot;LARP4B&quot;, &quot;LARP7&quot;, &quot;LAS1L&quot;, &quot;LASP1&quot;, &quot;LDHA&quot;, &quot;LDHB&quot;, &quot;LDLR&quot;, &quot;LEO1&quot;, &quot;LEPRE1&quot;, &quot;LGALS1&quot;, &quot;LGALS3BP&quot;, &quot;LIMA1&quot;, &quot;LIN7C&quot;, &quot;LLPH&quot;, &quot;LMAN1&quot;, &quot;LMNA&quot;, &quot;LMNA.1&quot;, &quot;LMNB1&quot;, &quot;LMNB2&quot;, &quot;LNPEP&quot;, &quot;LRPAP1&quot;, &quot;LRPPRC&quot;, &quot;LRRC47&quot;, &quot;LRRC59&quot;, &quot;LRRFIP1&quot;, &quot;LRRFIP1.1&quot;, &quot;LSM14A&quot;, &quot;LSM2&quot;, &quot;LSM4&quot;, &quot;LSM8&quot;, &quot;LUC7L&quot;, &quot;LUC7L2&quot;, &quot;LUC7L3&quot;, &quot;LUZP1&quot;, &quot;LYAR&quot;, &quot;M6PR&quot;, &quot;MACF1.1&quot;, &quot;MANF&quot;, &quot;MAP1B&quot;, &quot;MAP2&quot;, &quot;MAP4&quot;, &quot;MAP7D3&quot;, &quot;MAPRE1&quot;, &quot;MARCKS&quot;, &quot;MARCKSL1&quot;, &quot;MATR3&quot;, &quot;MBNL1&quot;, &quot;MCM5&quot;, &quot;MCM6&quot;, &quot;MDH2&quot;, &quot;MECP2&quot;, &quot;METAP1&quot;, &quot;MFAP1&quot;, &quot;MICAL3&quot;, &quot;MINOS1&quot;, &quot;MISP&quot;, &quot;MKI67&quot;, &quot;MKL1&quot;, &quot;MKRN2&quot;, &quot;MLEC&quot;, &quot;MLLT4&quot;, &quot;MMAB&quot;, &quot;MMTAG2&quot;, &quot;MNS1&quot;, &quot;MOV10&quot;, &quot;MPHOSPH10&quot;, &quot;MRPL1&quot;, &quot;MRPL12&quot;, &quot;MRPL13&quot;, &quot;MRPL2&quot;, &quot;MRPL27&quot;, &quot;MRPL28&quot;, &quot;MRPL3&quot;, &quot;MRPL37&quot;, &quot;MRPL39&quot;, &quot;MRPL41&quot;, &quot;MRPL42&quot;, &quot;MRPL43&quot;, &quot;MRPL45&quot;, &quot;MRPL49&quot;, &quot;MRPL57&quot;, &quot;MRPL9&quot;, &quot;MRPS18A&quot;, &quot;MRPS28&quot;, &quot;MRPS30&quot;, &quot;MRPS31&quot;, &quot;MRPS5&quot;, &quot;MRPS7&quot;, &quot;MSI2&quot;, &quot;MSN&quot;, &quot;MTDH&quot;, &quot;MTHFD1&quot;, &quot;MTHFD1L&quot;, &quot;MYH10&quot;, &quot;MYH14&quot;, &quot;MYH9&quot;, &quot;MYL12B&quot;, &quot;MYL6&quot;, &quot;MYOF&quot;, &quot;NAA15&quot;, &quot;NACA&quot;, &quot;NAP1L1&quot;, &quot;NAP1L4&quot;, &quot;NARS&quot;, &quot;NASP&quot;, &quot;NAT10&quot;, &quot;NCAPG&quot;, &quot;NCAPH&quot;, &quot;NCBP1&quot;, &quot;NCEH1&quot;, &quot;NCL&quot;, &quot;NCOA5&quot;, &quot;NCSTN&quot;, &quot;NDC1&quot;, &quot;NDUFS1&quot;, &quot;NEB&quot;, &quot;NEBL&quot;, &quot;NENF&quot;, &quot;NHP2&quot;, &quot;NIPBL&quot;, &quot;NKAP&quot;, &quot;NKRF&quot;, &quot;NKTR&quot;, &quot;NME1&quot;, &quot;NME2&quot;, &quot;NMT1&quot;, &quot;NNMT&quot;, &quot;NOB1&quot;, &quot;NOC2L&quot;, &quot;NOL6&quot;, &quot;NOL8&quot;, &quot;NOLC1&quot;, &quot;NOMO1&quot;, &quot;NOP16&quot;, &quot;NOP2&quot;, &quot;NOP56&quot;, &quot;NOP58&quot;, &quot;NPC1&quot;, &quot;NPLOC4&quot;, &quot;NPM1&quot;, &quot;NPTN&quot;, &quot;NSDHL&quot;, &quot;NSFL1C&quot;, &quot;NSRP1&quot;, &quot;NSUN2&quot;, &quot;NUCB1&quot;, &quot;NUCB2&quot;, &quot;NUCKS1&quot;, &quot;NUDC&quot;, &quot;NUDT16L1&quot;, &quot;NUDT21&quot;, &quot;NUMA1&quot;, &quot;NUMA1.1&quot;, &quot;NUP160&quot;, &quot;NUP214&quot;, &quot;NUP98&quot;, &quot;NXF1&quot;, &quot;ODF2&quot;, &quot;OGDH&quot;, &quot;OLA1&quot;, &quot;P09493-3&quot;, &quot;P09493-5&quot;, &quot;P0DP25&quot;, &quot;P4HA1&quot;, &quot;P4HB&quot;, &quot;PA2G4&quot;, &quot;PABPC1&quot;, &quot;PABPC4&quot;, &quot;PABPN1&quot;, &quot;PAF1&quot;, &quot;PAICS&quot;, &quot;PARK7&quot;, &quot;PARP1&quot;, &quot;PATL1&quot;, &quot;PCBP1&quot;, &quot;PCM1&quot;, &quot;PCNA&quot;, &quot;PCNP&quot;, &quot;PCNT&quot;, &quot;PDAP1&quot;, &quot;PDCD5&quot;, &quot;PDCD6IP&quot;, &quot;PDIA3&quot;, &quot;PDIA4&quot;, &quot;PDIA6&quot;, &quot;PDLIM5&quot;, &quot;PDLIM7&quot;, &quot;PEBP1&quot;, &quot;PEF1&quot;, &quot;PELO&quot;, &quot;PES1&quot;, &quot;PFN1&quot;, &quot;PFN2&quot;, &quot;PGAM1&quot;, &quot;PGAM5&quot;, &quot;PGD&quot;, &quot;PGK1&quot;, &quot;PGRMC1&quot;, &quot;PGRMC2&quot;, &quot;PHB&quot;, &quot;PHF3&quot;, &quot;PHF5A&quot;, &quot;PHF6&quot;, &quot;PHGDH&quot;, &quot;PHIP&quot;, &quot;PKM&quot;, &quot;PKP3&quot;, &quot;PLAUR&quot;, &quot;PLEC&quot;, &quot;PLIN3&quot;, &quot;PLOD1&quot;, &quot;PLS3&quot;, &quot;PLXNB2&quot;, &quot;PNISR&quot;, &quot;PNN&quot;, &quot;PODXL&quot;, &quot;PODXL2&quot;, &quot;POLDIP3&quot;, &quot;POLR2A&quot;, &quot;POLR2B&quot;, &quot;POLR2H&quot;, &quot;POLRMT&quot;, &quot;PPA1&quot;, &quot;PPFIA3&quot;, &quot;PPIA.1&quot;, &quot;PPIB&quot;, &quot;PPID&quot;, &quot;PPIG&quot;, &quot;PPIL4&quot;, &quot;PPM1G&quot;, &quot;PPP1CC&quot;, &quot;PPP1R12A&quot;, &quot;PPP1R1B&quot;, &quot;PPP1R8&quot;, &quot;PPP2R1A&quot;, &quot;PPP2R2A&quot;, &quot;PPP2R5D&quot;, &quot;PRC1&quot;, &quot;PRDM15&quot;, &quot;PRDX1&quot;, &quot;PRDX2&quot;, &quot;PRDX3&quot;, &quot;PRDX4&quot;, &quot;PRDX5&quot;, &quot;PRDX6&quot;, &quot;PRKCSH&quot;, &quot;PRKRA&quot;, &quot;PRMT1&quot;, &quot;PRNP&quot;, &quot;PROSC&quot;, &quot;PRPF19&quot;, &quot;PRPF3&quot;, &quot;PRPF31&quot;, &quot;PRPF38B&quot;, &quot;PRPF4&quot;, &quot;PRPF40A&quot;, &quot;PRPF4B&quot;, &quot;PRPF6&quot;, &quot;PRRC2A&quot;, &quot;PSAP&quot;, &quot;PSIP1&quot;, &quot;PSMA3&quot;, &quot;PSMA4&quot;, &quot;PSMA5&quot;, &quot;PSMA7&quot;, &quot;PSMB6&quot;, &quot;PSMB7&quot;, &quot;PSMC1&quot;, &quot;PSMC2&quot;, &quot;PSMC3&quot;, &quot;PSMC4&quot;, &quot;PSMC5&quot;, &quot;PSMC6&quot;, &quot;PSMD1&quot;, &quot;PSMD11&quot;, &quot;PSMD12&quot;, &quot;PSMD2&quot;, &quot;PSMD4&quot;, &quot;PSMD7&quot;, &quot;PSME1&quot;, &quot;PTBP1&quot;, &quot;PTCD1&quot;, &quot;PTCD3&quot;, &quot;PTGIS&quot;, &quot;PTK7&quot;, &quot;PTPRB&quot;, &quot;PTPRG&quot;, &quot;PTPRJ&quot;, &quot;PTRF&quot;, &quot;PTRH1&quot;, &quot;PUF60&quot;, &quot;PUM2&quot;, &quot;PURA&quot;, &quot;PVR&quot;, &quot;PVRL3&quot;, &quot;PYCR2&quot;, &quot;PYGB&quot;, &quot;PYGL&quot;, &quot;Q15366-3&quot;, &quot;QSOX2&quot;, &quot;R3HDM1&quot;, &quot;RAB10&quot;, &quot;RAB11B&quot;, &quot;RAB11FIP1&quot;, &quot;RAB14&quot;, &quot;RAB1A&quot;, &quot;RAB5C&quot;, &quot;RAB6A&quot;, &quot;RAB7A&quot;, &quot;RABGGTA&quot;, &quot;RAD17&quot;, &quot;RAD50&quot;, &quot;RALY&quot;, &quot;RAN&quot;, &quot;RANBP2&quot;, &quot;RANGAP1&quot;, &quot;RAP1B&quot;, &quot;RARS&quot;, &quot;RAVER1&quot;, &quot;RBBP4&quot;, &quot;RBBP6&quot;, &quot;RBBP7&quot;, &quot;RBFOX2&quot;, &quot;RBM12&quot;, &quot;RBM12B&quot;, &quot;RBM14&quot;, &quot;RBM15&quot;, &quot;RBM17&quot;, &quot;RBM25&quot;, &quot;RBM27&quot;, &quot;RBM3&quot;, &quot;RBM34&quot;, &quot;RBM39&quot;, &quot;RBM4&quot;, &quot;RBM8A&quot;, &quot;RBMS1&quot;, &quot;RBMS2&quot;, &quot;RBMX&quot;, &quot;RBMX2&quot;, &quot;RBPMS&quot;, &quot;RCC1&quot;, &quot;RCN1&quot;, &quot;RCN2&quot;, &quot;RDX&quot;, &quot;REEP5&quot;, &quot;REPS1&quot;, &quot;RHOA&quot;, &quot;RIF1&quot;, &quot;RNH1&quot;, &quot;RNMT&quot;, &quot;RNPS1&quot;, &quot;RPA1&quot;, &quot;RPF2&quot;, &quot;RPL10&quot;, &quot;RPL11&quot;, &quot;RPL12&quot;, &quot;RPL13&quot;, &quot;RPL13A&quot;, &quot;RPL14&quot;, &quot;RPL15&quot;, &quot;RPL17&quot;, &quot;RPL18&quot;, &quot;RPL18A&quot;, &quot;RPL19&quot;, &quot;RPL21&quot;, &quot;RPL22&quot;, &quot;RPL23&quot;, &quot;RPL23A&quot;, &quot;RPL24&quot;, &quot;RPL26&quot;, &quot;RPL27&quot;, &quot;RPL27A&quot;, &quot;RPL28&quot;, &quot;RPL3&quot;, &quot;RPL30&quot;, &quot;RPL31&quot;, &quot;RPL34&quot;, &quot;RPL35&quot;, &quot;RPL36AL&quot;, &quot;RPL38&quot;, &quot;RPL4&quot;, &quot;RPL5&quot;, &quot;RPL6&quot;, &quot;RPL7&quot;, &quot;RPL8&quot;, &quot;RPL9&quot;, &quot;RPLP0&quot;, &quot;RPLP2&quot;, &quot;RPN1&quot;, &quot;RPN2&quot;, &quot;RPS10&quot;, &quot;RPS11&quot;, &quot;RPS12&quot;, &quot;RPS13&quot;, &quot;RPS14&quot;, &quot;RPS15&quot;, &quot;RPS15A&quot;, &quot;RPS16&quot;, &quot;RPS17&quot;, &quot;RPS18&quot;, &quot;RPS19&quot;, &quot;RPS2&quot;, &quot;RPS20&quot;, &quot;RPS21&quot;, &quot;RPS24&quot;, &quot;RPS25&quot;, &quot;RPS26&quot;, &quot;RPS27A&quot;, &quot;RPS27L&quot;, &quot;RPS28&quot;, &quot;RPS3&quot;, &quot;RPS3A&quot;, &quot;RPS4X&quot;, &quot;RPS6&quot;, &quot;RPS7&quot;, &quot;RPS8&quot;, &quot;RPS9&quot;, &quot;RPSA&quot;, &quot;RRBP1&quot;, &quot;RRP12&quot;, &quot;RRP15&quot;, &quot;RRP1B&quot;, &quot;RRP9&quot;, &quot;RSF1&quot;, &quot;RSRC2&quot;, &quot;RTCB&quot;, &quot;RTF1&quot;, &quot;RTN4&quot;, &quot;RUVBL1&quot;, &quot;RUVBL2&quot;, &quot;S100A10&quot;, &quot;S100A11&quot;, &quot;S100A16&quot;, &quot;S100A2&quot;, &quot;S100A3&quot;, &quot;S100A4&quot;, &quot;SAFB&quot;, &quot;SAFB2&quot;, &quot;SAP18&quot;, &quot;SARNP&quot;, &quot;SARS&quot;, &quot;SART1&quot;, &quot;SCAF1&quot;, &quot;SCAF11&quot;, &quot;SCAF8&quot;, &quot;SCARB2&quot;, &quot;SCP2&quot;, &quot;SDAD1&quot;, &quot;SDC1&quot;, &quot;SDC2&quot;, &quot;SDC4&quot;, &quot;SDF4&quot;, &quot;SDHA&quot;, &quot;SDK2&quot;, &quot;SEC22B&quot;, &quot;SEC31A&quot;, &quot;SEC61B&quot;, &quot;SEC62&quot;, &quot;SEC63&quot;, &quot;SEPT11&quot;, &quot;SEPT7&quot;, &quot;SEPT9&quot;, &quot;SERBP1&quot;, &quot;SERBP1.1&quot;, &quot;SERPINB1&quot;, &quot;SERPINB6&quot;, &quot;SERPINH1&quot;, &quot;SERPINI2&quot;, &quot;SETD1A&quot;, &quot;SF1&quot;, &quot;SF3A1&quot;, &quot;SF3A2&quot;, &quot;SF3A3&quot;, &quot;SF3B1&quot;, &quot;SF3B2&quot;, &quot;SF3B3&quot;, &quot;SF3B4&quot;, &quot;SF3B6&quot;, &quot;SFN&quot;, &quot;SH3GL1&quot;, &quot;SH3GLB2&quot;, &quot;SHMT2&quot;, &quot;SKIV2L2&quot;, &quot;SKP1&quot;, &quot;SLC1A5&quot;, &quot;SLC25A6&quot;, &quot;SLC26A2&quot;, &quot;SLC29A1&quot;, &quot;SLC2A1&quot;, &quot;SLC39A10&quot;, &quot;SLC39A14&quot;, &quot;SLC39A6&quot;, &quot;SLC3A2&quot;, &quot;SLC4A2&quot;, &quot;SLC4A7&quot;, &quot;SLC7A1&quot;, &quot;SLC7A5&quot;, &quot;SLC9A3R1&quot;, &quot;SLC9A3R2&quot;, &quot;SLIRP&quot;, &quot;SLK&quot;, &quot;SLTM&quot;, &quot;SMARCA4&quot;, &quot;SMARCC2&quot;, &quot;SMC1A&quot;, &quot;SMC2&quot;, &quot;SMC3&quot;, &quot;SMC4&quot;, &quot;SMCHD1&quot;, &quot;SMNDC1&quot;, &quot;SNRNP200&quot;, &quot;SNRNP70&quot;, &quot;SNRPA&quot;, &quot;SNRPB&quot;, &quot;SNRPC&quot;, &quot;SNRPD1&quot;, &quot;SNRPD2&quot;, &quot;SNRPD3&quot;, &quot;SNRPF&quot;, &quot;SNRPG&quot;, &quot;SNW1&quot;, &quot;SON&quot;, &quot;SORBS2&quot;, &quot;SORD&quot;, &quot;SPANXB1&quot;, &quot;SPATS2&quot;, &quot;SPINT2&quot;, &quot;SPTAN1&quot;, &quot;SPTBN1&quot;, &quot;SQSTM1&quot;, &quot;SREK1&quot;, &quot;SRP14&quot;, &quot;SRP68&quot;, &quot;SRP9&quot;, &quot;SRPK1&quot;, &quot;SRPR&quot;, &quot;SRPRB&quot;, &quot;SRRM1&quot;, &quot;SRRM2&quot;, &quot;SRRT&quot;, &quot;SRSF1&quot;, &quot;SRSF10&quot;, &quot;SRSF11&quot;, &quot;SRSF2&quot;, &quot;SRSF3&quot;, &quot;SRSF4&quot;, &quot;SRSF5&quot;, &quot;SRSF6&quot;, &quot;SRSF7&quot;, &quot;SRSF9&quot;, &quot;SSB&quot;, &quot;SSBP1&quot;, &quot;SSR1&quot;, &quot;SSRP1&quot;, &quot;ST13&quot;, &quot;STAT1&quot;, &quot;STAU1&quot;, &quot;STAU2&quot;, &quot;STIM1&quot;, &quot;STIP1&quot;, &quot;STMN1&quot;, &quot;STMN2&quot;, &quot;STOML2&quot;, &quot;STRAP&quot;, &quot;STT3A&quot;, &quot;STUB1&quot;, &quot;STX7&quot;, &quot;SUB1&quot;, &quot;SUMO1&quot;, &quot;SUMO2&quot;, &quot;SUMO3&quot;, &quot;SUPT16H&quot;, &quot;SUPT5H&quot;, &quot;SUPT6H&quot;, &quot;SWAP70&quot;, &quot;SYNCRIP&quot;, &quot;SYPL1&quot;, &quot;SZRD1&quot;, &quot;TACC1&quot;, &quot;TACSTD2&quot;, &quot;TAF15&quot;, &quot;TAGLN&quot;, &quot;TALDO1&quot;, &quot;TARDBP&quot;, &quot;TARS&quot;, &quot;TBCB&quot;, &quot;TBL2&quot;, &quot;TBL3&quot;, &quot;TCEA1&quot;, &quot;TCOF1&quot;, &quot;TCP1&quot;, &quot;TES&quot;, &quot;TFRC&quot;, &quot;TGOLN2&quot;, &quot;THRAP3&quot;, &quot;TIA1&quot;, &quot;TIAL1.1&quot;, &quot;TIMM10&quot;, &quot;TIMM44&quot;, &quot;TIMM50&quot;, &quot;TINAGL1&quot;, &quot;TJP1&quot;, &quot;TKT&quot;, &quot;TLN1&quot;, &quot;TLN2&quot;, &quot;TMED10&quot;, &quot;TMED9&quot;, &quot;TMEM109&quot;, &quot;TMEM87A&quot;, &quot;TMPO&quot;, &quot;TMPO.1&quot;, &quot;TMSB15B&quot;, &quot;TMX1&quot;, &quot;TMX3&quot;, &quot;TNPO1&quot;, &quot;TNS1&quot;, &quot;TOP1&quot;, &quot;TOP2A&quot;, &quot;TOP2B&quot;, &quot;TOR1AIP1.1&quot;, &quot;TP53BP1&quot;, &quot;TPD52&quot;, &quot;TPD52L2&quot;, &quot;TPI1&quot;, &quot;TPM4&quot;, &quot;TPPP3&quot;, &quot;TPR&quot;, &quot;TPX2&quot;, &quot;TRA2A&quot;, &quot;TRA2B&quot;, &quot;TRAP1&quot;, &quot;TRIM25&quot;, &quot;TRIM28&quot;, &quot;TRIP12&quot;, &quot;TRMT1&quot;, &quot;TRMT10C&quot;, &quot;TSN&quot;, &quot;TSR1&quot;, &quot;TUBA1B&quot;, &quot;TUBA4A&quot;, &quot;TUBB&quot;, &quot;TUBB4A&quot;, &quot;TUBB4B&quot;, &quot;TUBB6&quot;, &quot;TUBG1&quot;, &quot;TUFM&quot;, &quot;TWF1&quot;, &quot;TWSG1&quot;, &quot;TXN&quot;, &quot;TXNDC12&quot;, &quot;TXNDC15&quot;, &quot;TXNDC5&quot;, &quot;TXNL1&quot;, &quot;TXNRD1&quot;, &quot;U2AF1&quot;, &quot;U2AF2&quot;, &quot;U2SURP&quot;, &quot;UBA1&quot;, &quot;UBE2L3&quot;, &quot;UBR4&quot;, &quot;UBTF&quot;, &quot;UBXN4&quot;, &quot;UFL1&quot;, &quot;UGDH&quot;, &quot;UGGT1&quot;, &quot;UQCRC1&quot;, &quot;USP10&quot;, &quot;USP14&quot;, &quot;USP36&quot;, &quot;UTP18&quot;, &quot;VAMP3&quot;, &quot;VAMP5&quot;, &quot;VAPA&quot;, &quot;VAPB&quot;, &quot;VARS&quot;, &quot;VASP&quot;, &quot;VAT1&quot;, &quot;VCL&quot;, &quot;VCP&quot;, &quot;VCPIP1&quot;, &quot;VDAC1&quot;, &quot;VDAC2&quot;, &quot;VIM&quot;, &quot;VIMP&quot;, &quot;VPS35&quot;, &quot;WARS&quot;, &quot;WBP11&quot;, &quot;WDR1&quot;, &quot;WDR3&quot;, &quot;WDR33&quot;, &quot;WDR43&quot;, &quot;WDR74&quot;, &quot;WHSC1&quot;, &quot;WIBG&quot;, &quot;XIRP2&quot;, &quot;XPO1&quot;, &quot;XPO5&quot;, &quot;XRCC6&quot;, &quot;YARS&quot;, &quot;YARS2&quot;, &quot;YBX1&quot;, &quot;YBX3&quot;, &quot;YTHDC1&quot;, &quot;YTHDF1&quot;, &quot;YTHDF3&quot;, &quot;YWHAB&quot;, &quot;YWHAE&quot;, &quot;YWHAG&quot;, &quot;YWHAH&quot;, &quot;YWHAQ&quot;, &quot;YWHAZ&quot;, &quot;ZC3H11A&quot;, &quot;ZC3H13&quot;, &quot;ZC3H15&quot;, &quot;ZC3H18&quot;, &quot;ZC3HAV1&quot;, &quot;ZCCHC3&quot;, &quot;ZCRB1&quot;, &quot;ZNF207&quot;, &quot;ZNF326&quot;, &quot;ZRANB2&quot;)

pino &lt;- assay(data_filt) %&gt;% data.frame() %&gt;% rownames_to_column() %&gt;% filter(rowname %in% vettore)  
data_filt &lt;- data_filt[pino$rowname, ]</code></pre>
</div>
<div id="quality-control-plots"
class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>1. Quality control
plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3 class="tabset">Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples
<em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt, lab_size = 3)</code></pre>
<p><img src="figure/ANOVA.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se, lab_size = 3)</code></pre>
<p><img src="figure/ANOVA.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3 class="tabset">Distributions</h3>
<p>This plot is useful for checking the distributions of the samples
<em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/ANOVA.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/ANOVA.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3 class="tabset">Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and
before User define</em> filters. Notice that the percentages of
missingness are provided in the data. These are accurate to 1 decimal
place. Also the dataset is arranged by columns with most
missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/ANOVA.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/ANOVA.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3 class="tabset">Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the
dispersion of data points in a data series around the mean. The
coefficient of variation represents the ratio of the standard deviation
to the mean, and it is a useful statistic for comparing the degree of
variation from one data series to another, even if the means are
drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/ANOVA.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/ANOVA.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2">
<h2><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within
the 2 conditions or in common between all condition.</p>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 4

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = rev(define_set),
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;#df57bc&quot;, &quot;#3fa7d6&quot;, &quot;#60d394&quot;, &quot;#c5d86d&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry
proteomics experiment would provide absolute abundance measurements for
a whole proteome, across varying conditions. Unfortunately, this ideal
is not realized. Measurements are made on peptides requiring an
inferential step to obtain protein level estimates. The inference is
complicated by experimental factors that necessitate relative abundance
estimation and result in widespread non-ignorable missing data. Relative
abundance on the log scale takes the form of parameter contrasts. In a
complete-case analysis, contrast estimates may be biased by missing data
and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_filt &lt;- MakeMeanImputation(data_filt)

data_imputed &lt;- impute(data_filt, fun = &quot;man&quot;, shift = 1.8, scale = 0.3)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/ANOVA.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics
are used for the differential enrichment analysis (or differential
expression analysis) and False discovery rate (FDR) is ajusted with
<em>Benjamini-Hochberg</em> procedure. Significant proteins are are
define with <strong>FDR = 0.05</strong> and <strong>Fold change =
1</strong></p>
<pre class="r"><code>data_diff &lt;- anova_test(data_filt)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>1338</strong> significant proteins.</p>
<p>The results from the previous analysis can be easily visualized by a
number of functions. These visualizations assist in the determination of
the optimal cutoffs to be used, highlight the most interesting samples
and contrasts, and pinpoint differentially enriched/expressed
proteins.</p>
<div id="visualization-of-the-results"
class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>4.1. Visualization of the
results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data.
This can be very useful to observe batch effects, such as clear
differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/ANOVA.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4 tabset tabset-fade">
<h4 class="tabset tabset-fade">Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant
proteins (rows) in all samples (columns). This allows to see general
trends, for example if one sample or replicate is really different
compared to the others. Additionally, the clustering of samples
(columns) can indicate closer related samples and clustering of proteins
(rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = T, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5, clustering_distance = &quot;euclidean&quot;)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = T, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>table_info &lt;- plot_heatmap(dep, type = &quot;centered&quot;, kmeans = T, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5, plot = F)</code></pre>
<div id="cluster-1" class="section level5">
<h5>Cluster 1</h5>
<pre class="r"><code>ClusterPlot(table_info, 1, &quot;#6d6875&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-2" class="section level5">
<h5>Cluster 2</h5>
<pre class="r"><code>ClusterPlot(table_info, 2, &quot;#b5838d&quot;) </code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-3" class="section level5">
<h5>Cluster 3</h5>
<pre class="r"><code>ClusterPlot(table_info, 3, &quot;#e5989b&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-4" class="section level5">
<h5>Cluster 4</h5>
<pre class="r"><code>ClusterPlot(table_info, 4, &quot;#ffb4a2&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-5" class="section level5">
<h5>Cluster 5</h5>
<pre class="r"><code>ClusterPlot(table_info, 5, &quot;#ffcdb2&quot;)</code></pre>
<p><img src="figure/ANOVA.Rmd/cluster-5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by
p.value. Proteins that are UP regulated are colored with a red box in
Fold change column, in the orhter hand proteins that are DOWN regulated
are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  #select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(significant) %&gt;%
  select(name, significant) %&gt;% 
  # view()
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:2)),
  pageLength = 10))</code></pre>
<div id="htmlwidget-2a0dd1ba32e0541469b5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a0dd1ba32e0541469b5">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879","880","881","882","883","884","885","886","887","888","889","890","891","892","893","894","895","896","897","898","899","900","901","902","903","904","905","906","907","908","909","910","911","912","913","914","915","916","917","918","919","920","921","922","923","924","925","926","927","928","929","930","931","932","933","934","935","936","937","938","939","940","941","942","943","944","945","946","947","948","949","950","951","952","953","954","955","956","957","958","959","960","961","962","963","964","965","966","967","968","969","970","971","972","973","974","975","976","977","978","979","980","981","982","983","984","985","986","987","988","989","990","991","992","993","994","995","996","997","998","999","1000","1001","1002","1003","1004","1005","1006","1007","1008","1009","1010","1011","1012","1013","1014","1015","1016","1017","1018","1019","1020","1021","1022","1023","1024","1025","1026","1027","1028","1029","1030","1031","1032","1033","1034","1035","1036","1037","1038","1039","1040","1041","1042","1043","1044","1045","1046","1047","1048","1049","1050","1051","1052","1053","1054","1055","1056","1057","1058","1059","1060","1061","1062","1063","1064","1065","1066","1067","1068","1069","1070","1071","1072","1073","1074","1075","1076","1077","1078","1079","1080","1081","1082","1083","1084","1085","1086","1087","1088","1089","1090","1091","1092","1093","1094","1095","1096","1097","1098","1099","1100","1101","1102","1103","1104","1105","1106","1107","1108","1109","1110","1111","1112","1113","1114","1115","1116","1117","1118","1119","1120","1121","1122","1123","1124","1125","1126","1127","1128","1129","1130","1131","1132","1133","1134","1135","1136","1137","1138","1139","1140","1141","1142","1143","1144","1145","1146","1147","1148","1149","1150","1151","1152","1153","1154","1155","1156","1157","1158","1159","1160","1161","1162","1163","1164","1165","1166","1167","1168","1169","1170","1171","1172","1173","1174","1175","1176","1177","1178","1179","1180","1181","1182","1183","1184","1185","1186","1187","1188","1189","1190","1191","1192","1193","1194","1195","1196","1197","1198","1199","1200","1201","1202","1203","1204","1205","1206","1207","1208","1209","1210","1211","1212","1213","1214","1215","1216","1217","1218","1219","1220","1221","1222","1223","1224","1225","1226","1227","1228","1229","1230","1231","1232","1233","1234","1235","1236","1237","1238","1239","1240","1241","1242","1243","1244","1245","1246","1247","1248","1249","1250","1251","1252","1253","1254","1255","1256","1257","1258","1259","1260","1261","1262","1263","1264","1265","1266","1267","1268","1269","1270","1271","1272","1273","1274","1275","1276","1277","1278","1279","1280","1281","1282","1283","1284","1285","1286","1287","1288","1289","1290","1291","1292","1293","1294","1295","1296","1297","1298","1299","1300","1301","1302","1303","1304","1305","1306","1307","1308","1309","1310","1311","1312","1313","1314","1315","1316","1317","1318","1319","1320","1321","1322","1323","1324","1325","1326","1327","1328","1329","1330","1331","1332","1333","1334","1335","1336","1337","1338"],["AARS","AATF","ABCC1","ABCE1","ABT1","ACADVL","ACAT1","ACLY","ACO2","ACP1","ACTA1","ACTG1","ACTN1","ACTN4","ACTR1A","ACTR2","ADAM10","ADAMTS1","ADH5","ADNP","AFG3L2","AGPS","AGRN","AHCTF1","AHCY","AHNAK","AHNAK2","AIFM1","AIMP1","AK2","AKAP12","AKAP8","AKAP8L","ALCAM","ALDH18A1","ALDH6A1","ALDH7A1","ALDH9A1","ALDOA","ALDOC","ALYREF","ANKRD17","ANKRD20A3","ANLN","ANP32A","ANP32E.1","ANXA1","ANXA11","ANXA2","ANXA3","ANXA4","ANXA5","ANXA6","ANXA7","AP2B1","AP3B1","AP3D1","AP3S1","APEX1","API5","APLP2","APMAP","APP","AQR","ARCN1","ARF3","ARF4","ARFGEF3","ARGLU1","ARHGDIA","ARPC2","ARPP19","ASPH","ATAD3A","ATG9A","ATL3","ATP1A1","ATP1B1","ATP1B3","ATP2A2","ATP5A1","ATP5B","ATP5D","ATP5F1","ATP5H","ATP6V1H","ATRX","ATXN2","ATXN2L","AXL","B2M","BAG2","BAG6","BAIAP2","BASP1","BAZ2A","BCAP31","BCLAF1","BCLAF1.1","BIN1","BMS1","BOP1","BPTF","BRD2","BRD4","BSG","BST2","BTF3","BTF3L4","BUB3","BYSL","BZW1","C11orf98","C14orf166","C15orf52","C17orf85","C1QBP","C5orf24","C9orf114","CACYBP","CAD","CADM1","CADM4","CALD1","CALR","CALU","CAND1","CANX","CAP1","CAPN2","CAPRIN1","CAPZA1","CAPZB","CASC4","CAST","CAV1","CBX1","CBX3","CBX5","CCAR1","CCAR2","CCDC124","CCDC47","CCDC59","CCNT1","CCT2","CCT3","CCT4","CCT5","CCT6A","CCT7","CCT8","CD109","CD276","CD3EAP","CD44","CD46","CD46.1","CD47","CD55","CD58","CD63","CD81","CDC37","CDC42","CDC5L","CDH2","CDK1","CDK11A","CDKN2AIP","CDV3","CEBPZ","CELF1","CENPF","CEP131","CEP170","CEP290","CES2","CFL1","CFL2","CHAMP1","CHCHD3","CHD1","CHD4","CHERP","CHMP1A","CHORDC1","CHST3","CIRBP","CISD1","CKAP4","CKAP5","CLIC1","CLIC4","CLINT1","CLTA","CLTB","CLTC","CLUH","CMPK1","CNN2","CNOT1","CNP","CNPY2","CNTNAP3B","COIL","COL12A1","COL18A1","COL1A1","COL4A2","COPA","COPB1","COPG1","CORO1C","COTL1","COX4I1","COX5A","CPD","CPNE1","CPSF4","CPSF6","CPSF7","CRKL","CROCC","CS","CSDE1","CSTB","CSTF1","CSTF2","CSTF2T","CSTF3","CTGF","CTNNA1","CTNND1","CTR9","CTSB","CTTN","CWC15","CWC22","CYB5B","CYB5R3","CYC1","CYR61","DAG1","DAP","DARS","DAZAP1","DBI","DBN1","DBT","DCAF13","DCD","DCTN1","DCTN2","DDB1","DDOST","DDX1","DDX10","DDX17","DDX18","DDX21","DDX24","DDX27","DDX39B","DDX3X","DDX42","DDX46","DDX47","DDX5","DDX52","DDX54","DDX56","DDX6","DENR","DGCR8","DHRS2","DHX15","DHX30","DHX36","DHX9","DIABLO","DIDO1","DIMT1","DKC1","DLD","DNAJA1","DNAJA2","DNAJB1","DNAJB11","DNAJC1","DNAJC10","DNAJC5","DNAJC8","DNAJC9","DNM1L","DNMT1","DNTTIP2","DPYSL2","DPYSL3","DRG1","DSTN","DTYMK","DYNC1H1","DYNC1I2","DYNC1LI2","DYNLL2","EBNA1BP2","ECE1","ECHS1","EDF1","EEA1","EEF1A1","EEF1B2","EEF1D","EEF1G","EEF2","EFHD2","EFTUD2","EGFR","EHD4","EIF1","EIF1AX","EIF2AK2","EIF2S1","EIF2S2","EIF2S3","EIF3B","EIF3F","EIF3I","EIF3J","EIF4A1","EIF4A3","EIF4B","EIF4E","EIF4ENIF1","EIF4G2","EIF4H","EIF5","EIF5A","EIF5B","EIF6","ELAVL1","ENO1","ENSA","EPB41L2","EPS15","EPS8L2","ERAL1","ERH","ERI1","ERO1L","ERP29","ERP44","ESYT1","ETFB","EWSR1","EXOSC10","EZR","F3","FABP5","FAM120A","FAM129B","FAM208A","FAM98A","FAM98B","FARSB","FASN","FBL","FEN1","FH","FKBP1A","FKBP2","FKBP3","FKBP9","FLNA","FLNB.1","FMR1","FN1","FRAS1","FSCN1","FSTL1","FTH1","FTSJ3","FUBP1","FUBP3","FUS","FXR1","FXR2","G6PD","GADD45GIP1","GALNT2","GANAB","GAPDH","GAR1","GARS","GART","GCN1L1","GDI2","GGH","GIGYF2","GLG1","GLS","GLS.1","GLTSCR2","GLUD1","GMPS","GNAI2","GNAI3","GNAS","GNB2","GNB2L1","GNL2","GNL3","GNL3L","GNS","GOLGB1","GOLIM4","GOLM1","GOT2","GPATCH4","GPATCH8","GPC1","GPC4","GPC6","GPI","GRPEL1","GRSF1","GRWD1","GSTP1","GTPBP4","H1FX","H2AFV","H2AFY","HADH","HADHA","HADHB","HCCS","HDAC2","HDGF","HDGFRP3","HDLBP","HEATR1","HERC5","HEXIM1","HIRIP3","HIST1H1B","HIST1H1C","HIST1H2AJ","HIST1H2BL","HIST1H4A","HIST2H2AC","HIST2H3A","HIST2H3PS2","HLA-A","HLA-B","HM13","HMGA1","HMGB1","HMGB2","HMGB3","HMGCS1","HMGN5","HMOX2","HNRNPA0","HNRNPA1","HNRNPA2B1","HNRNPA3","HNRNPAB","HNRNPC","HNRNPC.1","HNRNPD","HNRNPDL","HNRNPF","HNRNPH1","HNRNPH2","HNRNPH3","HNRNPK","HNRNPK.1","HNRNPL","HNRNPLL","HNRNPM","HNRNPR","HNRNPR.1","HNRNPU","HNRNPUL1","HNRNPUL2","HOXB6","HP1BP3","HSD17B10","HSD17B4","HSP90AA1","HSP90AB1","HSP90B1","HSPA1A","HSPA4","HSPA5","HSPA6","HSPA8","HSPA9","HSPB1","HSPD1","HSPE1","HSPG2","HSPH1","HTATSF1","HUWE1","HYOU1","IARS","IDH1","IDH2","IGF2BP2","IGF2BP3","IGFBP3","IGFBP6","IGFBP7","IK","IKBIP.1","IL6ST","ILF2","ILF3.1","IMPDH2","INF2","INO80E","INPP5F","IPO5","IPO7","IQGAP1","ISCU","ITGA3","ITGA5","ITGA6","ITGAV","ITGB1","ITGB4","IWS1","JUP","KANK2","KARS","KDM4B","KHDRBS1","KHSRP","KIAA0020","KIAA0319L","KIAA1524","KIAA1598","KIF23","KIF2C","KIF4A","KIF5B","KLC1","KMT2A","KPNA2","KPNA3","KPNA4","KPNB1","KRI1","KRR1","KRT18","KTN1","L1CAM","L1RE1","LAMA3","LAMA5","LAMB1","LAMB2","LAMB3","LAMC1","LAMC2","LAMP1","LAMP2","LAP3","LARP1","LARP4","LARP4B","LARP7","LDHA","LDHB","LDLR","LEO1","LEPRE1","LGALS1","LGALS3BP","LGMN","LIMA1","LIN7C","LLPH","LMAN1","LMNA","LMNA.1","LMNB1","LMNB2","LNPEP","LOXL2","LRP8","LRPAP1","LRPPRC","LRRC47","LRRC59","LRRFIP1","LRRFIP1.1","LSM14A","LSM2","LSM8","LUC7L","LUC7L2","LUC7L3","LUZP1","LYAR","M6PR","MACF1.1","MANF","MAP1B","MAP2","MAP4","MAP7D1","MAP7D3","MAPRE1","MARCKSL1","MATR3","MBNL1","MCM5","MCM6","MCM7","MDH2","MDN1","MECP2","MEPCE","MET","METAP1","MEX3C","MEX3D","MICA","MINOS1","MISP","MKI67","MKRN2","MLEC","MMAB","MNS1","MOV10","MPHOSPH10","MRPL12","MRPL13","MRPL2","MRPL27","MRPL28","MRPL3","MRPL37","MRPL39","MRPL41","MRPL42","MRPL43","MRPL45","MRPL49","MRPL51","MRPL57","MRPS18A","MRPS28","MRPS30","MRPS31","MRPS35","MRPS9","MSI2","MSN","MTHFD1","MTHFD1L","MTHFSD","MYBBP1A","MYEF2","MYH10","MYH14","MYH9","MYL12B","MYL6","MYO1B","MYOF","NAA15","NACA","NAP1L1","NAP1L4","NARS","NASP","NAT10","NCAPG","NCAPH","NCBP1","NCEH1","NCL","NCSTN","NDC80","NDUFS1","NEB","NEBL","NENF","NGDN","NHP2","NID2","NIFK","NIPBL","NKRF","NME1","NME2","NMT1","NNMT","NOC3L","NOL11","NOL6","NOL8","NOLC1","NOMO1","NONO","NOP14","NOP16","NOP2","NOP58","NOP9","NPLOC4","NPTN","NRP1","NSA2","NSDHL","NSFL1C","NSUN5","NUCB1","NUCB2","NUDC","NUDT16L1","NUDT21","NUFIP2","NUMA1","NUP133","NUP153","NUP160","NUP214","NUP50","NUP98","NUSAP1","NUTF2","NXF1","ODF2","OGDH","OLA1","OTUD4","P09493-3","P09493-5","P0DP25","P4HA1","P4HB","PA2G4","PABPC1","PABPC4","PABPN1","PAF1","PAICS","PALLD","PARK7","PAWR","PAX8","PCBD1","PCBP1","PCM1","PCNA","PCNP","PDAP1","PDCD11","PDCD5","PDCD6IP","PDIA3","PDIA4","PDIA6","PDLIM5","PDLIM7","PEBP1","PELO","PES1","PFN1","PFN2","PGAM1","PGAM5","PGD","PGK1","PGRMC1","PGRMC2","PHB","PHB2","PHF3","PHF5A","PHF6","PHGDH","PHIP","PHLDB1","PKM","PLAU","PLAUR","PLEC","PLIN3","PLOD1","PLS3","PLXNB2","PNISR","PNN","PNO1","PODXL","PODXL2","POLDIP3","POLR1E","POLR2A","POLR2B","POLR2H","POLRMT","POP1","PPA1","PPAN","PPFIA3","PPIA","PPIA.1","PPIB","PPID","PPIE","PPIG","PPIL4","PPM1G","PPP1CC","PPP1R10","PPP1R12A","PPP2R1A","PPP2R2A","PPP2R5D","PRC1","PRDM15","PRDX1","PRDX2","PRDX3","PRDX4","PRDX5","PRDX6","PRKCSH","PRKDC","PRMT1","PRNP","PROSC","PRPF19","PRPF3","PRPF4","PRPF40A","PRPF4B","PRPF6","PRPF8","PRRC2A","PRRC2B","PRRC2C","PSIP1","PSMA2","PSMA3","PSMA4","PSMA5","PSMA7","PSMB6","PSMB7","PSMC1","PSMC2","PSMC3","PSMC4","PSMC5","PSMC6","PSMD1","PSMD11","PSMD12","PSMD2","PSMD3","PSMD4","PSMD6","PSMD7","PSME1","PSPC1","PTBP1","PTBP3","PTCD1","PTGES3","PTGIS","PTPRB","PTPRF","PTPRG","PTPRJ","PTRF","PUM1","PUM1.1","PUM2","PURA","PURB","PVR","PVRL3","PWP2","PYCR2","PYGB","PYGL","Q15366-3","QKI","R3HDM1","RAB10","RAB11B","RAB11FIP1","RAB14","RAB1A","RAB5C","RAB6A","RAB7A","RAB8A","RACGAP1","RAD50","RALY","RAN","RANBP1","RANBP2","RANGAP1","RAP1B","RARS","RAVER1","RBBP4","RBBP6","RBBP7","RBFOX2","RBM10","RBM12","RBM12B","RBM14","RBM15","RBM15B","RBM17","RBM19","RBM22","RBM25","RBM27","RBM28","RBM3","RBM34","RBM39","RBM4","RBM47","RBM5","RBM6","RBM7","RBM8A","RBMS1","RBMS2","RBMX","RBMX2","RBMXL1","RBPMS","RCC1","RCN1","RCN2","RDX","REEP5","REPS1","RFC1","RHOA","RHOC","RIF1","RNH1","RNPS1","RPF2","RPL10A","RPL11","RPL12","RPL13","RPL14","RPL15","RPL17","RPL18","RPL18A","RPL22","RPL23","RPL23A","RPL26","RPL27","RPL28","RPL3","RPL30","RPL36AL","RPL38","RPL4","RPL5","RPL7A","RPL7L1","RPL8","RPL9","RPLP0","RPLP2","RPN1","RPN2","RPS12","RPS13","RPS14","RPS15","RPS15A","RPS16","RPS17","RPS18","RPS19","RPS2","RPS20","RPS21","RPS24","RPS25","RPS26","RPS27","RPS27A","RPS28","RPS3","RPS3A","RPS5","RPS6","RPSA","RRBP1","RRP1","RRP12","RRP1B","RRP36","RRP9","RRS1","RSL1D1","RTCB","RTF1","RTN4","RUVBL1","RUVBL2","S100A10","S100A11","S100A16","S100A2","S100A3","S100A4","S100A6","SAFB","SAFB2","SAP18","SAP30BP","SARS","SART1","SART3","SCAF1","SCAF8","SCARB2","SCP2","SDAD1","SDC1","SDC2","SDC4","SDF4","SDHA","SDK2","SEC22B","SEC62","SEC63","SELH","SEPT11","SERBP1","SERBP1.1","SERPINB1","SERPINB6","SERPINH1","SERPINI2","SET","SETD2","SF1","SF3A1","SF3A2","SF3B1","SF3B2","SF3B3","SF3B4","SFN","SFPQ","SH3GL1","SH3GLB2","SHMT2","SKIV2L2","SKP1","SLC1A5","SLC25A24","SLC25A6","SLC26A2","SLC38A2","SLC39A10","SLC39A14","SLC39A6","SLC4A7","SLC9A3R1","SLIRP","SLK","SLTM","SMARCC2","SMC1A","SMC2","SMC3","SMC4","SMCHD1","SMNDC1","SND1","SNRNP200","SNRNP70","SNRPA","SNRPB","SNRPB2","SNRPD2","SNRPF","SNRPG","SNW1","SOD1","SON","SORBS2","SORD","SP100","SP3","SPANXB1","SPATS2L","SPEN","SPINT2","SPTAN1","SPTBN1","SQSTM1","SREK1","SRP14","SRP68","SRP72","SRP9","SRPK1","SRPRB","SRRM1","SRRM2","SRSF1","SRSF10","SRSF11","SRSF2","SRSF3","SRSF4","SRSF5","SRSF6","SRSF7","SRSF9","SSB","SSBP1","SSRP1","ST13","STAM","STAT1","STAU1","STAU2","STIM1","STIP1","STMN1","STMN2","STOML2","STRAP","STRBP","STT3A","STUB1","STX7","SUB1","SUGP2","SUMO1","SUPT16H","SUPT6H","SURF6","SWAP70","SYNCRIP","SYNE1","SYPL1","TACC1","TAF15","TAGLN","TAGLN2","TALDO1","TARDBP","TARS","TBCA","TBCB","TBL3","TCEA1","TCERG1","TCP1","TFB2M","TFRC","TGOLN2","THRAP3","TIA1","TIAL1.1","TIMM44","TIMM50","TIMP1","TIMP2","TINAGL1","TJP1","TKT","TLN1","TMED10","TMED9","TMEM109","TMEM43","TMPO","TMPO.1","TMSB10","TMSB15B","TMSB4X","TMX1","TMX3","TNFRSF10D","TNPO1","TNRC6B","TNS1","TOP2A","TOP2B","TOR1AIP1.1","TP53BP1","TPD52","TPD52L2","TPI1","TPM3.1","TPM4","TPPP3","TPT1","TPX2","TRA2A","TRA2B","TRAP1","TRIM25","TRIP12","TRIP13","TRMT1","TRMT1L","TSN","TUBA1B","TUBA4A","TUBB","TUBB4B","TUBB6","TUBG1","TUFM","TWF1","TWSG1","TXN","TXNDC12","TXNDC15","TXNDC5","TXNL1","TXNRD1","U2AF2","U2SURP","UBA1","UBAP2","UBAP2L","UBE2L3","UBE2V2","UBR4","UBTF","UBXN1","UFL1","UGDH","UGGT1","UHRF1","UPF1","UQCRC1","URB1","USP10","USP14","USP36","UTP11L","UTP14A","UTP15","UTP18","UTP20","UTP3","UTP6","VAMP3","VAPA","VAPB","VARS","VASP","VCAN","VCL","VCP","VDAC1","VDAC2","VIM","VPS35","VWA1","WARS","WBP11","WDR1","WDR3","WDR33","WDR46","WDR74","WDR75","WWC2","XIRP2","XPO1","XPO5","XRCC5","XRCC6","XRN2","YARS","YARS2","YBX1","YBX2","YBX3","YLPM1","YTHDC1","YTHDC2","YTHDF1","YTHDF2","YTHDF3","YWHAB","YWHAE","YWHAG","YWHAH","YWHAQ","YWHAZ","ZC3H11A","ZC3H13","ZC3H14","ZC3H15","ZC3H18","ZC3H4","ZC3H7A","ZC3H7B","ZC3HAV1","ZCCHC3","ZCCHC7","ZCCHC8","ZCRB1","ZFC3H1","ZFR","ZNF326","ZNF622","ZNF638","ZRANB2"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>significant<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF"
class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a
href="GO:CC" class="uri">GO:CC</a>) and a KEGG ontology.<br />
The enrichment results are visualized with an interactive
<strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="gostplot" class="section level3">
<h3>Gostplot</h3>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ed4906484ed42468d86d" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed4906484ed42468d86d">{"x":{"data":[{"x":[81.4050222982064,171.315254791692,117.814579288437,116.052221741021,81.742495020052,118.778251394152,109.928966687978,129.689869400492,119.055728965447,81.2512847249212,93.6665311919297,119.051979268537,120.053148343346,126.813851870986,75.5779933010058,109.685236388867,165.600716701773,109.962713960163,83.3473652972733,182.414357643503,81.3750247229312,81.420021085844,109.156529124643,131.32098755608,86.1296404040448,120.068147130984,131.092256044606,104.821879497381,150.665673911651,103.378246187264,120.079396221712,165.634463973958,81.3675253291125,86.1258907071354,116.104717497753,81.5362616900353,146.548506705135,97.7949474891742,81.3712750260218,165.593217307955,120.02690046498,128.103747607818,128.088748820181,72.9869527366135,93.5540402846479,159.424965891999,136.311834142485,131.084756650788,83.0961356043438,128.107497304728,131.096005741516,118.78950048488,119.78317016587,118.793250181789,139.052862583253,109.913967900341,129.626124553033,129.622374856123,116.100967800843,131.294739677714,96.6887869009025,132.442146931989,72.9907024335229,106.93670855428,90.2093106414671,103.381995884174,109.951464869434,120.064397434074,88.1957234011217,130.934768774412,126.082660973654,171.319004488602,107.116694005931,116.05597143793,131.328486949898,120.075646524802,86.122141010226,81.4087719951158,131.081006953878,131.410980281905,107.675398845431,101.927113483328,129.693619097402,143.799978870548,104.825629194291,126.596369450241,97.442475979691,136.626808682874,128.09249851709,83.5086022643773,130.653541506207,171.322754185511,90.2955536703832,119.996902889705,96.6775378101744,164.892023985898,89.8268415567087,93.3253087731748,109.85397274979,103.385745581083,81.3487768445655,93.6627814950204,109.160278821552,94.5027136027249,131.122253619882,109.771479417784,88.5481949106049,82.7586628824982,126.821351264805,96.4788038739764,106.944207948099,89.8305912536181,90.2130603383765,131.099755438425,182.418107340412,130.942268168231,73.1631884913551,73.3544230337342,103.400744368721,88.1919737042123,88.1994730980311,90.3330506394771,142.243854653149,83.3961113570954,120.323126520822,105.440579487432,132.472144507264,107.112944309022,131.343485737536,176.467338345202,83.1448816641659,91.5816997103058,85.6309307150952,176.647323796853,176.639824403034,108.11786308074,90.2243094291047,85.7471713192864,120.000652586614,105.166851613046,91.3792160771985,122.887919206849,117.897072620444,72.8332151633283,176.459838951383,165.630714277049,171.660226907357,176.632325009215,106.482995228244,106.479245531334,170.494071168535,118.122054435007,83.1036349981626,117.67209080588,88.1807246134841,100.824702591966,120.023150768071,176.587328646302,104.585648592089,93.5765384661043,90.0480736743631,129.697368794311,81.38252411675,104.600647379727,142.626323737907,103.389495277992,129.236156074456,144.216195227491,108.61282307278,129.911101518147,115.531013870615,176.651073493762,91.0229948708059,176.658572887581,180.689497065181,126.817601567896,129.281152437368,73.1931860666302,142.622574040998,103.96319890513,109.164028518461,153.312959929684,153.2792126575,153.271713263681,153.27546296059,123.600361619634,72.4469963816605,101.345910462372,80.5050950399515,169.804126937206,83.3736131756391,125.538954921792,86.1183913133166,80.9925556381729,80.9288107907132,83.5161016581961,129.989845153244,83.5123519612867,104.608146773546,83.212376208535,146.263529740021,126.360138544949,164.888274288988,75.5854926948246,82.4924344019311,174.888715946346,139.266595307089,173.092611126746,168.375492414726,142.60757525336,94.4614669367216,129.907351821237,103.059521949966,152.390534489973,117.904572014262,142.615074647179,154.351625973587,85.6384301089139,186.46403030565,119.786919862779,116.010975075018,131.324737252989,72.4432466847511,104.8293788912,117.893322923534,87.4757815945178,73.1556890975363,94.2214863345203,84.2697907369846,132.554637839271,129.284902134278,137.369248670935,83.4148598416424,73.1894363697208,146.26727943693,131.305988768442,144.542418858608,128.111247001637,132.438397235079,103.396994671811,144.129952198575,88.1882240073029,90.0518233712725,105.178100703774,120.368122883735,132.689626928009,73.5269090915664,104.623145561183,132.633381474368,106.947957645009,118.796999878699,83.8535743800417,73.1781872789926,90.0780712496382,100.918445014701,81.5100138116695,103.063271646875,174.896215340165,131.287240283895,80.9325604876226,129.896102730509,90.2993033672926,158.513789543016,109.763980023965,115.399774478786,72.7769697096873,93.659031798111,105.174351006865,127.35005852903,88.1769749165747,148.768327275497,88.1844743103935,133.754540850277,142.611324950269,183.029307936644,168.390491202364,101.192172889087,90.0555730681819,130.728535444395,73.5381581822946,81.3525265414749,121.785508315487,128.804940929875,130.736034838214,81.5400113869446,103.40449406563,73.2231836419054,73.2269333388148,72.5107412291202,100.895946833244,101.218420767452,131.088506347697,85.9271567709374,126.135156730386,103.873206179304,132.49839238563,84.1160531636994,82.7699119732264,123.596611922725,127.957509428352,83.5423495365619,104.630644955002,119.794419256598,103.393244974902,82.06496895426,90.7942633593328,100.85844986415,131.350985131355,81.3862738136594,120.341875005369,160.617369509187,81.2925313909245,83.3511149941827,106.81296855627,119.134472600544,74.5880733169254,88.2032227949405,118.208297463923,74.584323620016,83.051139241431,83.0061428785183,153.811669618634,93.9402590663156,169.991611782676,167.104345162441,84.4647749762731,155.367793836033,96.5462984183455,74.3443430178147,83.3211174189075,131.482224523184,144.981133397007,73.5456575761134,129.899852427419,160.609870115368,103.453240125452,109.651489116683,157.8688416746,142.247604350058,110.153948502542,105.459327971979,112.868729064944,83.4073604478236,88.4207052156854,157.872591371509,130.642292415479,88.5556943044237,156.072736854999,100.929694105429,146.690995187692,73.1669381882645,126.393885817134,83.8835719553168,109.344013970112,103.58822921419,129.95609788106,103.415743156358,132.487143294902,131.347235434445,140.93521043177,176.928551065057,166.009433664898,88.4244549125948,150.73691815293,100.892197136335,142.240104956239,110.258940016005,186.426533336556,83.3023689343605,81.2962810878339,87.6482676523499,91.8404287970541,128.070000335634,155.337796260758,101.195922585996,90.7980130562422,146.256030346202,121.613022257655,103.535733457459,130.938518471321,131.388482100449,73.1069430377141,130.799779685674,100.907195923972,103.52823406364,105.436829790522,106.254263716771,80.5838386750488,104.83312858811,131.455976644818,151.554352079178,152.949239329473,92.455379090195,74.348092714724,126.648865206973,155.334046563848,130.721036050576,87.3107949305044,106.831717040817,169.999111176494,112.928724215495,132.014681484318,95.2414038938759,127.421302770308,81.2175374527366,96.5500481152549,156.841424721425,131.309738465351,186.392786064372,129.903602124328,91.4729584999334,84.4610252793638,150.48568846,104.720637680828,157.857592583872,139.004116523431,122.407958002446,182.538097641513,150.740667849839,172.368919623232,103.460739519271,103.22075891707,130.803529382583,80.3476077697569,142.131363745867,184.101721252731,137.155515947099,142.618824344088,115.842238714095,148.243369708182,107.225435216304,132.637131171277,98.6498783845164,96.5537978121643,103.539483154368,129.95234818415,72.840714557147,119.809418044235,131.264742102439,153.822918709362,135.993109905186,101.709631062583,101.3571595531,184.330452764204,135.175675978938,123.105401627594,84.9934822404979,72.4844933507545,108.485333377861,86.388369490793,120.045648949527,84.6822573970181,129.937349396513,73.200685460449,73.6768969679422,93.5915372537418,85.5334385954509,132.948356014757,123.097902233775,131.365983918992,101.713380759493,110.157698199451,103.678221940016,87.232051295407,75.8292229939353,101.360909250009,131.279740890076,104.840627981928,72.6457303178585,168.817956650035,171.352751760786,128.489966389486,152.435530852886,138.992867432703,117.9195708019,186.396535761281,139.206600156538,138.876626828512,77.884056900284,102.22333953917,132.022180878137,116.997145362189,75.6754854206501,90.3368003363865,115.403524175696,87.8919979514607,128.182491242915,74.2993466549019,83.5011028705585,108.680317617149,171.349002063877,170.464073593259,83.287370146723,102.823291044674,83.0473895445216,80.3738556481226,139.000366826522,131.37723300972,73.039448493345,109.827724871424,101.349660159281,135.989360208277,76.5904114665426,100.869698954879,85.7734191976522,83.5985949902028,72.4020000187478,106.771721890267,154.962826569818,138.88412622233,126.442631876956,89.9018354948967,131.99968269668,120.979323479967,102.898284982862,77.8915562941028,94.9639263225806,182.545597035332,119.805668347326,164.895773682807,157.86509197769,143.47375523943,145.3373546034,122.119231340423,106.261763110589,160.051165275868,89.2568876264806,116.798411425991,103.524484366731,74.1793563538012,81.7087477478674,89.8905864041685,131.395981494267,130.848525745496,139.019115311069,87.3332931119607,74.2918472610831,151.843078741201,129.877354245962,139.0716110678,98.4886414174124,152.401783580701,139.056612280163,145.637330356152,167.081846980985,90.6217773015006,129.554880311754,114.81857145783,80.936310184532,160.276147090432,159.766188310754,74.5955727107442,162.660954324807,162.657204627898,130.717286353667,118.613264730138,171.502739637162,128.313730634744,130.852275442405,115.422272660243,148.835821819866,126.288894303671,103.408243762539,109.947715172525,128.298731847107,102.913283770499,108.684067314059,130.151082120348,161.273566468331,115.384775691149,106.835466737727,80.3176101944817,180.622002520812,136.926784435626,121.099313781067,167.070597890256,104.450659503351,102.90578437668,87.070814328303,97.4387262827816,139.555321969112,87.3820391717829,123.941584038389,82.083717438807,157.482622892932,115.598508414984,130.833526957858,85.6159319274576,98.7698686856171,128.564960327674,155.930248372442,148.265867889638,86.103392525679,165.476976703763,118.10330595046,84.4910228546389,167.243083948089,100.850950470332,128.598707599858,131.515971795368,96.6812875070838,93.9065117941311,134.91319719528,153.564189622614,127.383805801214,131.103505135335,86.5833537300816,155.664019891875,155.671519285694,107.874132781629,121.628021045292,94.1802396685169,152.394284186882,89.2756361110276,123.165396778144,126.311392485127,157.876341068419,130.94601786514,101.398406219103,123.585362831997,126.637616116245,178.765902550661,103.595728608009,138.996617129612,129.243655468274,120.101894403168,118.699507759054,101.237169251999,121.624271348383,100.700962593956,123.817844040379,175.953629868614,146.289777618386,134.91694689219,169.436656640085,109.831474568334,185.736589105227,145.104873395018,169.665388151558,109.688986085777,101.162175313811,128.684950628774,88.2069724918499,73.2306830357242,131.489723917002,72.4282478971135,135.381909308955,136.420575352857,103.0332740716,76.6054102541801,104.443160109532,108.526580043864,74.2206030198046,156.080236248818,82.9648962125149,169.267920279162,128.317480331654,98.6423789906976,99.7822868511538,86.8833294828333,157.853842886962,117.889573226625,106.722975830445,155.289050200936,135.329413552223,85.1922161766959,136.555564441596,181.581924929617,80.3288592852099,93.3290584700841,101.10592986017,155.067818083281,82.9573968186962,123.349131926705,74.689315133479,148.337112130916,87.3182943243231,128.013754881993,156.158979883915,156.166479277734,131.857194214123,101.177174101449,102.437072263006,123.930334947661,76.594161163452,139.01161591725,101.803373485318,101.29716440255,169.189176644065,129.258654255912,83.1936277239881],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.692792829972,15.6417275354497,15.6335786336437,15.6274419332804,15.3658872946987,15.3005782637733,14.9901476972966,14.935188139448,14.8826141279938,14.8108901233244,14.8007500707934,14.749156970094,14.6795041558285,14.6182508845729,14.5767564178448,14.4795187944033,14.3595685440706,14.3539853668425,14.3239000078514,14.3068342063336,14.2754655175377,14.2349932578453,14.0135212828382,14.0030596807967,13.9342558843502,13.6861412893446,13.6654153789167,13.6076353207655,13.5805022224533,13.5655866441149,13.441967465744,13.2166980469171,13.1195210941316,13.086559316053,13.0530138090503,12.9939960412652,12.9887154379843,12.9476299857215,12.9274770179058,12.9171578137255,12.8955509659005,12.7524604389372,12.5977049453186,12.5956330530859,12.2228824847472,12.1830586604012,11.9515725109091,11.9041939548363,11.8394287232785,11.6915532904736,11.6510779799823,11.6109194754334,11.5346170394111,11.3471373263009,11.2669498960871,11.2654091418933,11.2291044435028,11.2291044435028,11.1947999528763,11.186403578442,11.1666902226545,11.0844249161331,10.77772470169,10.694202091676,10.6592068808579,10.5988290605955,10.5341988396629,10.5277407065446,10.4886852549857,10.4616574785828,10.4616574785828,10.4507505801584,10.3934848508405,10.3816566177139,10.3699184690353,10.3387611628382,10.3271384775122,10.3267578796903,10.3006212611337,10.1908852221423,10.1388686248754,10.0479343804543,9.94154933417345,9.90282217528882,9.73093514610028,9.68762101180527,9.65773698012575,9.57939746819407,9.54690445861389,9.49014376039276,9.46437744449075,9.46004412100446,9.43062601707496,9.4197730897082,9.38119107790895,9.33186940976053,9.3313405420419,9.30217052604765,9.25606669621285,9.15583790443992,9.11872355607137,9.10707602014522,9.00260410150477,8.96909568841439,8.90218205966669,8.90218205966669,8.90218205966669,8.90218205966669,8.87912551563754,8.84789874427057,8.79584713522262,8.79100059679257,8.74274487053882,8.63762808400263,8.62883533025298,8.51030317053754,8.50535997752117,8.49666698600945,8.47710052763981,8.40354382214945,8.39580813738928,8.34080434134258,8.26904081710612,8.22919085128847,8.22867852401951,8.21043860660971,8.15475754972472,8.13736987833014,8.12867478654813,8.09325144432248,8.00120955531211,7.957070613819,7.93078692536467,7.90289004017884,7.85049181221673,7.71170986926538,7.70784414193419,7.68199200254897,7.61758900630121,7.56956398014715,7.46802307771296,7.43145354906999,7.38017842483014,7.34853520783033,7.31750453042469,7.28172071774077,7.27115454914083,7.24949670199928,7.19144553702332,7.14960655696362,7.13804887319192,7.11318764135715,7.09898863990966,7.09898863990966,7.09114003686481,7.05860881045013,7.04405674965367,7.03912012544712,7.02266044494128,7.00567173768268,6.99443304942828,6.99163087449083,6.98778560062295,6.91970583839309,6.88130050623485,6.88034088963389,6.84687279782362,6.83503027650852,6.8123363966414,6.80828835618185,6.73320095518489,6.69369553729597,6.69309317420152,6.68703548204723,6.68411372560419,6.65160803082599,6.64886084466185,6.61608771540012,6.54177410227229,6.51539189017693,6.50431698003005,6.50121319256187,6.46940041003046,6.40373876162373,6.34555550303358,6.34481772834332,6.23281447247783,6.21148657530019,6.20989647902494,6.20685688962739,6.19044710052349,6.1796050084321,6.17193465919177,6.15399713291661,6.15259687437469,6.13111253810845,6.11005371272441,6.08115872166656,6.07837862028241,6.06222758107569,6.0335846792516,6.01733456082682,6.01321542393436,5.95269480464366,5.93676791262734,5.91839932379133,5.80189493525046,5.73683719409945,5.73425393898305,5.68280609080429,5.68280609080429,5.63338283868546,5.63288501890952,5.56872926280324,5.49520753353232,5.48429389078905,5.4457417877693,5.43295889612561,5.36728504595853,5.25852940724824,5.25850260356501,5.2524085648736,5.25114522013359,5.22437103367752,5.2166627668356,5.19505203507353,5.16398623194481,5.13930800297013,5.13170982872195,5.11489362666532,5.10443364757799,5.06636718259367,5.04221242283643,5.02929756391576,5.00887366276119,5.00792498579446,4.94569375892082,4.8654412532724,4.84321866125292,4.84059947239349,4.80352281788801,4.73503505330048,4.67582587424892,4.66142251047706,4.64845661732545,4.64147968404562,4.61688682177452,4.61243222704991,4.61243222704991,4.60766163779769,4.58172988196448,4.56575767659057,4.54718821384447,4.52541707728104,4.51875621461636,4.51653043969216,4.50461723827574,4.48834227327616,4.48711269028624,4.48566242136192,4.46824688868062,4.43757301985748,4.42197567312127,4.40259246734528,4.39282829023742,4.39138026962645,4.38317470434599,4.38054577835741,4.34775805887908,4.34494645660857,4.30923421379986,4.280601064909,4.27735442120847,4.2564683881356,4.24463818560349,4.2395656151414,4.23863566256377,4.23219143325633,4.22586107194333,4.21033903673727,4.1760766526382,4.16662471066682,4.16235710279636,4.16141770329638,4.12080886982047,4.09526335955967,4.0861109803152,4.06293266687412,4.05054825357148,4.04952210141687,4.03030931112039,4.02328582664545,3.99200487612422,3.92982706387797,3.92270164967198,3.92117767324615,3.8996600320543,3.86096321335134,3.85325861959207,3.85025675109308,3.81343324028677,3.80903114877238,3.79983075064392,3.79183909911285,3.79036758552206,3.77605887699846,3.77500015185341,3.74961272917569,3.7359588265563,3.72732111960086,3.70937995170289,3.70516299480957,3.69873027258261,3.68599630125777,3.6681823985937,3.65340004288394,3.65032149455082,3.64435499952918,3.61078969853247,3.56040286276769,3.55630271792411,3.54884135931063,3.52129884585231,3.45323349056409,3.44949152588331,3.44849767363868,3.43604394000485,3.43007544518971,3.40217535179051,3.35312507394447,3.34719346966958,3.33826226361897,3.29839834701762,3.29225299527337,3.21366227867167,3.19364179971467,3.19182264744155,3.19013100210733,3.19012808168527,3.17983254990958,3.17286659515091,3.16406129205691,3.16149879831296,3.15396554333158,3.1538513048931,3.15298979245957,3.15227358516347,3.14349939507284,3.13732268102136,3.12806188094575,3.12558391666851,3.10529567268675,3.09186632063633,3.084607729079,3.06235933871176,2.98786440008656,2.97851642224454,2.96499405450109,2.96336966422138,2.96309617676459,2.93338872886013,2.93273821115517,2.89775976475264,2.8967573738637,2.87482850604378,2.8741594187399,2.86997604635585,2.86939534416842,2.8370801723929,2.83422348840162,2.82649938267094,2.82649938267094,2.82006164723088,2.81973800991387,2.81480427011745,2.81466862415737,2.80865005441705,2.80865005441705,2.80727513095258,2.80393636757718,2.79937156909513,2.78792456753173,2.78339020193261,2.76923297926418,2.76550797606751,2.75800447780276,2.74279575358929,2.73330640765156,2.71464857458853,2.69813418749268,2.6868525690989,2.68035501198856,2.67181542437102,2.65785528655223,2.65639062946393,2.6504964889146,2.62804815678219,2.61735479060699,2.60806962395228,2.60614338026328,2.59822955353519,2.5973369284995,2.59613895552541,2.59313389246718,2.58674500988567,2.58358879289183,2.58344630844071,2.5799189492405,2.57940947139026,2.5769664997466,2.57086534431571,2.53803201220859,2.5317749783696,2.52990729544566,2.52668382791732,2.51620665189661,2.48927375228506,2.47978252601104,2.47746003509902,2.47608142200359,2.4724613450111,2.47103448546796,2.4655961820584,2.44766901247657,2.44739194506101,2.43734370784645,2.42437415785343,2.41288115257883,2.40929949768489,2.4068015944449,2.37887472488564,2.37592512712344,2.37479393880563,2.37312459240263,2.36659626875549,2.36596286602822,2.35969930438182,2.35895875287973,2.35660665122191,2.34831445567172,2.33282238283287,2.33217393020298,2.31998267104992,2.31660681432168,2.31600732546804,2.31143830446253,2.31088867988007,2.30648635289647,2.29941884723477,2.29741212295581,2.27567829689503,2.27338639459395,2.26824993021771,2.2618842598344,2.25872348626709,2.2508244393563,2.24123514599679,2.23556389738645,2.23109860638081,2.21731205730691,2.20255596036282,2.20234012947869,2.19894423302379,2.18753525660579,2.18752298872362,2.18272243911393,2.1771558646712,2.17706408678092,2.17200646493933,2.16630362907533,2.15541039099379,2.15175503698786,2.13423888140205,2.10861650149521,2.10752837256334,2.09787941507708,2.09787941507708,2.08548441071977,2.08103548825029,2.079936772384,2.07478559158835,2.06630158047561,2.05881540072433,2.05816138473594,2.05075726182614,2.0493389828289,2.04665590391479,2.03122034915789,2.02402214890719,2.02308969603322,2.01951373249433,1.99913316618285,1.99261968682489,1.99194185227895,1.98183625664909,1.97759158541107,1.97320021020956,1.96578124789149,1.95126353594866,1.94599999791793,1.94572153495517,1.93797747336779,1.92205605365517,1.91666141818582,1.89360082722551,1.88972490210443,1.88357776016747,1.88070692250417,1.87998697369141,1.8685035790574,1.86558317073541,1.85779113863075,1.85308694248188,1.84167398990146,1.82259117069088,1.81875481234392,1.81875332255436,1.81638327377359,1.81508443842241,1.80908222875308,1.80414458184401,1.78390117073783,1.78295403291593,1.7777557505788,1.76873055654683,1.7655383355693,1.76352151290006,1.76090063827626,1.75869411148147,1.75817506604547,1.739654496433,1.739654496433,1.73550202882804,1.72933175118399,1.71625867572442,1.71595413638949,1.71464139448605,1.71429222617877,1.7115661967379,1.69345839813717,1.6821219096379,1.67594434481063,1.66128706210441,1.65590100540932,1.65525744512682,1.64568762653404,1.64250419161593,1.63256041415462,1.61423097571646,1.60516984121788,1.59513865421746,1.58710420465158,1.57961590889233,1.57751205370563,1.5730965232315,1.55446059755287,1.55182814101583,1.5437676317433,1.5437592208057,1.53858074151638,1.53799617113373,1.5344299308349,1.53139682132645,1.52980905040367,1.52979803201199,1.52872614700626,1.52840492089185,1.52507735169456,1.52040100238905,1.51977438331575,1.512957421167,1.51234620839753,1.50981305441823,1.50723231809188,1.49724118710383,1.49436990501004,1.48984530874176,1.48548312143279,1.47329433876215,1.46755435889016,1.46741046876444,1.46721130269794,1.46281547650746,1.45235205174767,1.45154118959297,1.44871908232769,1.44453843656396,1.43935092197203,1.43446301539231,1.42809935274022,1.42326910207916,1.41856123681019,1.41856123681019,1.41230858150987,1.41035880226227,1.40695583950049,1.40631922806105,1.40229263873242,1.38816290765472,1.38131441609132,1.37771381807076,1.37771381807076,1.37771381807076,1.37628634338726,1.37441680214253,1.36977396300178,1.36882488035202,1.36283672277662,1.34608908395091,1.33916942249574,1.32841223621803,1.32700400064417,1.32571673276713,1.30881909386027],"text":["GO:0006412 (713) <br> translation <br> 2.035e-63","GO:1903311 (318) <br> regulation of mRNA metabolic process <br> 2.264e-63","GO:0043043 (738) <br> peptide biosynthetic process <br> 3.666e-61","GO:0042254 (302) <br> ribosome biogenesis <br> 7.162e-61","GO:0006518 (895) <br> peptide metabolic process <br> 4.406e-58","GO:0043484 (151) <br> regulation of RNA splicing <br> 2.294e-56","GO:0034645 (1564) <br> cellular macromolecule biosynthetic process <br> 3.730e-56","GO:0050684 (141) <br> regulation of mRNA processing <br> 1.148e-54","GO:0043604 (867) <br> amide biosynthetic process <br> 5.532e-54","GO:0006364 (225) <br> rRNA processing <br> 1.296e-53","GO:0016072 (263) <br> rRNA metabolic process <br> 2.166e-49","GO:0043603 (1165) <br> cellular amide metabolic process <br> 1.185e-47","GO:0044260 (5746) <br> cellular macromolecule metabolic process <br> 3.007e-44","GO:0048024 (105) <br> regulation of mRNA splicing, via spliceosome <br> 3.032e-44","GO:0002181 (148) <br> cytoplasmic translation <br> 4.612e-43","GO:0034470 (399) <br> ncRNA processing <br> 2.450e-40","GO:1901566 (1698) <br> organonitrogen compound biosynthetic process <br> 3.306e-38","GO:0034660 (536) <br> ncRNA metabolic process <br> 7.825e-38","GO:0006996 (3805) <br> organelle organization <br> 8.111e-35","GO:2000112 (656) <br> regulation of cellular macromolecule biosynthetic process <br> 3.997e-34","GO:0006403 (198) <br> RNA localization <br> 1.313e-32","GO:0006417 (390) <br> regulation of translation <br> 6.567e-31","GO:0034248 (446) <br> regulation of cellular amide metabolic process <br> 4.525e-30","GO:0051246 (2472) <br> regulation of protein metabolic process <br> 2.087e-28","GO:0009059 (4880) <br> macromolecule biosynthetic process <br> 2.578e-27","GO:0044267 (4796) <br> cellular protein metabolic process <br> 6.797e-27","GO:0051171 (5529) <br> regulation of nitrogen compound metabolic process <br> 6.924e-27","GO:0032268 (2309) <br> regulation of cellular protein metabolic process <br> 7.061e-27","GO:0080090 (5694) <br> regulation of primary metabolic process <br> 7.655e-27","GO:0031323 (5921) <br> regulation of cellular metabolic process <br> 1.748e-26","GO:0044271 (4734) <br> cellular nitrogen compound biosynthetic process <br> 2.689e-26","GO:1901576 (5869) <br> organic substance biosynthetic process <br> 2.938e-26","GO:0006401 (308) <br> RNA catabolic process <br> 7.303e-26","GO:0009058 (5956) <br> biosynthetic process <br> 9.898e-26","GO:0042274 (73) <br> ribosomal small subunit biogenesis <br> 1.091e-25","GO:0006457 (214) <br> protein folding <br> 1.326e-25","GO:0071840 (7994) <br> cellular component organization or biogenesis <br> 1.903e-25","GO:0019538 (5389) <br> protein metabolic process <br> 5.436e-25","GO:0006402 (261) <br> mRNA catabolic process <br> 5.557e-24","GO:1901564 (6338) <br> organonitrogen compound metabolic process <br> 8.862e-24","GO:0044249 (5794) <br> cellular biosynthetic process <br> 1.229e-23","GO:0048522 (5561) <br> positive regulation of cellular process <br> 2.909e-23","GO:0048518 (6094) <br> positive regulation of biological process <br> 3.487e-23","GO:0000380 (79) <br> alternative mRNA splicing, via spliceosome <br> 1.165e-22","GO:0016032 (419) <br> viral process <br> 9.961e-22","GO:0140694 (373) <br> non-membrane-bounded organelle assembly <br> 4.442e-21","GO:0060255 (6710) <br> regulation of macromolecule metabolic process <br> 4.572e-21","GO:0051169 (311) <br> nuclear transport <br> 1.110e-20","GO:0006913 (311) <br> nucleocytoplasmic transport <br> 1.110e-20","GO:0048523 (4818) <br> negative regulation of cellular process <br> 1.464e-20","GO:0051172 (2330) <br> negative regulation of nitrogen compound metabolic process <br> 2.946e-20","GO:0043487 (197) <br> regulation of RNA stability <br> 3.657e-20","GO:0044085 (4801) <br> cellular component biogenesis <br> 3.731e-20","GO:0043488 (184) <br> regulation of mRNA stability <br> 3.950e-20","GO:0061013 (193) <br> regulation of mRNA catabolic process <br> 9.334e-20","GO:0034641 (8767) <br> cellular nitrogen compound metabolic process <br> 1.067e-19","GO:0050658 (162) <br> RNA transport <br> 1.340e-19","GO:0050657 (162) <br> nucleic acid transport <br> 1.340e-19","GO:0042273 (72) <br> ribosomal large subunit biogenesis <br> 2.033e-19","GO:0051236 (165) <br> establishment of RNA localization <br> 3.358e-19","GO:0019222 (7219) <br> regulation of metabolic process <br> 4.211e-19","GO:0051641 (3117) <br> cellular localization <br> 5.008e-19","GO:0000381 (62) <br> regulation of alternative mRNA splicing, via spliceosome <br> 9.486e-19","GO:0033036 (3147) <br> macromolecule localization <br> 1.505e-18","GO:0010604 (3378) <br> positive regulation of macromolecule metabolic process <br> 1.846e-18","GO:0031324 (2493) <br> negative regulation of cellular metabolic process <br> 2.248e-18","GO:0034655 (435) <br> nucleobase-containing compound catabolic process <br> 4.895e-18","GO:0044265 (1135) <br> cellular macromolecule catabolic process <br> 9.065e-18","GO:0009893 (3678) <br> positive regulation of metabolic process <br> 1.072e-17","GO:0051128 (2246) <br> regulation of cellular component organization <br> 1.080e-17","GO:0046700 (473) <br> heterocycle catabolic process <br> 2.431e-17","GO:1903312 (93) <br> negative regulation of mRNA metabolic process <br> 3.055e-17","GO:0033120 (36) <br> positive regulation of RNA splicing <br> 3.629e-17","GO:0042255 (61) <br> ribosome assembly <br> 3.885e-17","GO:0051248 (1022) <br> negative regulation of protein metabolic process <br> 5.032e-17","GO:0044270 (479) <br> cellular nitrogen compound catabolic process <br> 5.681e-17","GO:0009057 (1341) <br> macromolecule catabolic process <br> 5.748e-17","GO:0006413 (126) <br> translational initiation <br> 2.029e-16","GO:0051168 (154) <br> nuclear export <br> 2.282e-16","GO:0051276 (1034) <br> chromosome organization <br> 2.325e-16","GO:0033365 (1105) <br> protein localization to organelle <br> 2.358e-16","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 4.306e-16","GO:0050685 (32) <br> positive regulation of mRNA processing <br> 5.005e-16","GO:0070727 (1963) <br> cellular macromolecule localization <br> 1.023e-15","GO:0032269 (957) <br> negative regulation of cellular protein metabolic process <br> 1.161e-15","GO:0046907 (1826) <br> intracellular transport <br> 1.310e-15","GO:0019439 (494) <br> aromatic compound catabolic process <br> 1.546e-15","GO:0060341 (805) <br> regulation of cellular localization <br> 1.582e-15","GO:0048519 (5844) <br> negative regulation of biological process <br> 1.782e-15","GO:0007049 (1713) <br> cell cycle <br> 2.092e-15","GO:0051028 (129) <br> mRNA transport <br> 2.409e-15","GO:1903313 (116) <br> positive regulation of mRNA metabolic process <br> 2.650e-15","GO:0010628 (1117) <br> positive regulation of gene expression <br> 3.315e-15","GO:0044237 (12839) <br> cellular metabolic process <br> 4.369e-15","GO:0019219 (3945) <br> regulation of nucleobase-containing compound metabolic process <br> 4.426e-15","GO:1901361 (522) <br> organic cyclic compound catabolic process <br> 4.744e-15","GO:0010467 (8909) <br> gene expression <br> 4.934e-15","GO:0015931 (225) <br> nucleobase-containing compound transport <br> 5.303e-15","GO:0034613 (1954) <br> cellular protein localization <br> 5.821e-15","GO:0031325 (3176) <br> positive regulation of cellular metabolic process <br> 9.693e-15","GO:0006396 (3356) <br> RNA processing <br> 9.930e-15","GO:0016071 (2153) <br> mRNA metabolic process <br> 1.163e-14","GO:0034249 (194) <br> negative regulation of cellular amide metabolic process <br> 2.060e-14","GO:0017148 (172) <br> negative regulation of translation <br> 2.161e-14","GO:0051179 (6425) <br> localization <br> 2.468e-14","GO:0034504 (300) <br> protein localization to nucleus <br> 2.627e-14","GO:0010033 (3009) <br> response to organic substance <br> 2.719e-14","GO:0006807 (12062) <br> nitrogen compound metabolic process <br> 3.614e-14","GO:0048026 (23) <br> positive regulation of mRNA splicing, via spliceosome <br> 6.072e-14","GO:0019058 (312) <br> viral life cycle <br> 7.594e-14","GO:0033043 (1109) <br> regulation of organelle organization <br> 8.193e-14","GO:0010468 (5388) <br> regulation of gene expression <br> 8.851e-14","GO:0010605 (3353) <br> negative regulation of macromolecule metabolic process <br> 1.014e-13","GO:0051173 (2978) <br> positive regulation of nitrogen compound metabolic process <br> 1.026e-13","GO:2000113 (258) <br> negative regulation of cellular macromolecule biosynthetic process <br> 1.128e-13","GO:0051130 (1021) <br> positive regulation of cellular component organization <br> 1.182e-13","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.210e-13","GO:0000723 (137) <br> telomere maintenance <br> 1.272e-13","GO:0031329 (850) <br> regulation of cellular catabolic process <br> 1.768e-13","GO:0009892 (3558) <br> negative regulation of metabolic process <br> 2.525e-13","GO:0009894 (989) <br> regulation of catabolic process <br> 2.537e-13","GO:0010638 (488) <br> positive regulation of organelle organization <br> 5.986e-13","GO:0065008 (3741) <br> regulation of biological quality <br> 6.561e-13","GO:0007010 (1435) <br> cytoskeleton organization <br> 1.118e-12","GO:0044403 (281) <br> biological process involved in symbiotic interaction <br> 1.247e-12","GO:0032502 (6326) <br> developmental process <br> 1.447e-12","GO:0051649 (2405) <br> establishment of localization in cell <br> 2.034e-12","GO:0033119 (26) <br> negative regulation of RNA splicing <br> 2.233e-12","GO:0051252 (3700) <br> regulation of RNA metabolic process <br> 2.450e-12","GO:1904816 (13) <br> positive regulation of protein localization to chromosome, telomeric region <br> 2.920e-12","GO:0006928 (2074) <br> movement of cell or subcellular component <br> 4.496e-12","GO:0012501 (1908) <br> programmed cell death <br> 5.408e-12","GO:0008104 (2700) <br> protein localization <br> 5.427e-12","GO:1904871 (11) <br> positive regulation of protein localization to Cajal body <br> 5.901e-12","GO:1904869 (11) <br> regulation of protein localization to Cajal body <br> 5.901e-12","GO:0033554 (1852) <br> cellular response to stress <br> 6.386e-12","GO:0010608 (1126) <br> posttranscriptional regulation of gene expression <br> 6.510e-12","GO:0008219 (2057) <br> cell death <br> 6.813e-12","GO:0044238 (12572) <br> primary metabolic process <br> 8.233e-12","GO:0032386 (341) <br> regulation of intracellular transport <br> 1.668e-11","GO:0010941 (1575) <br> regulation of cell death <br> 2.022e-11","GO:0045727 (135) <br> positive regulation of translation <br> 2.192e-11","GO:0043067 (1439) <br> regulation of programmed cell death <br> 2.519e-11","GO:0000278 (895) <br> mitotic cell cycle <br> 2.923e-11","GO:1904814 (15) <br> regulation of protein localization to chromosome, telomeric region <br> 2.967e-11","GO:1901575 (2081) <br> organic substance catabolic process <br> 3.246e-11","GO:1903405 (12) <br> protein localization to nuclear body <br> 3.454e-11","GO:1904867 (12) <br> protein localization to Cajal body <br> 3.454e-11","GO:0032880 (829) <br> regulation of protein localization <br> 3.542e-11","GO:0032879 (2701) <br> regulation of localization <br> 4.041e-11","GO:1903047 (744) <br> mitotic cell cycle process <br> 4.153e-11","GO:0043170 (12328) <br> macromolecule metabolic process <br> 4.267e-11","GO:0006915 (1857) <br> apoptotic process <br> 4.584e-11","GO:0042981 (1411) <br> regulation of apoptotic process <br> 4.708e-11","GO:0009889 (4141) <br> regulation of biosynthetic process <br> 4.712e-11","GO:0022402 (1194) <br> cell cycle process <br> 5.005e-11","GO:0044248 (2188) <br> cellular catabolic process <br> 6.443e-11","GO:1904851 (10) <br> positive regulation of establishment of protein localization to telomere <br> 7.263e-11","GO:0032200 (164) <br> telomere organization <br> 8.955e-11","GO:0016043 (7793) <br> cellular component organization <br> 1.144e-10","GO:0010556 (3926) <br> regulation of macromolecule biosynthetic process <br> 1.251e-10","GO:0050686 (30) <br> negative regulation of mRNA processing <br> 1.858e-10","GO:0006405 (82) <br> RNA export from nucleus <br> 2.053e-10","GO:0032204 (82) <br> regulation of telomere maintenance <br> 2.199e-10","GO:0070203 (11) <br> regulation of establishment of protein localization to telomere <br> 2.634e-10","GO:0031326 (4081) <br> regulation of cellular biosynthetic process <br> 2.839e-10","GO:0048856 (5718) <br> anatomical structure development <br> 3.235e-10","GO:0070925 (891) <br> organelle assembly <br> 3.433e-10","GO:0034063 (27) <br> stress granule assembly <br> 3.467e-10","GO:0050794 (11067) <br> regulation of cellular process <br> 3.710e-10","GO:0042026 (21) <br> protein refolding <br> 3.804e-10","GO:1904872 (18) <br> regulation of telomerase RNA localization to Cajal body <br> 4.157e-10","GO:0010833 (84) <br> telomere maintenance via telomere lengthening <br> 4.657e-10","GO:1904874 (15) <br> positive regulation of telomerase RNA localization to Cajal body <br> 4.663e-10","GO:1990173 (14) <br> protein localization to nucleoplasm <br> 4.987e-10","GO:0048025 (21) <br> negative regulation of mRNA splicing, via spliceosome <br> 5.545e-10","GO:0048869 (4210) <br> cellular developmental process <br> 6.985e-10","GO:0000470 (29) <br> maturation of LSU-rRNA <br> 7.608e-10","GO:0070202 (12) <br> regulation of establishment of protein localization to chromosome <br> 7.815e-10","GO:0031647 (299) <br> regulation of protein stability <br> 9.940e-10","GO:0034250 (160) <br> positive regulation of cellular amide metabolic process <br> 1.074e-09","GO:0090685 (19) <br> RNA localization to nucleus <br> 1.253e-09","GO:0090672 (19) <br> telomerase RNA localization <br> 1.253e-09","GO:0090670 (19) <br> RNA localization to Cajal body <br> 1.253e-09","GO:0090671 (19) <br> telomerase RNA localization to Cajal body <br> 1.253e-09","GO:0045935 (1897) <br> positive regulation of nucleobase-containing compound metabolic process <br> 1.321e-09","GO:0000028 (18) <br> ribosomal small subunit assembly <br> 1.419e-09","GO:0030154 (4128) <br> cell differentiation <br> 1.600e-09","GO:0006139 (7940) <br> nucleobase-containing compound metabolic process <br> 1.618e-09","GO:1902850 (150) <br> microtubule cytoskeleton organization involved in mitosis <br> 1.808e-09","GO:0007004 (71) <br> telomere maintenance via telomerase <br> 2.303e-09","GO:0046483 (8099) <br> heterocycle metabolic process <br> 2.351e-09","GO:0009056 (2547) <br> catabolic process <br> 3.088e-09","GO:0006278 (72) <br> RNA-dependent DNA biosynthetic process <br> 3.123e-09","GO:0006259 (896) <br> DNA metabolic process <br> 3.187e-09","GO:0007052 (122) <br> mitotic spindle organization <br> 3.333e-09","GO:0050821 (190) <br> protein stabilization <br> 3.949e-09","GO:0007051 (187) <br> spindle organization <br> 4.020e-09","GO:0032206 (52) <br> positive regulation of telomere maintenance <br> 4.562e-09","GO:0006950 (3780) <br> response to stress <br> 5.382e-09","GO:0071704 (13835) <br> organic substance metabolic process <br> 5.899e-09","GO:0046822 (107) <br> regulation of nucleocytoplasmic transport <br> 5.906e-09","GO:1901360 (8395) <br> organic cyclic compound metabolic process <br> 6.160e-09","GO:0002183 (35) <br> cytoplasmic translational initiation <br> 7.002e-09","GO:0006725 (8159) <br> cellular aromatic compound metabolic process <br> 7.288e-09","GO:1904356 (63) <br> regulation of telomere maintenance via telomere lengthening <br> 7.436e-09","GO:0061077 (65) <br> chaperone-mediated protein folding <br> 8.068e-09","GO:1903829 (454) <br> positive regulation of protein localization <br> 9.972e-09","GO:1902369 (76) <br> negative regulation of RNA catabolic process <br> 1.104e-08","GO:0070198 (31) <br> protein localization to chromosome, telomeric region <br> 1.173e-08","GO:0017038 (214) <br> protein import <br> 1.251e-08","GO:0050793 (2421) <br> regulation of developmental process <br> 1.411e-08","GO:0031123 (126) <br> RNA 3'-end processing <br> 1.942e-08","GO:0090304 (7476) <br> nucleic acid metabolic process <br> 1.960e-08","GO:0043069 (883) <br> negative regulation of programmed cell death <br> 2.080e-08","GO:0070200 (17) <br> establishment of protein localization to telomere <br> 2.412e-08","GO:0097435 (755) <br> supramolecular fiber organization <br> 2.694e-08","GO:0008152 (14361) <br> metabolic process <br> 3.404e-08","GO:2001252 (84) <br> positive regulation of chromosome organization <br> 3.703e-08","GO:0044087 (940) <br> regulation of cellular component biogenesis <br> 4.167e-08","GO:0042221 (4316) <br> response to chemical <br> 4.482e-08","GO:0051247 (1448) <br> positive regulation of protein metabolic process <br> 4.814e-08","GO:0000027 (26) <br> ribosomal large subunit assembly <br> 5.227e-08","GO:0032270 (1361) <br> positive regulation of cellular protein metabolic process <br> 5.356e-08","GO:0043066 (866) <br> negative regulation of apoptotic process <br> 5.630e-08","GO:0009653 (2685) <br> anatomical structure morphogenesis <br> 6.435e-08","GO:0000460 (36) <br> maturation of 5.8S rRNA <br> 7.086e-08","GO:0016477 (1454) <br> cell migration <br> 7.277e-08","GO:0007275 (5114) <br> multicellular organism development <br> 7.706e-08","GO:0051674 (1645) <br> localization of cell <br> 7.962e-08","GO:0048870 (1645) <br> cell motility <br> 7.962e-08","GO:0060548 (971) <br> negative regulation of cell death <br> 8.107e-08","GO:0007017 (898) <br> microtubule-based process <br> 8.738e-08","GO:0000469 (27) <br> cleavage involved in rRNA processing <br> 9.035e-08","GO:0071705 (2278) <br> nitrogen compound transport <br> 9.139e-08","GO:0051239 (2623) <br> regulation of multicellular organismal process <br> 9.492e-08","GO:0071103 (329) <br> DNA conformation change <br> 9.870e-08","GO:0048524 (59) <br> positive regulation of viral process <br> 1.013e-07","GO:0051640 (541) <br> organelle localization <br> 1.019e-07","GO:0031328 (1942) <br> positive regulation of cellular biosynthetic process <br> 1.029e-07","GO:0070887 (3008) <br> cellular response to chemical stimulus <br> 1.203e-07","GO:0009891 (1975) <br> positive regulation of biosynthetic process <br> 1.314e-07","GO:0010557 (1839) <br> positive regulation of macromolecule biosynthetic process <br> 1.317e-07","GO:0032392 (92) <br> DNA geometric change <br> 1.423e-07","GO:0044419 (1634) <br> biological process involved in interspecies interaction between organisms <br> 1.462e-07","GO:0051726 (1011) <br> regulation of cell cycle <br> 1.541e-07","GO:0000819 (198) <br> sister chromatid segregation <br> 1.555e-07","GO:0032210 (54) <br> regulation of telomere maintenance via telomerase <br> 1.848e-07","GO:0051701 (183) <br> biological process involved in interaction with host <br> 2.024e-07","GO:0033044 (192) <br> regulation of chromosome organization <br> 2.027e-07","GO:0043489 (66) <br> RNA stabilization <br> 2.056e-07","GO:0007155 (1480) <br> cell adhesion <br> 2.070e-07","GO:0000466 (24) <br> maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.230e-07","GO:0010564 (675) <br> regulation of cell cycle process <br> 2.245e-07","GO:0022610 (1487) <br> biological adhesion <br> 2.421e-07","GO:0006446 (80) <br> regulation of translational initiation <br> 2.872e-07","GO:0031124 (73) <br> mRNA 3'-end processing <br> 3.052e-07","GO:1904358 (37) <br> positive regulation of telomere maintenance via telomere lengthening <br> 3.131e-07","GO:0051234 (4920) <br> establishment of localization <br> 3.153e-07","GO:0006260 (288) <br> DNA replication <br> 3.393e-07","GO:0050789 (12142) <br> regulation of biological process <br> 3.947e-07","GO:0010629 (1570) <br> negative regulation of gene expression <br> 4.513e-07","GO:0140014 (297) <br> mitotic nuclear division <br> 4.520e-07","GO:0034502 (94) <br> protein localization to chromosome <br> 5.850e-07","GO:0040011 (1821) <br> locomotion <br> 6.145e-07","GO:0000226 (627) <br> microtubule cytoskeleton organization <br> 6.167e-07","GO:0016070 (6948) <br> RNA metabolic process <br> 6.211e-07","GO:0032388 (204) <br> positive regulation of intracellular transport <br> 6.450e-07","GO:0048255 (57) <br> mRNA stabilization <br> 6.613e-07","GO:0009888 (1912) <br> tissue development <br> 6.731e-07","GO:0072594 (658) <br> establishment of protein localization to organelle <br> 7.015e-07","GO:0009890 (1610) <br> negative regulation of biosynthetic process <br> 7.037e-07","GO:0052126 (165) <br> movement in host environment <br> 7.394e-07","GO:0070199 (26) <br> establishment of protein localization to chromosome <br> 7.762e-07","GO:2000278 (113) <br> regulation of DNA biosynthetic process <br> 8.295e-07","GO:1902373 (64) <br> negative regulation of mRNA catabolic process <br> 8.349e-07","GO:0030029 (800) <br> actin filament-based process <br> 8.665e-07","GO:0010558 (1523) <br> negative regulation of macromolecule biosynthetic process <br> 9.256e-07","GO:0051052 (356) <br> regulation of DNA metabolic process <br> 9.609e-07","GO:0000902 (1010) <br> cell morphogenesis <br> 9.700e-07","GO:0006397 (1919) <br> mRNA processing <br> 1.115e-06","GO:0045184 (1942) <br> establishment of protein localization <br> 1.157e-06","GO:0048731 (4784) <br> system development <br> 1.207e-06","GO:0051054 (204) <br> positive regulation of DNA metabolic process <br> 1.578e-06","GO:0006458 (40) <br> 'de novo' protein folding <br> 1.833e-06","GO:0031330 (261) <br> negative regulation of cellular catabolic process <br> 1.844e-06","GO:0000478 (15) <br> endonucleolytic cleavage involved in rRNA processing <br> 2.076e-06","GO:0000479 (15) <br> endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.076e-06","GO:0000070 (170) <br> mitotic sister chromatid segregation <br> 2.326e-06","GO:0022604 (305) <br> regulation of cell morphogenesis <br> 2.329e-06","GO:0030036 (711) <br> actin cytoskeleton organization <br> 2.699e-06","GO:0051170 (166) <br> import into nucleus <br> 3.197e-06","GO:0008380 (1841) <br> RNA splicing <br> 3.279e-06","GO:0046718 (147) <br> viral entry into host cell <br> 3.583e-06","GO:0031589 (356) <br> cell-substrate adhesion <br> 3.690e-06","GO:0051656 (409) <br> establishment of organelle localization <br> 4.293e-06","GO:0007229 (112) <br> integrin-mediated signaling pathway <br> 5.514e-06","GO:0006810 (4760) <br> transport <br> 5.514e-06","GO:0045934 (1496) <br> negative regulation of nucleobase-containing compound metabolic process <br> 5.592e-06","GO:0048468 (2079) <br> cell development <br> 5.609e-06","GO:0007059 (340) <br> chromosome segregation <br> 5.965e-06","GO:0032212 (34) <br> positive regulation of telomere maintenance via telomerase <br> 6.072e-06","GO:0044089 (493) <br> positive regulation of cellular component biogenesis <br> 6.382e-06","GO:0031327 (1581) <br> negative regulation of cellular biosynthetic process <br> 6.855e-06","GO:0006606 (162) <br> protein import into nucleus <br> 7.256e-06","GO:0010769 (96) <br> regulation of cell morphogenesis involved in differentiation <br> 7.384e-06","GO:0022411 (449) <br> cellular component disassembly <br> 7.675e-06","GO:0051254 (1729) <br> positive regulation of RNA metabolic process <br> 7.863e-06","GO:0006406 (58) <br> mRNA export from nucleus <br> 8.583e-06","GO:0044409 (154) <br> entry into host <br> 9.074e-06","GO:1900182 (87) <br> positive regulation of protein localization to nucleus <br> 9.348e-06","GO:0006378 (52) <br> mRNA polyadenylation <br> 9.798e-06","GO:0006997 (139) <br> nucleus organization <br> 9.819e-06","GO:0032984 (230) <br> protein-containing complex disassembly <br> 1.133e-05","GO:0043631 (53) <br> RNA polyadenylation <br> 1.363e-05","GO:0001817 (718) <br> regulation of cytokine production <br> 1.435e-05","GO:0009895 (320) <br> negative regulation of catabolic process <br> 1.443e-05","GO:0043254 (402) <br> regulation of protein-containing complex assembly <br> 1.572e-05","GO:0001816 (724) <br> cytokine production <br> 1.841e-05","GO:0006898 (243) <br> receptor-mediated endocytosis <br> 2.109e-05","GO:0006886 (1136) <br> intracellular protein transport <br> 2.181e-05","GO:0097190 (593) <br> apoptotic signaling pathway <br> 2.247e-05","GO:0016192 (1597) <br> vesicle-mediated transport <br> 2.283e-05","GO:1902903 (373) <br> regulation of supramolecular fiber organization <br> 2.416e-05","GO:1901998 (41) <br> toxin transport <br> 2.441e-05","GO:0007339 (41) <br> binding of sperm to zona pellucida <br> 2.441e-05","GO:0098813 (278) <br> nuclear chromosome segregation <br> 2.468e-05","GO:0019079 (130) <br> viral genome replication <br> 2.620e-05","GO:0001731 (12) <br> formation of translation preinitiation complex <br> 2.718e-05","GO:0006986 (138) <br> response to unfolded protein <br> 2.837e-05","GO:0051301 (635) <br> cell division <br> 2.983e-05","GO:0071310 (2385) <br> cellular response to organic substance <br> 3.029e-05","GO:0000904 (718) <br> cell morphogenesis involved in differentiation <br> 3.044e-05","GO:0050790 (2314) <br> regulation of catalytic activity <br> 3.129e-05","GO:1900180 (137) <br> regulation of protein localization to nucleus <br> 3.248e-05","GO:0031344 (634) <br> regulation of cell projection organization <br> 3.258e-05","GO:0034446 (110) <br> substrate adhesion-dependent cell spreading <br> 3.268e-05","GO:0120035 (618) <br> regulation of plasma membrane bounded cell projection organization <br> 3.402e-05","GO:0065009 (3017) <br> regulation of molecular function <br> 3.651e-05","GO:0034975 (11) <br> protein folding in endoplasmic reticulum <br> 3.785e-05","GO:0032508 (86) <br> DNA duplex unwinding <br> 3.957e-05","GO:0035966 (159) <br> response to topologically incorrect protein <br> 4.047e-05","GO:0007015 (444) <br> actin filament organization <br> 4.061e-05","GO:0009987 (19329) <br> cellular process <br> 4.138e-05","GO:0120036 (1479) <br> plasma membrane bounded cell projection organization <br> 4.163e-05","GO:0051017 (157) <br> actin filament bundle assembly <br> 4.490e-05","GO:0010035 (534) <br> response to inorganic substance <br> 4.519e-05","GO:0099173 (163) <br> postsynapse organization <br> 4.906e-05","GO:0022613 (1870) <br> ribonucleoprotein complex biogenesis <br> 5.241e-05","GO:0071897 (185) <br> DNA biosynthetic process <br> 5.280e-05","GO:0000463 (15) <br> maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 5.540e-05","GO:0046831 (12) <br> regulation of RNA export from nucleus <br> 5.693e-05","GO:0007163 (222) <br> establishment or maintenance of cell polarity <br> 5.760e-05","GO:0034330 (687) <br> cell junction organization <br> 5.773e-05","GO:0031440 (28) <br> regulation of mRNA 3'-end processing <br> 5.859e-05","GO:0050808 (422) <br> synapse organization <br> 5.945e-05","GO:0031334 (195) <br> positive regulation of protein-containing complex assembly <br> 6.161e-05","GO:0051653 (57) <br> spindle localization <br> 6.667e-05","GO:0051253 (1395) <br> negative regulation of RNA metabolic process <br> 6.814e-05","GO:0061572 (160) <br> actin filament bundle organization <br> 6.881e-05","GO:1904951 (316) <br> positive regulation of establishment of protein localization <br> 6.896e-05","GO:1901698 (1088) <br> response to nitrogen compound <br> 7.572e-05","GO:0009988 (69) <br> cell-cell recognition <br> 8.030e-05","GO:0080134 (1273) <br> regulation of response to stress <br> 8.201e-05","GO:0022603 (926) <br> regulation of anatomical structure morphogenesis <br> 8.651e-05","GO:0065007 (12859) <br> biological regulation <br> 8.901e-05","GO:0035036 (49) <br> sperm-egg recognition <br> 8.922e-05","GO:2001242 (167) <br> regulation of intrinsic apoptotic signaling pathway <br> 9.326e-05","GO:0006979 (423) <br> response to oxidative stress <br> 9.478e-05","GO:0006379 (20) <br> mRNA cleavage <br> 1.019e-04","GO:0009719 (1551) <br> response to endogenous stimulus <br> 1.175e-04","GO:0014070 (901) <br> response to organic cyclic compound <br> 1.195e-04","GO:0048513 (3494) <br> animal organ development <br> 1.199e-04","GO:0098789 (13) <br> pre-mRNA cleavage required for polyadenylation <br> 1.260e-04","GO:0030030 (1521) <br> cell projection organization <br> 1.377e-04","GO:0010770 (79) <br> positive regulation of cell morphogenesis involved in differentiation <br> 1.402e-04","GO:0071702 (2689) <br> organic substance transport <br> 1.412e-04","GO:0045070 (31) <br> positive regulation of viral genome replication <br> 1.537e-04","GO:0031399 (1477) <br> regulation of protein modification process <br> 1.552e-04","GO:0051129 (680) <br> negative regulation of cellular component organization <br> 1.586e-04","GO:0051270 (1029) <br> regulation of cellular component movement <br> 1.615e-04","GO:0000447 (9) <br> endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.620e-04","GO:0051084 (36) <br> 'de novo' posttranslational protein folding <br> 1.675e-04","GO:0022607 (4556) <br> cellular component assembly <br> 1.679e-04","GO:0031397 (80) <br> negative regulation of protein ubiquitination <br> 1.780e-04","GO:0032501 (7474) <br> multicellular organismal process <br> 1.837e-04","GO:0032781 (44) <br> positive regulation of ATP-dependent activity <br> 1.874e-04","GO:0006165 (101) <br> nucleoside diphosphate phosphorylation <br> 1.953e-04","GO:0032271 (205) <br> regulation of protein polymerization <br> 1.972e-04","GO:0051293 (52) <br> establishment of spindle localization <br> 2.001e-04","GO:0090068 (235) <br> positive regulation of cell cycle process <br> 2.061e-04","GO:0090501 (155) <br> RNA phosphodiester bond hydrolysis <br> 2.147e-04","GO:0015031 (1829) <br> protein transport <br> 2.221e-04","GO:0001732 (16) <br> formation of cytoplasmic translation initiation complex <br> 2.237e-04","GO:0046939 (102) <br> nucleotide phosphorylation <br> 2.268e-04","GO:0098787 (14) <br> mRNA cleavage involved in mRNA processing <br> 2.450e-04","GO:0051050 (892) <br> positive regulation of transport <br> 2.752e-04","GO:0009605 (2752) <br> response to external stimulus <br> 2.778e-04","GO:0032989 (749) <br> cellular component morphogenesis <br> 2.826e-04","GO:1902905 (178) <br> positive regulation of supramolecular fiber organization <br> 3.011e-04","GO:0035987 (46) <br> endodermal cell differentiation <br> 3.522e-04","GO:0051493 (523) <br> regulation of cytoskeleton organization <br> 3.552e-04","GO:0018208 (62) <br> peptidyl-proline modification <br> 3.560e-04","GO:0048285 (496) <br> organelle fission <br> 3.664e-04","GO:0006354 (80) <br> DNA-templated transcription, elongation <br> 3.715e-04","GO:0019080 (98) <br> viral gene expression <br> 3.961e-04","GO:0106027 (89) <br> neuron projection organization <br> 4.435e-04","GO:0051240 (1421) <br> positive regulation of multicellular organismal process <br> 4.496e-04","GO:2001233 (359) <br> regulation of apoptotic signaling pathway <br> 4.589e-04","GO:0050792 (158) <br> regulation of viral process <br> 5.030e-04","GO:0010970 (156) <br> transport along microtubule <br> 5.102e-04","GO:0007338 (156) <br> single fertilization <br> 6.114e-04","GO:0075522 (11) <br> IRES-dependent viral translational initiation <br> 6.403e-04","GO:0032239 (15) <br> regulation of nucleobase-containing compound transport <br> 6.430e-04","GO:0120032 (190) <br> regulation of plasma membrane bounded cell projection assembly <br> 6.455e-04","GO:0060999 (36) <br> positive regulation of dendritic spine development <br> 6.455e-04","GO:0045595 (1541) <br> regulation of cell differentiation <br> 6.609e-04","GO:2000145 (953) <br> regulation of cell motility <br> 6.716e-04","GO:0080135 (648) <br> regulation of cellular response to stress <br> 6.854e-04","GO:1903608 (7) <br> protein localization to cytoplasmic stress granule <br> 6.894e-04","GO:0031346 (336) <br> positive regulation of cell projection organization <br> 7.015e-04","GO:0031175 (953) <br> neuron projection development <br> 7.017e-04","GO:0051085 (31) <br> chaperone cofactor-dependent protein refolding <br> 7.031e-04","GO:0006091 (499) <br> generation of precursor metabolites and energy <br> 7.042e-04","GO:0062197 (322) <br> cellular response to chemical stress <br> 7.186e-04","GO:2000573 (70) <br> positive regulation of DNA biosynthetic process <br> 7.289e-04","GO:0060491 (192) <br> regulation of cell projection assembly <br> 7.446e-04","GO:0070201 (522) <br> regulation of establishment of protein localization <br> 7.489e-04","GO:0042176 (401) <br> regulation of protein catabolic process <br> 7.847e-04","GO:0072350 (14) <br> tricarboxylic acid metabolic process <br> 8.093e-04","GO:0033157 (232) <br> regulation of intracellular protein transport <br> 8.230e-04","GO:0051702 (101) <br> biological process involved in interaction with symbiont <br> 8.662e-04","GO:0019884 (70) <br> antigen processing and presentation of exogenous antigen <br> 1.028e-03","GO:0019081 (16) <br> viral translation <br> 1.051e-03","GO:0031400 (493) <br> negative regulation of protein modification process <br> 1.084e-03","GO:0050807 (209) <br> regulation of synapse organization <br> 1.088e-03","GO:0000280 (450) <br> nuclear division <br> 1.089e-03","GO:0044093 (1491) <br> positive regulation of molecular function <br> 1.166e-03","GO:0051225 (118) <br> spindle assembly <br> 1.168e-03","GO:0097193 (296) <br> intrinsic apoptotic signaling pathway <br> 1.265e-03","GO:0060148 (30) <br> positive regulation of posttranscriptional gene silencing <br> 1.268e-03","GO:0030334 (894) <br> regulation of cell migration <br> 1.334e-03","GO:0030162 (728) <br> regulation of proteolysis <br> 1.336e-03","GO:2000637 (29) <br> positive regulation of gene silencing by miRNA <br> 1.349e-03","GO:0055001 (174) <br> muscle cell development <br> 1.351e-03","GO:0045787 (314) <br> positive regulation of cell cycle <br> 1.455e-03","GO:0007492 (81) <br> endoderm development <br> 1.465e-03","GO:0000054 (15) <br> ribosomal subunit export from nucleus <br> 1.491e-03","GO:0033750 (15) <br> ribosome localization <br> 1.491e-03","GO:0009132 (129) <br> nucleoside diphosphate metabolic process <br> 1.513e-03","GO:0044257 (821) <br> cellular protein catabolic process <br> 1.514e-03","GO:0007399 (2428) <br> nervous system development <br> 1.532e-03","GO:0050803 (215) <br> regulation of synapse structure or activity <br> 1.532e-03","GO:0000472 (5) <br> endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 1.554e-03","GO:0000967 (5) <br> rRNA 5'-end processing <br> 1.554e-03","GO:0016049 (484) <br> cell growth <br> 1.559e-03","GO:0008037 (224) <br> cell recognition <br> 1.571e-03","GO:0051851 (64) <br> modulation by host of symbiont process <br> 1.587e-03","GO:0045785 (439) <br> positive regulation of cell adhesion <br> 1.630e-03","GO:0051258 (278) <br> protein polymerization <br> 1.647e-03","GO:0030335 (524) <br> positive regulation of cell migration <br> 1.701e-03","GO:0034976 (257) <br> response to endoplasmic reticulum stress <br> 1.716e-03","GO:0031503 (152) <br> protein-containing complex localization <br> 1.746e-03","GO:0009566 (197) <br> fertilization <br> 1.808e-03","GO:0002262 (161) <br> myeloid cell homeostasis <br> 1.848e-03","GO:0030163 (985) <br> protein catabolic process <br> 1.929e-03","GO:0051231 (10) <br> spindle elongation <br> 2.004e-03","GO:0032273 (95) <br> positive regulation of protein polymerization <br> 2.057e-03","GO:0000154 (34) <br> rRNA modification <br> 2.088e-03","GO:1902533 (971) <br> positive regulation of intracellular signal transduction <br> 2.129e-03","GO:1903321 (93) <br> negative regulation of protein modification by small protein conjugation or removal <br> 2.199e-03","GO:0048646 (1123) <br> anatomical structure formation involved in morphogenesis <br> 2.206e-03","GO:0090316 (161) <br> positive regulation of intracellular protein transport <br> 2.236e-03","GO:0060996 (94) <br> dendritic spine development <br> 2.355e-03","GO:0043086 (682) <br> negative regulation of catalytic activity <br> 2.413e-03","GO:2001234 (225) <br> negative regulation of apoptotic signaling pathway <br> 2.466e-03","GO:0061061 (616) <br> muscle structure development <br> 2.477e-03","GO:0060964 (45) <br> regulation of gene silencing by miRNA <br> 2.522e-03","GO:0002831 (347) <br> regulation of response to biotic stimulus <br> 2.527e-03","GO:0030705 (194) <br> cytoskeleton-dependent intracellular transport <br> 2.534e-03","GO:0051495 (196) <br> positive regulation of cytoskeleton organization <br> 2.552e-03","GO:0042692 (357) <br> muscle cell differentiation <br> 2.590e-03","GO:0002218 (73) <br> activation of innate immune response <br> 2.609e-03","GO:0010639 (352) <br> negative regulation of organelle organization <br> 2.609e-03","GO:0040012 (997) <br> regulation of locomotion <br> 2.631e-03","GO:0009790 (1088) <br> embryo development <br> 2.634e-03","GO:0048545 (333) <br> response to steroid hormone <br> 2.649e-03","GO:0001706 (55) <br> endoderm formation <br> 2.686e-03","GO:0007044 (93) <br> cell-substrate junction assembly <br> 2.897e-03","GO:0034097 (922) <br> response to cytokine <br> 2.939e-03","GO:1903320 (241) <br> regulation of protein modification by small protein conjugation or removal <br> 2.952e-03","GO:1903039 (239) <br> positive regulation of leukocyte cell-cell adhesion <br> 2.974e-03","GO:0006974 (807) <br> cellular response to DNA damage stimulus <br> 3.046e-03","GO:0031032 (193) <br> actomyosin structure organization <br> 3.241e-03","GO:0006897 (638) <br> endocytosis <br> 3.313e-03","GO:0006101 (5) <br> citrate metabolic process <br> 3.331e-03","GO:0060998 (55) <br> regulation of dendritic spine development <br> 3.341e-03","GO:0051261 (120) <br> protein depolymerization <br> 3.369e-03","GO:0000413 (45) <br> protein peptidyl-prolyl isomerization <br> 3.380e-03","GO:0034599 (273) <br> cellular response to oxidative stress <br> 3.423e-03","GO:0030155 (733) <br> regulation of cell adhesion <br> 3.567e-03","GO:0060147 (47) <br> regulation of posttranscriptional gene silencing <br> 3.570e-03","GO:0002474 (48) <br> antigen processing and presentation of peptide antigen via MHC class I <br> 3.653e-03","GO:0022414 (1521) <br> reproductive process <br> 3.764e-03","GO:0008283 (1940) <br> cell population proliferation <br> 3.865e-03","GO:0007076 (19) <br> mitotic chromosome condensation <br> 3.897e-03","GO:0000003 (1524) <br> reproduction <br> 3.919e-03","GO:0032970 (391) <br> regulation of actin filament-based process <br> 4.180e-03","GO:0098609 (888) <br> cell-cell adhesion <br> 4.208e-03","GO:0060966 (48) <br> regulation of gene silencing by RNA <br> 4.219e-03","GO:0046847 (64) <br> filopodium assembly <br> 4.235e-03","GO:0010501 (8) <br> RNA secondary structure unwinding <br> 4.299e-03","GO:0051489 (48) <br> regulation of filopodium assembly <br> 4.306e-03","GO:0044703 (1054) <br> multi-organism reproductive process <br> 4.368e-03","GO:0031056 (158) <br> regulation of histone modification <br> 4.376e-03","GO:0002833 (185) <br> positive regulation of response to biotic stimulus <br> 4.399e-03","GO:0018130 (4024) <br> heterocycle biosynthetic process <br> 4.484e-03","GO:2000147 (548) <br> positive regulation of cell motility <br> 4.647e-03","GO:0044092 (1041) <br> negative regulation of molecular function <br> 4.654e-03","GO:1901362 (4181) <br> organic cyclic compound biosynthetic process <br> 4.786e-03","GO:0120034 (105) <br> positive regulation of plasma membrane bounded cell projection assembly <br> 4.824e-03","GO:0070585 (129) <br> protein localization to mitochondrion <br> 4.831e-03","GO:0071407 (567) <br> cellular response to organic cyclic compound <br> 4.882e-03","GO:0045454 (35) <br> cell redox homeostasis <br> 4.888e-03","GO:0032784 (53) <br> regulation of DNA-templated transcription, elongation <br> 4.938e-03","GO:1900024 (59) <br> regulation of substrate adhesion-dependent cell spreading <br> 5.019e-03","GO:0010243 (1001) <br> response to organonitrogen compound <br> 5.042e-03","GO:0042590 (32) <br> antigen processing and presentation of exogenous peptide antigen via MHC class I <br> 5.301e-03","GO:0031396 (208) <br> regulation of protein ubiquitination <br> 5.329e-03","GO:0001649 (226) <br> osteoblast differentiation <br> 5.392e-03","GO:0006508 (1755) <br> proteolysis <br> 5.472e-03","GO:0010498 (503) <br> proteasomal protein catabolic process <br> 5.512e-03","GO:0051272 (561) <br> positive regulation of cellular component movement <br> 5.613e-03","GO:0051098 (367) <br> regulation of binding <br> 5.738e-03","GO:0061003 (18) <br> positive regulation of dendritic spine morphogenesis <br> 5.813e-03","GO:0009611 (543) <br> response to wounding <br> 5.874e-03","GO:0001704 (121) <br> formation of primary germ layer <br> 6.063e-03","GO:0090151 (34) <br> establishment of protein localization to mitochondrial membrane <br> 6.273e-03","GO:0050779 (86) <br> RNA destabilization <br> 6.276e-03","GO:0061024 (825) <br> membrane organization <br> 6.325e-03","GO:0019752 (933) <br> carboxylic acid metabolic process <br> 6.493e-03","GO:0090307 (67) <br> mitotic spindle assembly <br> 6.493e-03","GO:0061014 (86) <br> positive regulation of mRNA catabolic process <br> 6.566e-03","GO:0071495 (1310) <br> cellular response to endogenous stimulus <br> 6.650e-03","GO:1901992 (85) <br> positive regulation of mitotic cell cycle phase transition <br> 6.652e-03","GO:0010720 (298) <br> positive regulation of cell development <br> 6.730e-03","GO:0050000 (84) <br> chromosome localization <br> 6.819e-03","GO:0039529 (28) <br> RIG-I signaling pathway <br> 6.992e-03","GO:0006261 (181) <br> DNA-dependent DNA replication <br> 7.051e-03","GO:1900087 (37) <br> positive regulation of G1/S transition of mitotic cell cycle <br> 7.341e-03","GO:0150115 (99) <br> cell-substrate junction organization <br> 7.787e-03","GO:0001819 (465) <br> positive regulation of cytokine production <br> 7.807e-03","GO:1900740 (9) <br> positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway <br> 7.982e-03","GO:1900739 (9) <br> regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway <br> 7.982e-03","GO:0051049 (1719) <br> regulation of transport <br> 8.213e-03","GO:0043436 (957) <br> oxoacid metabolic process <br> 8.298e-03","GO:1903362 (260) <br> regulation of cellular protein catabolic process <br> 8.319e-03","GO:0048588 (221) <br> developmental cell growth <br> 8.418e-03","GO:0051099 (174) <br> positive regulation of binding <br> 8.584e-03","GO:0040017 (564) <br> positive regulation of locomotion <br> 8.733e-03","GO:0072655 (124) <br> establishment of protein localization to mitochondrion <br> 8.747e-03","GO:0046785 (85) <br> microtubule polymerization <br> 8.897e-03","GO:0031331 (437) <br> positive regulation of cellular catabolic process <br> 8.926e-03","GO:0034654 (3953) <br> nucleobase-containing compound biosynthetic process <br> 8.981e-03","GO:0048584 (2111) <br> positive regulation of response to stimulus <br> 9.306e-03","GO:0031060 (72) <br> regulation of histone methylation <br> 9.462e-03","GO:0034101 (132) <br> erythrocyte homeostasis <br> 9.482e-03","GO:0050870 (218) <br> positive regulation of T cell activation <br> 9.561e-03","GO:1900363 (18) <br> regulation of mRNA polyadenylation <br> 1.002e-02","GO:0040007 (951) <br> growth <br> 1.017e-02","GO:0032990 (657) <br> cell part morphogenesis <br> 1.019e-02","GO:0006082 (980) <br> organic acid metabolic process <br> 1.043e-02","GO:1990138 (173) <br> neuron projection extension <br> 1.053e-02","GO:0060429 (1160) <br> epithelium development <br> 1.064e-02","GO:0044770 (529) <br> cell cycle phase transition <br> 1.082e-02","GO:1901989 (110) <br> positive regulation of cell cycle phase transition <br> 1.119e-02","GO:0032103 (435) <br> positive regulation of response to external stimulus <br> 1.132e-02","GO:0031058 (96) <br> positive regulation of histone modification <br> 1.133e-02","GO:0009408 (105) <br> response to heat <br> 1.154e-02","GO:0019438 (4035) <br> aromatic compound biosynthetic process <br> 1.197e-02","GO:0061157 (82) <br> mRNA destabilization <br> 1.212e-02","GO:0009628 (1121) <br> response to abiotic stimulus <br> 1.278e-02","GO:0046034 (276) <br> ATP metabolic process <br> 1.289e-02","GO:0006611 (58) <br> protein export from nucleus <br> 1.307e-02","GO:0110053 (267) <br> regulation of actin filament organization <br> 1.316e-02","GO:0042060 (415) <br> wound healing <br> 1.318e-02","GO:0051094 (1271) <br> positive regulation of developmental process <br> 1.354e-02","GO:0008088 (76) <br> axo-dendritic transport <br> 1.363e-02","GO:0019953 (898) <br> sexual reproduction <br> 1.387e-02","GO:0048666 (1104) <br> neuron development <br> 1.403e-02","GO:0099111 (196) <br> microtubule-based transport <br> 1.440e-02","GO:0072359 (1089) <br> circulatory system development <br> 1.505e-02","GO:0009048 (19) <br> dosage compensation by inactivation of X chromosome <br> 1.518e-02","GO:1901532 (55) <br> regulation of hematopoietic progenitor cell differentiation <br> 1.518e-02","GO:0043161 (438) <br> proteasome-mediated ubiquitin-dependent protein catabolic process <br> 1.526e-02","GO:0007346 (466) <br> regulation of mitotic cell cycle <br> 1.531e-02","GO:1902036 (36) <br> regulation of hematopoietic stem cell differentiation <br> 1.552e-02","GO:0022409 (284) <br> positive regulation of cell-cell adhesion <br> 1.570e-02","GO:0048675 (125) <br> axon extension <br> 1.645e-02","GO:0051310 (65) <br> metaphase plate congression <br> 1.648e-02","GO:0019220 (1367) <br> regulation of phosphate metabolic process <br> 1.668e-02","GO:0016182 (2) <br> synaptic vesicle budding from endosome <br> 1.703e-02","GO:0052547 (439) <br> regulation of peptidase activity <br> 1.716e-02","GO:0097061 (80) <br> dendritic spine organization <br> 1.724e-02","GO:0048268 (18) <br> clathrin coat assembly <br> 1.734e-02","GO:0051174 (1369) <br> regulation of phosphorus metabolic process <br> 1.743e-02","GO:0009185 (110) <br> ribonucleoside diphosphate metabolic process <br> 1.745e-02","GO:0098961 (4) <br> dendritic transport of ribonucleoprotein complex <br> 1.821e-02","GO:0098963 (4) <br> dendritic transport of messenger ribonucleoprotein complex <br> 1.821e-02","GO:0033484 (3) <br> nitric oxide homeostasis <br> 1.839e-02","GO:0045089 (148) <br> positive regulation of innate immune response <br> 1.865e-02","GO:0016358 (235) <br> dendrite development <br> 1.922e-02","GO:0090305 (272) <br> nucleic acid phosphodiester bond hydrolysis <br> 1.923e-02","GO:0010248 (11) <br> establishment or maintenance of transmembrane electrochemical gradient <br> 1.929e-02","GO:0045807 (99) <br> positive regulation of endocytosis <br> 1.931e-02","GO:0046794 (24) <br> transport of virus <br> 1.943e-02","GO:0120039 (634) <br> plasma membrane bounded cell projection morphogenesis <br> 2.026e-02","GO:0051131 (23) <br> chaperone-mediated protein complex assembly <br> 2.079e-02","GO:0030182 (1364) <br> neuron differentiation <br> 2.109e-02","GO:0045931 (113) <br> positive regulation of mitotic cell cycle <br> 2.181e-02","GO:0046931 (20) <br> pore complex assembly <br> 2.209e-02","GO:1905475 (181) <br> regulation of protein localization to membrane <br> 2.212e-02","GO:0031442 (10) <br> positive regulation of mRNA 3'-end processing <br> 2.261e-02","GO:0060997 (59) <br> dendritic spine morphogenesis <br> 2.278e-02","GO:0048858 (638) <br> cell projection morphogenesis <br> 2.330e-02","GO:0044281 (1779) <br> small molecule metabolic process <br> 2.431e-02","GO:0043462 (72) <br> regulation of ATP-dependent activity <br> 2.482e-02","GO:0030043 (4) <br> actin filament fragmentation <br> 2.540e-02","GO:0045088 (234) <br> regulation of innate immune response <br> 2.588e-02","GO:0022008 (1639) <br> neurogenesis <br> 2.633e-02","GO:0045995 (64) <br> regulation of embryonic development <br> 2.645e-02","GO:1904667 (13) <br> negative regulation of ubiquitin protein ligase activity <br> 2.672e-02","GO:0071711 (32) <br> basement membrane organization <br> 2.790e-02","GO:0052548 (412) <br> regulation of endopeptidase activity <br> 2.807e-02","GO:1902743 (53) <br> regulation of lamellipodium organization <br> 2.859e-02","GO:0034605 (65) <br> cellular response to heat <br> 2.859e-02","GO:2001022 (108) <br> positive regulation of response to DNA damage stimulus <br> 2.893e-02","GO:0071345 (828) <br> cellular response to cytokine stimulus <br> 2.897e-02","GO:1902808 (52) <br> positive regulation of cell cycle G1/S phase transition <br> 2.921e-02","GO:0034471 (21) <br> ncRNA 5'-end processing <br> 2.942e-02","GO:0030003 (646) <br> cellular cation homeostasis <br> 2.953e-02","GO:0048699 (1519) <br> generation of neurons <br> 2.953e-02","GO:0009896 (505) <br> positive regulation of catabolic process <br> 2.960e-02","GO:0000480 (7) <br> endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 2.962e-02","GO:0051303 (82) <br> establishment of chromosome localization <br> 2.985e-02","GO:0000022 (9) <br> mitotic spindle elongation <br> 3.017e-02","GO:0055080 (728) <br> cation homeostasis <br> 3.022e-02","GO:0060284 (499) <br> regulation of cell development <br> 3.069e-02","GO:0031116 (35) <br> positive regulation of microtubule polymerization <br> 3.074e-02","GO:0002478 (59) <br> antigen processing and presentation of exogenous peptide antigen <br> 3.092e-02","GO:0032101 (927) <br> regulation of response to external stimulus <br> 3.110e-02","GO:0033962 (21) <br> P-body assembly <br> 3.182e-02","GO:0001667 (416) <br> ameboidal-type cell migration <br> 3.204e-02","GO:0099175 (86) <br> regulation of postsynapse organization <br> 3.237e-02","GO:0006875 (578) <br> cellular metal ion homeostasis <br> 3.270e-02","GO:1902683 (18) <br> regulation of receptor localization to synapse <br> 3.363e-02","GO:0048589 (631) <br> developmental growth <br> 3.408e-02","GO:0019882 (127) <br> antigen processing and presentation <br> 3.409e-02","GO:0021762 (47) <br> substantia nigra development <br> 3.410e-02","GO:0009266 (178) <br> response to temperature stimulus <br> 3.445e-02","GO:0120031 (548) <br> plasma membrane bounded cell projection assembly <br> 3.529e-02","GO:0043065 (501) <br> positive regulation of apoptotic process <br> 3.536e-02","GO:0032956 (350) <br> regulation of actin cytoskeleton organization <br> 3.559e-02","GO:0098771 (740) <br> inorganic ion homeostasis <br> 3.593e-02","GO:0055065 (657) <br> metal ion homeostasis <br> 3.636e-02","GO:0007549 (21) <br> dosage compensation <br> 3.677e-02","GO:0060322 (812) <br> head development <br> 3.732e-02","GO:1990778 (337) <br> protein localization to cell periphery <br> 3.773e-02","GO:0006085 (18) <br> acetyl-CoA biosynthetic process <br> 3.815e-02","GO:0015936 (18) <br> coenzyme A metabolic process <br> 3.815e-02","GO:0023056 (1693) <br> positive regulation of signaling <br> 3.870e-02","GO:0098696 (9) <br> regulation of neurotransmitter receptor localization to postsynaptic specialization membrane <br> 3.887e-02","GO:0006873 (661) <br> cellular ion homeostasis <br> 3.918e-02","GO:0045862 (364) <br> positive regulation of proteolysis <br> 3.924e-02","GO:0001844 (11) <br> protein insertion into mitochondrial membrane involved in apoptotic signaling pathway <br> 3.960e-02","GO:0072384 (85) <br> organelle transport along microtubule <br> 4.091e-02","GO:0009607 (1529) <br> response to biotic stimulus <br> 4.156e-02","GO:0048490 (19) <br> anterograde synaptic vesicle transport <br> 4.191e-02","GO:0099514 (19) <br> synaptic vesicle cytoskeletal transport <br> 4.191e-02","GO:0099517 (19) <br> synaptic vesicle transport along microtubule <br> 4.191e-02","GO:0051444 (20) <br> negative regulation of ubiquitin-protein transferase activity <br> 4.204e-02","GO:0030007 (14) <br> cellular potassium ion homeostasis <br> 4.223e-02","GO:0030832 (160) <br> regulation of actin filament length <br> 4.268e-02","GO:0046031 (91) <br> ADP metabolic process <br> 4.277e-02","GO:0002475 (13) <br> antigen processing and presentation via MHC class Ib <br> 4.337e-02","GO:0061001 (44) <br> regulation of dendritic spine morphogenesis <br> 4.507e-02","GO:0030433 (87) <br> ubiquitin-dependent ERAD pathway <br> 4.580e-02","GO:0030100 (205) <br> regulation of endocytosis <br> 4.694e-02","GO:1902652 (146) <br> secondary alcohol metabolic process <br> 4.710e-02","GO:0048863 (234) <br> stem cell differentiation <br> 4.724e-02","GO:0006942 (93) <br> regulation of striated muscle contraction <br> 4.911e-02"],"key":["GO:0006412","GO:1903311","GO:0043043","GO:0042254","GO:0006518","GO:0043484","GO:0034645","GO:0050684","GO:0043604","GO:0006364","GO:0016072","GO:0043603","GO:0044260","GO:0048024","GO:0002181","GO:0034470","GO:1901566","GO:0034660","GO:0006996","GO:2000112","GO:0006403","GO:0006417","GO:0034248","GO:0051246","GO:0009059","GO:0044267","GO:0051171","GO:0032268","GO:0080090","GO:0031323","GO:0044271","GO:1901576","GO:0006401","GO:0009058","GO:0042274","GO:0006457","GO:0071840","GO:0019538","GO:0006402","GO:1901564","GO:0044249","GO:0048522","GO:0048518","GO:0000380","GO:0016032","GO:0140694","GO:0060255","GO:0051169","GO:0006913","GO:0048523","GO:0051172","GO:0043487","GO:0044085","GO:0043488","GO:0061013","GO:0034641","GO:0050658","GO:0050657","GO:0042273","GO:0051236","GO:0019222","GO:0051641","GO:0000381","GO:0033036","GO:0010604","GO:0031324","GO:0034655","GO:0044265","GO:0009893","GO:0051128","GO:0046700","GO:1903312","GO:0033120","GO:0042255","GO:0051248","GO:0044270","GO:0009057","GO:0006413","GO:0051168","GO:0051276","GO:0033365","GO:0030490","GO:0050685","GO:0070727","GO:0032269","GO:0046907","GO:0019439","GO:0060341","GO:0048519","GO:0007049","GO:0051028","GO:1903313","GO:0010628","GO:0044237","GO:0019219","GO:1901361","GO:0010467","GO:0015931","GO:0034613","GO:0031325","GO:0006396","GO:0016071","GO:0034249","GO:0017148","GO:0051179","GO:0034504","GO:0010033","GO:0006807","GO:0048026","GO:0019058","GO:0033043","GO:0010468","GO:0010605","GO:0051173","GO:2000113","GO:0051130","GO:0000462","GO:0000723","GO:0031329","GO:0009892","GO:0009894","GO:0010638","GO:0065008","GO:0007010","GO:0044403","GO:0032502","GO:0051649","GO:0033119","GO:0051252","GO:1904816","GO:0006928","GO:0012501","GO:0008104","GO:1904871","GO:1904869","GO:0033554","GO:0010608","GO:0008219","GO:0044238","GO:0032386","GO:0010941","GO:0045727","GO:0043067","GO:0000278","GO:1904814","GO:1901575","GO:1903405","GO:1904867","GO:0032880","GO:0032879","GO:1903047","GO:0043170","GO:0006915","GO:0042981","GO:0009889","GO:0022402","GO:0044248","GO:1904851","GO:0032200","GO:0016043","GO:0010556","GO:0050686","GO:0006405","GO:0032204","GO:0070203","GO:0031326","GO:0048856","GO:0070925","GO:0034063","GO:0050794","GO:0042026","GO:1904872","GO:0010833","GO:1904874","GO:1990173","GO:0048025","GO:0048869","GO:0000470","GO:0070202","GO:0031647","GO:0034250","GO:0090685","GO:0090672","GO:0090670","GO:0090671","GO:0045935","GO:0000028","GO:0030154","GO:0006139","GO:1902850","GO:0007004","GO:0046483","GO:0009056","GO:0006278","GO:0006259","GO:0007052","GO:0050821","GO:0007051","GO:0032206","GO:0006950","GO:0071704","GO:0046822","GO:1901360","GO:0002183","GO:0006725","GO:1904356","GO:0061077","GO:1903829","GO:1902369","GO:0070198","GO:0017038","GO:0050793","GO:0031123","GO:0090304","GO:0043069","GO:0070200","GO:0097435","GO:0008152","GO:2001252","GO:0044087","GO:0042221","GO:0051247","GO:0000027","GO:0032270","GO:0043066","GO:0009653","GO:0000460","GO:0016477","GO:0007275","GO:0051674","GO:0048870","GO:0060548","GO:0007017","GO:0000469","GO:0071705","GO:0051239","GO:0071103","GO:0048524","GO:0051640","GO:0031328","GO:0070887","GO:0009891","GO:0010557","GO:0032392","GO:0044419","GO:0051726","GO:0000819","GO:0032210","GO:0051701","GO:0033044","GO:0043489","GO:0007155","GO:0000466","GO:0010564","GO:0022610","GO:0006446","GO:0031124","GO:1904358","GO:0051234","GO:0006260","GO:0050789","GO:0010629","GO:0140014","GO:0034502","GO:0040011","GO:0000226","GO:0016070","GO:0032388","GO:0048255","GO:0009888","GO:0072594","GO:0009890","GO:0052126","GO:0070199","GO:2000278","GO:1902373","GO:0030029","GO:0010558","GO:0051052","GO:0000902","GO:0006397","GO:0045184","GO:0048731","GO:0051054","GO:0006458","GO:0031330","GO:0000478","GO:0000479","GO:0000070","GO:0022604","GO:0030036","GO:0051170","GO:0008380","GO:0046718","GO:0031589","GO:0051656","GO:0007229","GO:0006810","GO:0045934","GO:0048468","GO:0007059","GO:0032212","GO:0044089","GO:0031327","GO:0006606","GO:0010769","GO:0022411","GO:0051254","GO:0006406","GO:0044409","GO:1900182","GO:0006378","GO:0006997","GO:0032984","GO:0043631","GO:0001817","GO:0009895","GO:0043254","GO:0001816","GO:0006898","GO:0006886","GO:0097190","GO:0016192","GO:1902903","GO:1901998","GO:0007339","GO:0098813","GO:0019079","GO:0001731","GO:0006986","GO:0051301","GO:0071310","GO:0000904","GO:0050790","GO:1900180","GO:0031344","GO:0034446","GO:0120035","GO:0065009","GO:0034975","GO:0032508","GO:0035966","GO:0007015","GO:0009987","GO:0120036","GO:0051017","GO:0010035","GO:0099173","GO:0022613","GO:0071897","GO:0000463","GO:0046831","GO:0007163","GO:0034330","GO:0031440","GO:0050808","GO:0031334","GO:0051653","GO:0051253","GO:0061572","GO:1904951","GO:1901698","GO:0009988","GO:0080134","GO:0022603","GO:0065007","GO:0035036","GO:2001242","GO:0006979","GO:0006379","GO:0009719","GO:0014070","GO:0048513","GO:0098789","GO:0030030","GO:0010770","GO:0071702","GO:0045070","GO:0031399","GO:0051129","GO:0051270","GO:0000447","GO:0051084","GO:0022607","GO:0031397","GO:0032501","GO:0032781","GO:0006165","GO:0032271","GO:0051293","GO:0090068","GO:0090501","GO:0015031","GO:0001732","GO:0046939","GO:0098787","GO:0051050","GO:0009605","GO:0032989","GO:1902905","GO:0035987","GO:0051493","GO:0018208","GO:0048285","GO:0006354","GO:0019080","GO:0106027","GO:0051240","GO:2001233","GO:0050792","GO:0010970","GO:0007338","GO:0075522","GO:0032239","GO:0120032","GO:0060999","GO:0045595","GO:2000145","GO:0080135","GO:1903608","GO:0031346","GO:0031175","GO:0051085","GO:0006091","GO:0062197","GO:2000573","GO:0060491","GO:0070201","GO:0042176","GO:0072350","GO:0033157","GO:0051702","GO:0019884","GO:0019081","GO:0031400","GO:0050807","GO:0000280","GO:0044093","GO:0051225","GO:0097193","GO:0060148","GO:0030334","GO:0030162","GO:2000637","GO:0055001","GO:0045787","GO:0007492","GO:0000054","GO:0033750","GO:0009132","GO:0044257","GO:0007399","GO:0050803","GO:0000472","GO:0000967","GO:0016049","GO:0008037","GO:0051851","GO:0045785","GO:0051258","GO:0030335","GO:0034976","GO:0031503","GO:0009566","GO:0002262","GO:0030163","GO:0051231","GO:0032273","GO:0000154","GO:1902533","GO:1903321","GO:0048646","GO:0090316","GO:0060996","GO:0043086","GO:2001234","GO:0061061","GO:0060964","GO:0002831","GO:0030705","GO:0051495","GO:0042692","GO:0002218","GO:0010639","GO:0040012","GO:0009790","GO:0048545","GO:0001706","GO:0007044","GO:0034097","GO:1903320","GO:1903039","GO:0006974","GO:0031032","GO:0006897","GO:0006101","GO:0060998","GO:0051261","GO:0000413","GO:0034599","GO:0030155","GO:0060147","GO:0002474","GO:0022414","GO:0008283","GO:0007076","GO:0000003","GO:0032970","GO:0098609","GO:0060966","GO:0046847","GO:0010501","GO:0051489","GO:0044703","GO:0031056","GO:0002833","GO:0018130","GO:2000147","GO:0044092","GO:1901362","GO:0120034","GO:0070585","GO:0071407","GO:0045454","GO:0032784","GO:1900024","GO:0010243","GO:0042590","GO:0031396","GO:0001649","GO:0006508","GO:0010498","GO:0051272","GO:0051098","GO:0061003","GO:0009611","GO:0001704","GO:0090151","GO:0050779","GO:0061024","GO:0019752","GO:0090307","GO:0061014","GO:0071495","GO:1901992","GO:0010720","GO:0050000","GO:0039529","GO:0006261","GO:1900087","GO:0150115","GO:0001819","GO:1900740","GO:1900739","GO:0051049","GO:0043436","GO:1903362","GO:0048588","GO:0051099","GO:0040017","GO:0072655","GO:0046785","GO:0031331","GO:0034654","GO:0048584","GO:0031060","GO:0034101","GO:0050870","GO:1900363","GO:0040007","GO:0032990","GO:0006082","GO:1990138","GO:0060429","GO:0044770","GO:1901989","GO:0032103","GO:0031058","GO:0009408","GO:0019438","GO:0061157","GO:0009628","GO:0046034","GO:0006611","GO:0110053","GO:0042060","GO:0051094","GO:0008088","GO:0019953","GO:0048666","GO:0099111","GO:0072359","GO:0009048","GO:1901532","GO:0043161","GO:0007346","GO:1902036","GO:0022409","GO:0048675","GO:0051310","GO:0019220","GO:0016182","GO:0052547","GO:0097061","GO:0048268","GO:0051174","GO:0009185","GO:0098961","GO:0098963","GO:0033484","GO:0045089","GO:0016358","GO:0090305","GO:0010248","GO:0045807","GO:0046794","GO:0120039","GO:0051131","GO:0030182","GO:0045931","GO:0046931","GO:1905475","GO:0031442","GO:0060997","GO:0048858","GO:0044281","GO:0043462","GO:0030043","GO:0045088","GO:0022008","GO:0045995","GO:1904667","GO:0071711","GO:0052548","GO:1902743","GO:0034605","GO:2001022","GO:0071345","GO:1902808","GO:0034471","GO:0030003","GO:0048699","GO:0009896","GO:0000480","GO:0051303","GO:0000022","GO:0055080","GO:0060284","GO:0031116","GO:0002478","GO:0032101","GO:0033962","GO:0001667","GO:0099175","GO:0006875","GO:1902683","GO:0048589","GO:0019882","GO:0021762","GO:0009266","GO:0120031","GO:0043065","GO:0032956","GO:0098771","GO:0055065","GO:0007549","GO:0060322","GO:1990778","GO:0006085","GO:0015936","GO:0023056","GO:0098696","GO:0006873","GO:0045862","GO:0001844","GO:0072384","GO:0009607","GO:0048490","GO:0099514","GO:0099517","GO:0051444","GO:0030007","GO:0030832","GO:0046031","GO:0002475","GO:0061001","GO:0030433","GO:0030100","GO:1902652","GO:0048863","GO:0006942"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[18.9022152589507,17.8248844913663,18.9464947605076,17.7531788301542,19.1919709229045,16.7520742647617,19.8812864049344,16.6488827426211,19.1517885876194,17.337119573383,17.5592811806137,19.5213275498429,21.383762350468,16.1951334195648,16.7219375695761,18.1358138355731,19.9803262322112,18.5304771554215,20.9220920001977,18.7946200908942,17.1523833138406,18.1048565973137,18.2859667437542,20.4253256600867,21.2022561920127,21.182853073392,21.3411577022457,20.3453923006955,21.3737106708018,21.41689355837,21.1682982884454,21.4071586924438,17.7805473874915,21.4233948591627,15.6117086396742,17.2649847629917,21.7450317386835,21.3127165481918,17.5484971573843,21.4918216770413,21.3929553847006,21.3475510340267,21.44863885784,15.7409065320485,18.2019552296956,18.0441880299229,21.5543716601559,17.7940127181872,17.7940127181872,21.1879695938973,20.3560238343394,17.1450135964435,21.1840181091261,17.0452487313639,17.1151141341332,21.8447320305701,16.8571248761385,16.8571248761385,15.5890025441251,16.8843993611985,21.6342377369221,20.694180918159,15.3399864812078,20.7051968135798,20.7864382496039,20.4352116585987,18.2524291177122,19.4890455319957,20.8835144799693,20.3128653159365,18.3645967823511,16.0034441398068,14.3855512064191,15.3125836213341,19.3586023106706,18.3814046595272,19.6942951765898,16.4776721074215,16.7815502640699,19.3731760097155,19.4558295422027,14.9720468128971,14.1672052233937,20.1535890150701,19.2763056482958,20.0673854756342,18.4224388449725,19.0575776826448,21.4024457937121,19.9908867658992,16.5136799415609,16.3503139956675,19.4692313587364,22.2510889448021,20.9630586685685,18.4955217699218,21.8620307679571,17.337119573383,20.1481269457774,20.7157394263062,20.77896047022,20.2630182245997,17.1226530103951,16.945949724284,21.5067932132317,17.7439228900073,20.6535644760044,22.1852067135967,13.5289620440801,17.7984694613409,19.4603141925348,21.3125105823352,20.7779367209327,20.6416179244059,17.5321490806206,19.357379501947,14.4357019782045,16.6052938368976,19.1267008744113,20.8457349517513,19.3175513586338,18.4061904586819,20.9028259189286,19.7769179488223,17.6524512589461,21.4897395239242,20.3931730773862,13.7706696626195,20.8902979721036,12.3145705419034,20.2188275889889,20.1197815677148,20.5281428078753,11.9248043149337,11.9248043149337,20.0842677653338,19.4791811647674,20.2090823369005,22.2289368985162,17.9213005709739,19.8897541373588,16.5829624692697,19.7803035039439,19.1919709229045,12.6348180515352,20.2228154765536,12.130087403251,12.130087403251,19.0949494063364,20.5285730300795,18.9568798319921,22.2082537382267,20.0874851884684,19.7563889521972,21.0178805892433,19.5516923455637,20.282043869896,11.6938338220078,16.875369651513,21.7174281368269,20.9575906157535,14.0455945528341,15.8013875772071,15.8013875772071,11.9248043149337,21.00139726203,21.3783619786669,19.1863145380992,13.8439054273732,22.0939611368224,13.3456670761075,13.0267853035918,15.840329725114,12.6348180515352,12.4818892601862,13.3456670761075,21.0365246884531,13.9811327271558,12.130087403251,17.7392694374499,16.8386274852372,13.1398677547467,13.1398677547467,13.1398677547467,13.1398677547467,20.1128946730897,13.0267853035918,21.0143308934023,21.7376887739924,16.7421035046212,15.5659342061357,21.7591603598476,20.4602290949778,15.5890025441251,19.1933807458677,16.4281379155417,17.0922347975337,17.0689509834861,15.04002854028,20.9146076086643,22.3296303538895,16.2246709565678,21.7979677618228,14.3337390874724,21.7671469300927,15.3668876189073,15.4192527423607,18.3097950422393,15.6777581925644,14.1075597373662,17.2649847629917,20.400937681522,16.4776721074215,21.6723218250093,19.1749188813886,12.9057140413384,18.9756853515877,22.3687321136376,15.840329725114,19.2537830240764,21.0645413749139,19.7878841234893,13.7706696626195,19.7123859440358,19.1503276359467,20.5216689495945,14.3855512064191,19.7929097594161,21.2544886771381,19.9421931110287,19.9421931110287,19.2945313136595,19.1961952918413,13.8439054273732,20.3295073744326,20.4944936958485,17.8719196243201,15.2561992217656,18.5427207437881,20.1408020607539,20.6531811595984,20.1608301380075,20.0758586423466,15.9862222962431,19.9341133232308,19.3450797828416,17.1523833138406,15.1050590561126,17.037252478234,17.1075317971132,15.4447486517457,19.8144326825285,13.6135405510155,18.8315768920833,19.8201578587349,15.7613525063434,15.6117086396742,14.4357019782045,21.2113712207536,17.6869221048388,22.1921942104392,19.8859130752885,17.7299109631303,16.0204579468312,20.0641093111529,18.7359091976879,21.5924882541332,17.1957524839532,15.1975771506459,20.1222753372909,18.7985646765168,19.9162799396758,16.8843993611985,13.7706696626195,16.3096881902664,15.3933042519638,19.0496400179017,19.8491509451787,17.9804113192125,19.3438425884973,20.1266259955052,20.1408020607539,21.1800517054597,17.1957524839532,14.5771276281867,17.5484971573843,12.6348180515352,12.6348180515352,16.9286524404976,17.7669371033375,18.8986003722095,16.8933683490854,20.0771564985104,16.7117402249923,17.9804113192125,18.169326856153,16.2958762527023,21.1744264651765,19.827476350021,20.2216775491875,17.91725888209,14.2801580888833,18.4197457790742,19.8943461462169,16.8571248761385,16.0538802978005,18.2949566663389,20.0020424161915,15.2271784019298,16.7815502640699,15.8968142150176,15.04002854028,16.6272635745276,17.3686290089728,15.07289736813,18.9112045527137,17.8335679993185,18.1459633822643,18.9219024744874,17.4471344333468,19.4901363603888,18.6631959596551,19.9065005054007,18.0441880299229,14.6215364357113,14.6215364357113,17.6373875142509,16.5254745288737,12.130087403251,16.6163232063064,18.7523954558061,20.3833891900803,18.9112045527137,20.3479329828902,16.6052938368976,18.7503470563095,16.2678316420048,18.7170866230792,20.6566261165412,11.9248043149337,15.8782345485842,16.8292819453059,18.2799366835378,22.6771653543307,19.8136124230792,16.8103928012089,18.5255448400669,16.8662784262297,20.0958071715212,17.0531968661889,12.6348180515352,12.130087403251,17.3178399709782,18.8543405969197,13.913976779011,18.2115778943649,17.1301489018925,15.1975771506459,19.7424930759613,16.8386274852372,17.8161407746662,19.4365723184185,15.5186655459237,19.630538849964,19.2349017499212,22.2527286257592,14.9368675888327,16.9022773939471,18.2147688364887,13.245899907445,19.871196151379,19.2004044202613,20.8250260232441,12.3145705419034,19.8475596428723,15.7409065320485,20.5233990863871,14.1075597373662,19.8119701129419,18.8411147522054,19.3671259505437,11.4303977042964,14.3855512064191,21.1253568475022,15.7613525063434,21.6720308433301,14.7475473009664,16.1341095793971,17.202843099759,15.04002854028,17.3993902076886,16.7912335309883,20.0693465557303,12.7755203026965,16.149617672749,12.4818892601862,19.187731206266,20.5502914024592,18.9654649144497,16.9965294034581,14.8261304120748,18.498052281756,15.3399864812078,18.4278071827661,15.7613525063434,16.086525501471,15.9332585063981,19.7649880266315,17.9919055031525,16.8198706814625,16.8008473940455,16.8008473940455,11.9248043149337,12.6348180515352,17.0922347975337,14.3855512064191,19.8633723463948,19.2710453419392,18.778710044007,10.7620154654366,17.9009606491458,19.2710453419392,14.1075597373662,18.435815580693,17.8421920921248,15.5424924404583,17.1075317971132,18.4955217699218,18.1425894363292,12.4818892601862,17.3810213538362,16.1341095793971,15.5424924404583,12.7755203026965,18.4197457790742,17.2308272666479,18.2979387338881,19.8234163870359,16.3767476598635,17.7252056958466,14.0455945528341,19.1905593946504,18.9289811247792,13.9811327271558,16.9630247323137,17.807336039845,15.7815103024704,12.6348180515352,12.6348180515352,16.5136799415609,19.0826238437656,20.4043174396193,17.2717102078241,9.75115060553602,9.75115060553602,18.395236652471,17.3307247916066,15.3933042519638,18.264730755316,17.6373875142509,18.5005775260106,17.5266530886386,16.7619716287489,17.1450135964435,16.8479081762206,19.3124751428022,11.6938338220078,16.0372684684923,14.2801580888833,19.2945313136595,16.0034441398068,19.4758741266029,16.8479081762206,16.0204579468312,18.8449085942359,17.337119573383,18.7128633568874,14.7873455809501,17.9452808027599,17.1226530103951,17.1376022769618,17.984254475954,15.6117086396742,17.9649194289544,19.3276375650685,19.4365723184185,17.888596594439,15.1365420018419,16.0034441398068,19.2294503599098,17.4353628964611,17.4234829789244,19.0607378043956,17.1151141341332,18.7585196835198,9.75115060553602,15.1365420018419,16.4026830738168,14.7873455809501,17.6118789785835,18.9377703385878,14.8639497336129,14.9008482243626,19.8475596428723,20.1395765274585,13.1398677547467,19.8499457538561,18.1083347076545,19.1820541891873,14.9008482243626,15.3933042519638,11.124724876289,14.9008482243626,19.3970642410911,16.8198706814625,17.0531968661889,20.9854971310272,18.5596562541288,19.38159321904,21.0287287147562,16.1951334195648,16.5136799415609,18.604462262134,14.3337390874724,15.07289736813,15.2561992217656,19.3326479257464,14.1672052233937,17.2238871933875,17.3434830020115,20.0199359909179,18.4464121453598,18.5904918372769,18.022049120071,13.0267853035918,18.5475837055952,16.4154694975993,14.2801580888833,15.8782345485842,19.0888028055985,19.2443806838824,15.4698067922852,15.8782345485842,19.6656722750918,15.8594084911348,17.7345988358629,15.840329725114,13.913976779011,17.0211133572295,14.4357019782045,16.1025673311142,18.3418206544775,11.4303977042964,11.4303977042964,19.9950831985299,19.2763056482958,17.5430709244817,17.311349322699,16.9630247323137,18.5974972738938,16.4531289535021,15.8594084911348,18.2585953181141,20.965352613998,20.2397450766346,15.5890025441251,16.5487618726662,17.2916806776005,13.0267853035918,19.2684062305978,18.7965940142072,19.3060984551992,16.9545146694682,19.5160099192676,18.5131254689391,16.2678316420048,18.2524291177122,16.0538802978005,16.1951334195648,20.9885841917155,15.8013875772071,19.4736641344545,17.6272452159053,15.2271784019298,17.5805805564662,18.1890071497803,19.6286085094532,15.6777581925644,19.1961952918413,19.4547056420288,17.1376022769618,19.4377140590351,13.1398677547467,15.1365420018419,18.2616668635935,18.3446909683926,14.3855512064191,17.6673389517636,16.4654555525795,15.4192527423607,19.7177574017963,3.77952755905512,18.264730755316,15.7613525063434,13.0267853035918,19.7195422490489,16.2678316420048,8.9733663912244,8.9733663912244,7.75192306836651,16.7219375695761,17.3993902076886,17.6067155221808,11.9248043149337,16.1025673311142,13.6135405510155,18.7503470563095,13.5289620440801,19.7150748527546,16.3096881902664,13.245899907445,17.0211133572295,11.6938338220078,15.2561992217656,18.7585196835198,20.0362022791806,15.5890025441251,8.9733663912244,17.3932961896552,19.9377931767086,15.3933042519638,12.3145705419034,14.1672052233937,18.1792061911917,15.07289736813,15.4192527423607,16.2392075511492,19.0934157720723,15.04002854028,13.3456670761075,18.7746991791997,19.8459660888245,18.4516760220843,10.7620154654366,15.8013875772071,11.4303977042964,18.9289811247792,18.435815580693,14.3337390874724,15.2561992217656,19.2362606185863,13.3456670761075,18.1922569255497,15.8782345485842,18.6296616678071,13.0267853035918,18.7441807244137,16.4897804845379,14.8639497336129,16.9965294034581,18.5596562541288,18.4411253957884,17.957100920704,18.9499665937121,18.7965940142072,13.3456670761075,19.0686011350516,17.9050550937873,13.0267853035918,13.0267853035918,19.9767837713122,11.4303977042964,18.804457201168,18.0108304342482,11.9248043149337,15.8594084911348,19.8539114031617,13.1398677547467,13.1398677547467,13.1398677547467,13.245899907445,12.4818892601862,16.8386274852372,15.9687874995265,12.3145705419034,14.7475473009664,15.8968142150176,17.202843099759,16.7014651436091,17.3932961896552,16.0034441398068],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[62.1567120180552,67.1372606392799,62.0667021032138,59.8614591895993,62.0704525163322,59.8727104289545,57.2999270297375,57.3261799215663,53.0507089665993,53.3244891242419,59.8502079502441,52.9231949205739,53.6432742393052,59.8464575371257,59.8577087764809,57.3224295084479,59.8652096027177,52.7844296351934,57.7649782564181,59.8689600158361,59.8539583633625,52.9456973992843,53.9433072887766,55.7322543462496,53.1369684683223,60.6227930526329,55.6872493888289,65.9333780282761,52.9869519435866,62.2392211066599,55.2634527064506,62.8730409236681,58.9126046706461,55.9310262415244,55.2522014670954,54.9371667651505,68.5811696898609,59.2313897857094,67.2085184885294,66.0458904218278,59.4789170515233,61.4403831124424,54.9446675913873,55.0271766799919,56.1447997892727,66.1208986841957,57.3299303346846,55.7285039331312,58.6313236867667,61.6204029421252,53.1482197076775,56.8348758030569,64.9845235093229,53.6807783704892,55.6834989757105,55.0309270931103,53.2982362324132,55.0684312242942,53.3957469734913,55.6909998019473,58.290036092993,53.6845287836075,62.843037618721,62.8655400974313,53.7257833279099,53.3282395373603,66.9309879177683,55.0196758537551,57.3674344658686,61.4028789812584,55.1359386604253,65.9371284413944,66.0496408349462,53.5007585408063,57.3036774428559,57.2774245510271,65.415821017938,55.6797485625921,52.2481205592634,66.1359003366692,56.1523006155095,53.6732775442524,53.4820064752144,52.990702356705,61.9166855784781,67.2610242721869,61.0465897350112,59.7001914255084,53.650775065542,62.8505384449578,53.9845618330789,52.1168561001197,53.4032477997281,55.0234262668735,67.2572738590685,66.0533912480646,53.4069982128465,52.1243569263564,54.9484180045057,56.606100602835,65.4683268015954,53.4219998653201,59.5276724220624,52.2706230379737,56.0735419400233,56.0022840907738,53.1857238388614,54.2770940563135,59.6139319237854,53.5120097801615,56.1935551598118,56.1898047466934,56.3735749894947,55.9835320251819,56.4410824256257,54.2283386857744,52.88569078939,53.4520031702672,56.0510394613129,53.7745386984489,52.2968759298025,55.8447667398014,61.4066293943768,65.3033086243862,52.1318577525932,53.4144990390833,56.1560510286279,66.353424297536,65.4795780409506,59.7189434911004,55.7247535200128,58.4100493127816,53.4445023440305,56.0097849170106,56.099794831852,54.8471568503091,59.5989302713118,55.7135022806577,57.6074609054456,56.013535330129,62.861789684313,62.8392872056026,66.7472176749671,55.946027893998,52.5744065005635,55.8560179791566,59.9327170388487,57.2699237247904,59.8952129076648,52.2593717986186,59.7226939042188,53.0169552485338,52.5706560874451,64.5869797187733,52.4131387364726,55.5597353428036,66.0871449661301,65.4720772147138,66.8709813078741,54.970920483216,53.1669717732694,59.4639153990497,57.5362030561962,53.3432411898339,53.5232610195167,52.2668726248553,53.6095205212397,53.8045420033961,65.7383565461197,55.1134361817149,58.8563484738702,65.187045817716,52.7169221990624,53.5382626719902,62.824285553129,59.523922008944,55.9235254152876,65.76085902483,60.5177814853179,56.0847931793785,66.1171482710773,55.9122741759324,56.0060345038922,54.8359056109539,53.5870180425293,64.459465672748,53.7032808491995,62.8692905105497,62.8467880318394,65.5920904345024,66.4246821467854,58.7475864934368,67.0547515506753,61.035338495656,55.0084246143999,55.717252693776,63.8143946163845,53.1969750782166,53.3319899504787,56.0772923531417,66.075893726775,55.1059353554781,65.588340021384,53.8795502657639,58.0575104796527,52.2631222117369,65.1420408602953,65.4758276278322,63.713133462188,53.8945519182375,57.2736741379088,53.3057370586499,62.7267748120508,61.2303599778124,53.7145320885547,55.8672692185117,57.8249848663124,52.0380974246334,55.9947832645371,66.0646424874198,67.204768075411,66.7397168487304,52.6381635235762,55.8485171529198,61.0090856038273,55.1096857685965,62.8017830744187,53.6957800229627,55.094684116123,65.2770557325574,55.8522675660382,56.8686295211224,65.3070590375046,55.9010229365773,63.7093830490696,55.8747700447485,53.5945188687661,55.8785204578669,56.4448328387441,53.7782891115673,53.4182494522017,65.5733383689104,65.1907962308344,63.3868475208878,52.2931255166841,56.1598014417463,63.6156227211098,59.2351401988278,59.4601649859313,53.5195106063983,58.5413137719253,53.8270444821064,53.8757998526455,60.5627864427386,53.7182825016731,66.057141661183,56.1973055729302,53.5082593670431,53.2644825143476,68.4874093619011,59.4714162252865,61.4591351780343,58.8525980607518,57.1724129837122,61.0315880825376,63.7018822228328,55.7210031068944,52.904442854982,59.90646414702,59.9102145601384,55.5672361690404,57.6749683415767,55.1246874210701,56.4748361436912,66.5709482584027,60.5890393345674,56.0885435924969,53.4482527571488,57.2886757903823,57.2136675280145,67.5310540167111,64.5719780662997,57.734974951471,53.009454422297,59.8127038190602,53.7070312623179,55.9047733496956,56.4523336649809,52.503148651314,65.2245499489,54.1533304234066,56.1110460712072,59.8052029928234,59.0138658248427,55.9197750021692,53.755786632857,58.2862856798746,52.5931585661554,54.205836207064,66.1058970317221,59.7301947304556,55.9685303727083,53.8983023313559,58.863849300107,52.5106494775508,58.5375633588069,60.4952790066076,63.3755962815327,63.7056326359512,52.120606513238,53.4107486259649,55.9160245890508,63.4431037176637,56.864879108004,63.6043714817546,59.7076922517452,55.5709865821588,61.0053351907089,66.7509680880855,65.3333119293333,53.6770279573708,52.9982031829418,65.4083201917012],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.9952351669425,15.3508329797852,15.0572856894512,14.8181130379814,14.7113206099705,14.2274406173666,14.1227825810091,13.9599914244894,13.6917798769865,13.6814317055365,13.4744427990217,13.4744427990217,13.1929598182073,12.5517965008905,12.3754745328387,12.3588808107065,12.1053660151381,12.1052329435425,12.0999677592754,11.7769048052851,11.7264104416758,11.4755122034479,11.0431772429947,10.8077373902907,10.8019948812698,10.6033176773833,10.4038017730419,10.1548449713949,10.1548449713949,9.92146306452783,9.80097434021279,9.36639945660178,9.27207077110822,9.27207077110822,9.19665515948023,9.10161145058715,8.96592329837049,8.9487885005749,8.9487885005749,8.89850411899272,8.7761424821365,8.56390173521625,8.5459985807144,8.49268626155191,8.49268626155191,8.39207861885423,8.29251833273119,8.23876282197173,8.10013462360587,8.07472710748105,8.06587572425707,8.04224919044229,7.99170002494884,7.97137877137754,7.89133537960367,7.8869629133964,7.57385570033472,7.5662921292108,7.5662921292108,7.51731433843159,7.46025629499539,7.46025629499539,7.43215737097354,7.37363312351706,7.21809955712464,7.17791934746262,7.17702335778626,6.96405788772349,6.92008087400357,6.76132678205834,6.75825892227911,6.69195305364942,6.67581458719755,6.56575884376343,6.44050110423261,6.43458577982968,6.3885566771335,6.3885566771335,6.34012225290908,6.33390296202916,6.24278694984543,6.17458264432772,6.02419788515694,5.98355322263183,5.98317870392692,5.97071751476309,5.90140142343617,5.85123233798323,5.55124652784435,5.55124652784435,5.52844120382663,5.52665331841377,5.44312671333865,5.41576714095386,5.39006041580195,5.34812982883494,5.23893407360219,5.14953503569385,5.12636067161272,5.00382940148152,4.81124172231472,4.7409315834582,4.69812478838071,4.69046988790902,4.66051583236165,4.64814632824019,4.55988727816914,4.47747212638714,4.42138461826821,4.36409461112245,4.33770553076836,4.30928936759369,4.30475375430312,4.300972339186,4.28397776698132,4.22803037647078,4.22005751638094,4.19492405722692,4.15696230068975,4.14893164008558,4.12584530303336,4.10780389319826,4.02234981394459,3.93693709020957,3.93693709020957,3.90566165203569,3.87057738152468,3.85943020046936,3.83539343901295,3.82433488675535,3.81280493194071,3.77485263838339,3.74107389666731,3.73485296404798,3.72416390465396,3.69560879697319,3.64129173791344,3.57991174391258,3.42371624099262,3.35790602442601,3.26964865973652,3.25186905111891,3.23439050547413,3.22866602840582,3.22098746313705,3.20354917500334,3.19206828593675,3.18540434905159,3.14098966935814,3.13310382988909,3.1120807304911,3.07938450696881,3.07916194078185,3.07297306714631,3.06536501923564,3.01911972971164,3.0160939013045,2.98999129691148,2.98278213009337,2.98166759878495,2.9678770827699,2.96597079908682,2.95787069879264,2.94596775959365,2.92392455003387,2.8966900257973,2.8925812200634,2.87939063818764,2.87330404313028,2.86026618159939,2.85668035778821,2.85623040020118,2.82034948671653,2.81514835695365,2.77795438752946,2.77726034204782,2.72811427992529,2.72811427992529,2.66476449075306,2.66476449075306,2.66168326314074,2.65635784722415,2.65486341637076,2.64698694445836,2.62528898365564,2.6212302995391,2.54450562533497,2.52999759655264,2.50235757330712,2.47625523910836,2.47615557511929,2.46310195346884,2.45394681269166,2.44364484023423,2.44208401254027,2.43042042892795,2.42709694963641,2.4173531942542,2.41637185872579,2.41538624023639,2.41340178945842,2.41279198173075,2.35610074687435,2.34717379397054,2.34465613774929,2.34465613774929,2.34014208468309,2.30774737461727,2.28788915335179,2.28739090941685,2.21545703943066,2.20065007336296,2.19937700772179,2.17395020945453,2.15088572464225,2.13119921870251,2.12631281625587,2.11890280646384,2.11051688137021,2.11024950579,2.06431370535071,2.03124151406351,1.99599335962178,1.95305613219378,1.92777582614713,1.87541689426434,1.87402379766647,1.86971218604137,1.86645790337737,1.86250351818811,1.83338400008534,1.81477712138717,1.80821954442084,1.76760294297633,1.70426719694358,1.6292624094016,1.58234639668458,1.57389871113949,1.55090447812221,1.54888020406788,1.52624287559983,1.51877492270083,1.50737547985035,1.48395896412296,1.47876459079985,1.45942689246311,1.45942689246311,1.45502111768087,1.43464216746677,1.4068603786326,1.38955660283063,1.37661325168639,1.35297999857911,1.34835574757592,1.33195472413304,1.32339282075964,1.31797738581251,1.30916482434028,1.30533307281446],"text":["GO:0070062 (2110) <br> extracellular exosome <br> 1.748e-126","GO:1903561 (2132) <br> extracellular vesicle <br> 1.543e-125","GO:0065010 (2133) <br> extracellular membrane-bounded organelle <br> 1.841e-125","GO:0043230 (2133) <br> extracellular organelle <br> 1.841e-125","GO:0070013 (6569) <br> intracellular organelle lumen <br> 8.694e-118","GO:0043233 (6570) <br> organelle lumen <br> 9.531e-118","GO:0031974 (6570) <br> membrane-enclosed lumen <br> 9.531e-118","GO:0031982 (3942) <br> vesicle <br> 1.668e-81","GO:0005654 (4161) <br> nucleoplasm <br> 1.039e-79","GO:0005737 (12193) <br> cytoplasm <br> 3.021e-79","GO:0043227 (15557) <br> membrane-bounded organelle <br> 5.129e-79","GO:0005615 (3353) <br> extracellular space <br> 3.300e-78","GO:0005829 (5373) <br> cytosol <br> 7.628e-77","GO:0043226 (16674) <br> organelle <br> 1.781e-74","GO:0043229 (15870) <br> intracellular organelle <br> 7.098e-68","GO:0031981 (5442) <br> nuclear lumen <br> 2.694e-67","GO:0043231 (14514) <br> intracellular membrane-bounded organelle <br> 4.960e-62","GO:0005576 (4303) <br> extracellular region <br> 5.671e-59","GO:0032991 (7228) <br> protein-containing complex <br> 2.691e-56","GO:0043232 (6362) <br> intracellular non-membrane-bounded organelle <br> 1.836e-55","GO:0043228 (6362) <br> non-membrane-bounded organelle <br> 1.836e-55","GO:0005622 (17758) <br> intracellular anatomical structure <br> 2.620e-55","GO:0005925 (420) <br> focal adhesion <br> 6.071e-55","GO:0030055 (428) <br> cell-substrate junction <br> 6.804e-54","GO:0005681 (195) <br> spliceosomal complex <br> 2.070e-50","GO:0044391 (189) <br> ribosomal subunit <br> 3.009e-48","GO:0022626 (104) <br> cytosolic ribosome <br> 1.556e-41","GO:0099080 (1330) <br> supramolecular complex <br> 2.064e-39","GO:0005634 (10053) <br> nucleus <br> 8.225e-39","GO:0070161 (839) <br> anchoring junction <br> 3.668e-36","GO:0016607 (414) <br> nuclear speck <br> 4.343e-35","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.303e-34","GO:0035770 (260) <br> ribonucleoprotein granule <br> 1.040e-33","GO:0030141 (860) <br> secretory granule <br> 3.721e-32","GO:0016604 (869) <br> nuclear body <br> 8.157e-32","GO:0010494 (79) <br> cytoplasmic stress granule <br> 1.437e-31","GO:1990904 (2377) <br> ribonucleoprotein complex <br> 7.219e-31","GO:0036464 (242) <br> cytoplasmic ribonucleoprotein granule <br> 8.452e-31","GO:1904813 (124) <br> ficolin-1-rich granule lumen <br> 1.103e-30","GO:0099503 (1025) <br> secretory vesicle <br> 1.510e-29","GO:0042470 (109) <br> melanosome <br> 2.004e-29","GO:0048770 (109) <br> pigment granule <br> 2.004e-29","GO:0012505 (4674) <br> endomembrane system <br> 2.055e-29","GO:0015934 (115) <br> large ribosomal subunit <br> 3.179e-29","GO:0030684 (79) <br> preribosome <br> 2.337e-27","GO:0101002 (184) <br> ficolin-1-rich granule <br> 3.832e-27","GO:0031983 (325) <br> vesicle lumen <br> 1.277e-26","GO:0030054 (2076) <br> cell junction <br> 1.419e-26","GO:0034774 (320) <br> secretory granule lumen <br> 2.379e-26","GO:0060205 (323) <br> cytoplasmic vesicle lumen <br> 4.646e-26","GO:0005684 (93) <br> U2-type spliceosomal complex <br> 1.802e-25","GO:0031410 (2447) <br> cytoplasmic vesicle <br> 9.159e-25","GO:0097708 (2449) <br> intracellular vesicle <br> 1.020e-24","GO:0005840 (520) <br> ribosome <br> 5.107e-23","GO:0022625 (55) <br> cytosolic large ribosomal subunit <br> 1.256e-21","GO:0015935 (76) <br> small ribosomal subunit <br> 4.705e-20","GO:0005730 (1942) <br> nucleolus <br> 8.137e-20","GO:0016020 (9818) <br> membrane <br> 8.185e-20","GO:0005759 (483) <br> mitochondrial matrix <br> 1.035e-19","GO:0022627 (47) <br> cytosolic small ribosomal subunit <br> 1.599e-19","GO:0034399 (130) <br> nuclear periphery <br> 4.465e-19","GO:0005844 (71) <br> polysome <br> 1.024e-18","GO:0071005 (50) <br> U2-type precatalytic spliceosome <br> 1.770e-17","GO:0071011 (52) <br> precatalytic spliceosome <br> 6.085e-17","GO:0005856 (2334) <br> cytoskeleton <br> 1.011e-16","GO:0005739 (1674) <br> mitochondrion <br> 4.458e-16","GO:1902494 (1424) <br> catalytic complex <br> 8.764e-16","GO:0015629 (501) <br> actin cytoskeleton <br> 1.520e-15","GO:0032040 (39) <br> small-subunit processome <br> 1.944e-15","GO:0048471 (719) <br> perinuclear region of cytoplasm <br> 5.923e-15","GO:0016363 (109) <br> nuclear matrix <br> 7.537e-15","GO:0099081 (996) <br> supramolecular polymer <br> 1.096e-14","GO:0099512 (988) <br> supramolecular fiber <br> 2.033e-14","GO:0005788 (311) <br> endoplasmic reticulum lumen <br> 2.082e-14","GO:0031975 (1245) <br> envelope <br> 3.354e-14","GO:0031967 (1245) <br> organelle envelope <br> 3.354e-14","GO:0098687 (359) <br> chromosomal region <br> 6.413e-14","GO:0022624 (25) <br> proteasome accessory complex <br> 2.807e-13","GO:0000502 (66) <br> proteasome complex <br> 4.212e-13","GO:0101031 (28) <br> chaperone complex <br> 4.376e-13","GO:0030686 (27) <br> 90S preribosome <br> 7.846e-13","GO:0005838 (22) <br> proteasome regulatory particle <br> 7.848e-13","GO:0005783 (2007) <br> endoplasmic reticulum <br> 7.944e-13","GO:0005635 (473) <br> nuclear envelope <br> 1.671e-12","GO:0062023 (421) <br> collagen-containing extracellular matrix <br> 1.878e-12","GO:1905369 (72) <br> endopeptidase complex <br> 3.346e-12","GO:0045202 (1305) <br> synapse <br> 9.054e-12","GO:0042788 (32) <br> polysomal ribosome <br> 1.557e-11","GO:0005832 (10) <br> chaperonin-containing T-complex <br> 1.578e-11","GO:0071007 (30) <br> U2-type catalytic step 2 spliceosome <br> 2.493e-11","GO:0005938 (319) <br> cell cortex <br> 3.946e-11","GO:0000313 (86) <br> organellar ribosome <br> 7.001e-11","GO:0005761 (86) <br> mitochondrial ribosome <br> 7.001e-11","GO:0015630 (1317) <br> microtubule cytoskeleton <br> 1.198e-10","GO:1905368 (103) <br> peptidase complex <br> 1.581e-10","GO:0099513 (757) <br> polymeric cytoskeletal fiber <br> 4.301e-10","GO:0005762 (56) <br> mitochondrial large ribosomal subunit <br> 5.345e-10","GO:0000315 (56) <br> organellar large ribosomal subunit <br> 5.345e-10","GO:0012506 (1190) <br> vesicle membrane <br> 6.358e-10","GO:0031252 (421) <br> cell leading edge <br> 7.914e-10","GO:0098794 (613) <br> postsynapse <br> 1.082e-09","GO:0005766 (154) <br> primary lysosome <br> 1.125e-09","GO:0042582 (154) <br> azurophil granule <br> 1.125e-09","GO:0000781 (169) <br> chromosome, telomeric region <br> 1.263e-09","GO:0030659 (1171) <br> cytoplasmic vesicle membrane <br> 1.674e-09","GO:0030424 (633) <br> axon <br> 2.730e-09","GO:0005694 (1795) <br> chromosome <br> 2.844e-09","GO:0009295 (45) <br> nucleoid <br> 3.216e-09","GO:0042645 (45) <br> mitochondrial nucleoid <br> 3.216e-09","GO:0005791 (85) <br> rough endoplasmic reticulum <br> 4.054e-09","GO:0030864 (80) <br> cortical actin cytoskeleton <br> 5.099e-09","GO:0030863 (110) <br> cortical cytoskeleton <br> 5.771e-09","GO:0031012 (560) <br> extracellular matrix <br> 7.941e-09","GO:0030312 (561) <br> external encapsulating structure <br> 8.419e-09","GO:0031090 (3657) <br> organelle membrane <br> 8.593e-09","GO:0008540 (12) <br> proteasome regulatory particle, base subcomplex <br> 9.073e-09","GO:0005604 (97) <br> basement membrane <br> 1.019e-08","GO:0005775 (175) <br> vacuolar lumen <br> 1.068e-08","GO:0030496 (199) <br> midbody <br> 1.284e-08","GO:0005874 (432) <br> microtubule <br> 1.297e-08","GO:0000793 (245) <br> condensed chromosome <br> 2.668e-08","GO:0030117 (96) <br> membrane coat <br> 2.715e-08","GO:0048475 (96) <br> coated membrane <br> 2.715e-08","GO:0098588 (2124) <br> bounding membrane of organelle <br> 3.039e-08","GO:0000323 (729) <br> lytic vacuole <br> 3.465e-08","GO:0005764 (729) <br> lysosome <br> 3.465e-08","GO:0030687 (24) <br> preribosome, large subunit precursor <br> 3.697e-08","GO:0120025 (2205) <br> plasma membrane bounded cell projection <br> 4.230e-08","GO:0098798 (277) <br> mitochondrial protein-containing complex <br> 6.052e-08","GO:0042995 (2312) <br> cell projection <br> 6.639e-08","GO:0030027 (203) <br> lamellipodium <br> 6.652e-08","GO:0034663 (11) <br> endoplasmic reticulum chaperone complex <br> 1.086e-07","GO:0005773 (827) <br> vacuole <br> 1.202e-07","GO:0030426 (163) <br> growth cone <br> 1.732e-07","GO:0030667 (309) <br> secretory granule membrane <br> 1.745e-07","GO:0009986 (908) <br> cell surface <br> 2.033e-07","GO:0042641 (79) <br> actomyosin <br> 2.110e-07","GO:0030016 (232) <br> myofibril <br> 2.718e-07","GO:0032432 (79) <br> actin filament bundle <br> 3.627e-07","GO:0030427 (169) <br> site of polarized growth <br> 3.676e-07","GO:0071010 (16) <br> prespliceosome <br> 4.087e-07","GO:0071004 (16) <br> U2-type prespliceosome <br> 4.087e-07","GO:0140534 (127) <br> endoplasmic reticulum protein-containing complex <br> 4.570e-07","GO:0030175 (104) <br> filopodium <br> 4.636e-07","GO:0001726 (181) <br> ruffle <br> 5.718e-07","GO:0030120 (56) <br> vesicle coat <br> 6.690e-07","GO:0043292 (242) <br> contractile fiber <br> 9.458e-07","GO:0031965 (303) <br> nuclear membrane <br> 1.039e-06","GO:0043256 (12) <br> laminin complex <br> 1.039e-06","GO:0000775 (202) <br> chromosome, centromeric region <br> 1.070e-06","GO:0043005 (1347) <br> neuron projection <br> 1.255e-06","GO:0005643 (85) <br> nuclear pore <br> 1.409e-06","GO:0001725 (71) <br> stress fiber <br> 2.810e-06","GO:0097517 (71) <br> contractile actin filament bundle <br> 2.810e-06","GO:0000932 (93) <br> P-body <br> 2.962e-06","GO:0019866 (562) <br> organelle inner membrane <br> 2.974e-06","GO:0099572 (346) <br> postsynaptic specialization <br> 3.605e-06","GO:0098796 (1323) <br> membrane protein complex <br> 3.839e-06","GO:0150034 (273) <br> distal axon <br> 4.073e-06","GO:0014069 (322) <br> postsynaptic density <br> 4.486e-06","GO:0005689 (28) <br> U12-type spliceosomal complex <br> 5.769e-06","GO:0042383 (135) <br> sarcolemma <br> 7.087e-06","GO:0032279 (328) <br> asymmetric synapse <br> 7.475e-06","GO:0005743 (500) <br> mitochondrial inner membrane <br> 9.912e-06","GO:0005794 (1606) <br> Golgi apparatus <br> 1.544e-05","GO:0000779 (147) <br> condensed chromosome, centromeric region <br> 1.816e-05","GO:0005819 (397) <br> spindle <br> 2.004e-05","GO:0005884 (114) <br> actin filament <br> 2.040e-05","GO:0098978 (332) <br> glutamatergic synapse <br> 2.185e-05","GO:0016323 (224) <br> basolateral plasma membrane <br> 2.248e-05","GO:0035578 (91) <br> azurophil granule lumen <br> 2.755e-05","GO:0098552 (625) <br> side of membrane <br> 3.331e-05","GO:0002199 (8) <br> zona pellucida receptor complex <br> 3.790e-05","GO:0005798 (89) <br> Golgi-associated vesicle <br> 4.324e-05","GO:0070993 (18) <br> translation preinitiation complex <br> 4.595e-05","GO:0042581 (158) <br> specific granule <br> 4.906e-05","GO:0030139 (337) <br> endocytic vesicle <br> 4.957e-05","GO:0098984 (352) <br> neuron to neuron synapse <br> 5.001e-05","GO:0044297 (554) <br> cell body <br> 5.200e-05","GO:0030662 (178) <br> coated vesicle membrane <br> 5.915e-05","GO:0099738 (43) <br> cell cortex region <br> 6.025e-05","GO:0030135 (294) <br> coated vesicle <br> 6.384e-05","GO:0030425 (612) <br> dendrite <br> 6.967e-05","GO:0009925 (249) <br> basal plasma membrane <br> 7.097e-05","GO:0005813 (623) <br> centrosome <br> 7.484e-05","GO:0097447 (614) <br> dendritic tree <br> 7.802e-05","GO:0005849 (28) <br> mRNA cleavage factor complex <br> 9.498e-05","GO:0071012 (12) <br> catalytic step 1 spliceosome <br> 1.156e-04","GO:0071006 (12) <br> U2-type catalytic step 1 spliceosome <br> 1.156e-04","GO:0098858 (213) <br> actin-based cell projection <br> 1.243e-04","GO:0120111 (88) <br> neuron projection cytoplasm <br> 1.347e-04","GO:0035145 (20) <br> exon-exon junction complex <br> 1.382e-04","GO:1902737 (4) <br> dendritic filopodium <br> 1.461e-04","GO:0045178 (267) <br> basal part of cell <br> 1.499e-04","GO:0015030 (102) <br> Cajal body <br> 1.539e-04","GO:0030017 (211) <br> sarcomere <br> 1.679e-04","GO:0090533 (19) <br> cation-transporting ATPase complex <br> 1.815e-04","GO:0005697 (23) <br> telomerase holoenzyme complex <br> 1.841e-04","GO:0005740 (796) <br> mitochondrial envelope <br> 1.887e-04","GO:0030660 (54) <br> Golgi-associated vesicle membrane <br> 2.016e-04","GO:0099568 (265) <br> cytoplasmic region <br> 2.284e-04","GO:0016281 (13) <br> eukaryotic translation initiation factor 4F complex <br> 2.631e-04","GO:0098857 (331) <br> membrane microdomain <br> 3.770e-04","GO:0005905 (72) <br> clathrin-coated pit <br> 4.386e-04","GO:0033290 (15) <br> eukaryotic 48S preinitiation complex <br> 5.375e-04","GO:0000776 (137) <br> kinetochore <br> 5.599e-04","GO:0098533 (25) <br> ATPase dependent transmembrane transport complex <br> 5.829e-04","GO:0098797 (692) <br> plasma membrane protein complex <br> 5.907e-04","GO:0072686 (162) <br> mitotic spindle <br> 6.012e-04","GO:0005911 (495) <br> cell-cell junction <br> 6.258e-04","GO:0031966 (745) <br> mitochondrial membrane <br> 6.426e-04","GO:0005732 (28) <br> sno(s)RNA-containing ribonucleoprotein complex <br> 6.525e-04","GO:0070820 (163) <br> tertiary granule <br> 7.228e-04","GO:0045335 (140) <br> phagocytic vesicle <br> 7.360e-04","GO:0005852 (16) <br> eukaryotic translation initiation factor 3 complex <br> 7.725e-04","GO:0030123 (11) <br> AP-3 adaptor complex <br> 8.329e-04","GO:0033018 (10) <br> sarcoplasmic reticulum lumen <br> 8.334e-04","GO:0000228 (235) <br> nuclear chromosome <br> 8.453e-04","GO:0030315 (52) <br> T-tubule <br> 8.603e-04","GO:0099524 (12) <br> postsynaptic cytosol <br> 9.569e-04","GO:1904724 (55) <br> tertiary granule lumen <br> 9.636e-04","GO:0140513 (2671) <br> nuclear protein-containing complex <br> 1.023e-03","GO:0002102 (30) <br> podosome <br> 1.040e-03","GO:0030118 (48) <br> clathrin coat <br> 1.043e-03","GO:0045121 (330) <br> membrane raft <br> 1.077e-03","GO:0016282 (17) <br> eukaryotic 43S preinitiation complex <br> 1.082e-03","GO:0070937 (6) <br> CRD-mediated mRNA stability complex <br> 1.102e-03","GO:0005847 (24) <br> mRNA cleavage and polyadenylation specificity factor complex <br> 1.132e-03","GO:0016234 (75) <br> inclusion body <br> 1.191e-03","GO:0098576 (32) <br> lumenal side of membrane <br> 1.269e-03","GO:0030119 (42) <br> AP-type membrane coat adaptor complex <br> 1.281e-03","GO:0031429 (5) <br> box H/ACA snoRNP complex <br> 1.320e-03","GO:0098590 (1237) <br> plasma membrane region <br> 1.339e-03","GO:0030132 (17) <br> clathrin coat of coated pit <br> 1.380e-03","GO:0072669 (8) <br> tRNA-splicing ligase complex <br> 1.391e-03","GO:0030125 (29) <br> clathrin vesicle coat <br> 1.392e-03","GO:0005815 (820) <br> microtubule organizing center <br> 1.512e-03","GO:0030126 (13) <br> COPI vesicle coat <br> 1.531e-03","GO:0031091 (90) <br> platelet alpha granule <br> 1.667e-03","GO:0005875 (162) <br> microtubule associated complex <br> 1.670e-03","GO:0005765 (400) <br> lysosomal membrane <br> 1.870e-03","GO:0098852 (400) <br> lytic vacuole membrane <br> 1.870e-03","GO:0098553 (25) <br> lumenal side of endoplasmic reticulum membrane <br> 2.164e-03","GO:0071556 (25) <br> integral component of lumenal side of endoplasmic reticulum membrane <br> 2.164e-03","GO:0000792 (72) <br> heterochromatin <br> 2.179e-03","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 2.206e-03","GO:0071944 (6181) <br> cell periphery <br> 2.214e-03","GO:0036477 (835) <br> somatodendritic compartment <br> 2.254e-03","GO:0042382 (12) <br> paraspeckles <br> 2.370e-03","GO:0005793 (132) <br> endoplasmic reticulum-Golgi intermediate compartment <br> 2.392e-03","GO:0034709 (12) <br> methylosome <br> 2.854e-03","GO:0005890 (12) <br> sodium:potassium-exchanging ATPase complex <br> 2.951e-03","GO:0005903 (105) <br> brush border <br> 3.145e-03","GO:0044309 (176) <br> neuron spine <br> 3.340e-03","GO:0005853 (4) <br> eukaryotic translation elongation factor 1 complex <br> 3.341e-03","GO:0099522 (15) <br> cytosolic region <br> 3.443e-03","GO:0030867 (28) <br> rough endoplasmic reticulum membrane <br> 3.516e-03","GO:0005790 (32) <br> smooth endoplasmic reticulum <br> 3.600e-03","GO:0005721 (24) <br> pericentric heterochromatin <br> 3.613e-03","GO:1990812 (3) <br> growth cone filopodium <br> 3.712e-03","GO:0042405 (12) <br> nuclear inclusion body <br> 3.740e-03","GO:0051233 (36) <br> spindle midzone <br> 3.825e-03","GO:0035577 (57) <br> azurophil granule membrane <br> 3.834e-03","GO:0031674 (142) <br> I band <br> 3.842e-03","GO:0045177 (429) <br> apical part of cell <br> 3.860e-03","GO:0072588 (6) <br> box H/ACA RNP complex <br> 3.866e-03","GO:0030018 (129) <br> Z disc <br> 4.405e-03","GO:0005610 (3) <br> laminin-5 complex <br> 4.496e-03","GO:0043259 (3) <br> laminin-10 complex <br> 4.522e-03","GO:0043260 (3) <br> laminin-11 complex <br> 4.522e-03","GO:0019897 (173) <br> extrinsic component of plasma membrane <br> 4.569e-03","GO:0032587 (97) <br> ruffle membrane <br> 4.923e-03","GO:0016328 (70) <br> lateral plasma membrane <br> 5.154e-03","GO:0031143 (18) <br> pseudopodium <br> 5.160e-03","GO:0120259 (3) <br> 7SK snRNP <br> 6.089e-03","GO:0044326 (4) <br> dendritic spine neck <br> 6.300e-03","GO:0030663 (18) <br> COPI-coated vesicle membrane <br> 6.319e-03","GO:0005774 (454) <br> vacuolar membrane <br> 6.700e-03","GO:0031970 (95) <br> organelle envelope lumen <br> 7.065e-03","GO:0031904 (35) <br> endosome lumen <br> 7.393e-03","GO:1990124 (12) <br> messenger ribonucleoprotein complex <br> 7.476e-03","GO:0097513 (3) <br> myosin II filament <br> 7.605e-03","GO:0032839 (29) <br> dendrite cytoplasm <br> 7.753e-03","GO:0005641 (10) <br> nuclear envelope lumen <br> 7.758e-03","GO:0043202 (98) <br> lysosomal lumen <br> 8.624e-03","GO:0005850 (5) <br> eukaryotic translation initiation factor 2 complex <br> 9.306e-03","GO:0030133 (409) <br> transport vesicle <br> 1.009e-02","GO:0031093 (66) <br> platelet alpha granule lumen <br> 1.114e-02","GO:0001650 (140) <br> fibrillar center <br> 1.181e-02","GO:0098562 (194) <br> cytoplasmic side of membrane <br> 1.332e-02","GO:0008180 (34) <br> COP9 signalosome <br> 1.337e-02","GO:0030670 (77) <br> phagocytic vesicle membrane <br> 1.350e-02","GO:0043197 (173) <br> dendritic spine <br> 1.360e-02","GO:0036019 (27) <br> endolysosome <br> 1.372e-02","GO:0030137 (22) <br> COPI-coated vesicle <br> 1.468e-02","GO:0005869 (16) <br> dynactin complex <br> 1.532e-02","GO:0034388 (5) <br> Pwp2p-containing subcomplex of 90S preribosome <br> 1.555e-02","GO:0001772 (44) <br> immunological synapse <br> 1.708e-02","GO:0008305 (31) <br> integrin complex <br> 1.976e-02","GO:0099631 (2) <br> postsynaptic endocytic zone cytoplasmic component <br> 2.348e-02","GO:0043025 (488) <br> neuronal cell body <br> 2.616e-02","GO:0030286 (54) <br> dynein complex <br> 2.667e-02","GO:0005912 (172) <br> adherens junction <br> 2.813e-02","GO:0035580 (61) <br> specific granule lumen <br> 2.826e-02","GO:0001652 (6) <br> granular component <br> 2.977e-02","GO:0034708 (101) <br> methyltransferase complex <br> 3.028e-02","GO:0044291 (73) <br> cell-cell contact zone <br> 3.109e-02","GO:0071541 (7) <br> eukaryotic translation initiation factor 3 complex, eIF3m <br> 3.281e-02","GO:0072589 (4) <br> box H/ACA scaRNP complex <br> 3.321e-02","GO:0000314 (28) <br> organellar small ribosomal subunit <br> 3.472e-02","GO:0005763 (28) <br> mitochondrial small ribosomal subunit <br> 3.472e-02","GO:0030136 (197) <br> clathrin-coated vesicle <br> 3.507e-02","GO:0071682 (23) <br> endocytic vesicle lumen <br> 3.676e-02","GO:0031428 (7) <br> box C/D RNP complex <br> 3.919e-02","GO:0071920 (2) <br> cleavage body <br> 4.078e-02","GO:0042824 (9) <br> MHC class I peptide loading complex <br> 4.201e-02","GO:0019898 (310) <br> extrinsic component of membrane <br> 4.436e-02","GO:0045120 (14) <br> pronucleus <br> 4.484e-02","GO:0140535 (773) <br> intracellular protein-containing complex <br> 4.656e-02","GO:0098636 (36) <br> protein complex involved in cell adhesion <br> 4.749e-02","GO:0005839 (23) <br> proteasome core complex <br> 4.809e-02","GO:0005638 (5) <br> lamin filament <br> 4.907e-02","GO:0098685 (79) <br> Schaffer collateral - CA1 synapse <br> 4.951e-02"],"key":["GO:0070062","GO:1903561","GO:0065010","GO:0043230","GO:0070013","GO:0043233","GO:0031974","GO:0031982","GO:0005654","GO:0005737","GO:0043227","GO:0005615","GO:0005829","GO:0043226","GO:0043229","GO:0031981","GO:0043231","GO:0005576","GO:0032991","GO:0043232","GO:0043228","GO:0005622","GO:0005925","GO:0030055","GO:0005681","GO:0044391","GO:0022626","GO:0099080","GO:0005634","GO:0070161","GO:0016607","GO:0071013","GO:0035770","GO:0030141","GO:0016604","GO:0010494","GO:1990904","GO:0036464","GO:1904813","GO:0099503","GO:0042470","GO:0048770","GO:0012505","GO:0015934","GO:0030684","GO:0101002","GO:0031983","GO:0030054","GO:0034774","GO:0060205","GO:0005684","GO:0031410","GO:0097708","GO:0005840","GO:0022625","GO:0015935","GO:0005730","GO:0016020","GO:0005759","GO:0022627","GO:0034399","GO:0005844","GO:0071005","GO:0071011","GO:0005856","GO:0005739","GO:1902494","GO:0015629","GO:0032040","GO:0048471","GO:0016363","GO:0099081","GO:0099512","GO:0005788","GO:0031975","GO:0031967","GO:0098687","GO:0022624","GO:0000502","GO:0101031","GO:0030686","GO:0005838","GO:0005783","GO:0005635","GO:0062023","GO:1905369","GO:0045202","GO:0042788","GO:0005832","GO:0071007","GO:0005938","GO:0000313","GO:0005761","GO:0015630","GO:1905368","GO:0099513","GO:0005762","GO:0000315","GO:0012506","GO:0031252","GO:0098794","GO:0005766","GO:0042582","GO:0000781","GO:0030659","GO:0030424","GO:0005694","GO:0009295","GO:0042645","GO:0005791","GO:0030864","GO:0030863","GO:0031012","GO:0030312","GO:0031090","GO:0008540","GO:0005604","GO:0005775","GO:0030496","GO:0005874","GO:0000793","GO:0030117","GO:0048475","GO:0098588","GO:0000323","GO:0005764","GO:0030687","GO:0120025","GO:0098798","GO:0042995","GO:0030027","GO:0034663","GO:0005773","GO:0030426","GO:0030667","GO:0009986","GO:0042641","GO:0030016","GO:0032432","GO:0030427","GO:0071010","GO:0071004","GO:0140534","GO:0030175","GO:0001726","GO:0030120","GO:0043292","GO:0031965","GO:0043256","GO:0000775","GO:0043005","GO:0005643","GO:0001725","GO:0097517","GO:0000932","GO:0019866","GO:0099572","GO:0098796","GO:0150034","GO:0014069","GO:0005689","GO:0042383","GO:0032279","GO:0005743","GO:0005794","GO:0000779","GO:0005819","GO:0005884","GO:0098978","GO:0016323","GO:0035578","GO:0098552","GO:0002199","GO:0005798","GO:0070993","GO:0042581","GO:0030139","GO:0098984","GO:0044297","GO:0030662","GO:0099738","GO:0030135","GO:0030425","GO:0009925","GO:0005813","GO:0097447","GO:0005849","GO:0071012","GO:0071006","GO:0098858","GO:0120111","GO:0035145","GO:1902737","GO:0045178","GO:0015030","GO:0030017","GO:0090533","GO:0005697","GO:0005740","GO:0030660","GO:0099568","GO:0016281","GO:0098857","GO:0005905","GO:0033290","GO:0000776","GO:0098533","GO:0098797","GO:0072686","GO:0005911","GO:0031966","GO:0005732","GO:0070820","GO:0045335","GO:0005852","GO:0030123","GO:0033018","GO:0000228","GO:0030315","GO:0099524","GO:1904724","GO:0140513","GO:0002102","GO:0030118","GO:0045121","GO:0016282","GO:0070937","GO:0005847","GO:0016234","GO:0098576","GO:0030119","GO:0031429","GO:0098590","GO:0030132","GO:0072669","GO:0030125","GO:0005815","GO:0030126","GO:0031091","GO:0005875","GO:0005765","GO:0098852","GO:0098553","GO:0071556","GO:0000792","GO:0030688","GO:0071944","GO:0036477","GO:0042382","GO:0005793","GO:0034709","GO:0005890","GO:0005903","GO:0044309","GO:0005853","GO:0099522","GO:0030867","GO:0005790","GO:0005721","GO:1990812","GO:0042405","GO:0051233","GO:0035577","GO:0031674","GO:0045177","GO:0072588","GO:0030018","GO:0005610","GO:0043259","GO:0043260","GO:0019897","GO:0032587","GO:0016328","GO:0031143","GO:0120259","GO:0044326","GO:0030663","GO:0005774","GO:0031970","GO:0031904","GO:1990124","GO:0097513","GO:0032839","GO:0005641","GO:0043202","GO:0005850","GO:0030133","GO:0031093","GO:0001650","GO:0098562","GO:0008180","GO:0030670","GO:0043197","GO:0036019","GO:0030137","GO:0005869","GO:0034388","GO:0001772","GO:0008305","GO:0099631","GO:0043025","GO:0030286","GO:0005912","GO:0035580","GO:0001652","GO:0034708","GO:0044291","GO:0071541","GO:0072589","GO:0000314","GO:0005763","GO:0030136","GO:0071682","GO:0031428","GO:0071920","GO:0042824","GO:0019898","GO:0045120","GO:0140535","GO:0098636","GO:0005839","GO:0005638","GO:0098685"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[20.2391849219751,20.2514430713508,20.2519970347108,20.2519970347108,21.5311068212147,21.5312736804037,21.5312736804037,20.9621971604282,21.0233185405979,22.1966233770388,22.4522820101636,20.7779367209327,21.3094162076497,22.524402481486,22.4730291225783,21.3235749743744,22.3798221927753,21.0611450344719,21.635595572086,21.4959734519815,21.4959734519815,22.5896740478699,18.2051711326577,18.2306007999516,17.1301489018925,17.0845191431789,16.1801260995578,19.6842209919621,21.991572505069,19.1101764583058,18.1857487980281,15.8594084911348,17.5430709244817,19.14152340129,19.1547050272983,15.7409065320485,20.3794510162394,17.4412620934695,16.4531289535021,19.3622629970042,16.2535934593326,16.2535934593326,21.1540185325086,16.3369045304455,15.7409065320485,17.0452487313639,17.8550185262182,20.2199684559896,17.8335679993185,17.8464821016361,16.0034441398068,20.4134387528307,20.4143944784611,18.4904448583444,15.1365420018419,15.6777581925644,20.1408020607539,21.9662816912276,18.3924827650377,14.8639497336129,16.5254745288737,15.5659342061357,14.9720468128971,15.04002854028,20.3580372522075,19.9632191707246,19.7675550546094,18.4411253957884,14.5314039820807,18.9129942601168,16.2535934593326,19.3263815809387,19.3162843873573,17.7940127181872,19.6032120216723,19.6032120216723,17.9919055031525,13.6939879797272,15.4447486517457,13.913976779011,13.8439054273732,13.43983269111,20.1799114508644,18.3645967823511,18.2083786735995,15.5890025441251,19.6609874234229,14.1672052233937,11.6938338220078,14.0455945528341,17.8292337214709,15.8782345485842,15.8782345485842,19.6721988113624,16.1649553304206,18.9790726054027,15.1673729569525,15.1673729569525,19.547551668787,18.2083786735995,18.706500422945,16.7815502640699,16.7815502640699,16.9199186967909,19.5276763094948,18.7482951423705,20.0469161156022,14.7873455809501,14.7873455809501,15.8594084911348,15.7613525063434,16.2678316420048,18.5881476245701,18.5904918372769,20.8769988379845,12.130087403251,16.0702978764781,16.9714805839255,17.1597118754719,18.2431214808468,17.4587994754946,16.0538802978005,16.0538802978005,20.2470013142082,18.9307441955874,18.9307441955874,13.6135405510155,20.2911675732325,17.6323264470017,20.3469174161514,17.1886232475471,11.9248043149337,19.0918801305911,16.8662784262297,17.7850517897527,19.2101670450722,15.7409065320485,17.3810213538362,15.7409065320485,16.9199186967909,12.7755203026965,12.7755203026965,16.4897804845379,16.1801260995578,17.0211133572295,15.1673729569525,17.4412620934695,17.7577815556852,12.130087403251,17.1814549843141,19.6997527479627,15.8594084911348,15.5659342061357,15.5659342061357,16.0034441398068,18.5928315050878,17.9413158350498,19.6777633174944,17.6118789785835,17.8421920921248,13.913976779011,16.5829624692697,17.8677155875114,18.4384733835673,19.9132799491073,16.7117402249923,18.1290009629389,16.3233634273442,17.8844480508839,17.3307247916066,15.9687874995265,18.7317518979519,11.124724876289,15.9332585063981,13.0267853035918,16.8198706814625,17.9050550937873,17.9649194289544,18.5739859419917,16.9965294034581,14.7066840512049,17.7157424774458,18.7043719177881,17.4818173804193,18.7275801117362,18.7086251559259,13.913976779011,12.130087403251,12.130087403251,17.2582244377334,15.9151535780747,13.245899907445,8.9733663912244,17.5805805564662,16.149617672749,17.2445977432898,13.1398677547467,13.5289620440801,19.043251088076,15.1050590561126,17.5699751684816,12.3145705419034,17.8802857667904,15.5890025441251,12.6348180515352,16.6052938368976,13.6939879797272,18.8636983678417,16.8571248761385,18.4251259711228,18.9586018440716,13.913976779011,16.8662784262297,16.638116308046,12.7755203026965,11.9248043149337,11.6938338220078,17.3993902076886,15.04002854028,12.130087403251,15.1365420018419,20.5155916904179,14.0455945528341,14.9008482243626,17.8761096542001,12.9057140413384,10.3183262175968,13.6135405510155,15.6560764403598,14.1672052233937,14.6647003677091,9.75115060553602,19.5952825837126,12.9057140413384,11.124724876289,13.9811327271558,19.0810740081694,12.3145705419034,15.9511346599151,16.8571248761385,18.139206270338,18.139206270338,13.6939879797272,13.6939879797272,15.5890025441251,12.6348180515352,21.4642433652279,19.104109343005,12.130087403251,16.5487618726662,12.130087403251,12.130087403251,16.1951334195648,16.9798828833657,8.9733663912244,12.6348180515352,13.913976779011,14.1672052233937,13.6135405510155,7.75192306836651,12.130087403251,14.3855512064191,15.1975771506459,16.6595641841931,18.2337429255293,10.3183262175968,16.5136799415609,7.75192306836651,7.75192306836651,7.75192306836651,16.9545146694682,16.0702978764781,15.5424924404583,13.0267853035918,7.75192306836651,8.9733663912244,13.0267853035918,18.3097950422393,16.0372684684923,14.3337390874724,12.130087403251,7.75192306836651,13.9811327271558,11.6938338220078,16.086525501471,9.75115060553602,18.169326856153,15.4447486517457,16.638116308046,17.1226530103951,14.2801580888833,15.6991178459387,16.9545146694682,13.8439054273732,13.43983269111,12.7755203026965,9.75115060553602,14.7475473009664,14.1075597373662,3.77952755905512,18.4061904586819,15.1050590561126,16.945949724284,15.3125836213341,10.3183262175968,16.1341095793971,15.6117086396742,10.7620154654366,8.9733663912244,13.913976779011,13.913976779011,17.1450135964435,13.5289620440801,10.7620154654366,3.77952755905512,11.4303977042964,17.789540196972,12.4818892601862,19.005826266795,14.3855512064191,13.5289620440801,9.75115060553602,15.7409065320485],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[3.62744733685985,3.48120206926184,46.8972971525874,40.5412528300588,3.64619673014164,9.02727260201694,28.8491311795311,35.9963998985515,28.6803866399949,8.96352466485884,16.4145335550444,19.2794408485027,16.902017780371,8.33354505059051,3.63119721551621,28.8041326356548,46.6573049185805,9.57100500718901,27.281681901173,15.2708205648548,16.1632916850683,16.1595418064119,3.649946608798,9.45475876884188,36.116396015555,40.1775146003919,9.32726289452567,2.12749587431621,46.8860475166184,3.64244685148528,5.75362853501545,3.67244588073616,25.9167260702583,28.4291447700189,40.6649988257186,16.8157705712748,27.0304400311969,3.6686960020798,33.3452356885057,3.63494709417257,22.23809510837,22.2305953510573,25.7404817734094,19.2981902417845,3.73994369655062,27.3529295956438,20.2731586924379,22.2530946229955,39.1762969991441,19.1969435180628,46.0798236055011,36.1201458942114,20.2131606339361,9.48100791943639,9.06102150992417,9.12101956842592,46.6460552826114,3.55619964238902,36.0413984424279,7.29107878412268,20.8506400055172,9.57475488584537,2.75747548858454,2.75372560992818,37.8450900761366,28.4703934352388,21.0756327248987,16.0620449613466,33.4389826549146,22.8455754507002,28.4966425858334,9.45100889018552,47.5835249467011,27.2854317798294,26.2017168481416,36.6301293914762,3.58994855029626,15.4883135269237,38.7675602256009,3.04996602378055,46.162320935941,27.2929315371421,9.74349942538153,9.91224396491769,16.4745316135461,3.14371299018952,8.2697971134324,15.3045694727621,22.4668377064079,27.3491797169875,3.49995146254364,9.46600840481095,20.9406370932698,4.02493447443391,19.178194124781,25.7292321374403,36.5738812116308,38.4000721172777,46.7060533411131,25.4517411168698,41.1149842644817,9.06477138858053,9.33476265183839,16.2907875593845,7.28357902680996,2.51748325457755,18.6307118409526,22.2605943803082,46.1210722707211,40.9274903316638,15.4770638909546,3.90493835743042,19.658178592795,9.01977284470422,40.8074942146603,20.2356599058743,3.69119527401795,40.6424995537805,7.87980973317106,46.6910538264877,8.34479468655959,27.3604293529566,3.67619575939251,40.9312402103201,4.56116712229326,8.34104480790323,14.5583436201466,38.7150619244119,15.4695641336419,4.56491700094962,16.3132868313227,12.7471522291252,46.1848202078792,9.04602199529874,26.7604487679391,15.6533081878035,24.3417770345875,27.0341899098533,26.7266998600319,9.22601617080397,19.1819440034374,19.6469289568259,19.2869406058154,39.3450415386802,21.9043559079541,19.2831907271591,5.55113508757206,40.9537394822583,46.976044604371,46.1885700865356,24.034286984766,39.5962834086563,25.8604778904129,20.4194039600359,24.3342772772748,27.0604390604478,46.0760737268448,40.533753072746,9.11726968976956,33.390234232382,16.22328974357,3.59369842895261,40.5150036794643,48.0372602641206,38.8238084054463,3.48870182657456,41.1337336577635,36.1351454088368,19.1706943674683,2.72747645933366,38.0738326741745,19.9281698560528,19.8644219188947,37.8563397121057,12.7434023504688,5.43863872788129,27.4204274114583,16.2270396222264,28.159153506761,9.3422624091511,28.4478941633007,2.73872609530274,16.2007904716319,3.34995631628927,46.3085662035391,14.6483407078992,41.0399866913545,8.00355572883091,26.9779417300079,45.9973262750612,36.0001497772079,36.0451483210842,3.53370037045087,3.69494515267431,16.3320362246045,38.058833159549,35.9776505052698,29.4678611578304],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.8655157486559,14.9294122874875,14.5413512959958,14.5059456109321,14.2456383715685,14.0619241203351,14.045568381369,13.8204452882288,13.5522111021032,12.6225616595401,12.3774235959091,11.6402939611241,11.5227028865135,11.0133398073822,10.4033021760914,10.1205990397451,10.0934663115418,9.74554221451061,9.6876364970402,9.62301118003798,9.54971093837945,9.53906943782994,9.38119130298052,8.5001658307974,8.38281689344831,8.1698232785866,7.53500437130342,7.51779212440684,7.3228408157474,7.2760581078681,7.2760581078681,6.79660893289478,6.49250830852958,6.42020664619402,6.12801458965389,6.03857991869598,5.98221073243015,5.9345279567423,5.91897803168411,5.75750173386681,5.74219959677153,5.66554898536826,5.6448569701131,5.4092204770659,5.387452874934,5.38560441118656,5.27566970885971,5.19897619951966,4.96905413368404,4.95146919289314,4.924124894777,4.90747250231672,4.76428412587226,4.62542274164014,4.62109538674786,4.46020745375707,4.44299209983263,4.28203550324861,4.2654147266065,4.23922698806409,4.22522393981074,4.19196699846778,4.17370336739082,4.11493502161611,3.90298609494654,3.85604515061073,3.75572673110908,3.74633822678524,3.74414539011712,3.60651351777365,3.53572418362936,3.47595320693672,3.45307613538924,3.40545157491318,3.40545157491318,3.40115385295637,3.34849068025037,3.32180818929672,3.30465414436464,3.26295511765973,3.23979418419074,3.21851933248544,3.00817538417321,2.97968758663696,2.94538759696857,2.94076652114794,2.92479320523195,2.85185086488849,2.8290426802682,2.82047724382606,2.78321720675233,2.7689950360203,2.76741420611764,2.74399805470104,2.71686295302831,2.70433625402148,2.68522342542948,2.68244399703284,2.66190161506701,2.60036911919307,2.58664965937257,2.5460607178854,2.54424802888814,2.53987274019725,2.53534571605717,2.49947547357444,2.493261224234,2.44684898121607,2.38245493187236,2.36608466785228,2.33001517273964,2.30195520516839,2.24532059521765,2.22659795248154,2.21394043451386,2.14011360451984,2.09595429053726,2.08498612992518,2.05654571236862,2.04134575658202,2.02358156657103,2.02141818960132,2.00256798771283,2.00167024131219,1.99440375645423,1.96892639405088,1.94883666439862,1.94364875271812,1.90518434366663,1.8999348909523,1.89915741198404,1.88127209661315,1.8346283872323,1.79614378293246,1.77480800805714,1.75221574980451,1.71184908250273,1.67305116833536,1.63175783943902,1.6258724443867,1.61512939500961,1.61341749357512,1.60034649172508,1.58248267987663,1.57765420136871,1.57545139099626,1.57325766321193,1.56062330336284,1.56021549134501,1.53208629643509,1.50473775375412,1.49966865045928,1.49966865045928,1.47729318873223,1.46038742637444,1.45262597391298,1.45034085041266,1.42339224028569,1.40887627318864,1.33988154896485,1.32875113298272,1.32875113298272,1.32871374668925,1.31621917985531,1.31329639076356],"text":["GO:0003723 (3142) <br> RNA binding <br> 8.306e-197","GO:0003676 (5447) <br> nucleic acid binding <br> 1.977e-102","GO:1901363 (7416) <br> heterocyclic compound binding <br> 6.199e-93","GO:0097159 (7497) <br> organic cyclic compound binding <br> 3.181e-92","GO:0003729 (350) <br> mRNA binding <br> 1.048e-69","GO:0005515 (14799) <br> protein binding <br> 3.720e-68","GO:0045296 (334) <br> cadherin binding <br> 3.634e-64","GO:0050839 (550) <br> cell adhesion molecule binding <br> 2.105e-57","GO:0044877 (1275) <br> protein-containing complex binding <br> 8.274e-36","GO:0005488 (18292) <br> binding <br> 4.720e-28","GO:0016887 (316) <br> ATP hydrolysis activity <br> 4.372e-27","GO:0019899 (2030) <br> enzyme binding <br> 3.236e-24","GO:0017111 (656) <br> nucleoside-triphosphatase activity <br> 4.617e-24","GO:0005198 (980) <br> structural molecule activity <br> 3.375e-23","GO:0003724 (76) <br> RNA helicase activity <br> 1.819e-22","GO:0045182 (151) <br> translation regulator activity <br> 2.455e-22","GO:0140657 (550) <br> ATP-dependent activity <br> 2.838e-22","GO:0008186 (78) <br> ATP-dependent activity, acting on RNA <br> 3.712e-22","GO:0043021 (157) <br> ribonucleoprotein complex binding <br> 4.619e-22","GO:0016462 (712) <br> pyrophosphatase activity <br> 5.050e-22","GO:0016818 (715) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 7.295e-22","GO:0016817 (715) <br> hydrolase activity, acting on acid anhydrides <br> 7.295e-22","GO:0003730 (97) <br> mRNA 3'-UTR binding <br> 2.538e-21","GO:0008135 (98) <br> translation factor activity, RNA binding <br> 1.427e-20","GO:0051082 (112) <br> unfolded protein binding <br> 1.154e-19","GO:0090079 (121) <br> translation regulator activity, nucleic acid binding <br> 2.210e-19","GO:0008092 (995) <br> cytoskeletal protein binding <br> 1.167e-18","GO:0000166 (2174) <br> nucleotide binding <br> 3.103e-18","GO:1901265 (2175) <br> nucleoside phosphate binding <br> 3.225e-18","GO:0003727 (84) <br> single-stranded RNA binding <br> 6.229e-18","GO:0004386 (160) <br> helicase activity <br> 1.364e-17","GO:0003743 (58) <br> translation initiation factor activity <br> 1.409e-17","GO:0036094 (2516) <br> small molecule binding <br> 1.684e-17","GO:0044183 (41) <br> protein folding chaperone <br> 1.363e-16","GO:0097367 (2288) <br> carbohydrate derivative binding <br> 1.176e-15","GO:0017076 (1930) <br> purine nucleotide binding <br> 2.875e-15","GO:0042802 (2091) <br> identical protein binding <br> 3.119e-15","GO:0003735 (449) <br> structural constituent of ribosome <br> 5.680e-15","GO:0048027 (26) <br> mRNA 5'-UTR binding <br> 8.671e-15","GO:0003725 (74) <br> double-stranded RNA binding <br> 9.004e-15","GO:0032555 (1917) <br> purine ribonucleotide binding <br> 1.512e-14","GO:0032553 (1934) <br> ribonucleotide binding <br> 2.804e-14","GO:0035639 (1848) <br> purine ribonucleoside triphosphate binding <br> 2.385e-13","GO:0019904 (690) <br> protein domain specific binding <br> 4.193e-13","GO:0003779 (449) <br> actin binding <br> 2.289e-12","GO:0043168 (2410) <br> anion binding <br> 3.001e-12","GO:0030554 (1576) <br> adenyl nucleotide binding <br> 9.698e-12","GO:0032559 (1565) <br> adenyl ribonucleotide binding <br> 3.951e-11","GO:0070717 (30) <br> poly-purine tract binding <br> 7.575e-11","GO:0019843 (66) <br> rRNA binding <br> 8.064e-11","GO:0140098 (400) <br> catalytic activity, acting on RNA <br> 1.797e-10","GO:0051087 (106) <br> chaperone binding <br> 2.053e-10","GO:0030515 (32) <br> snoRNA binding <br> 2.382e-10","GO:0008143 (24) <br> poly(A) binding <br> 2.820e-10","GO:0005524 (1502) <br> ATP binding <br> 2.890e-10","GO:0005544 (53) <br> calcium-dependent phospholipid binding <br> 4.157e-10","GO:0140640 (607) <br> catalytic activity, acting on a nucleic acid <br> 3.161e-09","GO:0003697 (123) <br> single-stranded DNA binding <br> 4.142e-09","GO:0051015 (218) <br> actin filament binding <br> 6.764e-09","GO:0004859 (12) <br> phospholipase inhibitor activity <br> 2.917e-08","GO:0031072 (128) <br> heat shock protein binding <br> 3.035e-08","GO:0008187 (28) <br> poly-pyrimidine tract binding <br> 4.755e-08","GO:0001099 (57) <br> basal RNA polymerase II transcription machinery binding <br> 5.296e-08","GO:0001098 (57) <br> basal transcription machinery binding <br> 5.296e-08","GO:0055102 (16) <br> lipase inhibitor activity <br> 1.597e-07","GO:0044389 (314) <br> ubiquitin-like protein ligase binding <br> 3.217e-07","GO:0031625 (296) <br> ubiquitin protein ligase binding <br> 3.800e-07","GO:0016787 (2425) <br> hydrolase activity <br> 7.447e-07","GO:0048306 (84) <br> calcium-dependent protein binding <br> 9.150e-07","GO:0033592 (5) <br> RNA strand annealing activity <br> 1.042e-06","GO:0044548 (14) <br> S100 protein binding <br> 1.163e-06","GO:0008134 (602) <br> transcription factor binding <br> 1.205e-06","GO:1990247 (7) <br> N6-methyladenosine-containing RNA binding <br> 1.748e-06","GO:0043022 (76) <br> ribosome binding <br> 1.811e-06","GO:0036402 (6) <br> proteasome-activating activity <br> 2.160e-06","GO:0051920 (9) <br> peroxiredoxin activity <br> 2.265e-06","GO:0003712 (512) <br> transcription coregulator activity <br> 3.897e-06","GO:0016620 (38) <br> oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor <br> 4.098e-06","GO:0070063 (58) <br> RNA polymerase binding <br> 4.115e-06","GO:0001618 (76) <br> virus receptor activity <br> 5.301e-06","GO:0140272 (77) <br> exogenous protein binding <br> 6.324e-06","GO:0043024 (17) <br> ribosomal small subunit binding <br> 1.074e-05","GO:0008266 (25) <br> poly(U) RNA binding <br> 1.118e-05","GO:0008379 (7) <br> thioredoxin peroxidase activity <br> 1.191e-05","GO:0016903 (47) <br> oxidoreductase activity, acting on the aldehyde or oxo group of donors <br> 1.237e-05","GO:0001671 (25) <br> ATPase activator activity <br> 1.721e-05","GO:0005178 (147) <br> integrin binding <br> 2.369e-05","GO:0016491 (756) <br> oxidoreductase activity <br> 2.393e-05","GO:0033218 (389) <br> amide binding <br> 3.466e-05","GO:0043167 (6060) <br> ion binding <br> 3.606e-05","GO:0003682 (580) <br> chromatin binding <br> 5.224e-05","GO:0008139 (26) <br> nuclear localization sequence binding <br> 5.427e-05","GO:0031369 (34) <br> translation initiation factor binding <br> 5.765e-05","GO:0003857 (7) <br> 3-hydroxyacyl-CoA dehydrogenase activity <br> 5.954e-05","GO:0019834 (5) <br> phospholipase A2 inhibitor activity <br> 6.427e-05","GO:0035613 (15) <br> RNA stem-loop binding <br> 6.703e-05","GO:0051787 (28) <br> misfolded protein binding <br> 7.675e-05","GO:0061980 (44) <br> regulatory RNA binding <br> 1.250e-04","GO:0140677 (33) <br> molecular function activator activity <br> 1.393e-04","GO:0035198 (33) <br> miRNA binding <br> 1.755e-04","GO:0099122 (8) <br> RNA polymerase II C-terminal domain binding <br> 1.793e-04","GO:0005525 (382) <br> GTP binding <br> 1.802e-04","GO:0008094 (108) <br> ATP-dependent activity, acting on DNA <br> 2.474e-04","GO:0016853 (170) <br> isomerase activity <br> 2.913e-04","GO:0004857 (389) <br> enzyme inhibitor activity <br> 3.342e-04","GO:0000993 (36) <br> RNA polymerase II complex binding <br> 3.523e-04","GO:0019001 (404) <br> guanyl nucleotide binding <br> 3.931e-04","GO:0032561 (404) <br> guanyl ribonucleotide binding <br> 3.931e-04","GO:0140142 (30) <br> nucleocytoplasmic carrier activity <br> 3.971e-04","GO:0098631 (61) <br> cell adhesion mediator activity <br> 4.482e-04","GO:0016616 (128) <br> oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor <br> 4.766e-04","GO:0003824 (5716) <br> catalytic activity <br> 4.958e-04","GO:0023026 (25) <br> MHC class II protein complex binding <br> 5.458e-04","GO:0005509 (724) <br> calcium ion binding <br> 5.757e-04","GO:0097718 (33) <br> disordered domain specific binding <br> 6.046e-04","GO:0030544 (43) <br> Hsp70 protein binding <br> 9.814e-04","GO:0003755 (45) <br> peptidyl-prolyl cis-trans isomerase activity <br> 1.048e-03","GO:0097322 (9) <br> 7SK snRNA binding <br> 1.134e-03","GO:0005048 (49) <br> signal sequence binding <br> 1.146e-03","GO:0140666 (13) <br> annealing activity <br> 1.189e-03","GO:0005201 (173) <br> extracellular matrix structural constituent <br> 1.407e-03","GO:0043175 (41) <br> RNA polymerase core enzyme binding <br> 1.482e-03","GO:0003746 (26) <br> translation elongation factor activity <br> 1.512e-03","GO:0098632 (50) <br> cell-cell adhesion mediator activity <br> 1.647e-03","GO:0004029 (16) <br> aldehyde dehydrogenase (NAD+) activity <br> 1.702e-03","GO:0005200 (104) <br> structural constituent of cytoskeleton <br> 1.708e-03","GO:0015631 (378) <br> tubulin binding <br> 1.803e-03","GO:0070034 (22) <br> telomerase RNA binding <br> 1.919e-03","GO:0016614 (141) <br> oxidoreductase activity, acting on CH-OH group of donors <br> 1.975e-03","GO:0004030 (17) <br> aldehyde dehydrogenase [NAD(P)+] activity <br> 2.064e-03","GO:0016859 (48) <br> cis-trans isomerase activity <br> 2.078e-03","GO:0015036 (43) <br> disulfide oxidoreductase activity <br> 2.178e-03","GO:0140296 (52) <br> general transcription initiation factor binding <br> 2.510e-03","GO:0005520 (29) <br> insulin-like growth factor binding <br> 2.590e-03","GO:0042277 (313) <br> peptide binding <br> 2.844e-03","GO:0016667 (59) <br> oxidoreductase activity, acting on a sulfur group of donors <br> 2.856e-03","GO:0034513 (5) <br> box H/ACA snoRNA binding <br> 2.885e-03","GO:0042803 (690) <br> protein homodimerization activity <br> 2.915e-03","GO:0042162 (36) <br> telomeric DNA binding <br> 3.166e-03","GO:0008022 (190) <br> protein C-terminus binding <br> 3.212e-03","GO:0019838 (137) <br> growth factor binding <br> 3.574e-03","GO:0023023 (33) <br> MHC protein complex binding <br> 4.145e-03","GO:0019901 (677) <br> protein kinase binding <br> 4.304e-03","GO:0070990 (7) <br> snRNP binding <br> 4.677e-03","GO:0031994 (13) <br> insulin-like growth factor I binding <br> 4.989e-03","GO:0019900 (758) <br> kinase binding <br> 5.684e-03","GO:0004332 (3) <br> fructose-bisphosphate aldolase activity <br> 5.935e-03","GO:0098641 (18) <br> cadherin binding involved in cell-cell adhesion <br> 6.110e-03","GO:1901681 (265) <br> sulfur compound binding <br> 7.242e-03","GO:0140297 (454) <br> DNA-binding transcription factor binding <br> 8.018e-03","GO:0034046 (8) <br> poly(G) binding <br> 8.223e-03","GO:0071987 (5) <br> WD40-repeat domain binding <br> 8.779e-03","GO:0035925 (26) <br> mRNA 3'-UTR AU-rich region binding <br> 9.092e-03","GO:0030619 (8) <br> U1 snRNA binding <br> 9.471e-03","GO:0034511 (11) <br> U3 snoRNA binding <br> 9.519e-03","GO:0042826 (123) <br> histone deacetylase binding <br> 9.941e-03","GO:0140097 (226) <br> catalytic activity, acting on DNA <br> 9.962e-03","GO:0097157 (12) <br> pre-mRNA intronic binding <br> 1.013e-02","GO:0005543 (464) <br> phospholipid binding <br> 1.074e-02","GO:0048156 (43) <br> tau protein binding <br> 1.125e-02","GO:0016835 (80) <br> carbon-oxygen lyase activity <br> 1.139e-02","GO:0003713 (283) <br> transcription coactivator activity <br> 1.244e-02","GO:0097100 (5) <br> supercoiled DNA binding <br> 1.259e-02","GO:1990948 (9) <br> ubiquitin ligase inhibitor activity <br> 1.261e-02","GO:0070181 (9) <br> small ribosomal subunit rRNA binding <br> 1.314e-02","GO:0003678 (73) <br> DNA helicase activity <br> 1.463e-02","GO:0099186 (14) <br> structural constituent of postsynapse <br> 1.599e-02","GO:0051117 (82) <br> ATPase binding <br> 1.680e-02","GO:0019829 (47) <br> ATPase-coupled cation transmembrane transporter activity <br> 1.769e-02","GO:0001091 (21) <br> RNA polymerase II general transcription initiation factor binding <br> 1.942e-02","GO:0061629 (328) <br> RNA polymerase II-specific DNA-binding transcription factor binding <br> 2.123e-02","GO:0030276 (70) <br> clathrin binding <br> 2.335e-02","GO:0030235 (7) <br> nitric-oxide synthase regulator activity <br> 2.367e-02","GO:0055105 (10) <br> ubiquitin-protein transferase inhibitor activity <br> 2.426e-02","GO:0015035 (38) <br> protein-disulfide reductase activity <br> 2.435e-02","GO:0004300 (10) <br> enoyl-CoA hydratase activity <br> 2.510e-02","GO:0043236 (26) <br> laminin binding <br> 2.615e-02","GO:0016836 (64) <br> hydro-lyase activity <br> 2.645e-02","GO:0043878 (6) <br> glyceraldehyde-3-phosphate dehydrogenase (NAD+) (non-phosphorylating) activity <br> 2.658e-02","GO:0008097 (10) <br> 5S rRNA binding <br> 2.671e-02","GO:0044325 (131) <br> transmembrane transporter binding <br> 2.750e-02","GO:0001094 (10) <br> TFIID-class transcription factor complex binding <br> 2.753e-02","GO:0016829 (203) <br> lyase activity <br> 2.937e-02","GO:0002020 (136) <br> protease binding <br> 3.128e-02","GO:0140358 (20) <br> P-type transmembrane transporter activity <br> 3.165e-02","GO:0015662 (20) <br> P-type ion transporter activity <br> 3.165e-02","GO:0098918 (20) <br> structural constituent of synapse <br> 3.332e-02","GO:0005102 (1546) <br> signaling receptor binding <br> 3.464e-02","GO:0042626 (109) <br> ATPase-coupled transmembrane transporter activity <br> 3.527e-02","GO:0140030 (161) <br> modification-dependent protein binding <br> 3.545e-02","GO:0050840 (53) <br> extracellular matrix binding <br> 3.772e-02","GO:0051018 (51) <br> protein kinase A binding <br> 3.901e-02","GO:0003691 (9) <br> double-stranded telomeric DNA binding <br> 4.572e-02","GO:0003756 (20) <br> protein disulfide isomerase activity <br> 4.691e-02","GO:0016864 (20) <br> intramolecular oxidoreductase activity, transposing S-S bonds <br> 4.691e-02","GO:0061608 (19) <br> nuclear import signal receptor activity <br> 4.691e-02","GO:0050815 (9) <br> phosphoserine residue binding <br> 4.828e-02","GO:0046912 (5) <br> acyltransferase, acyl groups converted into alkyl on transfer <br> 4.861e-02"],"key":["GO:0003723","GO:0003676","GO:1901363","GO:0097159","GO:0003729","GO:0005515","GO:0045296","GO:0050839","GO:0044877","GO:0005488","GO:0016887","GO:0019899","GO:0017111","GO:0005198","GO:0003724","GO:0045182","GO:0140657","GO:0008186","GO:0043021","GO:0016462","GO:0016818","GO:0016817","GO:0003730","GO:0008135","GO:0051082","GO:0090079","GO:0008092","GO:0000166","GO:1901265","GO:0003727","GO:0004386","GO:0003743","GO:0036094","GO:0044183","GO:0097367","GO:0017076","GO:0042802","GO:0003735","GO:0048027","GO:0003725","GO:0032555","GO:0032553","GO:0035639","GO:0019904","GO:0003779","GO:0043168","GO:0030554","GO:0032559","GO:0070717","GO:0019843","GO:0140098","GO:0051087","GO:0030515","GO:0008143","GO:0005524","GO:0005544","GO:0140640","GO:0003697","GO:0051015","GO:0004859","GO:0031072","GO:0008187","GO:0001099","GO:0001098","GO:0055102","GO:0044389","GO:0031625","GO:0016787","GO:0048306","GO:0033592","GO:0044548","GO:0008134","GO:1990247","GO:0043022","GO:0036402","GO:0051920","GO:0003712","GO:0016620","GO:0070063","GO:0001618","GO:0140272","GO:0043024","GO:0008266","GO:0008379","GO:0016903","GO:0001671","GO:0005178","GO:0016491","GO:0033218","GO:0043167","GO:0003682","GO:0008139","GO:0031369","GO:0003857","GO:0019834","GO:0035613","GO:0051787","GO:0061980","GO:0140677","GO:0035198","GO:0099122","GO:0005525","GO:0008094","GO:0016853","GO:0004857","GO:0000993","GO:0019001","GO:0032561","GO:0140142","GO:0098631","GO:0016616","GO:0003824","GO:0023026","GO:0005509","GO:0097718","GO:0030544","GO:0003755","GO:0097322","GO:0005048","GO:0140666","GO:0005201","GO:0043175","GO:0003746","GO:0098632","GO:0004029","GO:0005200","GO:0015631","GO:0070034","GO:0016614","GO:0004030","GO:0016859","GO:0015036","GO:0140296","GO:0005520","GO:0042277","GO:0016667","GO:0034513","GO:0042803","GO:0042162","GO:0008022","GO:0019838","GO:0023023","GO:0019901","GO:0070990","GO:0031994","GO:0019900","GO:0004332","GO:0098641","GO:1901681","GO:0140297","GO:0034046","GO:0071987","GO:0035925","GO:0030619","GO:0034511","GO:0042826","GO:0140097","GO:0097157","GO:0005543","GO:0048156","GO:0016835","GO:0003713","GO:0097100","GO:1990948","GO:0070181","GO:0003678","GO:0099186","GO:0051117","GO:0019829","GO:0001091","GO:0061629","GO:0030276","GO:0030235","GO:0055105","GO:0015035","GO:0004300","GO:0043236","GO:0016836","GO:0043878","GO:0008097","GO:0044325","GO:0001094","GO:0016829","GO:0002020","GO:0140358","GO:0015662","GO:0098918","GO:0005102","GO:0042626","GO:0140030","GO:0050840","GO:0051018","GO:0003691","GO:0003756","GO:0016864","GO:0061608","GO:0050815","GO:0046912"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[20.7033686393477,21.3245935449235,21.6635562527598,21.6753721565568,17.957100920704,22.4001547791002,17.892731484589,18.5644517215941,19.6324659256161,22.6202988760265,17.8161407746662,20.1934257134088,18.7946200908942,19.3060984551992,15.6777581925644,16.7520742647617,18.5644517215941,15.7201644110226,16.8103928012089,18.900409192877,18.9058191596849,18.9058191596849,16.0702978764781,16.086525501471,16.2958762527023,16.4154694975993,19.3251242335188,20.2744730170638,20.2750155224465,15.840329725114,16.8386274852372,15.2271784019298,20.4459374425421,14.6215364357113,20.3346567105196,20.1334284618906,20.228487584005,18.2949566663389,13.7706696626195,15.63406319171,20.1253846898753,20.1358917796467,20.0816871076476,18.8599640949483,18.2949566663389,20.3956054416575,19.8905207767085,19.8820588403789,14.0455945528341,15.4447486517457,18.139206270338,16.2099806263843,14.1672052233937,13.6135405510155,19.8323290840716,15.07289736813,18.6936723533036,16.4406903957948,17.2916806776005,12.130087403251,16.5017825041227,13.913976779011,15.1975771506459,15.1975771506459,12.7755203026965,17.807336039845,17.7252056958466,20.4028702506821,15.840329725114,9.75115060553602,12.4818892601862,18.6828765114146,10.7620154654366,15.6777581925644,10.3183262175968,11.4303977042964,18.4699221935228,14.4842896018858,15.2271784019298,15.6777581925644,15.6991178459387,12.9057140413384,13.6939879797272,10.7620154654366,14.8639497336129,13.6939879797272,16.7117402249923,18.9773801929868,18.101368709356,21.4424761251133,18.6341872744013,13.7706696626195,14.2801580888833,10.7620154654366,9.75115060553602,12.6348180515352,13.913976779011,14.7475473009664,14.2246901392547,14.2246901392547,11.124724876289,18.076675201769,16.2392075511492,16.9286524404976,18.101368709356,14.3855512064191,18.152683806952,18.152683806952,14.0455945528341,15.3125836213341,16.5017825041227,21.3779751629097,13.6939879797272,18.9219024744874,14.2246901392547,14.7066840512049,14.7873455809501,11.4303977042964,14.9368675888327,12.3145705419034,16.9545146694682,14.6215364357113,13.7706696626195,14.9720468128971,12.7755203026965,16.1801260995578,18.062341107798,13.43983269111,16.6488827426211,12.9057140413384,14.9008482243626,14.7066840512049,15.04002854028,13.9811327271558,17.802910533295,15.2561992217656,9.75115060553602,18.8599640949483,14.3855512064191,17.0922347975337,16.6052938368976,14.2246901392547,18.8354012127213,10.7620154654366,12.3145705419034,18.9807625955603,7.75192306836651,13.0267853035918,17.5699751684816,18.3097950422393,11.124724876289,9.75115060553602,13.7706696626195,11.124724876289,11.9248043149337,16.4406903957948,17.3434830020115,12.130087403251,18.3389435934989,14.7066840512049,15.7613525063434,17.6623957042418,9.75115060553602,11.4303977042964,11.4303977042964,15.6117086396742,12.4818892601862,15.8013875772071,14.8639497336129,13.3456670761075,17.8677155875114,15.5424924404583,10.7620154654366,11.6938338220078,14.4842896018858,11.6938338220078,13.7706696626195,15.3933042519638,10.3183262175968,11.6938338220078,16.5371679561074,11.6938338220078,17.1886232475471,16.59417406815,13.245899907445,13.245899907445,13.245899907445,19.8672910503509,16.2535934593326,16.8479081762206,15.07289736813,15.0064223927283,11.4303977042964,13.245899907445,13.245899907445,13.1398677547467,11.4303977042964,9.75115060553602],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[195.356202782784,199.834952474961,199.831201428317,196.684073294535,199.936230734332,199.928728641045,194.002074944529,194.545976707817,195.266177663343,194.93983660537,193.480679461102,191.526384159909,197.456788903068,198.383297423979,193.476928414458,199.33231222475,196.413997936213,193.769510052641,199.628644909576,196.008884898729,196.005133852086,196.012635945373,194.549727754461,197.644341235236,197.861901940551,193.484430507745,197.974433339852,197.970682293209,199.336063271393,197.708109028173,190.828689484243,191.143777402286,195.922610825932,193.810771565718,199.924977594401,191.642666605853,195.363704876071,195.971374432296,194.53847461453,191.65016869914,191.646417652497,190.584871452425,194.369677515579,196.927891326353,196.920389233067,194.977347071804,197.501801462788,197.047924818941,193.73575063285,199.8124461951,197.434282623208,191.901488824245,197.513054602718,199.519864556918,199.883716081324,191.132524262356,199.523615603561,199.879965034681,199.688661655869,191.098764842565,199.617391769646,195.896353499428,197.306747037333,197.389270063487,195.993880712156,196.128918391317,192.970537117604,197.764374727824,193.544447254039,199.114751519435,194.335918095789,193.330637595367,198.55209452293,197.107941565235,192.974288164248,195.408717435791,191.001237629838,191.417603807251,195.72005430719,194.332167049145,194.305909722642,197.329253317193,195.146144170755,194.294656582712,192.985541304178,193.968315524739,197.76062368118,192.05153068998,192.981790257534,193.938307151592,192.227829882218,195.551257208239,195.558759301525,195.555008254882,195.573763488099,191.578898812916,195.566261394812,196.185184090967,199.838703521604,195.577514534742,190.547360985991,191.110017982496,191.128773215712,190.539858892704,198.034450086146,190.712407038299,195.547506161595,199.876213988038,199.872462941394,196.170179904394,192.014020223546,190.761170644663,197.378016923557,197.385519016844,191.443861133755,193.214355149423,193.912049825089,190.536107846061,197.30299599069,198.323280677685,195.176152543902,197.374265876914,194.260897162921,192.989292350821,193.814522612361,199.396080017687,198.600858129294,196.654064921388,194.054589597536,191.413852760608,191.631413465923,190.543609939348,195.198658823763,191.63891555921,190.528605752774,197.505552509432,194.733529039985,195.112384750965,190.472340053124,199.403582110974,190.464837959837,191.878982544385,191.920244057462,191.938999290679,194.737280086629,191.912741964175,191.102515889209,191.875231497742,198.642119642371,194.197129369984,193.915800871732,194.302158675998,196.695326434465,191.019992863055,194.726026946699,199.29855280496,196.92414027971,194.617246594041,195.209911963693,191.21504728851,195.416219529078,196.710330621038,195.798826286701,192.010269176903,193.308131315507,196.56779084859,196.687824341178,195.570012441456,193.326886548724,199.294801758316,196.616554454954,197.651843328523,199.504860370345,191.590151952846,191.016241816411,196.90163399985,193.064313283688,191.635164512567,196.89038085992,196.87912771999,192.355365468092,192.351614421449,191.428856947182,191.586400906203,191.158781588859,195.626278141106,191.207545195223,191.200043101936,191.196292055293,197.344257503767,191.440110087112,193.206853056136,193.285625035647,195.585016628029,191.181287868719,198.394550563909,197.333004363837,198.90844395405,191.582649859559,194.298407629355,194.320913909215,198.398301610552,194.662259153761,191.676426025644,194.49346205481,193.773261099284,191.19254100865,196.346479096632,199.857458754821,199.493607230415,194.028332271033,198.274517071321,196.871625626703,194.876068812433,199.467349903911,192.996794444108,193.536945160752,197.72686426139,199.992496433982,191.545139393126,197.951927059992,199.707416889086,198.608360222581,191.211296241866,194.681014386978,190.993735536551,197.35926169034,196.939144466284,193.927054011662,193.593210860403,190.967478210048,190.963727163404],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.1650745462048,15.1378969989593,13.3079164314699,12.4879978214382,12.3383456371081,11.6769905032328,11.1262493565346,10.5036313040842,10.3456417569788,9.67224218342665,9.17451741487294,8.95295691682271,8.77831950353153,8.45053301131757,8.24670383321639,7.99122942341659,7.99122942341659,7.79244501598627,7.59946454193817,7.59946454193817,7.59946454193817,7.59946454193817,7.59946454193817,7.59946454193817,7.51789179878466,7.41199644544951,7.41199644544951,7.41199644544951,7.22977099619139,7.19074223182854,7.06598102633602,7.0525381267918,7.0525381267918,7.0525381267918,6.94034156943071,6.88006553563622,6.86506774517133,6.71213701636976,6.71213701636976,6.71213701636976,6.54855098194533,6.46867393602658,6.38911915631857,6.38911915631857,6.38911915631857,6.31140400478234,6.31140400478234,6.25273751512055,6.23539465365952,6.2336654111772,6.2336654111772,6.2336654111772,6.2336654111772,6.2336654111772,6.19132503092729,6.08202472860927,6.03080552577514,6.03080552577514,6.03080552577514,6.01881820409975,5.99660244071739,5.94556044960997,5.90959696833978,5.90395511549987,5.87059153910258,5.86663110725254,5.64847871371805,5.64847871371805,5.61440676593438,5.51295663786312,5.4919908120389,5.44384333994047,5.37591012927238,5.37591012927238,5.37591012927238,5.28277268053659,5.24419890909358,5.24419890909358,5.22677508850596,5.18973174255212,5.13862869695561,5.11538953485179,5.11538953485179,5.02523985277943,5.02523985277943,5.02181076409783,4.9893792423704,4.97092191352547,4.80467755763924,4.78715860078282,4.75080021842915,4.64820280382582,4.54132012738374,4.51677885923667,4.51145514574163,4.51026834072828,4.39807550239156,4.3551914746232,4.28697978038556,4.28689017006865,4.28689017006865,4.28559360020598,4.16927398092127,4.15721405015437,4.07331448820316,4.07135625617459,4.00762718364562,3.85376194412787,3.69986183923376,3.6606272202497,3.65468000279725,3.56075993383187,3.56075993383187,3.36786561360415,3.31986797527584,3.30140586732596,3.28735010572851,3.27063295521529,3.19295856192338,3.12957678232177,3.11401838201675,3.04968604369989,2.908618004901,2.86468584767054,2.85466505318488,2.85189450591701,2.84605737149135,2.81443680472336,2.80707406484675,2.78325251998132,2.73952984864368,2.72336128916885,2.67507640853102,2.64743705935481,2.60863132125173,2.60002845252565,2.57302494450565,2.56146903511392,2.55138781454315,2.51629856379391,2.46477515322895,2.45173611333698,2.45069072401498,2.35833431246554,2.35833431246554,2.31329118293141,2.31329118293141,2.30390433162406,2.3035186241658,2.26486662006414,2.26075255058267,2.25971666494841,2.25971666494841,2.25971666494841,2.24965493336788,2.24681477263655,2.23062345076755,2.19093963033709,2.12865511046285,2.12622774146312,2.12187199720208,2.11539755344199,2.08760133559814,2.04196570208002,1.98609746078894,1.94724480785202,1.89879680921484,1.88686061073177,1.88142879705884,1.87286520981127,1.85480814317317,1.79579577494502,1.79186771394901,1.78945247159318,1.70676536288174,1.68455006863883,1.65467743629536,1.65452945248901,1.65293546339809,1.64572719102083,1.62671077684097,1.61970831981301,1.59559885320584,1.58804455150335,1.56472157929838,1.56007847631864,1.53067945670717,1.52544424383415,1.52016025508201,1.51690127981106,1.48732329388549,1.45195767819649,1.44100638061421,1.39056589623509,1.35127976093377,1.31053984014653,1.31053984014653],"text":["REAC:R-HSA-8953854 (661) <br> Metabolism of RNA <br> 4.995e-74","REAC:R-HSA-72163 (178) <br> mRNA Splicing - Major Pathway <br> 9.065e-45","REAC:R-HSA-72172 (186) <br> mRNA Splicing <br> 1.520e-42","REAC:R-HSA-72203 (237) <br> Processing of Capped Intron-Containing Pre-mRNA <br> 8.528e-40","REAC:R-HSA-8868773 (192) <br> rRNA processing in the nucleus and cytosol <br> 3.585e-37","REAC:R-HSA-72312 (202) <br> rRNA processing <br> 1.105e-35","REAC:R-HSA-72706 (113) <br> GTP hydrolysis and joining of the 60S ribosomal subunit <br> 1.674e-34","REAC:R-HSA-168255 (152) <br> Influenza Infection <br> 1.927e-34","REAC:R-HSA-6791226 (182) <br> Major pathway of rRNA processing in the nucleolus and cytosol <br> 3.602e-34","REAC:R-HSA-156827 (112) <br> L13a-mediated translational silencing of Ceruloplasmin expression <br> 1.121e-33","REAC:R-HSA-72613 (120) <br> Eukaryotic Translation Initiation <br> 1.672e-33","REAC:R-HSA-72737 (120) <br> Cap-dependent Translation Initiation <br> 1.672e-33","REAC:R-HSA-9010553 (171) <br> Regulation of expression of SLITs and ROBOs <br> 5.408e-33","REAC:R-HSA-376176 (217) <br> Signaling by ROBO receptors <br> 2.202e-30","REAC:R-HSA-156842 (94) <br> Eukaryotic Translation Elongation <br> 3.089e-30","REAC:R-HSA-72766 (292) <br> Translation <br> 5.690e-30","REAC:R-HSA-156902 (90) <br> Peptide chain elongation <br> 4.612e-28","REAC:R-HSA-72689 (102) <br> Formation of a pool of free 40S subunits <br> 9.025e-28","REAC:R-HSA-192823 (90) <br> Viral mRNA Translation <br> 6.406e-27","REAC:R-HSA-975956 (96) <br> Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC) <br> 3.131e-26","REAC:R-HSA-975957 (116) <br> Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC) <br> 3.873e-26","REAC:R-HSA-927802 (116) <br> Nonsense-Mediated Decay (NMD) <br> 3.873e-26","REAC:R-HSA-168273 (133) <br> Influenza Viral RNA Transcription and Replication <br> 4.348e-26","REAC:R-HSA-9633012 (102) <br> Response of EIF2AK4 (GCN2) to amino acid deficiency <br> 1.256e-25","REAC:R-HSA-1799339 (113) <br> SRP-dependent cotranslational protein targeting to membrane <br> 2.054e-25","REAC:R-HSA-72764 (94) <br> Eukaryotic Translation Termination <br> 1.232e-24","REAC:R-HSA-2408557 (94) <br> Selenocysteine synthesis <br> 1.232e-24","REAC:R-HSA-2408522 (117) <br> Selenoamino acid metabolism <br> 5.254e-22","REAC:R-HSA-72649 (59) <br> Translation initiation complex formation <br> 6.765e-21","REAC:R-HSA-72702 (59) <br> Ribosomal scanning and start codon recognition <br> 6.765e-21","REAC:R-HSA-72662 (60) <br> Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S <br> 1.551e-20","REAC:R-HSA-422475 (549) <br> Axon guidance <br> 1.413e-19","REAC:R-HSA-9675108 (574) <br> Nervous system development <br> 2.451e-18","REAC:R-HSA-72695 (52) <br> Formation of the ternary complex, and subsequently, the 43S complex <br> 2.344e-17","REAC:R-HSA-6790901 (59) <br> rRNA modification in the nucleus and cytosol <br> 3.343e-17","REAC:R-HSA-9711097 (157) <br> Cellular response to starvation <br> 6.838e-16","REAC:R-HSA-71291 (370) <br> Metabolism of amino acids and derivatives <br> 7.280e-16","REAC:R-HSA-6798695 (476) <br> Neutrophil degranulation <br> 4.921e-14","REAC:R-HSA-5663205 (912) <br> Infectious disease <br> 3.251e-13","REAC:R-HSA-2262752 (751) <br> Cellular responses to stress <br> 4.588e-13","REAC:R-HSA-8953897 (765) <br> Cellular responses to stimuli <br> 2.104e-12","REAC:R-HSA-450408 (54) <br> AUF1 (hnRNP D0) binds and destabilizes mRNA <br> 7.477e-12","REAC:R-HSA-162909 (125) <br> Host Interactions of HIV factors <br> 3.136e-11","REAC:R-HSA-9013418 (23) <br> RHOBTB2 GTPase cycle <br> 4.512e-11","REAC:R-HSA-9706574 (35) <br> RHOBTB GTPase Cycle <br> 2.127e-10","REAC:R-HSA-3000157 (30) <br> Laminin interactions <br> 6.691e-10","REAC:R-HSA-450531 (86) <br> Regulation of mRNA stability by proteins that bind AU-rich elements <br> 1.114e-09","REAC:R-HSA-73856 (65) <br> RNA Polymerase II Transcription Termination <br> 1.666e-09","REAC:R-HSA-390450 (10) <br> Folding of actin by CCT/TriC <br> 3.544e-09","REAC:R-HSA-72187 (56) <br> mRNA 3'-end processing <br> 5.666e-09","REAC:R-HSA-211733 (50) <br> Regulation of activated PAK-2p34 by proteasome mediated degradation <br> 1.020e-08","REAC:R-HSA-1236978 (50) <br> Cross-presentation of soluble exogenous antigens (endosomes) <br> 1.020e-08","REAC:R-HSA-350562 (51) <br> Regulation of ornithine decarboxylase (ODC) <br> 1.613e-08","REAC:R-HSA-69601 (52) <br> Ubiquitin Mediated Degradation of Phosphorylated Cdc25A <br> 2.515e-08","REAC:R-HSA-69613 (52) <br> p53-Independent G1/S DNA damage checkpoint <br> 2.515e-08","REAC:R-HSA-349425 (52) <br> Autodegradation of the E3 ubiquitin ligase COP1 <br> 2.515e-08","REAC:R-HSA-75815 (52) <br> Ubiquitin-dependent degradation of Cyclin D <br> 2.515e-08","REAC:R-HSA-69610 (52) <br> p53-Independent DNA Damage Response <br> 2.515e-08","REAC:R-HSA-180534 (52) <br> Vpu mediated degradation of CD4 <br> 2.515e-08","REAC:R-HSA-68867 (68) <br> Assembly of the pre-replicative complex <br> 3.035e-08","REAC:R-HSA-180585 (53) <br> Vif-mediated degradation of APOBEC3G <br> 3.873e-08","REAC:R-HSA-9604323 (53) <br> Negative regulation of NOTCH4 signaling <br> 3.873e-08","REAC:R-HSA-169911 (53) <br> Regulation of Apoptosis <br> 3.873e-08","REAC:R-HSA-8941858 (54) <br> Regulation of RUNX3 expression and activity <br> 5.892e-08","REAC:R-HSA-3000171 (58) <br> Non-integrin membrane-ECM interactions <br> 6.446e-08","REAC:R-HSA-68949 (71) <br> Orc1 removal from chromatin <br> 8.591e-08","REAC:R-HSA-4641257 (55) <br> Degradation of AXIN <br> 8.861e-08","REAC:R-HSA-174113 (55) <br> SCF-beta-TrCP mediated degradation of Emi1 <br> 8.861e-08","REAC:R-HSA-8854050 (55) <br> FBXL7 down-regulates AURKA during mitotic entry and in early mitosis <br> 8.861e-08","REAC:R-HSA-8852276 (79) <br> The role of GTSE1 in G2/M progression after G2 checkpoint <br> 1.147e-07","REAC:R-HSA-5362768 (56) <br> Hh mutants are degraded by ERAD <br> 1.318e-07","REAC:R-HSA-1236974 (90) <br> ER-Phagosome pathway <br> 1.364e-07","REAC:R-HSA-69541 (57) <br> Stabilization of p53 <br> 1.940e-07","REAC:R-HSA-8932339 (57) <br> ROS sensing by NFE2L2 <br> 1.940e-07","REAC:R-HSA-4641258 (57) <br> Degradation of DVL <br> 1.940e-07","REAC:R-HSA-351202 (58) <br> Metabolism of polyamines <br> 2.828e-07","REAC:R-HSA-1236975 (106) <br> Antigen processing-Cross presentation <br> 3.399e-07","REAC:R-HSA-68827 (59) <br> CDT1 association with the CDC6:ORC:origin complex <br> 4.082e-07","REAC:R-HSA-5676590 (59) <br> NIK-->noncanonical NF-kB signaling <br> 4.082e-07","REAC:R-HSA-5387390 (59) <br> Hh mutants abrogate ligand secretion <br> 4.082e-07","REAC:R-HSA-5358346 (65) <br> Hedgehog ligand biogenesis <br> 4.882e-07","REAC:R-HSA-9707587 (65) <br> Regulation of HMOX1 expression and activity <br> 4.882e-07","REAC:R-HSA-5687128 (89) <br> MAPK6/MAPK4 signaling <br> 5.588e-07","REAC:R-HSA-3371497 (57) <br> HSP90 chaperone cycle for steroid hormone receptors (SHR) in the presence of ligand <br> 5.816e-07","REAC:R-HSA-5610783 (60) <br> Degradation of GLI2 by the proteasome <br> 5.839e-07","REAC:R-HSA-5610785 (60) <br> GLI3 is processed to GLI3R by the proteasome <br> 5.839e-07","REAC:R-HSA-187577 (60) <br> SCF(Skp2)-mediated degradation of p27/p21 <br> 5.839e-07","REAC:R-HSA-5607761 (60) <br> Dectin-1 mediated noncanonical NF-kB signaling <br> 5.839e-07","REAC:R-HSA-5610780 (60) <br> Degradation of GLI1 by the proteasome <br> 5.839e-07","REAC:R-HSA-69275 (195) <br> G2/M Transition <br> 6.437e-07","REAC:R-HSA-5678895 (61) <br> Defective CFTR causes cystic fibrosis <br> 8.279e-07","REAC:R-HSA-5389840 (87) <br> Mitochondrial translation elongation <br> 9.315e-07","REAC:R-HSA-5419276 (87) <br> Mitochondrial translation termination <br> 9.315e-07","REAC:R-HSA-5368286 (87) <br> Mitochondrial translation initiation <br> 9.315e-07","REAC:R-HSA-453274 (197) <br> Mitotic G2-G2/M phases <br> 9.576e-07","REAC:R-HSA-174184 (73) <br> Cdc20:Phospho-APC/C mediated degradation of Cyclin A <br> 1.008e-06","REAC:R-HSA-68882 (228) <br> Mitotic Anaphase <br> 1.134e-06","REAC:R-HSA-4086400 (92) <br> PCP/CE pathway <br> 1.231e-06","REAC:R-HSA-72165 (51) <br> mRNA Splicing - Minor Pathway <br> 1.248e-06","REAC:R-HSA-2555396 (229) <br> Mitotic Metaphase and Anaphase <br> 1.347e-06","REAC:R-HSA-179419 (74) <br> APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint <br> 1.359e-06","REAC:R-HSA-4608870 (64) <br> Asymmetric localization of PCP proteins <br> 2.247e-06","REAC:R-HSA-174084 (64) <br> Autodegradation of Cdh1 by Cdh1:APC/C <br> 2.247e-06","REAC:R-HSA-176409 (76) <br> APC/C:Cdc20 mediated degradation of mitotic proteins <br> 2.430e-06","REAC:R-HSA-2467813 (188) <br> Separation of Sister Chromatids <br> 3.069e-06","REAC:R-HSA-176814 (77) <br> Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins <br> 3.221e-06","REAC:R-HSA-5368287 (93) <br> Mitochondrial translation <br> 3.599e-06","REAC:R-HSA-69580 (66) <br> p53-Dependent G1/S DNA damage checkpoint <br> 4.208e-06","REAC:R-HSA-69563 (66) <br> p53-Dependent G1 DNA Damage Response <br> 4.208e-06","REAC:R-HSA-1234176 (66) <br> Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha <br> 4.208e-06","REAC:R-HSA-69002 (85) <br> DNA Replication Pre-Initiation <br> 5.215e-06","REAC:R-HSA-1169091 (67) <br> Activation of NF-kappaB in B cells <br> 5.699e-06","REAC:R-HSA-5658442 (67) <br> Regulation of RAS by GAPs <br> 5.699e-06","REAC:R-HSA-8939902 (71) <br> Regulation of RUNX2 expression and activity <br> 5.932e-06","REAC:R-HSA-6807878 (102) <br> COPI-mediated anterograde transport <br> 6.461e-06","REAC:R-HSA-1168372 (80) <br> Downstream signaling events of B Cell Receptor (BCR) <br> 7.267e-06","REAC:R-HSA-69615 (68) <br> G1/S DNA Damage Checkpoints <br> 7.667e-06","REAC:R-HSA-174154 (68) <br> APC/C:Cdc20 mediated degradation of Securin <br> 7.667e-06","REAC:R-HSA-176408 (81) <br> Regulation of APC/C activators between G1/S and early anaphase <br> 9.435e-06","REAC:R-HSA-9013694 (81) <br> Signaling by NOTCH4 <br> 9.435e-06","REAC:R-HSA-8874081 (30) <br> MET activates PTK2 signaling <br> 9.510e-06","REAC:R-HSA-8948751 (69) <br> Regulation of PTEN stability and activity <br> 1.025e-05","REAC:R-HSA-162906 (227) <br> HIV Infection <br> 1.069e-05","REAC:R-HSA-195253 (83) <br> Degradation of beta-catenin by the destruction complex <br> 1.568e-05","REAC:R-HSA-389960 (26) <br> Formation of tubulin folding intermediates by CCT/TriC <br> 1.632e-05","REAC:R-HSA-72202 (80) <br> Transport of Mature Transcript to Cytoplasm <br> 1.775e-05","REAC:R-HSA-69052 (91) <br> Switching of origins to a post-replicative state <br> 2.248e-05","REAC:R-HSA-389957 (28) <br> Prefoldin mediated transfer of substrate  to CCT/TriC <br> 2.875e-05","REAC:R-HSA-8950505 (38) <br> Gene and protein expression by JAK-STAT signaling after Interleukin-12 stimulation <br> 3.042e-05","REAC:R-HSA-69017 (73) <br> CDK-mediated phosphorylation and removal of Cdc6 <br> 3.080e-05","REAC:R-HSA-9711123 (157) <br> Cellular response to chemical stress <br> 3.088e-05","REAC:R-HSA-174178 (74) <br> APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1 <br> 3.999e-05","REAC:R-HSA-8875878 (40) <br> MET promotes cell motility <br> 4.414e-05","REAC:R-HSA-1234174 (75) <br> Cellular response to hypoxia <br> 5.164e-05","REAC:R-HSA-174143 (88) <br> APC/C-mediated degradation of cell cycle proteins <br> 5.165e-05","REAC:R-HSA-453276 (88) <br> Regulation of mitotic cell cycle <br> 5.165e-05","REAC:R-HSA-447115 (57) <br> Interleukin-12 family signaling <br> 5.181e-05","REAC:R-HSA-68886 (407) <br> M Phase <br> 6.772e-05","REAC:R-HSA-382556 (103) <br> ABC-family proteins mediated transport <br> 6.963e-05","REAC:R-HSA-159236 (71) <br> Transport of Mature mRNA derived from an Intron-Containing Transcript <br> 8.447e-05","REAC:R-HSA-5619084 (77) <br> ABC transporter disorders <br> 8.485e-05","REAC:R-HSA-389958 (33) <br> Cooperation of Prefoldin and TriC/CCT  in actin and tubulin folding <br> 9.826e-05","REAC:R-HSA-69202 (82) <br> Cyclin E associated events during G1/S transition  <br> 1.400e-04","REAC:R-HSA-9707564 (123) <br> Cytoprotection by HMOX1 <br> 1.996e-04","REAC:R-HSA-9020591 (47) <br> Interleukin-12 signaling <br> 2.185e-04","REAC:R-HSA-69656 (84) <br> Cyclin A:Cdk2-associated events at S phase entry <br> 2.215e-04","REAC:R-HSA-390471 (38) <br> Association of TriC/CCT with target proteins during biosynthesis <br> 2.749e-04","REAC:R-HSA-6814122 (38) <br> Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding <br> 2.749e-04","REAC:R-HSA-69239 (120) <br> Synthesis of DNA <br> 4.287e-04","REAC:R-HSA-8856688 (133) <br> Golgi-to-ER retrograde transport <br> 4.788e-04","REAC:R-HSA-69206 (130) <br> G1/S Transition <br> 4.996e-04","REAC:R-HSA-5632684 (85) <br> Hedgehog 'on' state <br> 5.160e-04","REAC:R-HSA-77595 (19) <br> Processing of Intronless Pre-mRNAs <br> 5.362e-04","REAC:R-HSA-109581 (176) <br> Apoptosis <br> 6.413e-04","REAC:R-HSA-9020702 (101) <br> Interleukin-1 signaling <br> 7.420e-04","REAC:R-HSA-8878159 (94) <br> Transcriptional regulation by RUNX3 <br> 7.691e-04","REAC:R-HSA-9013422 (23) <br> RHOBTB1 GTPase cycle <br> 8.919e-04","REAC:R-HSA-168249 (1093) <br> Innate Immune System <br> 1.234e-03","REAC:R-HSA-2132295 (123) <br> MHC class II antigen presentation <br> 1.366e-03","REAC:R-HSA-3858494 (143) <br> Beta-catenin independent WNT signaling <br> 1.397e-03","REAC:R-HSA-392499 (2011) <br> Metabolism of proteins <br> 1.406e-03","REAC:R-HSA-5357801 (205) <br> Programmed Cell Death <br> 1.425e-03","REAC:R-HSA-168276 (38) <br> NS1 Mediated Effects on Host Pathways <br> 1.533e-03","REAC:R-HSA-69306 (128) <br> DNA Replication <br> 1.559e-03","REAC:R-HSA-2682334 (90) <br> EPH-Ephrin signaling <br> 1.647e-03","REAC:R-HSA-114608 (125) <br> Platelet degranulation  <br> 1.822e-03","REAC:R-HSA-75067 (28) <br> Processing of Capped Intronless Pre-mRNA <br> 1.891e-03","REAC:R-HSA-453279 (147) <br> Mitotic G1 phase and G1/S transition <br> 2.113e-03","REAC:R-HSA-199977 (155) <br> ER to Golgi Anterograde Transport <br> 2.252e-03","REAC:R-HSA-8878166 (118) <br> Transcriptional regulation by RUNX2 <br> 2.462e-03","REAC:R-HSA-8957275 (107) <br> Post-translational protein phosphorylation <br> 2.512e-03","REAC:R-HSA-76005 (130) <br> Response to elevated platelet cytosolic Ca2+ <br> 2.673e-03","REAC:R-HSA-5689603 (101) <br> UCH proteinases <br> 2.745e-03","REAC:R-HSA-69278 (548) <br> Cell Cycle, Mitotic <br> 2.809e-03","REAC:R-HSA-1169410 (78) <br> Antiviral mechanism by IFN-stimulated genes <br> 3.046e-03","REAC:R-HSA-5627123 (21) <br> RHO GTPases activate PAKs <br> 3.429e-03","REAC:R-HSA-1643685 (1639) <br> Disease <br> 3.534e-03","REAC:R-HSA-3371556 (87) <br> Cellular response to heat stress <br> 3.542e-03","REAC:R-HSA-5625900 (19) <br> RHO GTPases activate CIT <br> 4.382e-03","REAC:R-HSA-5627117 (19) <br> RHO GTPases Activate ROCKs <br> 4.382e-03","REAC:R-HSA-3656237 (14) <br> Defective EXT2 causes exostoses 2 <br> 4.861e-03","REAC:R-HSA-3656253 (14) <br> Defective EXT1 causes exostoses 1, TRPS2 and CHDS <br> 4.861e-03","REAC:R-HSA-5607764 (97) <br> CLEC7A (Dectin-1) signaling <br> 4.967e-03","REAC:R-HSA-69620 (290) <br> Cell Cycle Checkpoints <br> 4.971e-03","REAC:R-HSA-5620922 (23) <br> BBSome-mediated cargo-targeting to cilium <br> 5.434e-03","REAC:R-HSA-397014 (195) <br> Muscle contraction <br> 5.486e-03","REAC:R-HSA-77348 (5) <br> Beta oxidation of octanoyl-CoA to hexanoyl-CoA <br> 5.499e-03","REAC:R-HSA-77310 (5) <br> Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA <br> 5.499e-03","REAC:R-HSA-77350 (5) <br> Beta oxidation of hexanoyl-CoA to butanoyl-CoA <br> 5.499e-03","REAC:R-HSA-381426 (124) <br> Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs) <br> 5.628e-03","REAC:R-HSA-6811436 (53) <br> COPI-independent Golgi-to-ER retrograde traffic <br> 5.665e-03","REAC:R-HSA-202424 (98) <br> Downstream TCR signaling <br> 5.880e-03","REAC:R-HSA-3000178 (75) <br> ECM proteoglycans <br> 6.443e-03","REAC:R-HSA-68877 (199) <br> Mitotic Prometaphase <br> 7.436e-03","REAC:R-HSA-210991 (25) <br> Basigin interactions <br> 7.478e-03","REAC:R-HSA-194315 (697) <br> Signaling by Rho GTPases <br> 7.553e-03","REAC:R-HSA-3371453 (67) <br> Regulation of HSF1-mediated heat shock response <br> 7.667e-03","REAC:R-HSA-5668541 (100) <br> TNFR2 non-canonical NF-kB pathway <br> 8.173e-03","REAC:R-HSA-1640170 (678) <br> Cell Cycle <br> 9.079e-03","REAC:R-HSA-5610787 (113) <br> Hedgehog 'off' state <br> 1.033e-02","REAC:R-HSA-109582 (672) <br> Hemostasis <br> 1.129e-02","REAC:R-HSA-9716542 (712) <br> Signaling by Rho GTPases, Miro GTPases and RHOBTB3 <br> 1.262e-02","REAC:R-HSA-216083 (84) <br> Integrin cell surface interactions <br> 1.298e-02","REAC:R-HSA-75035 (13) <br> Chk1/Chk2(Cds1) mediated inactivation of Cyclin B:Cdk1 complex <br> 1.314e-02","REAC:R-HSA-1169408 (70) <br> ISG15 antiviral mechanism <br> 1.340e-02","REAC:R-HSA-196025 (11) <br> Formation of annular gap junctions <br> 1.397e-02","REAC:R-HSA-77346 (6) <br> Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA <br> 1.600e-02","REAC:R-HSA-6807070 (138) <br> PTEN Regulation <br> 1.615e-02","REAC:R-HSA-77286 (11) <br> mitochondrial fatty acid beta-oxidation of saturated fatty acids <br> 1.624e-02","REAC:R-HSA-446107 (11) <br> Type I hemidesmosome assembly <br> 1.964e-02","REAC:R-HSA-190873 (12) <br> Gap junction degradation <br> 2.068e-02","REAC:R-HSA-6806834 (76) <br> Signaling by MET <br> 2.215e-02","REAC:R-HSA-5663220 (136) <br> RHO GTPases Activate Formins <br> 2.215e-02","REAC:R-HSA-5578775 (54) <br> Ion homeostasis <br> 2.224e-02","REAC:R-HSA-948021 (186) <br> Transport to the Golgi and subsequent modification <br> 2.261e-02","REAC:R-HSA-1474228 (140) <br> Degradation of the extracellular matrix <br> 2.362e-02","REAC:R-HSA-1474244 (298) <br> Extracellular matrix organization <br> 2.400e-02","REAC:R-HSA-69242 (161) <br> S Phase <br> 2.537e-02","REAC:R-HSA-199992 (72) <br> trans-Golgi Network Vesicle Budding <br> 2.582e-02","REAC:R-HSA-5620920 (51) <br> Cargo trafficking to the periciliary membrane <br> 2.724e-02","REAC:R-HSA-3000484 (6) <br> Scavenging by Class F Receptors <br> 2.754e-02","REAC:R-HSA-5140745 (13) <br> WNT5A-dependent internalization of FZD2, FZD5 and ROR2 <br> 2.947e-02","REAC:R-HSA-3000170 (26) <br> Syndecan interactions <br> 2.982e-02","REAC:R-HSA-77305 (3) <br> Beta oxidation of palmitoyl-CoA to myristoyl-CoA <br> 3.019e-02","REAC:R-HSA-177243 (34) <br> Interactions of Rev with host cellular proteins <br> 3.042e-02","REAC:R-HSA-983170 (25) <br> Antigen Presentation: Folding, assembly and peptide loading of class I MHC <br> 3.256e-02","REAC:R-HSA-2565942 (85) <br> Regulation of PLK1 Activity at G2/M Transition <br> 3.532e-02","REAC:R-HSA-9013405 (50) <br> RHOD GTPase cycle <br> 3.622e-02","REAC:R-HSA-69481 (168) <br> G2/M Checkpoints <br> 4.068e-02","REAC:R-HSA-6803529 (25) <br> FGFR2 alternative splicing <br> 4.454e-02","REAC:R-HSA-141424 (92) <br> Amplification of signal from the kinetochores <br> 4.892e-02","REAC:R-HSA-141444 (92) <br> Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal <br> 4.892e-02"],"key":["REAC:R-HSA-8953854","REAC:R-HSA-72163","REAC:R-HSA-72172","REAC:R-HSA-72203","REAC:R-HSA-8868773","REAC:R-HSA-72312","REAC:R-HSA-72706","REAC:R-HSA-168255","REAC:R-HSA-6791226","REAC:R-HSA-156827","REAC:R-HSA-72613","REAC:R-HSA-72737","REAC:R-HSA-9010553","REAC:R-HSA-376176","REAC:R-HSA-156842","REAC:R-HSA-72766","REAC:R-HSA-156902","REAC:R-HSA-72689","REAC:R-HSA-192823","REAC:R-HSA-975956","REAC:R-HSA-975957","REAC:R-HSA-927802","REAC:R-HSA-168273","REAC:R-HSA-9633012","REAC:R-HSA-1799339","REAC:R-HSA-72764","REAC:R-HSA-2408557","REAC:R-HSA-2408522","REAC:R-HSA-72649","REAC:R-HSA-72702","REAC:R-HSA-72662","REAC:R-HSA-422475","REAC:R-HSA-9675108","REAC:R-HSA-72695","REAC:R-HSA-6790901","REAC:R-HSA-9711097","REAC:R-HSA-71291","REAC:R-HSA-6798695","REAC:R-HSA-5663205","REAC:R-HSA-2262752","REAC:R-HSA-8953897","REAC:R-HSA-450408","REAC:R-HSA-162909","REAC:R-HSA-9013418","REAC:R-HSA-9706574","REAC:R-HSA-3000157","REAC:R-HSA-450531","REAC:R-HSA-73856","REAC:R-HSA-390450","REAC:R-HSA-72187","REAC:R-HSA-211733","REAC:R-HSA-1236978","REAC:R-HSA-350562","REAC:R-HSA-69601","REAC:R-HSA-69613","REAC:R-HSA-349425","REAC:R-HSA-75815","REAC:R-HSA-69610","REAC:R-HSA-180534","REAC:R-HSA-68867","REAC:R-HSA-180585","REAC:R-HSA-9604323","REAC:R-HSA-169911","REAC:R-HSA-8941858","REAC:R-HSA-3000171","REAC:R-HSA-68949","REAC:R-HSA-4641257","REAC:R-HSA-174113","REAC:R-HSA-8854050","REAC:R-HSA-8852276","REAC:R-HSA-5362768","REAC:R-HSA-1236974","REAC:R-HSA-69541","REAC:R-HSA-8932339","REAC:R-HSA-4641258","REAC:R-HSA-351202","REAC:R-HSA-1236975","REAC:R-HSA-68827","REAC:R-HSA-5676590","REAC:R-HSA-5387390","REAC:R-HSA-5358346","REAC:R-HSA-9707587","REAC:R-HSA-5687128","REAC:R-HSA-3371497","REAC:R-HSA-5610783","REAC:R-HSA-5610785","REAC:R-HSA-187577","REAC:R-HSA-5607761","REAC:R-HSA-5610780","REAC:R-HSA-69275","REAC:R-HSA-5678895","REAC:R-HSA-5389840","REAC:R-HSA-5419276","REAC:R-HSA-5368286","REAC:R-HSA-453274","REAC:R-HSA-174184","REAC:R-HSA-68882","REAC:R-HSA-4086400","REAC:R-HSA-72165","REAC:R-HSA-2555396","REAC:R-HSA-179419","REAC:R-HSA-4608870","REAC:R-HSA-174084","REAC:R-HSA-176409","REAC:R-HSA-2467813","REAC:R-HSA-176814","REAC:R-HSA-5368287","REAC:R-HSA-69580","REAC:R-HSA-69563","REAC:R-HSA-1234176","REAC:R-HSA-69002","REAC:R-HSA-1169091","REAC:R-HSA-5658442","REAC:R-HSA-8939902","REAC:R-HSA-6807878","REAC:R-HSA-1168372","REAC:R-HSA-69615","REAC:R-HSA-174154","REAC:R-HSA-176408","REAC:R-HSA-9013694","REAC:R-HSA-8874081","REAC:R-HSA-8948751","REAC:R-HSA-162906","REAC:R-HSA-195253","REAC:R-HSA-389960","REAC:R-HSA-72202","REAC:R-HSA-69052","REAC:R-HSA-389957","REAC:R-HSA-8950505","REAC:R-HSA-69017","REAC:R-HSA-9711123","REAC:R-HSA-174178","REAC:R-HSA-8875878","REAC:R-HSA-1234174","REAC:R-HSA-174143","REAC:R-HSA-453276","REAC:R-HSA-447115","REAC:R-HSA-68886","REAC:R-HSA-382556","REAC:R-HSA-159236","REAC:R-HSA-5619084","REAC:R-HSA-389958","REAC:R-HSA-69202","REAC:R-HSA-9707564","REAC:R-HSA-9020591","REAC:R-HSA-69656","REAC:R-HSA-390471","REAC:R-HSA-6814122","REAC:R-HSA-69239","REAC:R-HSA-8856688","REAC:R-HSA-69206","REAC:R-HSA-5632684","REAC:R-HSA-77595","REAC:R-HSA-109581","REAC:R-HSA-9020702","REAC:R-HSA-8878159","REAC:R-HSA-9013422","REAC:R-HSA-168249","REAC:R-HSA-2132295","REAC:R-HSA-3858494","REAC:R-HSA-392499","REAC:R-HSA-5357801","REAC:R-HSA-168276","REAC:R-HSA-69306","REAC:R-HSA-2682334","REAC:R-HSA-114608","REAC:R-HSA-75067","REAC:R-HSA-453279","REAC:R-HSA-199977","REAC:R-HSA-8878166","REAC:R-HSA-8957275","REAC:R-HSA-76005","REAC:R-HSA-5689603","REAC:R-HSA-69278","REAC:R-HSA-1169410","REAC:R-HSA-5627123","REAC:R-HSA-1643685","REAC:R-HSA-3371556","REAC:R-HSA-5625900","REAC:R-HSA-5627117","REAC:R-HSA-3656237","REAC:R-HSA-3656253","REAC:R-HSA-5607764","REAC:R-HSA-69620","REAC:R-HSA-5620922","REAC:R-HSA-397014","REAC:R-HSA-77348","REAC:R-HSA-77310","REAC:R-HSA-77350","REAC:R-HSA-381426","REAC:R-HSA-6811436","REAC:R-HSA-202424","REAC:R-HSA-3000178","REAC:R-HSA-68877","REAC:R-HSA-210991","REAC:R-HSA-194315","REAC:R-HSA-3371453","REAC:R-HSA-5668541","REAC:R-HSA-1640170","REAC:R-HSA-5610787","REAC:R-HSA-109582","REAC:R-HSA-9716542","REAC:R-HSA-216083","REAC:R-HSA-75035","REAC:R-HSA-1169408","REAC:R-HSA-196025","REAC:R-HSA-77346","REAC:R-HSA-6807070","REAC:R-HSA-77286","REAC:R-HSA-446107","REAC:R-HSA-190873","REAC:R-HSA-6806834","REAC:R-HSA-5663220","REAC:R-HSA-5578775","REAC:R-HSA-948021","REAC:R-HSA-1474228","REAC:R-HSA-1474244","REAC:R-HSA-69242","REAC:R-HSA-199992","REAC:R-HSA-5620920","REAC:R-HSA-3000484","REAC:R-HSA-5140745","REAC:R-HSA-3000170","REAC:R-HSA-77305","REAC:R-HSA-177243","REAC:R-HSA-983170","REAC:R-HSA-2565942","REAC:R-HSA-9013405","REAC:R-HSA-69481","REAC:R-HSA-6803529","REAC:R-HSA-141424","REAC:R-HSA-141444"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(51,102,204,1)","opacity":0.8,"size":[18.804457201168,16.9965294034581,17.0610974356821,17.4114927450624,17.1075317971132,17.1814549843141,16.3096881902664,16.7619716287489,17.0292075442823,16.2958762527023,16.4026830738168,16.4026830738168,16.9373292133199,17.285057828302,16.0204579468312,17.7062081669264,15.9511346599151,16.149617672749,15.9511346599151,16.0538802978005,16.3503139956675,16.3503139956675,16.5602578885939,16.149617672749,16.3096881902664,16.0204579468312,16.0204579468312,16.3635942512243,15.2561992217656,15.2561992217656,15.2846609061879,18.5620563660477,18.6205591092323,15.04002854028,15.2561992217656,16.8103928012089,18.0331677498869,18.3730296238723,19.2157092026555,18.9688815606914,18.9925252625689,15.1050590561126,16.4654555525795,13.5289620440801,14.3337390874724,14.0455945528341,15.8782345485842,15.4192527423607,11.6938338220078,15.1673729569525,14.9720468128971,14.9720468128971,15.0064223927283,15.04002854028,15.04002854028,15.04002854028,15.04002854028,15.04002854028,15.04002854028,15.4944412735692,15.07289736813,15.07289736813,15.07289736813,15.1050590561126,15.2271784019298,15.5659342061357,15.1365420018419,15.1365420018419,15.1365420018419,15.7409065320485,15.1673729569525,15.9511346599151,15.1975771506459,15.1975771506459,15.1975771506459,15.2271784019298,16.2099806263843,15.2561992217656,15.2561992217656,15.2561992217656,15.4192527423607,15.4192527423607,15.9332585063981,15.1975771506459,15.2846609061879,15.2846609061879,15.2846609061879,15.2846609061879,15.2846609061879,17.1301489018925,15.3125836213341,15.8968142150176,15.8968142150176,15.8968142150176,17.1450135964435,15.6117086396742,17.3561169726422,15.9862222962431,15.0064223927283,17.3623880909764,15.63406319171,15.3933042519638,15.3933042519638,15.6777581925644,17.0767580441196,15.6991178459387,16.0034441398068,15.4447486517457,15.4447486517457,15.4447486517457,15.8594084911348,15.4698067922852,15.4698067922852,15.5659342061357,16.149617672749,15.7613525063434,15.4944412735692,15.4944412735692,15.7815103024704,15.7815103024704,14.0455945528341,15.5186655459237,17.3498153717115,15.8209916914484,13.7706696626195,15.7613525063434,15.9687874995265,13.913976779011,14.4842896018858,15.6117086396742,16.8103928012089,15.63406319171,14.5771276281867,15.6560764403598,15.9151535780747,15.9151535780747,15.1975771506459,18.1626964899875,16.1649553304206,15.5659342061357,15.6991178459387,14.2246901392547,15.8013875772071,16.4406903957948,14.8639497336129,15.840329725114,14.4842896018858,14.4842896018858,16.4026830738168,16.5602578885939,16.5254745288737,15.8594084911348,13.1398677547467,16.9798828833657,16.1341095793971,16.0204579468312,13.5289620440801,19.4422697323539,16.4406903957948,16.6701619097561,20.1822736493696,17.202843099759,14.4842896018858,16.5017825041227,15.9511346599151,16.4654555525795,13.913976779011,16.7117402249923,16.7912335309883,16.3767476598635,16.2246709565678,16.5254745288737,16.1341095793971,18.5596562541288,15.7201644110226,13.3456670761075,19.9377931767086,15.8968142150176,13.1398677547467,13.1398677547467,12.4818892601862,12.4818892601862,16.0702978764781,17.6966017287282,13.5289620440801,17.1301489018925,9.75115060553602,9.75115060553602,9.75115060553602,16.4531289535021,15.07289736813,16.086525501471,15.6560764403598,17.1597118754719,13.6939879797272,18.8729830284117,15.4698067922852,16.1184273909758,18.837308775272,16.3096881902664,18.8258172788651,18.900409192877,15.840329725114,12.3145705419034,15.5424924404583,11.9248043149337,10.3183262175968,16.6163232063064,11.9248043149337,11.9248043149337,12.130087403251,15.6777581925644,16.59417406815,15.1050590561126,17.0610974356821,16.638116308046,17.7345988358629,16.8479081762206,15.5890025441251,15.0064223927283,10.3183262175968,12.3145705419034,13.7706696626195,7.75192306836651,14.2801580888833,13.6939879797272,15.8594084911348,14.9720468128971,16.911127259905,13.6939879797272,15.9862222962431,15.9862222962431],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(51,102,204,1)"}},"hoveron":"points","set":"SharedData2a0dd1ba","name":"REAC","legendgroup":"REAC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,48.1385069878423],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[51.8880808998977,68.6449267128735],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[72.394500624929,186.704010907851],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[190.453584819907,200.003749573912],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(51,102,204,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","REAC"],"tickvals":[25.0692534939211,60.2665038063856,129.54925576639,195.228667196909],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","REAC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"98074f647bf":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"9804d267d":{"x":{},"xend":{},"y":{},"yend":{}},"98010fa35c8":{"x":{},"xend":{},"y":{},"yend":{}},"98015e31f54":{"x":{},"xend":{},"y":{},"yend":{}},"98037d3915":{"x":{},"xend":{},"y":{},"yend":{}},"98043bd2bca":{"x":{},"y":{}},"98047a330a":{"yintercept":{}}},"cur_data":"98074f647bf","visdat":{"98074f647bf":["function (y) ","x"],"9804d267d":["function (y) ","x"],"98010fa35c8":["function (y) ","x"],"98015e31f54":["function (y) ","x"],"98037d3915":["function (y) ","x"],"98043bd2bca":["function (y) ","x"],"98047a330a":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2a0dd1ba"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level3">
<h3>Table results</h3>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;REAC&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;#3366cc&#39;)))
}</code></pre>
<div id="htmlwidget-e4a421697d58abce2053" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e4a421697d58abce2053">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879","880","881","882","883","884","885","886","887","888","889","890","891","892","893","894","895","896","897","898","899","900","901","902","903","904","905","906","907","908","909","910","911","912","913","914","915","916","917","918","919","920","921","922","923","924","925","926","927","928","929","930","931","932","933","934","935","936","937","938","939","940","941","942","943","944","945","946","947","948","949","950","951","952","953","954","955","956","957","958","959","960","961","962","963","964","965","966","967","968","969","970","971","972","973","974","975","976","977","978","979","980","981","982","983","984","985","986","987","988","989","990","991","992","993","994","995","996","997","998","999","1000","1001","1002","1003","1004","1005","1006","1007","1008","1009","1010","1011","1012","1013","1014","1015","1016","1017","1018","1019","1020","1021","1022","1023","1024","1025","1026","1027","1028","1029","1030","1031","1032","1033","1034","1035","1036","1037","1038","1039","1040","1041","1042","1043","1044","1045","1046","1047","1048","1049","1050","1051","1052","1053","1054","1055","1056","1057","1058","1059","1060","1061","1062","1063","1064","1065","1066","1067","1068","1069","1070","1071","1072","1073","1074","1075","1076","1077","1078","1079","1080","1081","1082","1083","1084","1085","1086","1087","1088","1089","1090","1091","1092","1093","1094","1095","1096","1097","1098","1099","1100","1101","1102","1103","1104","1105","1106","1107","1108","1109","1110","1111","1112","1113","1114","1115","1116","1117","1118","1119","1120","1121","1122","1123","1124","1125","1126","1127","1128","1129","1130","1131","1132","1133","1134","1135","1136","1137","1138","1139","1140","1141","1142","1143","1144","1145","1146","1147","1148","1149","1150","1151","1152","1153","1154","1155","1156","1157","1158","1159","1160","1161","1162","1163","1164","1165","1166","1167","1168","1169","1170","1171","1172","1173","1174","1175","1176","1177","1178","1179","1180","1181","1182","1183","1184","1185","1186","1187","1188","1189","1190","1191","1192","1193","1194","1195","1196","1197","1198","1199","1200","1201","1202","1203","1204","1205","1206","1207","1208","1209","1210","1211","1212","1213","1214","1215","1216","1217","1218","1219","1220","1221","1222","1223","1224","1225","1226","1227","1228","1229","1230","1231","1232","1233","1234","1235","1236","1237","1238","1239","1240","1241","1242","1243","1244","1245","1246","1247","1248","1249","1250","1251","1252","1253","1254","1255","1256","1257","1258","1259","1260","1261","1262","1263","1264","1265","1266","1267","1268","1269","1270","1271","1272","1273","1274","1275","1276","1277","1278","1279","1280","1281","1282","1283","1284","1285","1286","1287","1288","1289","1290","1291","1292","1293","1294","1295","1296","1297","1298","1299","1300","1301","1302","1303","1304","1305","1306","1307","1308","1309","1310","1311","1312","1313","1314","1315","1316","1317","1318","1319","1320","1321","1322","1323","1324","1325","1326","1327","1328","1329","1330","1331","1332","1333","1334","1335","1336","1337","1338","1339","1340","1341","1342","1343","1344","1345","1346","1347","1348","1349","1350","1351","1352","1353","1354","1355","1356","1357","1358","1359","1360","1361","1362","1363","1364","1365","1366","1367","1368","1369","1370","1371","1372","1373","1374","1375","1376","1377","1378","1379","1380","1381","1382","1383","1384","1385","1386","1387","1388","1389","1390","1391","1392","1393","1394","1395","1396","1397","1398","1399","1400","1401","1402","1403","1404","1405","1406","1407","1408","1409","1410","1411","1412","1413","1414","1415","1416","1417","1418","1419","1420","1421","1422","1423","1424","1425","1426","1427","1428","1429","1430","1431","1432","1433","1434","1435","1436","1437","1438","1439","1440","1441","1442","1443","1444"],["GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:MF","GO:MF","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:MF","REAC","GO:CC","GO:BP","GO:CC","REAC","REAC","REAC","GO:BP","GO:CC","REAC","REAC","REAC","REAC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","REAC","REAC","GO:BP","REAC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","REAC","GO:MF","REAC","GO:CC","GO:BP","GO:CC","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:CC","REAC","REAC","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","GO:MF","REAC","GO:MF","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","REAC","GO:MF","GO:CC","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","REAC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","REAC","REAC","REAC","GO:CC","GO:BP","GO:CC","GO:CC","GO:MF","REAC","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","REAC","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:CC","GO:MF","GO:BP","REAC","REAC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","REAC","REAC","REAC","GO:CC","REAC","REAC","REAC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","REAC","REAC","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","REAC","GO:BP","GO:MF","GO:MF","GO:BP","REAC","REAC","GO:CC","REAC","REAC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","REAC","REAC","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","REAC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:MF","GO:MF","REAC","REAC","REAC","GO:BP","GO:CC","REAC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:CC","REAC","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","GO:BP","GO:BP","GO:CC","REAC","REAC","GO:MF","GO:MF","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","REAC","GO:MF","REAC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:BP","REAC","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","REAC","REAC","REAC","REAC","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:MF","REAC","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","REAC","GO:MF","REAC","GO:BP","GO:BP","REAC","REAC","GO:CC","GO:MF","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","REAC","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:CC","GO:CC","REAC","REAC","GO:MF","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:MF","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:CC","GO:CC","REAC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","REAC","GO:BP","GO:CC","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:BP","REAC","REAC","GO:MF","GO:CC","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","REAC","GO:CC","GO:CC","REAC","REAC","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","REAC","GO:BP","GO:BP","GO:CC","GO:BP","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","REAC","GO:BP","REAC","REAC","REAC","REAC","GO:BP","GO:BP","REAC","REAC","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","REAC","GO:CC","REAC","REAC","GO:CC","REAC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","REAC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","REAC","GO:BP","GO:BP","REAC","GO:BP","REAC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","REAC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:CC","GO:BP","REAC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","REAC","REAC","REAC","REAC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","REAC","GO:MF","REAC","GO:MF","GO:BP","GO:MF","GO:BP","GO:MF","REAC","GO:BP","REAC","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","REAC","GO:MF","GO:MF","REAC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","REAC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:BP","GO:BP","REAC","GO:BP","GO:CC","REAC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","REAC","GO:BP","GO:MF","GO:BP","GO:BP","REAC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","REAC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","REAC","GO:CC","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:MF","REAC","REAC","GO:CC","GO:BP","GO:CC"],[435,2739,4067,2715,2127,2716,2130,1444,396,11974,10279,1451,311,384,2124,277,469,2123,1308,440,1875,2126,1450,7161,2404,26382,2128,12114,11644,240,2494,9067,12371,1568,10011,2129,2125,283,549,15281,12445,1026,2363,334,5674,2330,12444,2502,12711,14514,850,2501,1014,9946,1662,3746,294,24858,10020,2529,2761,7116,2527,901,2922,2930,947,1092,1284,29342,1874,1197,808,287,1868,2396,1079,898,814,2408,4452,1959,4086,2115,807,9805,2368,3776,2025,2548,816,838,15716,1590,1858,885,1136,3664,3796,3845,2447,12715,15655,8649,20875,1452,1025,8264,1799,12718,24867,1482,1481,1483,1093,2596,2394,3663,11658,1918,2439,337,19777,1976,6775,1320,3493,809,2006,4609,3975,2395,24856,12704,14858,1690,14854,479,159,436,7149,11910,2020,6743,3540,2005,3778,3777,5644,1013,441,23211,17047,2369,1935,15653,2855,1989,14859,101,15656,12374,12639,12375,839,377,849,17778,403,10007,9099,15264,15263,185,1015,11657,10182,15709,6480,1708,16015,160,480,1955,9213,4752,8265,1459,35,11971,10017,439,12714,4215,15613,1002,447,6379,2922,896,14319,26383,2526,9261,11645,15718,12717,3662,2928,491,7049,2405,15652,15740,9410,7877,385,15282,318,1310,4012,19044,8650,10312,14456,836,6681,17131,14855,1462,2965,15538,26384,3952,6676,4775,12696,6477,24669,4650,5583,446,9991,2538,867,8360,437,8266,2389,3747,5673,5398,3777,9806,431,5897,15663,9969,4309,5396,1445,1438,2765,1472,14516,3608,6424,9215,4651,4753,15657,29343,15615,206,257,8270,6332,4214,4216,1012,1090,4614,97,3800,320,4785,18629,1138,477,426,2935,12783,8814,295,2675,16023,319,9260,465,15722,27756,6762,4100,2868,5118,3531,27804,27802,9528,4756,3562,36,12697,2443,4874,2084,471,8741,5064,13467,2924,12136,118,27754,1045,24866,26474,27800,9092,560,5402,9091,26163,12196,1727,2857,12076,4211,7583,12703,62,405,27788,9915,4587,8586,5650,837,4709,4099,11756,15283,9100,2398,1725,8590,4858,18731,1996,8267,1884,15160,19155,9660,15340,11505,1900,27805,3778,4969,27807,28882,406,64,14515,817,1207,15172,214,18730,1259,8420,9807,3622,1880,410,2038,21581,21572,21570,21571,103,13657,15,7722,2164,1759,1117,25979,2929,14174,1098,347,3661,2294,11907,2277,638,2061,2967,876,15361,2966,434,416,8592,1149,2886,2496,1148,19701,14393,24668,9079,852,2694,27335,1197,17835,1093,1215,625,26856,267,1862,387,418,25598,18726,5886,1111,504,15339,1883,8179,21335,12138,18728,2418,2515,182,2419,2514,2463,110,21858,1056,2539,1412,173,5028,3578,3533,66,408,1139,30422,2444,1452,1828,12640,3858,11633,2021,15717,14,203,202,8651,12135,1850,3625,4023,1478,2089,1024,204,5822,3168,16045,15173,17329,1514,2940,672,1950,825,213,19702,15712,19242,14860,16014,8269,1740,2310,416,19132,4213,4710,1036,768,8744,12795,16081,303,9560,1100,1124,8596,2160,1775,673,16066,9216,790,12376,3057,2057,210,4717,7608,1021,1322,2432,8180,27337,15707,7060,2278,147,1526,1101,5088,15336,258,1405,1035,2927,2921,4776,22968,3963,1083,1028,1834,1252,184,1025,676,938,1949,427,675,9967,11470,103,5672,930,8743,14657,1059,4210,20369,4212,16365,3751,18727,474,29506,25602,7681,8385,4711,1360,1362,1361,2146,1366,15558,306,301,1436,2136,5560,100,2390,1364,13173,7067,1988,15045,1529,2503,2090,1367,26,302,15560,12157,6744,2440,8271,222,223,6455,176,181,9236,32,7602,24,7688,183,3387,141,980,2022,15654,70,3610,14333,1359,3787,8396,3623,425,3996,3598,9806,2513,2512,1525,16030,823,417,281,3127,2768,13656,14819,83,1847,342,1849,2974,8598,11778,12642,265,8268,2021,2580,737,4908,1507,923,23,7592,15724,2399,12788,23529,1827,2099,1260,2374,2923,389,1846,1016,6746,2066,9180,2111,3861,12466,586,4217,437,677,12219,897,306,2385,102,585,460,512,2843,2831,3694,21714,2173,861,5747,1673,3549,26029,25259,3220,22129,6442,521,1859,2915,1654,15759,19359,961,308,257,315,15337,23527,8284,9937,3547,22796,5459,6761,18630,10071,222,8819,25,10795,2938,4275,22797,441,1266,15535,4311,2917,2037,22317,1077,3700,317,21,1881,1142,2302,401,7611,19815,1992,207,14402,3065,5052,9855,8320,1120,15352,541,3795,8274,1074,4582,16027,6329,1243,15723,18280,27879,6,1099,787,454,24967,9221,3353,4276,20894,2387,4,7601,18628,10099,30412,2910,485,381,2375,2929,2923,4069,5187,14849,3655,9708,22121,3877,7682,1830,11923,392,4909,19699,4045,2440,13127,833,8306,15614,15734,191,15577,7605,1022,6255,8304,10432,1885,3181,8813,350,9031,386,2185,8652,397,15752,1118,21112,21484,1143,390,5351,522,14470,3784,22120,1957,859,174,380,15556,3979,9185,3812,26031,1410,10811,139,15901,6094,14676,2354,3654,4436,5404,6443,11767,2184,532,22522,10382,15713,30403,3595,999,509,924,15338,5089,1027,1665,1646,4710,101,1873,3535,3624,3154,10350,3219,536,20827,152,1437,8622,22793,17765,379,13339,29375,20895,26663,8286,8222,15578,2122,18599,2891,29792,2492,1140,17272,18729,2359,488,11588,20229,9290,1062,1584,41,1096,16067,1726,3781,4085,4864,3961,7003,201,1057,452,6444,2433,860,8307,15351,120,2911,483,10306,1569,12646,15701,11919,856,1111,21717,16962,3571,1058,1329,7819,7725,3579,29853,16744,1269,1071,3153,1064,204,1324,1693,1669,13525,3361,6764,25,9626,448,456,3733,12709,1065,3278,15347,1426,216,343,5654,416,3505,16150,13523,15728,762,10383,1216,505,7820,684,1692,10072,8344,3350,3958,1631,917,409,3650,1663,9792,7726,3593,15705,8654,685,3818,68,1365,25716,3548,3067,2867,109,26392,14961,1140,3128,21347,767,1960,17762,2020,436,12142,2358,30404,17819,11784,1644,17731,1465,7956,15903,11896,1880,876,4786,11471,4134,14879,1920,509,2414,304,6604,1775,3642,5959,2963,6677,9678,518,26391,26155,151,2906,531,6595,1928,8116,2842,2129,2314,489,17764,15731,173,9984,1720,3779,1150,697,316,7723,16961,4583,433,368,1120,4427,2023,7595,2553,1858,1410,2439,3569,3151,2989,3,4707,9169,22021,17733,14415,4670,4611,15897,12958,8136,1717,1714,1467,1023,6020,272,2139,2140,982,29377,12645,9960,24670,22795,18957,508,507,19454,13262,1544,9033,261,303,5309,23378,4498,864,1224,11843,8303,477,189,2485,1380,202,200,199,4667,15736,1838,264,4610,15590,17769,3985,735,948,507,3916,10389,21189,15331,2321,1121,17783,756,6960,21338,17779,19534,25253,417,4862,15245,11315,2279,1441,11995,23438,1421,1369,4172,195,2118,3383,1835,1560,300,23302,588,24074,24073,11785,2255,15555,5877,12327,26432,14914,15591,2114,11476,20387,10027,14374,8272,10016,302,6364,486,14910,8140,4913,9679,5957,15404,6684,11755,23704,1072,10277,11466,9186,1026,2114,28864,17211,1899,12990,1218,25250,8372,1032,8550,8138,3794,3915,165,6680,17912,426,10272,12278,2119,3998,13748,1123,2585,327,9821,22693,11523,3557,605,1078,1127,15586,2112,3527,1901,7035,886,14981,22279,398,1076,20235,3657,24825,12191,3227,499,25296,1707,7590,10437,198,1572,2508,14990,15768,6478,9104,5738,189,16674,21648,14666,15658,3785,4580,22208,22210,9463,13131,5811,21336,4503,13541,195,14380,2411,619,22798,954,15616,7736,9621,13653,14467,28369,2086,1712,1180,2404,8322,17763,15162,4782,3792,679,4765,823,9563,12724,2866,12350,918,1940,7693,2544,13130,6780,2092,7550,3795,13715,6977,1089,1959,27619,292,7054,198,2000,19708,16675,537,1861,25881,9985,30228,19392,25942,3788,9947,2468,7673,15013,4218,224,167,2175,15761,10,203,16799,1774,1128,17076,8172,1124,2296,8548,361,11817,3374,9637,488,22319,145,2820,3064,3152,10412,25836,14915,7001,7305,3865,1602,63,407,1075,6662,22792,1842,12134,11734,9156,22108,1730,16785,3082,3414,17112,9068,29120,2117,5584,7658,22049,9080,2818,1328,13590,613,927,3125,20254,3981,14834,22340,22342,2086,15859,7677,8013,13745,1121,983,838,2432,17767,410,7844,3964,453,3823,9617,7709,25815,15166,3586,478,9062,7326,138,137,297,2881,3606],["RNA binding","extracellular exosome","extracellular vesicle","extracellular membrane-bounded organelle","extracellular organelle","intracellular organelle lumen","organelle lumen","membrane-enclosed lumen","nucleic acid binding","heterocyclic compound binding","organic cyclic compound binding","vesicle","nucleoplasm","cytoplasm","membrane-bounded organelle","extracellular space","cytosol","organelle","Metabolism of RNA","mRNA binding","protein binding","intracellular organelle","nuclear lumen","cadherin binding","translation","regulation of mRNA metabolic process","intracellular membrane-bounded organelle","peptide biosynthetic process","ribosome biogenesis","extracellular region","peptide metabolic process","cell adhesion molecule binding","regulation of RNA splicing","protein-containing complex","cellular macromolecule biosynthetic process","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","intracellular anatomical structure","focal adhesion","regulation of mRNA processing","amide biosynthetic process","cell-substrate junction","rRNA processing","spliceosomal complex","rRNA metabolic process","ribosomal subunit","cellular amide metabolic process","mRNA Splicing - Major Pathway","cellular macromolecule metabolic process","regulation of mRNA splicing, via spliceosome","cytoplasmic translation","mRNA Splicing","cytosolic ribosome","ncRNA processing","Processing of Capped Intron-Containing Pre-mRNA","supramolecular complex","nucleus","organonitrogen compound biosynthetic process","ncRNA metabolic process","rRNA processing in the nucleus and cytosol","anchoring junction","protein-containing complex binding","rRNA processing","nuclear speck","organelle organization","catalytic step 2 spliceosome","GTP hydrolysis and joining of the 60S ribosomal subunit","Influenza Infection","Major pathway of rRNA processing in the nucleolus and cytosol","regulation of cellular macromolecule biosynthetic process","ribonucleoprotein granule","L13a-mediated translational silencing of Ceruloplasmin expression","Eukaryotic Translation Initiation","Cap-dependent Translation Initiation","Regulation of expression of SLITs and ROBOs","RNA localization","secretory granule","nuclear body","cytoplasmic stress granule","regulation of translation","ribonucleoprotein complex","cytoplasmic ribonucleoprotein granule","ficolin-1-rich granule lumen","Signaling by ROBO receptors","Eukaryotic Translation Elongation","regulation of cellular amide metabolic process","Translation","secretory vesicle","melanosome","pigment granule","endomembrane system","large ribosomal subunit","regulation of protein metabolic process","Peptide chain elongation","binding","Formation of a pool of free 40S subunits","preribosome","macromolecule biosynthetic process","ficolin-1-rich granule","ATP hydrolysis activity","Viral mRNA Translation","cellular protein metabolic process","regulation of nitrogen compound metabolic process","regulation of cellular protein metabolic process","regulation of primary metabolic process","vesicle lumen","cell junction","regulation of cellular metabolic process","secretory granule lumen","cellular nitrogen compound biosynthetic process","organic substance biosynthetic process","Nonsense Mediated Decay (NMD) independent of the Exon Junction Complex (EJC)","Nonsense Mediated Decay (NMD) enhanced by the Exon Junction Complex (EJC)","Nonsense-Mediated Decay (NMD)","Influenza Viral RNA Transcription and Replication","cytoplasmic vesicle lumen","RNA catabolic process","biosynthetic process","ribosomal small subunit biogenesis","Response of EIF2AK4 (GCN2) to amino acid deficiency","protein folding","U2-type spliceosomal complex","cellular component organization or biogenesis","SRP-dependent cotranslational protein targeting to membrane","protein metabolic process","cytoplasmic vesicle","intracellular vesicle","Eukaryotic Translation Termination","Selenocysteine synthesis","enzyme binding","nucleoside-triphosphatase activity","mRNA catabolic process","organonitrogen compound metabolic process","cellular biosynthetic process","positive regulation of cellular process","structural molecule activity","positive regulation of biological process","ribosome","alternative mRNA splicing, via spliceosome","RNA helicase activity","translation regulator activity","ATP-dependent activity","ATP-dependent activity, acting on RNA","ribonucleoprotein complex binding","pyrophosphatase activity","Selenoamino acid metabolism","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","hydrolase activity, acting on acid anhydrides","viral process","cytosolic large ribosomal subunit","mRNA 3'-UTR binding","non-membrane-bounded organelle assembly","regulation of macromolecule metabolic process","Translation initiation complex formation","Ribosomal scanning and start codon recognition","nuclear transport","nucleocytoplasmic transport","translation factor activity, RNA binding","negative regulation of cellular process","Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S","negative regulation of nitrogen compound metabolic process","regulation of RNA stability","cellular component biogenesis","regulation of mRNA stability","small ribosomal subunit","nucleolus","membrane","regulation of mRNA catabolic process","mitochondrial matrix","cellular nitrogen compound metabolic process","unfolded protein binding","RNA transport","nucleic acid transport","Axon guidance","cytosolic small ribosomal subunit","ribosomal large subunit biogenesis","translation regulator activity, nucleic acid binding","establishment of RNA localization","regulation of metabolic process","nuclear periphery","cellular localization","regulation of alternative mRNA splicing, via spliceosome","polysome","cytoskeletal protein binding","macromolecule localization","positive regulation of macromolecule metabolic process","negative regulation of cellular metabolic process","Nervous system development","nucleotide binding","nucleoside phosphate binding","nucleobase-containing compound catabolic process","single-stranded RNA binding","cellular macromolecule catabolic process","positive regulation of metabolic process","regulation of cellular component organization","helicase activity","translation initiation factor activity","small molecule binding","U2-type precatalytic spliceosome","Formation of the ternary complex, and subsequently, the 43S complex","heterocycle catabolic process","negative regulation of mRNA metabolic process","rRNA modification in the nucleus and cytosol","positive regulation of RNA splicing","ribosome assembly","negative regulation of protein metabolic process","cellular nitrogen compound catabolic process","macromolecule catabolic process","precatalytic spliceosome","cytoskeleton","protein folding chaperone","translational initiation","nuclear export","chromosome organization","protein localization to organelle","maturation of SSU-rRNA","mitochondrion","positive regulation of mRNA processing","Cellular response to starvation","Metabolism of amino acids and derivatives","catalytic complex","cellular macromolecule localization","negative regulation of cellular protein metabolic process","carbohydrate derivative binding","intracellular transport","actin cytoskeleton","aromatic compound catabolic process","regulation of cellular localization","negative regulation of biological process","small-subunit processome","cell cycle","mRNA transport","positive regulation of mRNA metabolic process","purine nucleotide binding","identical protein binding","positive regulation of gene expression","cellular metabolic process","regulation of nucleobase-containing compound metabolic process","organic cyclic compound catabolic process","gene expression","nucleobase-containing compound transport","structural constituent of ribosome","cellular protein localization","perinuclear region of cytoplasm","nuclear matrix","mRNA 5'-UTR binding","double-stranded RNA binding","positive regulation of cellular metabolic process","RNA processing","supramolecular polymer","mRNA metabolic process","purine ribonucleotide binding","supramolecular fiber","negative regulation of cellular amide metabolic process","endoplasmic reticulum lumen","negative regulation of translation","localization","protein localization to nucleus","response to organic substance","ribonucleotide binding","envelope","organelle envelope","nitrogen compound metabolic process","Neutrophil degranulation","positive regulation of mRNA splicing, via spliceosome","chromosomal region","viral life cycle","regulation of organelle organization","regulation of gene expression","negative regulation of macromolecule metabolic process","positive regulation of nitrogen compound metabolic process","negative regulation of cellular macromolecule biosynthetic process","positive regulation of cellular component organization","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","telomere maintenance","regulation of cellular catabolic process","purine ribonucleoside triphosphate binding","negative regulation of metabolic process","regulation of catabolic process","proteasome accessory complex","Infectious disease","protein domain specific binding","proteasome complex","chaperone complex","Cellular responses to stress","positive regulation of organelle organization","regulation of biological quality","90S preribosome","proteasome regulatory particle","endoplasmic reticulum","cytoskeleton organization","biological process involved in symbiotic interaction","developmental process","nuclear envelope","collagen-containing extracellular matrix","establishment of localization in cell","Cellular responses to stimuli","negative regulation of RNA splicing","actin binding","regulation of RNA metabolic process","positive regulation of protein localization to chromosome, telomeric region","anion binding","endopeptidase complex","movement of cell or subcellular component","programmed cell death","protein localization","positive regulation of protein localization to Cajal body","regulation of protein localization to Cajal body","cellular response to stress","posttranscriptional regulation of gene expression","cell death","AUF1 (hnRNP D0) binds and destabilizes mRNA","primary metabolic process","synapse","adenyl nucleotide binding","polysomal ribosome","chaperonin-containing T-complex","regulation of intracellular transport","regulation of cell death","positive regulation of translation","U2-type catalytic step 2 spliceosome","regulation of programmed cell death","mitotic cell cycle","regulation of protein localization to chromosome, telomeric region","Host Interactions of HIV factors","organic substance catabolic process","protein localization to nuclear body","protein localization to Cajal body","regulation of protein localization","cell cortex","adenyl ribonucleotide binding","regulation of localization","mitotic cell cycle process","macromolecule metabolic process","RHOBTB2 GTPase cycle","apoptotic process","regulation of apoptotic process","regulation of biosynthetic process","cell cycle process","cellular catabolic process","organellar ribosome","mitochondrial ribosome","positive regulation of establishment of protein localization to telomere","poly-purine tract binding","rRNA binding","telomere organization","cellular component organization","microtubule cytoskeleton","regulation of macromolecule biosynthetic process","peptidase complex","catalytic activity, acting on RNA","negative regulation of mRNA processing","chaperone binding","RNA export from nucleus","RHOBTB GTPase Cycle","regulation of telomere maintenance","snoRNA binding","regulation of establishment of protein localization to telomere","poly(A) binding","regulation of cellular biosynthetic process","ATP binding","anatomical structure development","organelle assembly","stress granule assembly","regulation of cellular process","protein refolding","calcium-dependent phospholipid binding","regulation of telomerase RNA localization to Cajal body","polymeric cytoskeletal fiber","telomere maintenance via telomere lengthening","positive regulation of telomerase RNA localization to Cajal body","protein localization to nucleoplasm","mitochondrial large ribosomal subunit","organellar large ribosomal subunit","negative regulation of mRNA splicing, via spliceosome","vesicle membrane","Laminin interactions","cellular developmental process","maturation of LSU-rRNA","regulation of establishment of protein localization to chromosome","cell leading edge","regulation of protein stability","positive regulation of cellular amide metabolic process","postsynapse","Regulation of mRNA stability by proteins that bind AU-rich elements","primary lysosome","azurophil granule","RNA localization to nucleus","telomerase RNA localization","RNA localization to Cajal body","telomerase RNA localization to Cajal body","chromosome, telomeric region","positive regulation of nucleobase-containing compound metabolic process","ribosomal small subunit assembly","cell differentiation","nucleobase-containing compound metabolic process","RNA Polymerase II Transcription Termination","cytoplasmic vesicle membrane","microtubule cytoskeleton organization involved in mitosis","telomere maintenance via telomerase","heterocycle metabolic process","axon","chromosome","catabolic process","RNA-dependent DNA biosynthetic process","catalytic activity, acting on a nucleic acid","DNA metabolic process","nucleoid","mitochondrial nucleoid","mitotic spindle organization","Folding of actin by CCT/TriC","protein stabilization","spindle organization","rough endoplasmic reticulum","single-stranded DNA binding","positive regulation of telomere maintenance","cortical actin cytoskeleton","response to stress","mRNA 3'-end processing","cortical cytoskeleton","organic substance metabolic process","regulation of nucleocytoplasmic transport","organic cyclic compound metabolic process","actin filament binding","cytoplasmic translational initiation","cellular aromatic compound metabolic process","regulation of telomere maintenance via telomere lengthening","extracellular matrix","chaperone-mediated protein folding","external encapsulating structure","organelle membrane","proteasome regulatory particle, base subcomplex","positive regulation of protein localization","basement membrane","Regulation of activated PAK-2p34 by proteasome mediated degradation","Cross-presentation of soluble exogenous antigens (endosomes)","vacuolar lumen","negative regulation of RNA catabolic process","protein localization to chromosome, telomeric region","protein import","midbody","microtubule","regulation of developmental process","Regulation of ornithine decarboxylase (ODC)","RNA 3'-end processing","nucleic acid metabolic process","negative regulation of programmed cell death","establishment of protein localization to telomere","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","p53-Independent G1/S DNA damage checkpoint","Autodegradation of the E3 ubiquitin ligase COP1","Ubiquitin-dependent degradation of Cyclin D","p53-Independent DNA Damage Response","Vpu mediated degradation of CD4","condensed chromosome","supramolecular fiber organization","membrane coat","coated membrane","phospholipase inhibitor activity","Assembly of the pre-replicative complex","heat shock protein binding","bounding membrane of organelle","metabolic process","lytic vacuole","lysosome","preribosome, large subunit precursor","positive regulation of chromosome organization","Vif-mediated degradation of APOBEC3G","Negative regulation of NOTCH4 signaling","Regulation of Apoptosis","regulation of cellular component biogenesis","plasma membrane bounded cell projection","response to chemical","poly-pyrimidine tract binding","positive regulation of protein metabolic process","ribosomal large subunit assembly","basal RNA polymerase II transcription machinery binding","basal transcription machinery binding","positive regulation of cellular protein metabolic process","negative regulation of apoptotic process","Regulation of RUNX3 expression and activity","mitochondrial protein-containing complex","anatomical structure morphogenesis","Non-integrin membrane-ECM interactions","cell projection","lamellipodium","maturation of 5.8S rRNA","cell migration","multicellular organism development","localization of cell","cell motility","negative regulation of cell death","Orc1 removal from chromatin","microtubule-based process","Degradation of AXIN","SCF-beta-TrCP mediated degradation of Emi1","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","cleavage involved in rRNA processing","nitrogen compound transport","regulation of multicellular organismal process","DNA conformation change","positive regulation of viral process","organelle localization","positive regulation of cellular biosynthetic process","endoplasmic reticulum chaperone complex","The role of GTSE1 in G2/M progression after G2 checkpoint","vacuole","cellular response to chemical stimulus","positive regulation of biosynthetic process","positive regulation of macromolecule biosynthetic process","Hh mutants are degraded by ERAD","ER-Phagosome pathway","DNA geometric change","biological process involved in interspecies interaction between organisms","regulation of cell cycle","sister chromatid segregation","lipase inhibitor activity","growth cone","secretory granule membrane","regulation of telomere maintenance via telomerase","Stabilization of p53","ROS sensing by NFE2L2","Degradation of DVL","biological process involved in interaction with host","regulation of chromosome organization","cell surface","RNA stabilization","cell adhesion","actomyosin","maturation of 5.8S rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","regulation of cell cycle process","biological adhesion","myofibril","Metabolism of polyamines","regulation of translational initiation","mRNA 3'-end processing","positive regulation of telomere maintenance via telomere lengthening","establishment of localization","ubiquitin-like protein ligase binding","DNA replication","Antigen processing-Cross presentation","actin filament bundle","site of polarized growth","ubiquitin protein ligase binding","regulation of biological process","CDT1 association with the CDC6:ORC:origin complex","NIK--&gt;noncanonical NF-kB signaling","Hh mutants abrogate ligand secretion","prespliceosome","U2-type prespliceosome","negative regulation of gene expression","mitotic nuclear division","endoplasmic reticulum protein-containing complex","filopodium","Hedgehog ligand biogenesis","Regulation of HMOX1 expression and activity","MAPK6/MAPK4 signaling","ruffle","HSP90 chaperone cycle for steroid hormone receptors (SHR) in the presence of ligand","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","SCF(Skp2)-mediated degradation of p27/p21","Dectin-1 mediated noncanonical NF-kB signaling","Degradation of GLI1 by the proteasome","protein localization to chromosome","locomotion","microtubule cytoskeleton organization","RNA metabolic process","G2/M Transition","positive regulation of intracellular transport","mRNA stabilization","vesicle coat","tissue development","establishment of protein localization to organelle","negative regulation of biosynthetic process","movement in host environment","hydrolase activity","establishment of protein localization to chromosome","Defective CFTR causes cystic fibrosis","regulation of DNA biosynthetic process","negative regulation of mRNA catabolic process","actin filament-based process","calcium-dependent protein binding","negative regulation of macromolecule biosynthetic process","Mitochondrial translation elongation","Mitochondrial translation termination","Mitochondrial translation initiation","contractile fiber","Mitotic G2-G2/M phases","regulation of DNA metabolic process","cell morphogenesis","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","nuclear membrane","laminin complex","RNA strand annealing activity","chromosome, centromeric region","mRNA processing","Mitotic Anaphase","establishment of protein localization","S100 protein binding","transcription factor binding","system development","PCP/CE pathway","mRNA Splicing - Minor Pathway","neuron projection","Mitotic Metaphase and Anaphase","APC:Cdc20 mediated degradation of cell cycle proteins prior to satisfation of the cell cycle checkpoint","nuclear pore","positive regulation of DNA metabolic process","N6-methyladenosine-containing RNA binding","ribosome binding","'de novo' protein folding","negative regulation of cellular catabolic process","endonucleolytic cleavage involved in rRNA processing","endonucleolytic cleavage of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","proteasome-activating activity","Asymmetric localization of PCP proteins","Autodegradation of Cdh1 by Cdh1:APC/C","peroxiredoxin activity","mitotic sister chromatid segregation","regulation of cell morphogenesis","APC/C:Cdc20 mediated degradation of mitotic proteins","actin cytoskeleton organization","stress fiber","contractile actin filament bundle","P-body","organelle inner membrane","Separation of Sister Chromatids","import into nucleus","Activation of APC/C and APC/C:Cdc20 mediated degradation of mitotic proteins","RNA splicing","viral entry into host cell","Mitochondrial translation","postsynaptic specialization","cell-substrate adhesion","membrane protein complex","transcription coregulator activity","distal axon","oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor","RNA polymerase binding","p53-Dependent G1/S DNA damage checkpoint","p53-Dependent G1 DNA Damage Response","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","establishment of organelle localization","postsynaptic density","DNA Replication Pre-Initiation","virus receptor activity","integrin-mediated signaling pathway","transport","negative regulation of nucleobase-containing compound metabolic process","cell development","Activation of NF-kappaB in B cells","Regulation of RAS by GAPs","U12-type spliceosomal complex","Regulation of RUNX2 expression and activity","chromosome segregation","positive regulation of telomere maintenance via telomerase","exogenous protein binding","positive regulation of cellular component biogenesis","COPI-mediated anterograde transport","negative regulation of cellular biosynthetic process","sarcolemma","protein import into nucleus","Downstream signaling events of B Cell Receptor (BCR)","regulation of cell morphogenesis involved in differentiation","asymmetric synapse","G1/S DNA Damage Checkpoints","APC/C:Cdc20 mediated degradation of Securin","cellular component disassembly","positive regulation of RNA metabolic process","mRNA export from nucleus","entry into host","positive regulation of protein localization to nucleus","Regulation of APC/C activators between G1/S and early anaphase","Signaling by NOTCH4","MET activates PTK2 signaling","mRNA polyadenylation","nucleus organization","mitochondrial inner membrane","Regulation of PTEN stability and activity","HIV Infection","ribosomal small subunit binding","poly(U) RNA binding","protein-containing complex disassembly","thioredoxin peroxidase activity","oxidoreductase activity, acting on the aldehyde or oxo group of donors","RNA polyadenylation","regulation of cytokine production","negative regulation of catabolic process","Golgi apparatus","Degradation of beta-catenin by the destruction complex","regulation of protein-containing complex assembly","Formation of tubulin folding intermediates by CCT/TriC","ATPase activator activity","Transport of Mature Transcript to Cytoplasm","condensed chromosome, centromeric region","cytokine production","spindle","actin filament","receptor-mediated endocytosis","intracellular protein transport","glutamatergic synapse","apoptotic signaling pathway","Switching of origins to a post-replicative state","basolateral plasma membrane","vesicle-mediated transport","integrin binding","oxidoreductase activity","regulation of supramolecular fiber organization","toxin transport","binding of sperm to zona pellucida","nuclear chromosome segregation","viral genome replication","formation of translation preinitiation complex","azurophil granule lumen","response to unfolded protein","Prefoldin mediated transfer of substrate  to CCT/TriC","cell division","cellular response to organic substance","Gene and protein expression by JAK-STAT signaling after Interleukin-12 stimulation","cell morphogenesis involved in differentiation","CDK-mediated phosphorylation and removal of Cdc6","Cellular response to chemical stress","regulation of catalytic activity","regulation of protein localization to nucleus","regulation of cell projection organization","substrate adhesion-dependent cell spreading","side of membrane","regulation of plasma membrane bounded cell projection organization","amide binding","ion binding","regulation of molecular function","protein folding in endoplasmic reticulum","zona pellucida receptor complex","DNA duplex unwinding","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","response to topologically incorrect protein","actin filament organization","cellular process","plasma membrane bounded cell projection organization","Golgi-associated vesicle","MET promotes cell motility","actin filament bundle assembly","response to inorganic substance","translation preinitiation complex","specific granule","postsynapse organization","endocytic vesicle","neuron to neuron synapse","Cellular response to hypoxia","APC/C-mediated degradation of cell cycle proteins","Regulation of mitotic cell cycle","Interleukin-12 family signaling","cell body","chromatin binding","ribonucleoprotein complex biogenesis","DNA biosynthetic process","nuclear localization sequence binding","maturation of LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","regulation of RNA export from nucleus","establishment or maintenance of cell polarity","translation initiation factor binding","cell junction organization","regulation of mRNA 3'-end processing","coated vesicle membrane","synapse organization","3-hydroxyacyl-CoA dehydrogenase activity","cell cortex region","positive regulation of protein-containing complex assembly","coated vesicle","phospholipase A2 inhibitor activity","spindle localization","RNA stem-loop binding","M Phase","negative regulation of RNA metabolic process","actin filament bundle organization","positive regulation of establishment of protein localization","ABC-family proteins mediated transport","dendrite","basal plasma membrane","centrosome","response to nitrogen compound","misfolded protein binding","dendritic tree","cell-cell recognition","regulation of response to stress","Transport of Mature mRNA derived from an Intron-Containing Transcript","ABC transporter disorders","regulation of anatomical structure morphogenesis","biological regulation","sperm-egg recognition","regulation of intrinsic apoptotic signaling pathway","response to oxidative stress","mRNA cleavage factor complex","Cooperation of Prefoldin and TriC/CCT  in actin and tubulin folding","mRNA cleavage","catalytic step 1 spliceosome","U2-type catalytic step 1 spliceosome","response to endogenous stimulus","response to organic cyclic compound","animal organ development","actin-based cell projection","regulatory RNA binding","pre-mRNA cleavage required for polyadenylation","neuron projection cytoplasm","cell projection organization","exon-exon junction complex","molecular function activator activity","Cyclin E associated events during G1/S transition ","positive regulation of cell morphogenesis involved in differentiation","organic substance transport","dendritic filopodium","basal part of cell","positive regulation of viral genome replication","Cajal body","regulation of protein modification process","negative regulation of cellular component organization","regulation of cellular component movement","endonucleolytic cleavage in ITS1 to separate SSU-rRNA from 5.8S rRNA and LSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","'de novo' posttranslational protein folding","cellular component assembly","sarcomere","miRNA binding","negative regulation of protein ubiquitination","RNA polymerase II C-terminal domain binding","GTP binding","cation-transporting ATPase complex","multicellular organismal process","telomerase holoenzyme complex","positive regulation of ATP-dependent activity","mitochondrial envelope","nucleoside diphosphate phosphorylation","regulation of protein polymerization","Cytoprotection by HMOX1","establishment of spindle localization","Golgi-associated vesicle membrane","positive regulation of cell cycle process","RNA phosphodiester bond hydrolysis","Interleukin-12 signaling","Cyclin A:Cdk2-associated events at S phase entry","protein transport","formation of cytoplasmic translation initiation complex","nucleotide phosphorylation","cytoplasmic region","mRNA cleavage involved in mRNA processing","ATP-dependent activity, acting on DNA","eukaryotic translation initiation factor 4F complex","Association of TriC/CCT with target proteins during biosynthesis","Cooperation of PDCL (PhLP1) and TRiC/CCT in G-protein beta folding","positive regulation of transport","response to external stimulus","cellular component morphogenesis","isomerase activity","positive regulation of supramolecular fiber organization","enzyme inhibitor activity","endodermal cell differentiation","RNA polymerase II complex binding","regulation of cytoskeleton organization","peptidyl-proline modification","organelle fission","DNA-templated transcription, elongation","membrane microdomain","guanyl nucleotide binding","guanyl ribonucleotide binding","viral gene expression","nucleocytoplasmic carrier activity","Synthesis of DNA","clathrin-coated pit","neuron projection organization","cell adhesion mediator activity","positive regulation of multicellular organismal process","regulation of apoptotic signaling pathway","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","Golgi-to-ER retrograde transport","catalytic activity","G1/S Transition","regulation of viral process","transport along microtubule","Hedgehog 'on' state","Processing of Intronless Pre-mRNAs","eukaryotic 48S preinitiation complex","MHC class II protein complex binding","kinetochore","calcium ion binding","ATPase dependent transmembrane transport complex","plasma membrane protein complex","mitotic spindle","disordered domain specific binding","single fertilization","cell-cell junction","IRES-dependent viral translational initiation","Apoptosis","mitochondrial membrane","regulation of nucleobase-containing compound transport","regulation of plasma membrane bounded cell projection assembly","positive regulation of dendritic spine development","sno(s)RNA-containing ribonucleoprotein complex","regulation of cell differentiation","regulation of cell motility","regulation of cellular response to stress","protein localization to cytoplasmic stress granule","positive regulation of cell projection organization","neuron projection development","chaperone cofactor-dependent protein refolding","generation of precursor metabolites and energy","cellular response to chemical stress","tertiary granule","positive regulation of DNA biosynthetic process","phagocytic vesicle","Interleukin-1 signaling","regulation of cell projection assembly","regulation of establishment of protein localization","Transcriptional regulation by RUNX3","eukaryotic translation initiation factor 3 complex","regulation of protein catabolic process","tricarboxylic acid metabolic process","regulation of intracellular protein transport","AP-3 adaptor complex","sarcoplasmic reticulum lumen","nuclear chromosome","T-tubule","biological process involved in interaction with symbiont","RHOBTB1 GTPase cycle","postsynaptic cytosol","tertiary granule lumen","Hsp70 protein binding","nuclear protein-containing complex","antigen processing and presentation of exogenous antigen","podosome","clathrin coat","peptidyl-prolyl cis-trans isomerase activity","viral translation","membrane raft","eukaryotic 43S preinitiation complex","negative regulation of protein modification process","regulation of synapse organization","nuclear division","CRD-mediated mRNA stability complex","mRNA cleavage and polyadenylation specificity factor complex","7SK snRNA binding","signal sequence binding","positive regulation of molecular function","spindle assembly","annealing activity","inclusion body","Innate Immune System","intrinsic apoptotic signaling pathway","positive regulation of posttranscriptional gene silencing","lumenal side of membrane","AP-type membrane coat adaptor complex","box H/ACA snoRNP complex","regulation of cell migration","regulation of proteolysis","plasma membrane region","positive regulation of gene silencing by miRNA","muscle cell development","MHC class II antigen presentation","clathrin coat of coated pit","tRNA-splicing ligase complex","clathrin vesicle coat","Beta-catenin independent WNT signaling","Metabolism of proteins","extracellular matrix structural constituent","Programmed Cell Death","positive regulation of cell cycle","endoderm development","RNA polymerase core enzyme binding","ribosomal subunit export from nucleus","ribosome localization","translation elongation factor activity","microtubule organizing center","nucleoside diphosphate metabolic process","cellular protein catabolic process","COPI vesicle coat","nervous system development","regulation of synapse structure or activity","NS1 Mediated Effects on Host Pathways","endonucleolytic cleavage to generate mature 5'-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","rRNA 5'-end processing","cell growth","DNA Replication","cell recognition","modulation by host of symbiont process","positive regulation of cell adhesion","protein polymerization","EPH-Ephrin signaling","cell-cell adhesion mediator activity","platelet alpha granule","microtubule associated complex","positive regulation of cell migration","aldehyde dehydrogenase (NAD+) activity","structural constituent of cytoskeleton","response to endoplasmic reticulum stress","protein-containing complex localization","tubulin binding","fertilization","Platelet degranulation ","myeloid cell homeostasis","lysosomal membrane","lytic vacuole membrane","Processing of Capped Intronless Pre-mRNA","telomerase RNA binding","protein catabolic process","oxidoreductase activity, acting on CH-OH group of donors","spindle elongation","positive regulation of protein polymerization","aldehyde dehydrogenase [NAD(P)+] activity","cis-trans isomerase activity","rRNA modification","Mitotic G1 phase and G1/S transition","positive regulation of intracellular signal transduction","lumenal side of endoplasmic reticulum membrane","integral component of lumenal side of endoplasmic reticulum membrane","disulfide oxidoreductase activity","heterochromatin","negative regulation of protein modification by small protein conjugation or removal","anatomical structure formation involved in morphogenesis","preribosome, small subunit precursor","cell periphery","positive regulation of intracellular protein transport","ER to Golgi Anterograde Transport","somatodendritic compartment","dendritic spine development","paraspeckles","endoplasmic reticulum-Golgi intermediate compartment","negative regulation of catalytic activity","Transcriptional regulation by RUNX2","negative regulation of apoptotic signaling pathway","muscle structure development","general transcription initiation factor binding","Post-translational protein phosphorylation","regulation of gene silencing by miRNA","regulation of response to biotic stimulus","cytoskeleton-dependent intracellular transport","positive regulation of cytoskeleton organization","muscle cell differentiation","insulin-like growth factor binding","activation of innate immune response","negative regulation of organelle organization","regulation of locomotion","embryo development","response to steroid hormone","Response to elevated platelet cytosolic Ca2+","endoderm formation","UCH proteinases","Cell Cycle, Mitotic","peptide binding","methylosome","oxidoreductase activity, acting on a sulfur group of donors","box H/ACA snoRNA binding","cell-substrate junction assembly","protein homodimerization activity","response to cytokine","sodium:potassium-exchanging ATPase complex","regulation of protein modification by small protein conjugation or removal","positive regulation of leukocyte cell-cell adhesion","Antiviral mechanism by IFN-stimulated genes","cellular response to DNA damage stimulus","brush border","telomeric DNA binding","protein C-terminus binding","actomyosin structure organization","endocytosis","citrate metabolic process","neuron spine","eukaryotic translation elongation factor 1 complex","regulation of dendritic spine development","protein depolymerization","protein peptidyl-prolyl isomerization","cellular response to oxidative stress","RHO GTPases activate PAKs","cytosolic region","rough endoplasmic reticulum membrane","Disease","Cellular response to heat stress","regulation of cell adhesion","regulation of posttranscriptional gene silencing","growth factor binding","smooth endoplasmic reticulum","pericentric heterochromatin","antigen processing and presentation of peptide antigen via MHC class I","growth cone filopodium","nuclear inclusion body","reproductive process","spindle midzone","azurophil granule membrane","I band","apical part of cell","cell population proliferation","box H/ACA RNP complex","mitotic chromosome condensation","reproduction","MHC protein complex binding","regulation of actin filament-based process","cell-cell adhesion","regulation of gene silencing by RNA","filopodium assembly","RNA secondary structure unwinding","protein kinase binding","regulation of filopodium assembly","multi-organism reproductive process","regulation of histone modification","RHO GTPases activate CIT","RHO GTPases Activate ROCKs","positive regulation of response to biotic stimulus","Z disc","heterocycle biosynthetic process","laminin-5 complex","laminin-10 complex","laminin-11 complex","extrinsic component of plasma membrane","positive regulation of cell motility","negative regulation of molecular function","snRNP binding","organic cyclic compound biosynthetic process","positive regulation of plasma membrane bounded cell projection assembly","protein localization to mitochondrion","Defective EXT2 causes exostoses 2","Defective EXT1 causes exostoses 1, TRPS2 and CHDS","cellular response to organic cyclic compound","cell redox homeostasis","ruffle membrane","regulation of DNA-templated transcription, elongation","CLEC7A (Dectin-1) signaling","Cell Cycle Checkpoints","insulin-like growth factor I binding","regulation of substrate adhesion-dependent cell spreading","response to organonitrogen compound","lateral plasma membrane","pseudopodium","antigen processing and presentation of exogenous peptide antigen via MHC class I","regulation of protein ubiquitination","osteoblast differentiation","BBSome-mediated cargo-targeting to cilium","proteolysis","Muscle contraction","Beta oxidation of octanoyl-CoA to hexanoyl-CoA","Beta oxidation of lauroyl-CoA to decanoyl-CoA-CoA","Beta oxidation of hexanoyl-CoA to butanoyl-CoA","proteasomal protein catabolic process","positive regulation of cellular component movement","Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs)","COPI-independent Golgi-to-ER retrograde traffic","kinase binding","regulation of binding","positive regulation of dendritic spine morphogenesis","response to wounding","Downstream TCR signaling","fructose-bisphosphate aldolase activity","formation of primary germ layer","7SK snRNP","cadherin binding involved in cell-cell adhesion","establishment of protein localization to mitochondrial membrane","RNA destabilization","dendritic spine neck","COPI-coated vesicle membrane","membrane organization","ECM proteoglycans","carboxylic acid metabolic process","mitotic spindle assembly","positive regulation of mRNA catabolic process","cellular response to endogenous stimulus","positive regulation of mitotic cell cycle phase transition","vacuolar membrane","positive regulation of cell development","chromosome localization","RIG-I signaling pathway","DNA-dependent DNA replication","organelle envelope lumen","sulfur compound binding","positive regulation of G1/S transition of mitotic cell cycle","endosome lumen","Mitotic Prometaphase","messenger ribonucleoprotein complex","Basigin interactions","Signaling by Rho GTPases","myosin II filament","Regulation of HSF1-mediated heat shock response","dendrite cytoplasm","nuclear envelope lumen","cell-substrate junction organization","positive regulation of cytokine production","positive regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway","regulation of protein insertion into mitochondrial membrane involved in apoptotic signaling pathway","DNA-binding transcription factor binding","TNFR2 non-canonical NF-kB pathway","regulation of transport","poly(G) binding","oxoacid metabolic process","regulation of cellular protein catabolic process","developmental cell growth","positive regulation of binding","lysosomal lumen","positive regulation of locomotion","establishment of protein localization to mitochondrion","WD40-repeat domain binding","microtubule polymerization","positive regulation of cellular catabolic process","nucleobase-containing compound biosynthetic process","Cell Cycle","mRNA 3'-UTR AU-rich region binding","eukaryotic translation initiation factor 2 complex","positive regulation of response to stimulus","regulation of histone methylation","U1 snRNA binding","erythrocyte homeostasis","U3 snoRNA binding","positive regulation of T cell activation","histone deacetylase binding","catalytic activity, acting on DNA","regulation of mRNA polyadenylation","transport vesicle","pre-mRNA intronic binding","growth","cell part morphogenesis","Hedgehog 'off' state","organic acid metabolic process","neuron projection extension","epithelium development","phospholipid binding","cell cycle phase transition","platelet alpha granule lumen","positive regulation of cell cycle phase transition","tau protein binding","Hemostasis","positive regulation of response to external stimulus","positive regulation of histone modification","carbon-oxygen lyase activity","response to heat","fibrillar center","aromatic compound biosynthetic process","mRNA destabilization","transcription coactivator activity","supercoiled DNA binding","ubiquitin ligase inhibitor activity","Signaling by Rho GTPases, Miro GTPases and RHOBTB3","response to abiotic stimulus","ATP metabolic process","Integrin cell surface interactions","protein export from nucleus","Chk1/Chk2(Cds1) mediated inactivation of Cyclin B:Cdk1 complex","small ribosomal subunit rRNA binding","regulation of actin filament organization","wound healing","cytoplasmic side of membrane","COP9 signalosome","ISG15 antiviral mechanism","phagocytic vesicle membrane","positive regulation of developmental process","dendritic spine","axo-dendritic transport","endolysosome","sexual reproduction","Formation of annular gap junctions","neuron development","microtubule-based transport","DNA helicase activity","COPI-coated vesicle","circulatory system development","dosage compensation by inactivation of X chromosome","regulation of hematopoietic progenitor cell differentiation","proteasome-mediated ubiquitin-dependent protein catabolic process","regulation of mitotic cell cycle","dynactin complex","regulation of hematopoietic stem cell differentiation","Pwp2p-containing subcomplex of 90S preribosome","positive regulation of cell-cell adhesion","structural constituent of postsynapse","Beta oxidation of decanoyl-CoA to octanoyl-CoA-CoA","PTEN Regulation","mitochondrial fatty acid beta-oxidation of saturated fatty acids","axon extension","metaphase plate congression","regulation of phosphate metabolic process","ATPase binding","synaptic vesicle budding from endosome","immunological synapse","regulation of peptidase activity","dendritic spine organization","clathrin coat assembly","regulation of phosphorus metabolic process","ribonucleoside diphosphate metabolic process","ATPase-coupled cation transmembrane transporter activity","dendritic transport of ribonucleoprotein complex","dendritic transport of messenger ribonucleoprotein complex","nitric oxide homeostasis","positive regulation of innate immune response","dendrite development","nucleic acid phosphodiester bond hydrolysis","establishment or maintenance of transmembrane electrochemical gradient","positive regulation of endocytosis","RNA polymerase II general transcription initiation factor binding","transport of virus","Type I hemidesmosome assembly","integrin complex","plasma membrane bounded cell projection morphogenesis","Gap junction degradation","chaperone-mediated protein complex assembly","neuron differentiation","RNA polymerase II-specific DNA-binding transcription factor binding","positive regulation of mitotic cell cycle","pore complex assembly","regulation of protein localization to membrane","Signaling by MET","RHO GTPases Activate Formins","Ion homeostasis","Transport to the Golgi and subsequent modification","positive regulation of mRNA 3'-end processing","dendritic spine morphogenesis","cell projection morphogenesis","clathrin binding","postsynaptic endocytic zone cytoplasmic component","Degradation of the extracellular matrix","nitric-oxide synthase regulator activity","Extracellular matrix organization","ubiquitin-protein transferase inhibitor activity","small molecule metabolic process","protein-disulfide reductase activity","regulation of ATP-dependent activity","enoyl-CoA hydratase activity","S Phase","actin filament fragmentation","trans-Golgi Network Vesicle Budding","regulation of innate immune response","laminin binding","neuronal cell body","neurogenesis","hydro-lyase activity","regulation of embryonic development","glyceraldehyde-3-phosphate dehydrogenase (NAD+) (non-phosphorylating) activity","dynein complex","5S rRNA binding","negative regulation of ubiquitin protein ligase activity","Cargo trafficking to the periciliary membrane","transmembrane transporter binding","TFIID-class transcription factor complex binding","Scavenging by Class F Receptors","basement membrane organization","regulation of endopeptidase activity","adherens junction","specific granule lumen","regulation of lamellipodium organization","cellular response to heat","positive regulation of response to DNA damage stimulus","cellular response to cytokine stimulus","positive regulation of cell cycle G1/S phase transition","lyase activity","ncRNA 5'-end processing","WNT5A-dependent internalization of FZD2, FZD5 and ROR2","cellular cation homeostasis","generation of neurons","positive regulation of catabolic process","endonucleolytic cleavage in 5'-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","granular component","Syndecan interactions","establishment of chromosome localization","mitotic spindle elongation","Beta oxidation of palmitoyl-CoA to myristoyl-CoA","cation homeostasis","methyltransferase complex","Interactions of Rev with host cellular proteins","regulation of cell development","positive regulation of microtubule polymerization","antigen processing and presentation of exogenous peptide antigen","cell-cell contact zone","regulation of response to external stimulus","protease binding","P-type transmembrane transporter activity","P-type ion transporter activity","P-body assembly","ameboidal-type cell migration","regulation of postsynapse organization","Antigen Presentation: Folding, assembly and peptide loading of class I MHC","cellular metal ion homeostasis","eukaryotic translation initiation factor 3 complex, eIF3m","box H/ACA scaRNP complex","structural constituent of synapse","regulation of receptor localization to synapse","developmental growth","antigen processing and presentation","substantia nigra development","response to temperature stimulus","signaling receptor binding","organellar small ribosomal subunit","mitochondrial small ribosomal subunit","clathrin-coated vesicle","ATPase-coupled transmembrane transporter activity","plasma membrane bounded cell projection assembly","Regulation of PLK1 Activity at G2/M Transition","positive regulation of apoptotic process","modification-dependent protein binding","regulation of actin cytoskeleton organization","inorganic ion homeostasis","RHOD GTPase cycle","metal ion homeostasis","endocytic vesicle lumen","dosage compensation","head development","extracellular matrix binding","protein localization to cell periphery","acetyl-CoA biosynthetic process","coenzyme A metabolic process","positive regulation of signaling","regulation of neurotransmitter receptor localization to postsynaptic specialization membrane","protein kinase A binding","cellular ion homeostasis","box C/D RNP complex","positive regulation of proteolysis","protein insertion into mitochondrial membrane involved in apoptotic signaling pathway","G2/M Checkpoints","cleavage body","organelle transport along microtubule","response to biotic stimulus","anterograde synaptic vesicle transport","synaptic vesicle cytoskeletal transport","synaptic vesicle transport along microtubule","MHC class I peptide loading complex","negative regulation of ubiquitin-protein transferase activity","cellular potassium ion homeostasis","regulation of actin filament length","ADP metabolic process","antigen processing and presentation via MHC class Ib","extrinsic component of membrane","FGFR2 alternative splicing","pronucleus","regulation of dendritic spine morphogenesis","double-stranded telomeric DNA binding","ubiquitin-dependent ERAD pathway","intracellular protein-containing complex","protein disulfide isomerase activity","intramolecular oxidoreductase activity, transposing S-S bonds","nuclear import signal receptor activity","regulation of endocytosis","secondary alcohol metabolic process","stem cell differentiation","protein complex involved in cell adhesion","proteasome core complex","phosphoserine residue binding","acyltransferase, acyl groups converted into alkyl on transfer","Amplification of signal from the kinetochores","Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal","lamin filament","regulation of striated muscle contraction","Schaffer collateral - CA1 synapse"],["8.3e-197","1.7e-126","1.5e-125","1.8e-125","1.8e-125","8.7e-118","9.5e-118","9.5e-118","2.0e-102"," 6.2e-93"," 3.2e-92"," 1.7e-81"," 1.0e-79"," 3.0e-79"," 5.1e-79"," 3.3e-78"," 7.6e-77"," 1.8e-74"," 5.0e-74"," 1.0e-69"," 3.7e-68"," 7.1e-68"," 2.7e-67"," 3.6e-64"," 2.0e-63"," 2.3e-63"," 5.0e-62"," 3.7e-61"," 7.2e-61"," 5.7e-59"," 4.4e-58"," 2.1e-57"," 2.3e-56"," 2.7e-56"," 3.7e-56"," 1.8e-55"," 1.8e-55"," 2.6e-55"," 6.1e-55"," 1.1e-54"," 5.5e-54"," 6.8e-54"," 1.3e-53"," 2.1e-50"," 2.2e-49"," 3.0e-48"," 1.2e-47"," 9.1e-45"," 3.0e-44"," 3.0e-44"," 4.6e-43"," 1.5e-42"," 1.6e-41"," 2.4e-40"," 8.5e-40"," 2.1e-39"," 8.2e-39"," 3.3e-38"," 7.8e-38"," 3.6e-37"," 3.7e-36"," 8.3e-36"," 1.1e-35"," 4.3e-35"," 8.1e-35"," 1.3e-34"," 1.7e-34"," 1.9e-34"," 3.6e-34"," 4.0e-34"," 1.0e-33"," 1.1e-33"," 1.7e-33"," 1.7e-33"," 5.4e-33"," 1.3e-32"," 3.7e-32"," 8.2e-32"," 1.4e-31"," 6.6e-31"," 7.2e-31"," 8.5e-31"," 1.1e-30"," 2.2e-30"," 3.1e-30"," 4.5e-30"," 5.7e-30"," 1.5e-29"," 2.0e-29"," 2.0e-29"," 2.1e-29"," 3.2e-29"," 2.1e-28"," 4.6e-28"," 4.7e-28"," 9.0e-28"," 2.3e-27"," 2.6e-27"," 3.8e-27"," 4.4e-27"," 6.4e-27"," 6.8e-27"," 6.9e-27"," 7.1e-27"," 7.7e-27"," 1.3e-26"," 1.4e-26"," 1.7e-26"," 2.4e-26"," 2.7e-26"," 2.9e-26"," 3.1e-26"," 3.9e-26"," 3.9e-26"," 4.3e-26"," 4.6e-26"," 7.3e-26"," 9.9e-26"," 1.1e-25"," 1.3e-25"," 1.3e-25"," 1.8e-25"," 1.9e-25"," 2.1e-25"," 5.4e-25"," 9.2e-25"," 1.0e-24"," 1.2e-24"," 1.2e-24"," 3.2e-24"," 4.6e-24"," 5.6e-24"," 8.9e-24"," 1.2e-23"," 2.9e-23"," 3.4e-23"," 3.5e-23"," 5.1e-23"," 1.2e-22"," 1.8e-22"," 2.5e-22"," 2.8e-22"," 3.7e-22"," 4.6e-22"," 5.0e-22"," 5.3e-22"," 7.3e-22"," 7.3e-22"," 1.0e-21"," 1.3e-21"," 2.5e-21"," 4.4e-21"," 4.6e-21"," 6.8e-21"," 6.8e-21"," 1.1e-20"," 1.1e-20"," 1.4e-20"," 1.5e-20"," 1.6e-20"," 2.9e-20"," 3.7e-20"," 3.7e-20"," 3.9e-20"," 4.7e-20"," 8.1e-20"," 8.2e-20"," 9.3e-20"," 1.0e-19"," 1.1e-19"," 1.2e-19"," 1.3e-19"," 1.3e-19"," 1.4e-19"," 1.6e-19"," 2.0e-19"," 2.2e-19"," 3.4e-19"," 4.2e-19"," 4.5e-19"," 5.0e-19"," 9.5e-19"," 1.0e-18"," 1.2e-18"," 1.5e-18"," 1.8e-18"," 2.2e-18"," 2.5e-18"," 3.1e-18"," 3.2e-18"," 4.9e-18"," 6.2e-18"," 9.1e-18"," 1.1e-17"," 1.1e-17"," 1.4e-17"," 1.4e-17"," 1.7e-17"," 1.8e-17"," 2.3e-17"," 2.4e-17"," 3.1e-17"," 3.3e-17"," 3.6e-17"," 3.9e-17"," 5.0e-17"," 5.7e-17"," 5.7e-17"," 6.1e-17"," 1.0e-16"," 1.4e-16"," 2.0e-16"," 2.3e-16"," 2.3e-16"," 2.4e-16"," 4.3e-16"," 4.5e-16"," 5.0e-16"," 6.8e-16"," 7.3e-16"," 8.8e-16"," 1.0e-15"," 1.2e-15"," 1.2e-15"," 1.3e-15"," 1.5e-15"," 1.5e-15"," 1.6e-15"," 1.8e-15"," 1.9e-15"," 2.1e-15"," 2.4e-15"," 2.6e-15"," 2.9e-15"," 3.1e-15"," 3.3e-15"," 4.4e-15"," 4.4e-15"," 4.7e-15"," 4.9e-15"," 5.3e-15"," 5.7e-15"," 5.8e-15"," 5.9e-15"," 7.5e-15"," 8.7e-15"," 9.0e-15"," 9.7e-15"," 9.9e-15"," 1.1e-14"," 1.2e-14"," 1.5e-14"," 2.0e-14"," 2.1e-14"," 2.1e-14"," 2.2e-14"," 2.5e-14"," 2.6e-14"," 2.7e-14"," 2.8e-14"," 3.4e-14"," 3.4e-14"," 3.6e-14"," 4.9e-14"," 6.1e-14"," 6.4e-14"," 7.6e-14"," 8.2e-14"," 8.9e-14"," 1.0e-13"," 1.0e-13"," 1.1e-13"," 1.2e-13"," 1.2e-13"," 1.3e-13"," 1.8e-13"," 2.4e-13"," 2.5e-13"," 2.5e-13"," 2.8e-13"," 3.3e-13"," 4.2e-13"," 4.2e-13"," 4.4e-13"," 4.6e-13"," 6.0e-13"," 6.6e-13"," 7.8e-13"," 7.8e-13"," 7.9e-13"," 1.1e-12"," 1.2e-12"," 1.4e-12"," 1.7e-12"," 1.9e-12"," 2.0e-12"," 2.1e-12"," 2.2e-12"," 2.3e-12"," 2.4e-12"," 2.9e-12"," 3.0e-12"," 3.3e-12"," 4.5e-12"," 5.4e-12"," 5.4e-12"," 5.9e-12"," 5.9e-12"," 6.4e-12"," 6.5e-12"," 6.8e-12"," 7.5e-12"," 8.2e-12"," 9.1e-12"," 9.7e-12"," 1.6e-11"," 1.6e-11"," 1.7e-11"," 2.0e-11"," 2.2e-11"," 2.5e-11"," 2.5e-11"," 2.9e-11"," 3.0e-11"," 3.1e-11"," 3.2e-11"," 3.5e-11"," 3.5e-11"," 3.5e-11"," 3.9e-11"," 4.0e-11"," 4.0e-11"," 4.2e-11"," 4.3e-11"," 4.5e-11"," 4.6e-11"," 4.7e-11"," 4.7e-11"," 5.0e-11"," 6.4e-11"," 7.0e-11"," 7.0e-11"," 7.3e-11"," 7.6e-11"," 8.1e-11"," 9.0e-11"," 1.1e-10"," 1.2e-10"," 1.3e-10"," 1.6e-10"," 1.8e-10"," 1.9e-10"," 2.1e-10"," 2.1e-10"," 2.1e-10"," 2.2e-10"," 2.4e-10"," 2.6e-10"," 2.8e-10"," 2.8e-10"," 2.9e-10"," 3.2e-10"," 3.4e-10"," 3.5e-10"," 3.7e-10"," 3.8e-10"," 4.2e-10"," 4.2e-10"," 4.3e-10"," 4.7e-10"," 4.7e-10"," 5.0e-10"," 5.3e-10"," 5.3e-10"," 5.5e-10"," 6.4e-10"," 6.7e-10"," 7.0e-10"," 7.6e-10"," 7.8e-10"," 7.9e-10"," 9.9e-10"," 1.1e-09"," 1.1e-09"," 1.1e-09"," 1.1e-09"," 1.1e-09"," 1.3e-09"," 1.3e-09"," 1.3e-09"," 1.3e-09"," 1.3e-09"," 1.3e-09"," 1.4e-09"," 1.6e-09"," 1.6e-09"," 1.7e-09"," 1.7e-09"," 1.8e-09"," 2.3e-09"," 2.4e-09"," 2.7e-09"," 2.8e-09"," 3.1e-09"," 3.1e-09"," 3.2e-09"," 3.2e-09"," 3.2e-09"," 3.2e-09"," 3.3e-09"," 3.5e-09"," 3.9e-09"," 4.0e-09"," 4.1e-09"," 4.1e-09"," 4.6e-09"," 5.1e-09"," 5.4e-09"," 5.7e-09"," 5.8e-09"," 5.9e-09"," 5.9e-09"," 6.2e-09"," 6.8e-09"," 7.0e-09"," 7.3e-09"," 7.4e-09"," 7.9e-09"," 8.1e-09"," 8.4e-09"," 8.6e-09"," 9.1e-09"," 1.0e-08"," 1.0e-08"," 1.0e-08"," 1.0e-08"," 1.1e-08"," 1.1e-08"," 1.2e-08"," 1.3e-08"," 1.3e-08"," 1.3e-08"," 1.4e-08"," 1.6e-08"," 1.9e-08"," 2.0e-08"," 2.1e-08"," 2.4e-08"," 2.5e-08"," 2.5e-08"," 2.5e-08"," 2.5e-08"," 2.5e-08"," 2.5e-08"," 2.7e-08"," 2.7e-08"," 2.7e-08"," 2.7e-08"," 2.9e-08"," 3.0e-08"," 3.0e-08"," 3.0e-08"," 3.4e-08"," 3.5e-08"," 3.5e-08"," 3.7e-08"," 3.7e-08"," 3.9e-08"," 3.9e-08"," 3.9e-08"," 4.2e-08"," 4.2e-08"," 4.5e-08"," 4.8e-08"," 4.8e-08"," 5.2e-08"," 5.3e-08"," 5.3e-08"," 5.4e-08"," 5.6e-08"," 5.9e-08"," 6.1e-08"," 6.4e-08"," 6.4e-08"," 6.6e-08"," 6.7e-08"," 7.1e-08"," 7.3e-08"," 7.7e-08"," 8.0e-08"," 8.0e-08"," 8.1e-08"," 8.6e-08"," 8.7e-08"," 8.9e-08"," 8.9e-08"," 8.9e-08"," 9.0e-08"," 9.1e-08"," 9.5e-08"," 9.9e-08"," 1.0e-07"," 1.0e-07"," 1.0e-07"," 1.1e-07"," 1.1e-07"," 1.2e-07"," 1.2e-07"," 1.3e-07"," 1.3e-07"," 1.3e-07"," 1.4e-07"," 1.4e-07"," 1.5e-07"," 1.5e-07"," 1.6e-07"," 1.6e-07"," 1.7e-07"," 1.7e-07"," 1.8e-07"," 1.9e-07"," 1.9e-07"," 1.9e-07"," 2.0e-07"," 2.0e-07"," 2.0e-07"," 2.1e-07"," 2.1e-07"," 2.1e-07"," 2.2e-07"," 2.2e-07"," 2.4e-07"," 2.7e-07"," 2.8e-07"," 2.9e-07"," 3.1e-07"," 3.1e-07"," 3.2e-07"," 3.2e-07"," 3.4e-07"," 3.4e-07"," 3.6e-07"," 3.7e-07"," 3.8e-07"," 3.9e-07"," 4.1e-07"," 4.1e-07"," 4.1e-07"," 4.1e-07"," 4.1e-07"," 4.5e-07"," 4.5e-07"," 4.6e-07"," 4.6e-07"," 4.9e-07"," 4.9e-07"," 5.6e-07"," 5.7e-07"," 5.8e-07"," 5.8e-07"," 5.8e-07"," 5.8e-07"," 5.8e-07"," 5.8e-07"," 5.9e-07"," 6.1e-07"," 6.2e-07"," 6.2e-07"," 6.4e-07"," 6.4e-07"," 6.6e-07"," 6.7e-07"," 6.7e-07"," 7.0e-07"," 7.0e-07"," 7.4e-07"," 7.4e-07"," 7.8e-07"," 8.3e-07"," 8.3e-07"," 8.3e-07"," 8.7e-07"," 9.1e-07"," 9.3e-07"," 9.3e-07"," 9.3e-07"," 9.3e-07"," 9.5e-07"," 9.6e-07"," 9.6e-07"," 9.7e-07"," 1.0e-06"," 1.0e-06"," 1.0e-06"," 1.0e-06"," 1.1e-06"," 1.1e-06"," 1.1e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.2e-06"," 1.3e-06"," 1.3e-06"," 1.4e-06"," 1.4e-06"," 1.6e-06"," 1.7e-06"," 1.8e-06"," 1.8e-06"," 1.8e-06"," 2.1e-06"," 2.1e-06"," 2.2e-06"," 2.2e-06"," 2.2e-06"," 2.3e-06"," 2.3e-06"," 2.3e-06"," 2.4e-06"," 2.7e-06"," 2.8e-06"," 2.8e-06"," 3.0e-06"," 3.0e-06"," 3.1e-06"," 3.2e-06"," 3.2e-06"," 3.3e-06"," 3.6e-06"," 3.6e-06"," 3.6e-06"," 3.7e-06"," 3.8e-06"," 3.9e-06"," 4.1e-06"," 4.1e-06"," 4.1e-06"," 4.2e-06"," 4.2e-06"," 4.2e-06"," 4.3e-06"," 4.5e-06"," 5.2e-06"," 5.3e-06"," 5.5e-06"," 5.5e-06"," 5.6e-06"," 5.6e-06"," 5.7e-06"," 5.7e-06"," 5.8e-06"," 5.9e-06"," 6.0e-06"," 6.1e-06"," 6.3e-06"," 6.4e-06"," 6.5e-06"," 6.9e-06"," 7.1e-06"," 7.3e-06"," 7.3e-06"," 7.4e-06"," 7.5e-06"," 7.7e-06"," 7.7e-06"," 7.7e-06"," 7.9e-06"," 8.6e-06"," 9.1e-06"," 9.3e-06"," 9.4e-06"," 9.4e-06"," 9.5e-06"," 9.8e-06"," 9.8e-06"," 9.9e-06"," 1.0e-05"," 1.1e-05"," 1.1e-05"," 1.1e-05"," 1.1e-05"," 1.2e-05"," 1.2e-05"," 1.4e-05"," 1.4e-05"," 1.4e-05"," 1.5e-05"," 1.6e-05"," 1.6e-05"," 1.6e-05"," 1.7e-05"," 1.8e-05"," 1.8e-05"," 1.8e-05"," 2.0e-05"," 2.0e-05"," 2.1e-05"," 2.2e-05"," 2.2e-05"," 2.2e-05"," 2.2e-05"," 2.2e-05"," 2.3e-05"," 2.4e-05"," 2.4e-05"," 2.4e-05"," 2.4e-05"," 2.4e-05"," 2.5e-05"," 2.6e-05"," 2.7e-05"," 2.8e-05"," 2.8e-05"," 2.9e-05"," 3.0e-05"," 3.0e-05"," 3.0e-05"," 3.0e-05"," 3.1e-05"," 3.1e-05"," 3.1e-05"," 3.2e-05"," 3.3e-05"," 3.3e-05"," 3.3e-05"," 3.4e-05"," 3.5e-05"," 3.6e-05"," 3.7e-05"," 3.8e-05"," 3.8e-05"," 4.0e-05"," 4.0e-05"," 4.0e-05"," 4.1e-05"," 4.1e-05"," 4.2e-05"," 4.3e-05"," 4.4e-05"," 4.5e-05"," 4.5e-05"," 4.6e-05"," 4.9e-05"," 4.9e-05"," 5.0e-05"," 5.0e-05"," 5.2e-05"," 5.2e-05"," 5.2e-05"," 5.2e-05"," 5.2e-05"," 5.2e-05"," 5.2e-05"," 5.3e-05"," 5.4e-05"," 5.5e-05"," 5.7e-05"," 5.8e-05"," 5.8e-05"," 5.8e-05"," 5.9e-05"," 5.9e-05"," 5.9e-05"," 6.0e-05"," 6.0e-05"," 6.2e-05"," 6.4e-05"," 6.4e-05"," 6.7e-05"," 6.7e-05"," 6.8e-05"," 6.8e-05"," 6.9e-05"," 6.9e-05"," 7.0e-05"," 7.0e-05"," 7.1e-05"," 7.5e-05"," 7.6e-05"," 7.7e-05"," 7.8e-05"," 8.0e-05"," 8.2e-05"," 8.4e-05"," 8.5e-05"," 8.7e-05"," 8.9e-05"," 8.9e-05"," 9.3e-05"," 9.5e-05"," 9.5e-05"," 9.8e-05"," 1.0e-04"," 1.2e-04"," 1.2e-04"," 1.2e-04"," 1.2e-04"," 1.2e-04"," 1.2e-04"," 1.3e-04"," 1.3e-04"," 1.3e-04"," 1.4e-04"," 1.4e-04"," 1.4e-04"," 1.4e-04"," 1.4e-04"," 1.4e-04"," 1.5e-04"," 1.5e-04"," 1.5e-04"," 1.5e-04"," 1.6e-04"," 1.6e-04"," 1.6e-04"," 1.6e-04"," 1.7e-04"," 1.7e-04"," 1.7e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.9e-04"," 1.9e-04"," 2.0e-04"," 2.0e-04"," 2.0e-04"," 2.0e-04"," 2.0e-04"," 2.1e-04"," 2.1e-04"," 2.2e-04"," 2.2e-04"," 2.2e-04"," 2.2e-04"," 2.3e-04"," 2.3e-04"," 2.5e-04"," 2.5e-04"," 2.6e-04"," 2.7e-04"," 2.7e-04"," 2.8e-04"," 2.8e-04"," 2.8e-04"," 2.9e-04"," 3.0e-04"," 3.3e-04"," 3.5e-04"," 3.5e-04"," 3.6e-04"," 3.6e-04"," 3.7e-04"," 3.7e-04"," 3.8e-04"," 3.9e-04"," 3.9e-04"," 4.0e-04"," 4.0e-04"," 4.3e-04"," 4.4e-04"," 4.4e-04"," 4.5e-04"," 4.5e-04"," 4.6e-04"," 4.8e-04"," 4.8e-04"," 5.0e-04"," 5.0e-04"," 5.0e-04"," 5.1e-04"," 5.2e-04"," 5.4e-04"," 5.4e-04"," 5.5e-04"," 5.6e-04"," 5.8e-04"," 5.8e-04"," 5.9e-04"," 6.0e-04"," 6.0e-04"," 6.1e-04"," 6.3e-04"," 6.4e-04"," 6.4e-04"," 6.4e-04"," 6.4e-04"," 6.5e-04"," 6.5e-04"," 6.5e-04"," 6.6e-04"," 6.7e-04"," 6.9e-04"," 6.9e-04"," 7.0e-04"," 7.0e-04"," 7.0e-04"," 7.0e-04"," 7.2e-04"," 7.2e-04"," 7.3e-04"," 7.4e-04"," 7.4e-04"," 7.4e-04"," 7.5e-04"," 7.7e-04"," 7.7e-04"," 7.8e-04"," 8.1e-04"," 8.2e-04"," 8.3e-04"," 8.3e-04"," 8.5e-04"," 8.6e-04"," 8.7e-04"," 8.9e-04"," 9.6e-04"," 9.6e-04"," 9.8e-04"," 1.0e-03"," 1.0e-03"," 1.0e-03"," 1.0e-03"," 1.0e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.6e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.7e-03"," 1.8e-03"," 1.8e-03"," 1.8e-03"," 1.8e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 2.0e-03"," 2.0e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.3e-03"," 2.3e-03"," 2.4e-03"," 2.4e-03"," 2.4e-03"," 2.4e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.7e-03"," 2.7e-03"," 2.7e-03"," 2.8e-03"," 2.8e-03"," 2.9e-03"," 2.9e-03"," 2.9e-03"," 2.9e-03"," 2.9e-03"," 2.9e-03"," 3.0e-03"," 3.0e-03"," 3.0e-03"," 3.0e-03"," 3.0e-03"," 3.1e-03"," 3.2e-03"," 3.2e-03"," 3.2e-03"," 3.3e-03"," 3.3e-03"," 3.3e-03"," 3.3e-03"," 3.3e-03"," 3.4e-03"," 3.4e-03"," 3.4e-03"," 3.4e-03"," 3.4e-03"," 3.5e-03"," 3.5e-03"," 3.5e-03"," 3.6e-03"," 3.6e-03"," 3.6e-03"," 3.6e-03"," 3.6e-03"," 3.7e-03"," 3.7e-03"," 3.7e-03"," 3.8e-03"," 3.8e-03"," 3.8e-03"," 3.8e-03"," 3.9e-03"," 3.9e-03"," 3.9e-03"," 3.9e-03"," 3.9e-03"," 4.1e-03"," 4.2e-03"," 4.2e-03"," 4.2e-03"," 4.2e-03"," 4.3e-03"," 4.3e-03"," 4.3e-03"," 4.4e-03"," 4.4e-03"," 4.4e-03"," 4.4e-03"," 4.4e-03"," 4.4e-03"," 4.5e-03"," 4.5e-03"," 4.5e-03"," 4.5e-03"," 4.6e-03"," 4.6e-03"," 4.7e-03"," 4.7e-03"," 4.8e-03"," 4.8e-03"," 4.8e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 5.0e-03"," 5.0e-03"," 5.0e-03"," 5.0e-03"," 5.0e-03"," 5.2e-03"," 5.2e-03"," 5.3e-03"," 5.3e-03"," 5.4e-03"," 5.4e-03"," 5.5e-03"," 5.5e-03"," 5.5e-03"," 5.5e-03"," 5.5e-03"," 5.5e-03"," 5.6e-03"," 5.6e-03"," 5.7e-03"," 5.7e-03"," 5.7e-03"," 5.8e-03"," 5.9e-03"," 5.9e-03"," 5.9e-03"," 6.1e-03"," 6.1e-03"," 6.1e-03"," 6.3e-03"," 6.3e-03"," 6.3e-03"," 6.3e-03"," 6.3e-03"," 6.4e-03"," 6.5e-03"," 6.5e-03"," 6.6e-03"," 6.7e-03"," 6.7e-03"," 6.7e-03"," 6.7e-03"," 6.8e-03"," 7.0e-03"," 7.1e-03"," 7.1e-03"," 7.2e-03"," 7.3e-03"," 7.4e-03"," 7.4e-03"," 7.5e-03"," 7.5e-03"," 7.6e-03"," 7.6e-03"," 7.7e-03"," 7.8e-03"," 7.8e-03"," 7.8e-03"," 7.8e-03"," 8.0e-03"," 8.0e-03"," 8.0e-03"," 8.2e-03"," 8.2e-03"," 8.2e-03"," 8.3e-03"," 8.3e-03"," 8.4e-03"," 8.6e-03"," 8.6e-03"," 8.7e-03"," 8.7e-03"," 8.8e-03"," 8.9e-03"," 8.9e-03"," 9.0e-03"," 9.1e-03"," 9.1e-03"," 9.3e-03"," 9.3e-03"," 9.5e-03"," 9.5e-03"," 9.5e-03"," 9.5e-03"," 9.6e-03"," 9.9e-03"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.8e-02"," 1.8e-02"," 1.8e-02"," 1.8e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 2.0e-02"," 2.0e-02"," 2.0e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.4e-02"," 2.4e-02"," 2.4e-02"," 2.4e-02"," 2.4e-02"," 2.4e-02"," 2.5e-02"," 2.5e-02"," 2.5e-02"," 2.5e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.7e-02"," 2.7e-02"," 2.7e-02"," 2.7e-02"," 2.7e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.8e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.0e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.6e-02"," 3.6e-02"," 3.6e-02"," 3.6e-02"," 3.7e-02"," 3.7e-02"," 3.7e-02"," 3.8e-02"," 3.8e-02"," 3.8e-02"," 3.8e-02"," 3.9e-02"," 3.9e-02"," 3.9e-02"," 3.9e-02"," 3.9e-02"," 3.9e-02"," 4.0e-02"," 4.1e-02"," 4.1e-02"," 4.1e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.3e-02"," 4.3e-02"," 4.3e-02"," 4.4e-02"," 4.5e-02"," 4.5e-02"," 4.5e-02"," 4.6e-02"," 4.6e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.7e-02"," 4.8e-02"," 4.8e-02"," 4.9e-02"," 4.9e-02"," 4.9e-02"," 4.9e-02"," 4.9e-02"," 5.0e-02"],[3142,2110,2132,2133,2133,6569,6570,6570,5447,7416,7497,3942,4161,12193,15557,3353,5373,16674,661,350,14799,15870,5442,334,713,318,14514,738,302,4303,895,550,151,7228,1564,6362,6362,17758,420,141,867,428,225,195,263,189,1165,178,5746,105,148,186,104,399,237,1330,10053,1698,536,192,839,1275,202,414,3805,85,113,152,182,656,260,112,120,120,171,198,860,869,79,390,2377,242,124,217,94,446,292,1025,109,109,4674,115,2472,90,18292,102,79,4880,184,316,90,4796,5529,2309,5694,325,2076,5921,320,4734,5869,96,116,116,133,323,308,5956,73,102,214,93,7994,113,5389,2447,2449,94,94,2030,656,261,6338,5794,5561,980,6094,520,79,76,151,550,78,157,712,117,715,715,419,55,97,373,6710,59,59,311,311,98,4818,60,2330,197,4801,184,76,1942,9818,193,483,8767,112,162,162,549,47,72,121,165,7219,130,3117,62,71,995,3147,3378,2493,574,2174,2175,435,84,1135,3678,2246,160,58,2516,50,52,473,93,59,36,61,1022,479,1341,52,2334,41,126,154,1034,1105,50,1674,32,157,370,1424,1963,957,2288,1826,501,494,805,5844,39,1713,129,116,1930,2091,1117,12839,3945,522,8909,225,449,1954,719,109,26,74,3176,3356,996,2153,1917,988,194,311,172,6425,300,3009,1934,1245,1245,12062,476,23,359,312,1109,5388,3353,2978,258,1021,37,137,850,1848,3558,989,25,912,690,66,28,751,488,3741,27,22,2007,1435,281,6326,473,421,2405,765,26,449,3700,13,2410,72,2074,1908,2700,11,11,1852,1126,2057,54,12572,1305,1576,32,10,341,1575,135,30,1439,895,15,125,2081,12,12,829,319,1565,2701,744,12328,23,1857,1411,4141,1194,2188,86,86,10,30,66,164,7793,1317,3926,103,400,30,106,82,35,82,32,11,24,4081,1502,5718,891,27,11067,21,53,18,757,84,15,14,56,56,21,1190,30,4210,29,12,421,299,160,613,86,154,154,19,19,19,19,169,1897,18,4128,7940,65,1171,150,71,8099,633,1795,2547,72,607,896,45,45,122,10,190,187,85,123,52,80,3780,56,110,13835,107,8395,218,35,8159,63,560,65,561,3657,12,454,97,50,50,175,76,31,214,199,432,2421,51,126,7476,883,17,52,52,52,52,52,52,245,755,96,96,12,68,128,2124,14361,729,729,24,84,53,53,53,940,2205,4316,28,1448,26,57,57,1361,866,54,277,2685,58,2312,203,36,1454,5114,1645,1645,971,71,898,55,55,55,27,2278,2623,329,59,541,1942,11,79,827,3008,1975,1839,56,90,92,1634,1011,198,16,163,309,54,57,57,57,183,192,908,66,1480,79,24,675,1487,232,58,80,73,37,4920,314,288,106,79,169,296,12142,59,59,59,16,16,1570,297,127,104,65,65,89,181,57,60,60,60,60,60,94,1821,627,6948,195,204,57,56,1912,658,1610,165,2425,26,61,113,64,800,84,1523,87,87,87,242,197,356,1010,73,303,12,5,202,1919,228,1942,14,602,4784,92,51,1347,229,74,85,204,7,76,40,261,15,15,6,64,64,9,170,305,76,711,71,71,93,562,188,166,77,1841,147,93,346,356,1323,512,273,38,58,66,66,66,409,322,85,76,112,4760,1496,2079,67,67,28,71,340,34,77,493,102,1581,135,162,80,96,328,68,68,449,1729,58,154,87,81,81,30,52,139,500,69,227,17,25,230,7,47,53,718,320,1606,83,402,26,25,80,147,724,397,114,243,1136,332,593,91,224,1597,147,756,373,41,41,278,130,12,91,138,28,635,2385,38,718,73,157,2314,137,634,110,625,618,389,6060,3017,11,8,86,74,159,444,19329,1479,89,40,157,534,18,158,163,337,352,75,88,88,57,554,580,1870,185,26,15,12,222,34,687,28,178,422,7,43,195,294,5,57,15,407,1395,160,316,103,612,249,623,1088,28,614,69,1273,71,77,926,12859,49,167,423,28,33,20,12,12,1551,901,3494,213,44,13,88,1521,20,33,82,79,2689,4,267,31,102,1477,680,1029,9,36,4556,211,33,80,8,382,19,7474,23,44,796,101,205,123,52,54,235,155,47,84,1829,16,102,265,14,108,13,38,38,892,2752,749,170,178,389,46,36,523,62,496,80,331,404,404,98,30,120,72,89,61,1421,359,128,133,5716,130,158,156,85,19,15,25,137,724,25,692,162,33,156,495,11,176,745,15,190,36,28,1541,953,648,7,336,953,31,499,322,163,70,140,101,192,522,94,16,401,14,232,11,10,235,52,101,23,12,55,43,2671,70,30,48,45,16,330,17,493,209,450,6,24,9,49,1491,118,13,75,1093,296,30,32,42,5,894,728,1237,29,174,123,17,8,29,143,2011,173,205,314,81,41,15,15,26,820,129,821,13,2428,215,38,5,5,484,128,224,64,439,278,90,50,90,162,524,16,104,257,152,378,197,125,161,400,400,28,22,985,141,10,95,17,48,34,147,971,25,25,43,72,93,1123,15,6181,161,155,835,94,12,132,682,118,225,616,52,107,45,347,194,196,357,29,73,352,997,1088,333,130,55,101,548,313,12,59,5,93,690,922,12,241,239,78,807,105,36,190,193,638,5,176,4,55,120,45,273,21,15,28,1639,87,733,47,137,32,24,48,3,12,1521,36,57,142,429,1940,6,19,1524,33,391,888,48,64,8,677,48,1054,158,19,19,185,129,4024,3,3,3,173,548,1041,7,4181,105,129,14,14,567,35,97,53,97,290,13,59,1001,70,18,32,208,226,23,1755,195,5,5,5,503,561,124,53,758,367,18,543,98,3,121,3,18,34,86,4,18,825,75,933,67,86,1310,85,454,298,84,28,181,95,265,37,35,199,12,25,697,3,67,29,10,99,465,9,9,454,100,1719,8,957,260,221,174,98,564,124,5,85,437,3953,678,26,5,2111,72,8,132,11,218,123,226,18,409,12,951,657,113,980,173,1160,464,529,66,110,43,672,435,96,80,105,140,4035,82,283,5,9,712,1121,276,84,58,13,9,267,415,194,34,70,77,1271,173,76,27,898,11,1104,196,73,22,1089,19,55,438,466,16,36,5,284,14,6,138,11,125,65,1367,82,2,44,439,80,18,1369,110,47,4,4,3,148,235,272,11,99,21,24,11,31,634,12,23,1364,328,113,20,181,76,136,54,186,10,59,638,70,2,140,7,298,10,1779,38,72,10,161,4,72,234,26,488,1639,64,64,6,54,10,13,51,131,10,6,32,412,172,61,53,65,108,828,52,203,21,13,646,1519,505,7,6,26,82,9,3,728,101,34,499,35,59,73,927,136,20,20,21,416,86,25,578,7,4,20,18,631,127,47,178,1546,28,28,197,109,548,85,501,161,350,740,50,657,23,21,812,53,337,18,18,1693,9,51,661,7,364,11,168,2,85,1529,19,19,19,9,20,14,160,91,13,310,25,14,44,9,87,773,20,20,19,205,146,234,36,23,9,5,92,92,5,93,79],[636,419,420,420,420,771,771,771,695,816,820,510,529,1010,1170,457,611,1204,253,135,1166,1166,598,127,159,122,1100,159,117,492,172,154,82,621,227,624,624,1216,127,78,163,127,95,85,98,74,181,99,573,61,64,100,53,110,111,207,816,207,125,95,153,205,96,103,408,46,63,73,89,133,79,62,64,64,76,70,120,150,44,96,251,73,46,83,54,102,97,128,49,49,438,45,288,51,1241,54,41,384,51,75,50,456,509,271,519,79,203,533,78,373,435,51,56,56,60,78,79,438,39,52,53,40,660,57,400,206,206,49,49,247,114,70,441,423,496,127,531,84,38,24,32,71,24,52,116,52,116,116,87,28,39,80,567,36,36,72,72,25,438,36,256,56,432,54,31,214,599,55,60,695,41,50,50,121,25,34,26,50,592,43,311,31,29,140,205,330,263,122,122,122,83,35,151,349,241,27,19,132,25,31,86,36,36,23,27,138,86,167,25,167,18,25,45,137,144,26,133,21,53,88,147,213,129,120,202,60,85,115,488,22,186,40,38,107,226,143,914,358,87,684,53,68,210,63,35,18,30,300,317,90,227,105,89,48,47,45,261,61,144,105,144,144,860,93,17,64,62,138,451,311,284,55,130,21,39,115,100,324,127,15,153,100,22,17,133,79,248,17,14,196,160,56,504,73,43,236,133,17,55,330,10,116,22,126,197,167,10,10,192,135,208,25,878,118,87,16,7,61,166,36,16,155,108,10,42,209,10,10,107,29,66,141,89,864,18,190,152,323,137,216,20,20,8,17,22,39,586,134,308,24,33,17,29,27,22,18,17,8,15,316,102,453,94,16,782,11,8,13,67,18,12,10,16,16,14,40,15,320,15,8,62,45,32,68,29,34,34,13,13,13,13,36,189,13,345,597,30,39,35,21,606,58,170,236,21,39,107,18,18,31,7,35,39,24,31,14,14,171,27,16,810,29,622,30,11,607,15,44,19,44,195,9,67,18,21,21,35,24,11,40,32,59,172,21,31,562,76,7,21,21,21,21,21,21,41,92,12,12,5,24,29,52,831,51,51,13,16,21,21,21,107,131,171,14,125,13,18,18,119,74,21,34,159,18,135,18,16,103,284,93,93,81,24,99,21,21,21,14,209,125,50,20,57,184,8,30,54,194,186,176,21,27,25,120,90,37,5,24,38,13,21,21,21,35,23,56,21,46,17,13,68,46,39,21,16,22,11,91,50,44,29,17,24,48,609,21,21,21,10,10,155,43,27,18,22,22,26,30,16,21,21,21,21,21,11,118,75,520,53,38,19,9,127,81,157,30,162,7,21,17,20,32,7,150,22,22,22,39,53,53,76,23,45,7,5,27,164,56,117,9,72,264,26,23,72,56,23,18,22,7,21,11,43,10,10,6,21,21,7,30,39,23,59,15,15,22,42,40,31,23,157,23,22,41,31,64,64,26,5,18,21,21,21,49,39,24,14,19,85,145,187,21,21,12,25,48,7,14,63,16,151,12,30,23,12,39,21,21,59,162,18,23,21,23,23,12,17,28,38,21,44,8,11,38,6,5,17,44,47,61,23,54,7,7,28,21,44,35,18,25,115,37,72,24,12,41,18,61,51,7,7,41,27,6,12,17,7,71,100,17,48,21,33,96,26,56,16,39,55,39,19,117,7,4,21,21,18,57,1200,79,9,13,28,48,6,22,14,35,39,21,23,23,21,37,67,167,19,9,9,8,30,8,78,12,19,41,5,11,33,25,3,13,7,79,122,28,15,25,61,12,46,56,8,61,8,91,25,21,70,606,7,30,50,6,7,6,7,7,71,49,191,22,14,7,13,79,9,7,22,10,113,4,12,12,19,137,76,59,7,9,323,24,12,19,6,48,4,374,9,14,55,17,33,27,12,7,30,28,18,22,162,6,17,24,7,22,5,7,7,24,163,56,28,30,14,13,11,61,14,57,18,25,49,49,20,11,26,11,10,14,65,35,18,21,306,28,28,18,21,9,5,7,18,18,4,37,25,9,10,19,7,43,51,8,16,9,9,100,52,71,6,28,55,8,45,44,16,11,19,23,16,61,22,5,50,6,35,3,5,29,5,21,12,5,9,7,186,14,10,6,11,8,20,5,58,14,52,5,5,5,13,66,22,5,16,127,41,11,7,4,4,49,76,67,7,5,20,5,4,5,28,232,17,47,35,14,11,8,8,6,52,18,83,4,106,14,12,5,5,34,26,11,16,29,38,10,12,5,16,34,3,20,36,26,46,12,11,25,27,27,13,7,95,18,6,15,3,11,9,29,48,6,6,12,15,19,72,7,94,27,16,44,13,6,16,18,29,25,42,12,18,8,35,29,30,14,7,17,43,52,103,37,11,10,22,93,30,6,14,4,19,72,91,3,34,11,17,81,13,11,22,26,39,2,13,3,10,22,11,36,9,5,9,193,18,19,8,14,8,4,11,3,6,30,9,11,17,13,121,4,8,30,7,47,47,8,9,4,71,8,24,25,9,9,29,16,220,3,3,3,15,34,92,5,227,11,23,8,8,33,11,17,14,21,45,5,7,51,5,4,9,30,32,5,145,9,4,4,4,43,36,19,9,77,31,5,43,21,2,14,3,7,11,18,3,4,15,14,11,15,18,58,6,28,29,13,10,23,6,7,4,7,44,6,8,117,3,15,9,3,19,28,6,6,49,21,112,4,11,35,20,20,17,34,22,3,17,51,215,114,7,3,124,15,5,21,6,10,21,31,8,34,6,50,47,28,11,17,82,8,43,4,10,8,24,38,17,16,12,21,218,17,35,4,5,118,73,26,14,14,8,5,35,35,15,7,15,12,79,12,11,6,21,4,57,18,15,4,67,8,11,38,39,2,9,4,11,2,4,26,5,14,11,58,14,2,10,49,11,5,58,15,4,4,4,3,24,20,30,3,7,7,7,6,6,37,4,6,66,15,10,4,27,20,31,5,16,5,7,37,6,2,19,4,32,5,14,10,15,4,32,3,12,32,6,28,95,11,12,2,6,5,6,6,21,5,4,8,44,23,8,6,9,11,80,4,15,8,4,16,70,55,5,4,10,12,5,3,13,15,11,39,8,11,6,64,21,3,3,6,28,8,7,15,3,3,2,6,37,17,12,15,65,6,6,11,5,25,23,48,8,41,13,16,16,5,8,23,8,37,2,2,101,4,3,16,4,32,6,37,2,11,59,3,3,3,4,7,3,24,13,3,19,9,3,6,5,6,50,7,7,5,9,3,24,6,6,5,3,19,19,3,6,6]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"REAC\" ? \"#3366cc\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade">Treemap</h3>
<div id="bp" class="section level4">
<h4>BP</h4>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/ANOVA.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level4">
<h4>MF</h4>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/ANOVA.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level4">
<h4>CC</h4>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/ANOVA.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level3">
<h3>ScatterPlot</h3>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/ANOVA.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.8                 purrr_0.3.4                
 [5] readr_2.1.2                 tidyr_1.2.0                
 [7] tibble_3.1.6                tidyverse_1.3.1            
 [9] openxlsx_4.2.5              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              GGally_2.1.2               
[15] naniar_0.6.1                visdat_0.5.3               
[17] ggrepel_0.9.1               ggplot2_3.3.5              
[19] UpSetR_1.4.0                ggstatsplot_0.9.1          
[21] datapasta_3.1.0             patchwork_1.1.1            
[23] DT_0.21                     rrvgo_1.2.0                
[25] org.Hs.eg.db_3.12.0         AnnotationDbi_1.52.0       
[27] gprofiler2_0.2.1            enrichR_3.0                
[29] preprocessCore_1.52.1       SummarizedExperiment_1.20.0
[31] Biobase_2.50.0              GenomicRanges_1.42.0       
[33] GenomeInfoDb_1.26.2         IRanges_2.24.1             
[35] S4Vectors_0.28.1            BiocGenerics_0.36.0        
[37] MatrixGenerics_1.2.1        matrixStats_0.61.0         
[39] DEP_1.12.0                  MBQN_2.2.0                 
[41] workflowr_1.7.0            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3         R.methodsS3_1.8.1      bit64_4.0.5           
  [4] knitr_1.37             DelayedArray_0.16.2    R.utils_2.11.0        
  [7] data.table_1.14.2      RCurl_1.98-1.6         doParallel_1.0.17     
 [10] generics_0.1.2         callr_3.7.0            RSQLite_2.2.10        
 [13] correlation_0.8.0      chron_2.3-56           bit_4.0.4             
 [16] tzdb_0.2.0             lubridate_1.8.0        xml2_1.3.3            
 [19] httpuv_1.6.5           assertthat_0.2.1       WRS2_1.1-3            
 [22] xfun_0.30              hms_1.1.1              jquerylib_0.1.4       
 [25] evaluate_0.15          promises_1.2.0.1       fansi_1.0.2           
 [28] readxl_1.3.1           caTools_1.18.2         dbplyr_2.1.1          
 [31] igraph_1.2.11          DBI_1.1.2              htmlwidgets_1.5.4     
 [34] reshape_0.8.8          paletteer_1.4.0        hash_2.2.6.1          
 [37] ellipsis_0.3.2         crosstalk_1.2.0        backports_1.4.1       
 [40] insight_0.16.0         gridBase_0.4-7         vctrs_0.3.8           
 [43] imputeLCMD_2.0         Cairo_1.5-14           cachem_1.0.6          
 [46] withr_2.5.0            treemap_2.4-3          cluster_2.1.0         
 [49] lazyeval_0.2.2         crayon_1.5.1           labeling_0.4.2        
 [52] pkgconfig_2.0.3        slam_0.1-50            nlme_3.1-149          
 [55] statsExpressions_1.3.0 ProtGenerics_1.22.0    wordcloud_2.6         
 [58] rlang_1.0.2            RJSONIO_1.3-1.6        lifecycle_1.0.1       
 [61] sandwich_3.0-1         affyio_1.60.0          BiocFileCache_1.14.0  
 [64] modelr_0.1.8           cellranger_1.1.0       rprojroot_2.0.2       
 [67] graph_1.68.0           datawizard_0.3.0       Matrix_1.2-18         
 [70] mc2d_0.1-21            zoo_1.8-9              reprex_2.0.1          
 [73] whisker_0.4            GlobalOptions_0.1.2    processx_3.5.2        
 [76] pheatmap_1.0.12        png_0.1-7              viridisLite_0.4.0     
 [79] rjson_0.2.21           mzR_2.24.1             parameters_0.16.0     
 [82] bitops_1.0-7           shinydashboard_0.7.2   getPass_0.2-2         
 [85] R.oo_1.24.0            KernSmooth_2.23-17     blob_1.2.2            
 [88] shape_1.4.6            tmvtnorm_1.4-10        scales_1.1.1          
 [91] memoise_2.0.1          magrittr_2.0.3         plyr_1.8.6            
 [94] gplots_3.1.1           zlibbioc_1.36.0        compiler_4.0.3        
 [97] plotrix_3.8-2          pcaMethods_1.82.0      clue_0.3-60           
[100] cli_3.2.0              affy_1.68.0            XVector_0.30.0        
[103] ps_1.6.0               mgcv_1.8-33            MASS_7.3-53           
[106] tidyselect_1.1.2       vsn_3.58.0             stringi_1.6.2         
[109] highr_0.9              yaml_2.2.1             GOSemSim_2.16.1       
[112] norm_1.0-9.5           MALDIquant_1.21        grid_4.0.3            
[115] sass_0.4.1             tools_4.0.3            circlize_0.4.14       
[118] rstudioapi_0.13        foreach_1.5.2          git2r_0.29.0          
[121] gridExtra_2.3          farver_2.1.0           mzID_1.28.0           
[124] digest_0.6.29          BiocManager_1.30.16    shiny_1.7.1           
[127] proto_1.0.0            Rcpp_1.0.8.3           broom_0.7.12          
[130] performance_0.8.0      later_1.3.0            ncdf4_1.19            
[133] httr_1.4.2             MSnbase_2.15.7         ComplexHeatmap_2.6.2  
[136] colorspace_2.0-3       rvest_1.0.2            XML_3.99-0.9          
[139] fs_1.5.2               splines_4.0.3          rematch2_2.1.2        
[142] gmm_1.6-6              plotly_4.10.0          xtable_1.8-4          
[145] jsonlite_1.8.0         zeallot_0.1.0          NLP_0.2-1             
[148] R6_2.5.1               tm_0.7-8               gsubfn_0.7            
[151] pillar_1.7.0           htmltools_0.5.2        mime_0.12             
[154] glue_1.6.2             fastmap_1.1.0          BiocParallel_1.24.1   
[157] codetools_0.2-16       mvtnorm_1.1-3          utf8_1.2.2            
[160] lattice_0.20-41        bslib_0.3.1            sqldf_0.4-11          
[163] curl_4.3.2             gtools_3.9.2           zip_2.2.0             
[166] GO.db_3.12.1           limma_3.46.0           rmarkdown_2.12        
[169] munsell_0.5.0          GetoptLong_1.0.5       GenomeInfoDbData_1.2.4
[172] iterators_1.0.14       impute_1.64.0          haven_2.4.3           
[175] gtable_0.3.0           bayestestR_0.11.5     </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
