<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/header-attrs-2.12/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.21/datatables.js"></script>
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.5.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-12
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>MM_44/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220412code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220412)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220412code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220412)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongef7d0d7">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> ef7d0d7
</a>
</p>
</div>
<div id="strongRepositoryversionstrongef7d0d7"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version ef7d0d7.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups.txt
    Untracked:  output/results_table_OOPS_CDDP_vs_OOPS_NT.xlsx
    Untracked:  output/results_table_OOPS_CDDP_vs_OOPS_NTno_imp.xlsx
    Untracked:  output/results_table_WCE_CDDP_vs_WCE_NT.xlsx

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/WCE_CDDP_vs_WCE_NT.Rmd</code>)
and HTML (<code>docs/WCE_CDDP_vs_WCE_NT.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
ef7d0d7
</td>
<td>
FabioBedin
</td>
<td>
2022-04-12
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= params$n_peptides,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
if(max(expdesign$replicate)&lt;=3){
       threshold&lt;-0
     } else if(max(expdesign$replicate)&lt;6){
       threshold&lt;-1
     } else if (max(expdesign$replicate)&gt;=6){
       threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
     }

data_filt &lt;- filter_missval(data_se, thr = threshold)</code></pre>
<div id="wce_cddp-vs-wce_nt" class="section level1">
<h1><strong>WCE_CDDP</strong> vs <strong>WCE_NT</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the
<strong>proteinGroups</strong> obtained by <a
href="https://www.maxquant.org/">MaxQuant</a> software, I remove
proteins that are marked form the software as potential contaminant,
only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins
identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure
of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
</div>
<div id="quality-control-plots"
class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>1. Quality control
plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3 class="tabset">Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples
<em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3 class="tabset">Distributions</h3>
<p>This plot is useful for checking the distributions of the samples
<em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3 class="tabset">Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and
before User define</em> filters. Notice that the percentages of
missingness are provided in the data. These are accurate to 1 decimal
place. Also the dataset is arranged by columns with most
missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3 class="tabset">Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the
dispersion of data points in a data series around the mean. The
coefficient of variation represents the ratio of the standard deviation
to the mean, and it is a useful statistic for comparing the degree of
variation from one data series to another, even if the means are
drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within
the 2 conditions or in common between all condition.</p>
<div id="wce_cddp" class="section level3">
<h3>WCE_CDDP</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="wce_nt" class="section level3">
<h3>WCE_NT</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        #sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry
proteomics experiment would provide absolute abundance measurements for
a whole proteome, across varying conditions. Unfortunately, this ideal
is not realized. Measurements are made on peptides requiring an
inferential step to obtain protein level estimates. The inference is
complicated by experimental factors that necessitate relative abundance
estimation and result in widespread non-ignorable missing data. Relative
abundance on the log scale takes the form of parameter contrasts. In a
complete-case analysis, contrast estimates may be biased by missing data
and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- mix_imputation_mean(data_filt, cond1 = cond_1, cond2 = cond_2)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics
are used for the differential enrichment analysis (or differential
expression analysis) and False discovery rate (FDR) is ajusted with
<em>Benjamini-Hochberg</em> procedure. Significant proteins are are
define with <strong>FDR = 0.05</strong> and <strong>Fold change =
1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>320</strong> significant proteins in
<strong>WCE_CDDP</strong> vs <strong>WCE_NT</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a
number of functions. These visualizations assist in the determination of
the optimal cutoffs to be used, highlight the most interesting samples
and contrasts, and pinpoint differentially enriched/expressed
proteins.</p>
<div id="visualization-of-the-results"
class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>4.1. Visualization of the
results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data.
This can be very useful to observe batch effects, such as clear
differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between
the two samples (x axis) and their corresponding adjusted p value (y
axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-and-unique" class="section level4">
<h4>Volcano and unique</h4>
<pre class="r"><code>plot_volcano_2(dep, contrast=test, add_names=T, label_size=5, adjusted = F, unique_vec = uni_name)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/volcano-unique-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="correlation" class="section level4">
<h4>Correlation</h4>
<pre class="r"><code>plot_cor(dep, significant = TRUE, lower = 0, upper = 1, pal = &quot;Reds&quot;)</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/correlation-matrix-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cluster-heatmap" class="section level4">
<h4>Cluster heatmap</h4>
<p>The heatmap representation gives an overview of all significant
proteins (rows) in all samples (columns). This allows to see general
trends, for example if one sample or replicate is really different
compared to the others. Additionally, the clustering of samples
(columns) can indicate closer related samples and clustering of proteins
(rows) indicates similarly behaving proteins.</p>
<pre class="r"><code>if(results %&gt;% filter(significant) %&gt;% nrow() &lt;= 100){
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = T, indicate = &quot;condition&quot;, col_limit = 5)
}else {
  plot_heatmap(dep, type = &quot;centered&quot;, kmeans = TRUE, k = 5, show_row_names = F, indicate = &quot;condition&quot;, col_limit = 5)
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/cluster-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by
p.value. Proteins that are UP regulated are colored with a red box in
Fold change column, in the orhter hand proteins that are DOWN regulated
are colored with blue.</p>
<pre class="r"><code>results %&gt;% 
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-32e0541469b5ed490648" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-32e0541469b5ed490648">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320"],["RANBP6","IRF3","UBE2D3","LRRC8C","ANAPC2","FAM129A","NFYB","NSA2","FOS","UFC1","CCNDBP1","TSPAN4","CTDSP1","NRP2","MELK","PIK3CB","TENM3","FGFBP1","CEP135","MTCH1","FZD6","CLSTN1","TOP3A","THADA","TMEM9","NAGLT1","PLIN3.1","MED27","SCRN3","NEB","NUDCD3","BRPF3","C1orf131","COL12A1","RRAGA","DTNB","MIS18BP1","QRSL1","ATF7","TAF6","EEF2K","SCNM1","NPRL3","CACNA2D1","MECOM","NME7","C1orf50","COL1A1","AGRN","SEMA4B","BECN1","PSRC1","HSPG2","COL4A2","ZC3H13.1","ATAD5","SORL1","ARAP3","BIRC6","LAMA3","METTL15","SKP2","HNF1B","TANC2","FTL","SYDE1","BBX","USP6NL","ADCK4","AHDC1","TRIP4","CKAP2","SGCE","AMIGO2","TOP3B","BRWD1","CDCA5","HBEGF","MAFF","TRHDE","PIBF1","OPHN1","RPL7L1","ARL2BP","POLE","FOXP1","DCDC2","BMS1","UTP6","TMOD1","KIAA1217","APP","ARMC10","TIMP2","ASCC2","TMEM214","FTH1","VCAN","NXT1","TNFRSF10D","SURF6","LEPREL1","RPF1","FAM207A","SLBP","GALNT10","KIAA0020","COL5A1","TSPAN14","SRPRB","ENDOG","BICC1","PCCB","VLDLR","PAX8","TSPYL1","ITCH","HOXB7","AXL","NENF","NUSAP1","PNO1","PHKA2","MED18","NCOR2","ANKHD1","JAK1","WDR46","UQCC3","TMEM199","SLC20A1","DST","KIFC3","CDK2","NID2","LAMC1","LIMCH1","IGF2R","CALB1","ITGB4","PLK1","CCNA2","TRIP12","BRPF1","ITM2B","MGA","ECT2","PPHLN1","TRMT1","PTPRS","PLEC.1","DCAF13","USP34","LSR","DDX11","FAM25G","ZNF622","NXF1","FADS3","AP3M1","ARID2","KLHL42","POLR2A","PISD","ADAM10","IL6ST","LARS2","TM9SF2","SERF2","C9orf78","KIF22","NRP1","BAG3","MPDZ","TUBB2A","LAMA5","MARCKSL1","SHPK","UBE2S","PLEKHA7","TPRN","DUS2","CA2","CDK7","DDX10","TGFB1","CHD5","HMG20A","PLEKHA5","UTP20","COL4A1","JUND","FNDC3B","CALML3","SP100","PPFIA3","ORC3","GIT1","RSPRY1","TOR2A","RBM7","UTP14A","ADAMTS1","DDX56","RASAL2","ZPR1","COPRS","DDX52","KIRREL","PHF3","KIAA0391","C5orf22","CKAP2L","MYL6B","MED15","MBLAC2","PTPRF","DHFR","IGFBP3","LOXL2","MRPS6","NUP153","SLC30A1","UNG","NFKB2","CD70","INTS1","FRYL","HECTD1","SETMAR","POLR2D","CTNNA1","HMGXB4","MAN2A1","SDC1","SYNE2","HLTF","TMEM164","DOHH","MYL12A","PLCB4","TAGLN","ABCC4","CSNK1E","LSM14B","MET","NDC80","RBBP9","HAGH","TRMT5","NFS1","TERF2IP","FGD4","POLR2M","ARID1B","MTERF4","TUBA8","YAF2","PPP1R2","SNX18","UTP18","MED1","NXN","PCM1","APLP2","PTDSS2","DAPK1","RBMX2","CCSMST1","UBR7","PAXBP1","SOWAHC","TMX2","VTI1A","COL18A1","GLTSCR2","JUNB","CHD7","ZGPAT","GSTCD","C1orf123","SYF2","HMOX1","IGFBP7","METAP1","QKI","MTG1","COQ7","EIF4G3","SLC7A1","RBPJ","ORC1","PTPRJ","C12orf45","PTGFRN","PHLDA2","SMTN","AK3","PHLDA1","ZNF148","EPB41L1.1","ZNF66","VWA9","UNC13D","NOL10","FUT8","VWA8","MLLT1","SFSWAP","BUD13","CRELD2","ZMYND8","SPARC","KCTD5","SLC35F6","FRMD8","DNAJB4","USP48","RRP36","MAP4K3"],["1.5e-12","1.3e-11","2.0e-11","3.0e-11","1.5e-10","1.1e-10","1.4e-10","1.3e-10","3.4e-10","7.3e-10","1.5e-09","1.5e-09","1.7e-09","1.8e-09","2.1e-09","2.1e-09","2.8e-09","4.5e-09","5.5e-09","5.8e-09","6.6e-09","7.9e-09","9.4e-09","1.1e-08","1.1e-08","1.2e-08","1.7e-08","1.8e-08","2.1e-08","2.2e-08","3.3e-08","4.0e-08","4.2e-08","4.1e-08","4.8e-08","5.5e-08","5.9e-08","7.1e-08","7.7e-08","7.8e-08","8.1e-08","8.5e-08","1.0e-07","1.3e-07","1.3e-07","1.5e-07","1.9e-07","2.0e-07","2.2e-07","2.4e-07","2.7e-07","3.5e-07","3.6e-07","3.7e-07","4.9e-07","7.5e-07","7.6e-07","7.9e-07","8.2e-07","8.7e-07","9.1e-07","1.0e-06","1.2e-06","1.2e-06","1.4e-06","1.4e-06","1.6e-06","1.7e-06","1.8e-06","2.2e-06","2.3e-06","3.2e-06","3.2e-06","3.5e-06","3.6e-06","3.7e-06","3.8e-06","4.1e-06","4.0e-06","4.1e-06","4.5e-06","5.3e-06","5.4e-06","5.7e-06","5.9e-06","6.1e-06","6.5e-06","7.8e-06","8.4e-06","9.5e-06","1.0e-05","1.3e-05","1.3e-05","1.3e-05","1.3e-05","1.4e-05","1.5e-05","1.5e-05","1.6e-05","1.7e-05","1.7e-05","1.9e-05","1.9e-05","2.0e-05","2.0e-05","2.1e-05","2.3e-05","2.6e-05","2.6e-05","2.8e-05","3.0e-05","3.2e-05","3.2e-05","3.3e-05","3.5e-05","3.5e-05","3.7e-05","4.0e-05","4.1e-05","4.1e-05","4.1e-05","4.1e-05","4.5e-05","4.7e-05","4.8e-05","5.3e-05","5.6e-05","5.6e-05","5.7e-05","6.7e-05","7.3e-05","7.4e-05","7.6e-05","7.8e-05","7.8e-05","8.0e-05","9.2e-05","9.3e-05","9.5e-05","9.5e-05","9.6e-05","9.8e-05","1.0e-04","1.0e-04","1.0e-04","1.1e-04","1.1e-04","1.1e-04","1.1e-04","1.2e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.5e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.7e-04","1.8e-04","1.9e-04","1.9e-04","1.9e-04","2.0e-04","2.2e-04","2.3e-04","2.3e-04","2.2e-04","2.3e-04","2.4e-04","2.4e-04","2.5e-04","2.5e-04","2.6e-04","2.7e-04","2.7e-04","2.9e-04","3.2e-04","3.2e-04","3.2e-04","3.2e-04","3.3e-04","3.4e-04","3.5e-04","3.8e-04","3.9e-04","3.9e-04","4.0e-04","4.1e-04","4.1e-04","4.1e-04","4.3e-04","4.6e-04","4.7e-04","4.6e-04","4.7e-04","4.8e-04","4.9e-04","4.9e-04","4.9e-04","5.3e-04","5.6e-04","5.5e-04","5.6e-04","5.9e-04","6.1e-04","6.1e-04","6.4e-04","6.8e-04","6.6e-04","6.6e-04","6.7e-04","6.7e-04","6.6e-04","6.9e-04","7.0e-04","7.4e-04","7.6e-04","7.8e-04","7.9e-04","8.5e-04","9.1e-04","9.0e-04","9.1e-04","9.2e-04","9.2e-04","9.3e-04","9.3e-04","1.0e-03","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.6e-03","2.7e-03","2.8e-03","2.9e-03","2.9e-03","3.1e-03","3.1e-03","3.1e-03","3.1e-03","3.3e-03","3.4e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.6e-03","3.7e-03","3.8e-03","3.9e-03","3.9e-03","3.9e-03","4.0e-03","4.1e-03","4.3e-03","4.3e-03","4.5e-03","4.5e-03","4.6e-03","4.7e-03","4.8e-03","5.1e-03","5.3e-03","5.4e-03","5.7e-03","5.8e-03"],["7.4e-09","3.4e-08","3.4e-08","3.8e-08","9.4e-08","9.4e-08","9.4e-08","9.4e-08","1.9e-07","3.7e-07","6.5e-07","6.5e-07","6.5e-07","6.5e-07","6.7e-07","6.7e-07","8.3e-07","1.3e-06","1.4e-06","1.4e-06","1.6e-06","1.8e-06","2.0e-06","2.3e-06","2.3e-06","2.4e-06","3.1e-06","3.3e-06","3.7e-06","3.7e-06","5.4e-06","6.2e-06","6.2e-06","6.2e-06","6.9e-06","7.6e-06","8.1e-06","9.4e-06","9.8e-06","9.8e-06","9.9e-06","1.0e-05","1.2e-05","1.5e-05","1.5e-05","1.6e-05","2.1e-05","2.1e-05","2.2e-05","2.4e-05","2.6e-05","3.4e-05","3.4e-05","3.4e-05","4.5e-05","6.7e-05","6.7e-05","6.9e-05","7.0e-05","7.3e-05","7.5e-05","8.2e-05","9.8e-05","9.8e-05","1.1e-04","1.1e-04","1.2e-04","1.3e-04","1.3e-04","1.6e-04","1.6e-04","2.2e-04","2.2e-04","2.4e-04","2.4e-04","2.5e-04","2.5e-04","2.6e-04","2.6e-04","2.6e-04","2.8e-04","3.3e-04","3.3e-04","3.4e-04","3.5e-04","3.6e-04","3.8e-04","4.5e-04","4.8e-04","5.3e-04","5.6e-04","6.8e-04","7.0e-04","7.0e-04","7.1e-04","7.2e-04","7.7e-04","7.8e-04","8.1e-04","8.4e-04","8.6e-04","9.4e-04","9.4e-04","9.8e-04","9.8e-04","9.8e-04","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.4e-03","1.4e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.9e-03","1.9e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.6e-03","2.7e-03","2.8e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","3.3e-03","3.3e-03","3.3e-03","3.3e-03","3.3e-03","3.4e-03","3.4e-03","3.4e-03","3.4e-03","3.6e-03","3.6e-03","3.6e-03","3.6e-03","3.9e-03","4.0e-03","4.0e-03","4.2e-03","4.3e-03","4.3e-03","4.3e-03","4.4e-03","4.4e-03","4.6e-03","4.6e-03","4.6e-03","4.6e-03","4.6e-03","4.8e-03","4.8e-03","4.8e-03","4.8e-03","4.8e-03","4.9e-03","4.9e-03","5.2e-03","5.2e-03","5.2e-03","5.5e-03","5.8e-03","6.0e-03","6.0e-03","6.0e-03","6.2e-03","6.3e-03","6.4e-03","6.4e-03","6.6e-03","6.7e-03","6.9e-03","6.9e-03","7.3e-03","7.8e-03","7.8e-03","7.8e-03","7.9e-03","8.1e-03","8.2e-03","8.4e-03","9.1e-03","9.2e-03","9.3e-03","9.3e-03","9.5e-03","9.5e-03","9.6e-03","9.8e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.2e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","3.0e-02","3.1e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.4e-02","3.4e-02","3.4e-02","3.5e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.6e-02","3.7e-02","3.7e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","3.9e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.3e-02","4.3e-02","4.3e-02","4.5e-02","4.7e-02","4.7e-02","4.9e-02","5.0e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[-6.88,5.01,-5.82,-5.84,-3.53,-4.27,-4.02,-4.44,4.3,4.63,4.32,-3.16,-2.76,-3.56,-3.26,3.15,-2.62,-3.14,-2.62,-2.87,-3.03,-3.32,-3.08,-3.06,-3.44,-2.51,-5.44,-3.07,-3.03,-6.01,-2.43,2.61,-2.34,-6.15,-2.66,-2.21,2.27,-2.24,-3.56,2.57,-3.02,1.94,-2.08,-1.76,-3.76,-2.92,2.86,-2.88,-3.55,-2.42,-2.96,-2.11,-4.57,-1.97,-2.12,-1.77,-1.47,-2.28,-2.5,-2.63,-4.29,3.33,-4.71,-2.36,2.57,2.07,-2.27,-2.52,-2.02,-1.42,-1.92,2.53,-1.24,-2.43,-1.22,-2.99,2.13,1.87,1.78,-2.55,-2.18,-1.61,-1.86,1.46,-1.18,-2.43,-2.37,-1.4,-1.63,-1.28,4.1,-2.46,1.22,-4.81,-3.47,1.13,2.04,-4.4,1.14,3.46,-1.99,-1.33,-2.96,-1.55,1.31,-2.54,-1.46,-2.52,-1.26,1.1,1.12,-3.25,-2.48,-1.87,-1.67,1.4,-1.5,1.46,-1.85,1.26,-2.3,-1.07,-2.63,1.05,-1.26,-1.02,-1.45,-1.85,1.2,-1.65,-1.33,-1.22,-1.62,1.17,-2.97,-1.07,-3.03,-1.43,-2.04,-1.32,-1.98,1.28,-1.29,1.06,-2.09,-2.5,-1.13,-1.71,1.3,-2.79,-1.75,-2.03,-2.24,1.19,-1.38,-1.35,1.86,1.36,-1.05,-1.47,-1.22,1.12,-1.48,-1.16,-1.66,-1.17,-1.01,-1.53,-1.26,1.35,2.07,-1.37,1.05,-2.29,1.15,-3.94,1.53,1.22,2.33,-1.46,1.54,-1.31,1.81,-1.13,-1.24,1.31,-1.96,-1.68,-1.47,-2.04,-1.45,-1.91,-1.34,1.58,-1.32,-1.12,-1.88,1.04,-1.07,1.13,-1,-2.37,-1.56,-1.22,-1.44,1.09,1.01,-1.19,-1.68,-1.21,-1.11,1.28,2.36,1.2,-1.65,1.66,-1.05,1.37,-1.1,-2.22,-1.09,-1.15,1.51,1.09,1.1,-1.03,-1.47,-1.53,-1.42,2.34,-1.95,-1.02,-1.1,-1.55,-1.24,-2.06,-1.05,-1.31,1.34,1.36,-1.04,1.06,-1.21,1.02,1.03,-2.07,-1.37,1.05,1.05,1.14,-1.18,1.09,-1.33,1.63,-1.65,1.16,-1.55,-1.15,1.39,1.17,-2.59,-1.03,-1.4,-1,-2.97,-1.05,-2.43,1.05,1.14,1.36,-1.1,1.33,2.01,-1.03,-2.35,-1.45,1.64,-1.92,1.17,-1.63,1.16,-1.19,-1.48,-1.5,-1.23,-1.3,-1.08,-1.93,-1.15,-1.27,-1.32,1.02,-1.45,1.11,-1.24,2.44,1.02,1.14,-1.48,-1.01,-1.5,-1.27,-1.53,-1.22,-2.08,-1.19,-1.51,-1,-1.09,1.92,1.08,-1.92,-1.53,1.74,-1.05,1.43,1.06,-1.08,-1.07,-1.29]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF"
class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a
href="GO:CC" class="uri">GO:CC</a>) and a KEGG ontology.<br />
The enrichment results are visualized with an interactive
<strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="wce_cddp-1" class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>5.1. WCE_CDDP</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in WCE_CDDP were used to generate these gene
ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-4ed42468d86de4a42169" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4ed42468d86de4a42169">{"x":{"data":[{"x":[124.594205293434,84.087208186592,127.684646769836,86.1682873144594],"y":[1.81217497987998,1.69565103890337,1.46366478703859,1.30931438597011],"text":["GO:0044154 (14) <br> histone H3-K14 acetylation <br> 1.541e-02","GO:0006275 (105) <br> regulation of DNA replication <br> 2.015e-02","GO:0045740 (38) <br> positive regulation of DNA replication <br> 3.438e-02","GO:0006880 (5) <br> intracellular sequestering of iron ion <br> 4.906e-02"],"key":["GO:0044154","GO:0006275","GO:0045740","GO:0006880"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[12.7089527527314,16.5190835063999,14.7636000626305,9.90696320350249],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[55.6741209943464,55.058253379197,56.1808475131402,64.6626698584888,65.1109279328064,65.107030036508,66.5336600817274,61.2208274269893,64.4326939768824,59.4745698852999,62.1485267460118,55.6390399276607,55.3427998089812,63.2282440206725],"y":[5.81597399180799,3.44946371863083,3.24280370174795,3.24280370174795,3.14927334732391,3.05241678820367,2.69724298935487,2.30471012848074,1.97057290600104,1.96871762427646,1.96871762427646,1.59428619470826,1.54534143136469,1.50580841046848],"text":["GO:0005829 (5373) <br> cytosol <br> 1.528e-06","GO:0005654 (4161) <br> nucleoplasm <br> 3.553e-04","GO:0008043 (2) <br> intracellular ferritin complex <br> 5.717e-04","GO:0070288 (2) <br> ferritin complex <br> 5.717e-04","GO:0070776 (9) <br> MOZ/MORF histone acetyltransferase complex <br> 7.091e-04","GO:0070775 (10) <br> H3 histone acetyltransferase complex <br> 8.863e-04","GO:0097124 (5) <br> cyclin A2-CDK2 complex <br> 2.008e-03","GO:0035976 (4) <br> transcription factor AP-1 complex <br> 4.958e-03","GO:0070013 (6569) <br> intracellular organelle lumen <br> 1.070e-02","GO:0031974 (6570) <br> membrane-enclosed lumen <br> 1.075e-02","GO:0043233 (6570) <br> organelle lumen <br> 1.075e-02","GO:0005819 (397) <br> spindle <br> 2.545e-02","GO:0005737 (12193) <br> cytoplasm <br> 2.849e-02","GO:0044754 (11) <br> autolysosome <br> 3.120e-02"],"key":["GO:0005829","GO:0005654","GO:0008043","GO:0070288","GO:0070776","GO:0070775","GO:0097124","GO:0035976","GO:0070013","GO:0031974","GO:0043233","GO:0005819","GO:0005737","GO:0044754"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.7668090255052,21.4732464497629,3.77952755905512,3.77952755905512,11.6300255014221,11.9003352388478,9.90696320350249,9.10888491199513,21.9942840280337,21.9944552409408,21.9944552409408,18.5034098486149,22.6771653543307,12.1373322529182],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[30.5246374536771],"y":[1.30894113663473],"text":"GO:0046899 (2) <br> nucleoside triphosphate adenylate kinase activity <br> 4.910e-02","key":["GO:0046899"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":3.77952755905512,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData32e05414","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isSimpleKey":true,"_isNestedKey":false,"frame":null},{"x":[2,49.9528814359943],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8499055266887,71.2657061880019],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.1627302786963,193.967406707605],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.864430798299,200.003897024091],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9764407179972,62.5578058573453,134.565068493151,198.934163911195],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"23e836622a89":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"23e81039517b":{"x":{},"xend":{},"y":{},"yend":{}},"23e857bb12":{"x":{},"xend":{},"y":{},"yend":{}},"23e843618d6":{"x":{},"xend":{},"y":{},"yend":{}},"23e82b566132":{"x":{},"xend":{},"y":{},"yend":{}},"23e833262e50":{"x":{},"y":{}},"23e8d332585":{"yintercept":{}}},"cur_data":"23e836622a89","visdat":{"23e836622a89":["function (y) ","x"],"23e81039517b":["function (y) ","x"],"23e857bb12":["function (y) ","x"],"23e843618d6":["function (y) ","x"],"23e82b566132":["function (y) ","x"],"23e833262e50":["function (y) ","x"],"23e8d332585":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData32e05414"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-7d58abce2053a2454929" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7d58abce2053a2454929">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19"],["GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF"],[469,311,599,2775,2890,2889,3255,1892,2716,1444,2130,12685,2291,460,384,2407,13478,2825,7320],["cytosol","nucleoplasm","intracellular ferritin complex","ferritin complex","MOZ/MORF histone acetyltransferase complex","H3 histone acetyltransferase complex","cyclin A2-CDK2 complex","transcription factor AP-1 complex","intracellular organelle lumen","membrane-enclosed lumen","organelle lumen","histone H3-K14 acetylation","regulation of DNA replication","spindle","cytoplasm","autolysosome","positive regulation of DNA replication","intracellular sequestering of iron ion","nucleoside triphosphate adenylate kinase activity"],["1.5e-06","3.6e-04","5.7e-04","5.7e-04","7.1e-04","8.9e-04","2.0e-03","5.0e-03","1.1e-02","1.1e-02","1.1e-02","1.5e-02","2.0e-02","2.5e-02","2.8e-02","3.1e-02","3.4e-02","4.9e-02","4.9e-02"],[5373,4161,2,2,9,10,5,4,6569,6570,6570,14,105,397,12193,11,38,5,2],[35,38,2,2,2,2,2,2,47,47,47,2,4,6,46,2,3,2,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4 class="tabset tabset-fade">Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<pre><code>[1] &quot;NOT enough data.&quot;</code></pre>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="wce_nt-1" class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>5.2. WCE_NT</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in WCE_NT were used to generate these gene
ontologies.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-a56e939b13f92b785645" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-a56e939b13f92b785645">{"x":{"data":[{"x":[134.239656307426,109.508330192358,87.5050104246813,133.791483836069,84.3678031251809,97.2712731483436,114.208295413721,113.565265346122,120.53727013967,90.837075320424,113.919906171283,134.937246501973,109.504433040433,105.856698838778,75.9616464232896,120.591830266618,105.338377632773,122.43518312707,126.644107205903,115.295600800753,87.9336971364143,87.3452271957627,77.7738220683426,115.159200483384,87.0724265610235,133.464123074382,104.808364970994,113.561368194197,76.9632144679748,104.582330159353,154.017702326016,133.027642058799,134.941143653898,133.666774974474,133.062716426123,105.342274784698,189.17780699197,97.8480516332207,89.0677683465443,136.005066129381,107.852040624299,133.031539210724,88.2571607461765,117.290942586274,118.717300190767,91.5619455784453,91.5658427303701,91.2696591840819,134.987909476996,152.232806744437,137.688635760914,134.290319282449,133.047127818423,141.846896864724,136.390884169941,77.1424834565177,105.252640290427,138.43299177856,118.721197342692,134.984012325072,82.0801749452967,134.968423717372,134.286422130524,135.989477521682,172.470716690158,133.783689532219,138.444683234334,175.171442974076,172.069310041899,93.8378823025549,104.983736807612,87.4738332092826,134.925555046199,157.435504564105,77.134689152668,132.910727501054,120.213806529908,133.066613578048,87.1152952321968,89.4341006274798,89.4379977794046,105.307200417375,142.692578832416,89.075562650394,173.191689796255,82.3685641877352,135.89984302741,88.0038458710615,122.462463190544,86.3358648472277,119.859165704747,124.668251180006,138.452477538184,147.641961776969,122.454668886694],"y":[9.85740019574652,9.60791750557027,9.53327389624989,8.78873521776943,7.65561204952909,7.46102902465761,7.31072721475479,6.56228791311579,6.42808620468474,6.38011666531469,5.97633099612052,5.82422555057278,5.70966754228149,5.57058006197097,5.31580332030015,5.28278921175905,4.81470797859509,4.80305568705016,4.77986545091191,4.7063384086921,4.6579430733353,4.56171471658301,4.30999930286852,4.14314465951536,4.05060421414109,4.04196977210131,4.00842089548526,3.89861933710526,3.85667812685003,3.64439519932805,3.55986621903763,3.52615934921563,3.51766344459207,3.45307672948433,3.34747238811469,3.34062962793045,3.11958208486257,3.01417240387212,2.96104364249795,2.87127877755907,2.85150084263697,2.82801169212852,2.64198678940129,2.61200892815544,2.59091275823409,2.56615004742964,2.47859216815622,2.47738702461677,2.34849769426538,2.33234863444602,2.31411806001244,2.31411806001244,2.28480532914378,2.27403429328178,2.26568956723767,2.22008321456009,2.19879663457903,2.17362818304693,2.1036152885605,2.05464758345057,2.02783543503097,2.00592482052458,1.94640602936463,1.9024819425666,1.85101829440886,1.83653590515436,1.83440178942109,1.829909339834,1.81951397753839,1.77390460642052,1.72235988452334,1.72147672893784,1.7049663228131,1.69181178894135,1.68419462390087,1.64408586659167,1.62571292509331,1.61347732715764,1.59949994958389,1.59103753244423,1.58975736440341,1.58867035276241,1.57687702147761,1.54969826400359,1.53289801429032,1.52748450982008,1.5068375617766,1.46646965626245,1.44738564979004,1.44492890596015,1.3885155082818,1.358416546999,1.35179338545526,1.33333286007895,1.32809768985542],"text":["GO:0048856 (5718) <br> anatomical structure development <br> 1.389e-10","GO:0032502 (6326) <br> developmental process <br> 2.467e-10","GO:0007275 (5114) <br> multicellular organism development <br> 2.929e-10","GO:0048731 (4784) <br> system development <br> 1.627e-09","GO:0006364 (225) <br> rRNA processing <br> 2.210e-08","GO:0016072 (263) <br> rRNA metabolic process <br> 3.459e-08","GO:0034660 (536) <br> ncRNA metabolic process <br> 4.890e-08","GO:0034330 (687) <br> cell junction organization <br> 2.740e-07","GO:0042254 (302) <br> ribosome biogenesis <br> 3.732e-07","GO:0009653 (2685) <br> anatomical structure morphogenesis <br> 4.168e-07","GO:0034470 (399) <br> ncRNA processing <br> 1.056e-06","GO:0050793 (2421) <br> regulation of developmental process <br> 1.499e-06","GO:0032501 (7474) <br> multicellular organismal process <br> 1.951e-06","GO:0030490 (50) <br> maturation of SSU-rRNA <br> 2.688e-06","GO:0000462 (37) <br> maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA) <br> 4.833e-06","GO:0042274 (73) <br> ribosomal small subunit biogenesis <br> 5.214e-06","GO:0030198 (323) <br> extracellular matrix organization <br> 1.532e-05","GO:0043062 (324) <br> extracellular structure organization <br> 1.574e-05","GO:0045229 (326) <br> external encapsulating structure organization <br> 1.660e-05","GO:0035295 (1047) <br> tube development <br> 1.966e-05","GO:0007399 (2428) <br> nervous system development <br> 2.198e-05","GO:0007229 (112) <br> integrin-mediated signaling pathway <br> 2.743e-05","GO:0001944 (724) <br> vasculature development <br> 4.898e-05","GO:0035239 (849) <br> tube morphogenesis <br> 7.192e-05","GO:0007155 (1480) <br> cell adhesion <br> 8.900e-05","GO:0048646 (1123) <br> anatomical structure formation involved in morphogenesis <br> 9.079e-05","GO:0022610 (1487) <br> biological adhesion <br> 9.808e-05","GO:0034329 (420) <br> cell junction assembly <br> 1.263e-04","GO:0001568 (693) <br> blood vessel development <br> 1.391e-04","GO:0022008 (1639) <br> neurogenesis <br> 2.268e-04","GO:0072359 (1089) <br> circulatory system development <br> 2.755e-04","GO:0048513 (3494) <br> animal organ development <br> 2.977e-04","GO:0050794 (11067) <br> regulation of cellular process <br> 3.036e-04","GO:0048699 (1519) <br> generation of neurons <br> 3.523e-04","GO:0048522 (5561) <br> positive regulation of cellular process <br> 4.493e-04","GO:0030199 (71) <br> collagen fibril organization <br> 4.564e-04","GO:2000026 (1315) <br> regulation of multicellular organismal development <br> 7.593e-04","GO:0016477 (1454) <br> cell migration <br> 9.679e-04","GO:0008283 (1940) <br> cell population proliferation <br> 1.094e-03","GO:0051128 (2246) <br> regulation of cellular component organization <br> 1.345e-03","GO:0031581 (5) <br> hemidesmosome assembly <br> 1.408e-03","GO:0048514 (608) <br> blood vessel morphogenesis <br> 1.486e-03","GO:0007492 (81) <br> endoderm development <br> 2.280e-03","GO:0035987 (46) <br> endodermal cell differentiation <br> 2.443e-03","GO:0038063 (12) <br> collagen-activated tyrosine kinase receptor signaling pathway <br> 2.565e-03","GO:0009887 (1066) <br> animal organ morphogenesis <br> 2.716e-03","GO:0009888 (1912) <br> tissue development <br> 3.322e-03","GO:0009790 (1088) <br> embryo development <br> 3.331e-03","GO:0050808 (422) <br> synapse organization <br> 4.482e-03","GO:0071840 (7994) <br> cellular component organization or biogenesis <br> 4.652e-03","GO:0051674 (1645) <br> localization of cell <br> 4.852e-03","GO:0048870 (1645) <br> cell motility <br> 4.852e-03","GO:0048518 (6094) <br> positive regulation of biological process <br> 5.190e-03","GO:0060322 (812) <br> head development <br> 5.321e-03","GO:0051239 (2623) <br> regulation of multicellular organismal process <br> 5.424e-03","GO:0001706 (55) <br> endoderm formation <br> 6.024e-03","GO:0030154 (4128) <br> cell differentiation <br> 6.327e-03","GO:0051960 (438) <br> regulation of nervous system development <br> 6.705e-03","GO:0038065 (17) <br> collagen-activated signaling pathway <br> 7.877e-03","GO:0050807 (209) <br> regulation of synapse organization <br> 8.818e-03","GO:0003205 (167) <br> cardiac chamber development <br> 9.379e-03","GO:0050803 (215) <br> regulation of synapse structure or activity <br> 9.865e-03","GO:0048869 (4210) <br> cellular developmental process <br> 1.131e-02","GO:0051124 (7) <br> synaptic assembly at neuromuscular junction <br> 1.252e-02","GO:1901701 (1164) <br> cellular response to oxygen-containing compound <br> 1.409e-02","GO:0048729 (638) <br> tissue morphogenesis <br> 1.457e-02","GO:0051963 (103) <br> regulation of synapse assembly <br> 1.464e-02","GO:1902455 (7) <br> negative regulation of stem cell population maintenance <br> 1.479e-02","GO:1901576 (5869) <br> organic substance biosynthetic process <br> 1.515e-02","GO:0010646 (3313) <br> regulation of cell communication <br> 1.683e-02","GO:0023051 (3327) <br> regulation of signaling <br> 1.895e-02","GO:0007267 (1646) <br> cell-cell signaling <br> 1.899e-02","GO:0050789 (12142) <br> regulation of biological process <br> 1.973e-02","GO:0090068 (235) <br> positive regulation of cell cycle process <br> 2.033e-02","GO:0001704 (121) <br> formation of primary germ layer <br> 2.069e-02","GO:0048468 (2079) <br> cell development <br> 2.269e-02","GO:0042127 (1648) <br> regulation of cell population proliferation <br> 2.367e-02","GO:0048523 (4818) <br> negative regulation of cellular process <br> 2.435e-02","GO:0007166 (2783) <br> cell surface receptor signaling pathway <br> 2.515e-02","GO:0009058 (5956) <br> biosynthetic process <br> 2.564e-02","GO:0009059 (4880) <br> macromolecule biosynthetic process <br> 2.572e-02","GO:0030182 (1364) <br> neuron differentiation <br> 2.578e-02","GO:0060548 (971) <br> negative regulation of cell death <br> 2.649e-02","GO:0008285 (707) <br> negative regulation of cell population proliferation <br> 2.820e-02","GO:1901888 (200) <br> regulation of cell junction assembly <br> 2.932e-02","GO:0003279 (106) <br> cardiac septum development <br> 2.968e-02","GO:0051094 (1271) <br> positive regulation of developmental process <br> 3.113e-02","GO:0007420 (765) <br> brain development <br> 3.416e-02","GO:0043069 (883) <br> negative regulation of programmed cell death <br> 3.570e-02","GO:0006928 (2074) <br> movement of cell or subcellular component <br> 3.590e-02","GO:0040011 (1821) <br> locomotion <br> 4.088e-02","GO:0044249 (5794) <br> cellular biosynthetic process <br> 4.381e-02","GO:0051965 (63) <br> positive regulation of synapse assembly <br> 4.448e-02","GO:0062197 (322) <br> cellular response to chemical stress <br> 4.642e-02","GO:0043067 (1439) <br> regulation of programmed cell death <br> 4.698e-02"],"key":["GO:0048856","GO:0032502","GO:0007275","GO:0048731","GO:0006364","GO:0016072","GO:0034660","GO:0034330","GO:0042254","GO:0009653","GO:0034470","GO:0050793","GO:0032501","GO:0030490","GO:0000462","GO:0042274","GO:0030198","GO:0043062","GO:0045229","GO:0035295","GO:0007399","GO:0007229","GO:0001944","GO:0035239","GO:0007155","GO:0048646","GO:0022610","GO:0034329","GO:0001568","GO:0022008","GO:0072359","GO:0048513","GO:0050794","GO:0048699","GO:0048522","GO:0030199","GO:2000026","GO:0016477","GO:0008283","GO:0051128","GO:0031581","GO:0048514","GO:0007492","GO:0035987","GO:0038063","GO:0009887","GO:0009888","GO:0009790","GO:0050808","GO:0071840","GO:0051674","GO:0048870","GO:0048518","GO:0060322","GO:0051239","GO:0001706","GO:0030154","GO:0051960","GO:0038065","GO:0050807","GO:0003205","GO:0050803","GO:0048869","GO:0051124","GO:1901701","GO:0048729","GO:0051963","GO:1902455","GO:1901576","GO:0010646","GO:0023051","GO:0007267","GO:0050789","GO:0090068","GO:0001704","GO:0048468","GO:0042127","GO:0048523","GO:0007166","GO:0009058","GO:0009059","GO:0030182","GO:0060548","GO:0008285","GO:1901888","GO:0003279","GO:0051094","GO:0007420","GO:0043069","GO:0006928","GO:0040011","GO:0044249","GO:0051965","GO:0062197","GO:0043067"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[21.4602191304161,21.5721147241081,21.3357696591593,21.2609864600921,17.4001797647803,17.623374708422,18.5990879901373,18.9244578120153,17.8181742307197,20.5995413799917,18.2025889796692,20.4782485566766,21.7552539319502,15.024106382592,14.4852668565195,15.6667434572733,17.9119114820817,17.9162068147747,17.9247539681855,19.4613624114386,20.4816440349838,16.3540933271933,18.9923339389664,19.1965853459477,19.8890155572054,19.5488822697704,19.894767362824,18.2722688771406,18.9357305182298,20.0129498359015,19.5105447089455,20.9043094531098,22.1791467424615,19.9206956342843,21.4292648753663,15.62075611391,19.7442505555371,19.8673925248959,20.2156717380333,20.3897665416323,9.77892631171347,18.7651998057394,15.8373349155837,14.8775112833583,12.1689282166855,19.4838705477651,20.1982900751414,19.5093976577707,18.2787054385069,21.8285943750956,20.0173702355595,20.0173702355595,21.5308228872123,19.1397149364531,20.5722397264616,15.1893666851417,21.0944948337105,18.3290273856849,12.9481625134338,17.2933930624918,16.9633150114954,17.3344661617796,21.1167918583733,10.7944929897153,19.5934806177015,18.8281912084712,16.222563454708,10.7944929897153,21.4891497858458,20.8431930212115,20.8480460032252,20.0181052842958,22.2778367258612,17.4627400374017,16.4742428340656,20.2981546348135,20.0195739433602,21.2689411768662,20.6413494848875,21.5054614715639,21.2832942259864,19.7891955860899,19.3666959801624,18.961626677929,17.2292686365135,16.267798176046,19.7023270633931,19.0632852135909,19.2465271966704,20.2952914186446,20.13985350238,21.4748804145049,15.4207837047603,17.9076015185241,19.8547276341673],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData939b13f9","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[54.8867459420668,58.2740178253884,64.2728802286474,64.4326939768824,59.4745698852999,62.1485267460118,58.5117894995917,58.1064082845567,55.5260009350067,54.7970943272033,67.8394553416961,59.5447320186714,64.5223455917459,58.2818136179853,69.6987518760395,64.428796080584,62.1368330571166,59.5018551593888,54.9257249050509,55.058253379197,57.026691009896,62.1290372645198,62.1446288497134,62.1212414719229,55.3427998089812,55.5065114535146,62.140730953415,59.4979572630904,62.1719141238023,54.7815027420096,55.9469737352354,62.1251393682214,62.1329351608182,54.9062354235589,62.187505708996,62.1836078126976,55.0153765199145,54.8204817049938,66.0386272518289,54.9491122828414,67.8316595490993,67.8433532379945,68.2448365567311,67.8589448231882],"y":[13.4185653118216,8.33181562312951,8.11505224305031,6.45387767044995,6.44977612112436,6.44977612112436,6.3011873759671,6.29006313154711,5.81520526764684,5.18644597234887,4.67916474340618,4.53892001775491,4.31402329842653,4.24650837576879,4.22230654277127,4.21816848463722,4.21816848463722,4.21716151624043,3.72792865387988,3.13355725809858,3.00102080223972,2.89495967628644,2.89495967628644,2.88396054504716,2.66604123479432,2.55139438766739,2.33127071818483,2.32902200962254,2.30226736598326,2.13159609618781,2.12945951669403,2.10304801275218,1.9217322145156,1.76582094135919,1.7558501165408,1.7558501165408,1.69134245912971,1.62522787304263,1.62366139974941,1.52363460598568,1.50078350235214,1.50078350235214,1.44804149885136,1.3921333453329],"text":["GO:0005604 (97) <br> basement membrane <br> 3.814e-14","GO:0030684 (79) <br> preribosome <br> 4.658e-09","GO:0062023 (421) <br> collagen-containing extracellular matrix <br> 7.673e-09","GO:0070013 (6569) <br> intracellular organelle lumen <br> 3.517e-07","GO:0031974 (6570) <br> membrane-enclosed lumen <br> 3.550e-07","GO:0043233 (6570) <br> organelle lumen <br> 3.550e-07","GO:0031012 (560) <br> extracellular matrix <br> 4.998e-07","GO:0030312 (561) <br> external encapsulating structure <br> 5.128e-07","GO:0005788 (311) <br> endoplasmic reticulum lumen <br> 1.530e-06","GO:0005581 (93) <br> collagen trimer <br> 6.510e-06","GO:0098644 (21) <br> complex of collagen trimers <br> 2.093e-05","GO:0032040 (39) <br> small-subunit processome <br> 2.891e-05","GO:0070062 (2110) <br> extracellular exosome <br> 4.853e-05","GO:0030686 (27) <br> 90S preribosome <br> 5.669e-05","GO:1903561 (2132) <br> extracellular vesicle <br> 5.994e-05","GO:0065010 (2133) <br> extracellular membrane-bounded organelle <br> 6.051e-05","GO:0043230 (2133) <br> extracellular organelle <br> 6.051e-05","GO:0031982 (3942) <br> vesicle <br> 6.065e-05","GO:0005615 (3353) <br> extracellular space <br> 1.871e-04","GO:0005654 (4161) <br> nucleoplasm <br> 7.353e-04","GO:0012505 (4674) <br> endomembrane system <br> 9.977e-04","GO:0043228 (6362) <br> non-membrane-bounded organelle <br> 1.274e-03","GO:0043232 (6362) <br> intracellular non-membrane-bounded organelle <br> 1.274e-03","GO:0043226 (16674) <br> organelle <br> 1.306e-03","GO:0005737 (12193) <br> cytoplasm <br> 2.158e-03","GO:0005783 (2007) <br> endoplasmic reticulum <br> 2.809e-03","GO:0043231 (14514) <br> intracellular membrane-bounded organelle <br> 4.664e-03","GO:0031981 (5442) <br> nuclear lumen <br> 4.688e-03","GO:0043256 (12) <br> laminin complex <br> 4.986e-03","GO:0005576 (4303) <br> extracellular region <br> 7.386e-03","GO:0005915 (9) <br> zonula adherens <br> 7.422e-03","GO:0043227 (15557) <br> membrane-bounded organelle <br> 7.888e-03","GO:0043229 (15870) <br> intracellular organelle <br> 1.197e-02","GO:0005610 (3) <br> laminin-5 complex <br> 1.715e-02","GO:0043260 (3) <br> laminin-11 complex <br> 1.754e-02","GO:0043259 (3) <br> laminin-10 complex <br> 1.754e-02","GO:0005641 (10) <br> nuclear envelope lumen <br> 2.035e-02","GO:0005587 (7) <br> collagen type IV trimer <br> 2.370e-02","GO:0071944 (6181) <br> cell periphery <br> 2.379e-02","GO:0005622 (17758) <br> intracellular anatomical structure <br> 2.995e-02","GO:0098642 (8) <br> network-forming collagen trimer <br> 3.157e-02","GO:0098645 (8) <br> collagen network <br> 3.157e-02","GO:0098978 (332) <br> glutamatergic synapse <br> 3.564e-02","GO:0098651 (9) <br> basement membrane collagen trimer <br> 4.054e-02"],"key":["GO:0005604","GO:0030684","GO:0062023","GO:0070013","GO:0031974","GO:0043233","GO:0031012","GO:0030312","GO:0005788","GO:0005581","GO:0098644","GO:0032040","GO:0070062","GO:0030686","GO:1903561","GO:0065010","GO:0043230","GO:0031982","GO:0005615","GO:0005654","GO:0012505","GO:0043228","GO:0043232","GO:0043226","GO:0005737","GO:0005783","GO:0043231","GO:0031981","GO:0043256","GO:0005576","GO:0005915","GO:0043227","GO:0043229","GO:0005610","GO:0043260","GO:0043259","GO:0005641","GO:0005587","GO:0071944","GO:0005622","GO:0098642","GO:0098645","GO:0098978","GO:0098651"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[16.1274657222001,15.7965422855522,18.2754913372617,21.6136744323874,21.6138420676856,21.6138420676856,18.6570267009338,18.6593818172362,17.8591980487043,16.0603010299089,13.3901618940841,14.5814139974581,20.3157434393389,13.8907176927077,20.3280586048311,20.3286151448313,20.3286151448313,21.042118612013,20.8570011251474,21.1035242849928,21.2348321994975,21.5783776479263,21.5783776479263,22.611590191448,22.2822864937662,20.2561942705653,22.4663374189607,21.4051772962856,12.1689282166855,21.1415267205783,11.4659840632313,22.5391342674345,22.5599778805315,7.77039980235131,7.77039980235131,7.77039980235131,11.7306454967647,10.7944929897153,21.5464999755733,22.6771653543307,11.1588894645275,11.1588894645275,17.9500540216875,11.4659840632313],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData939b13f9","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[8.594300665613,20.3837566428467,19.8576156322924,9.30361669465648,42.1854811838863,48.4134318125208,16.9813781079293,34.6908947891028,5.90123815973914,9.62319864180794,28.3460239359007,9.33869276202677,9.32310339875108,9.92719122568372,23.0495377629881,8.58260864315624,20.9917418105982,9.38156351103489,6.98469890739896,48.7447057821291,48.4017397900641,6.98080156658003,19.5770070933302,17.4880324143889,9.86873111339991,8.5163538492346,17.5659792307673,15.7926891581586,16.7163589322427,16.7202562730617,23.0261537180746,28.3499212767196,16.5643626403048,26.6740647245839,5.37899449000383,15.6562822294964,13.4971554158146],"y":[8.22503614824662,7.08138115924684,6.59768797119709,5.19686196930582,3.46526994865766,3.41888487883269,3.32285315870537,3.25452251455411,3.23324662298227,3.05455213545042,3.02417577512198,2.7945905798668,2.60089502527691,2.5850405428731,2.50697967656088,2.50414323829725,2.45843337175241,2.35856347133661,2.33227076076123,2.20309157850576,2.12936554821041,1.9548832823936,1.93549939117779,1.85911156414116,1.70656655547398,1.70221700053229,1.69897775366721,1.58233054648277,1.56823125796393,1.56823125796393,1.5664551305943,1.51640174147013,1.5026560369699,1.3956384516413,1.3811669315856,1.32992562811685,1.32992562811685],"text":["GO:0005201 (173) <br> extracellular matrix structural constituent <br> 5.956e-09","GO:0030020 (41) <br> extracellular matrix structural constituent conferring tensile strength <br> 8.291e-08","GO:0019838 (137) <br> growth factor binding <br> 2.525e-07","GO:0005515 (14799) <br> protein binding <br> 6.355e-06","GO:0097367 (2288) <br> carbohydrate derivative binding <br> 3.426e-04","GO:0140657 (550) <br> ATP-dependent activity <br> 3.812e-04","GO:0016887 (316) <br> ATP hydrolysis activity <br> 4.755e-04","GO:0048407 (11) <br> platelet-derived growth factor binding <br> 5.565e-04","GO:0004386 (160) <br> helicase activity <br> 5.845e-04","GO:0008094 (108) <br> ATP-dependent activity, acting on DNA <br> 8.820e-04","GO:0043167 (6060) <br> ion binding <br> 9.459e-04","GO:0005524 (1502) <br> ATP binding <br> 1.605e-03","GO:0005520 (29) <br> insulin-like growth factor binding <br> 2.507e-03","GO:0008201 (170) <br> heparin binding <br> 2.600e-03","GO:0032559 (1565) <br> adenyl ribonucleotide binding <br> 3.112e-03","GO:0005198 (980) <br> structural molecule activity <br> 3.132e-03","GO:0030554 (1576) <br> adenyl nucleotide binding <br> 3.480e-03","GO:0005539 (242) <br> glycosaminoglycan binding <br> 4.380e-03","GO:0004714 (126) <br> transmembrane receptor protein tyrosine kinase activity <br> 4.653e-03","GO:1901681 (265) <br> sulfur compound binding <br> 6.265e-03","GO:0140640 (607) <br> catalytic activity, acting on a nucleic acid <br> 7.424e-03","GO:0004713 (144) <br> protein tyrosine kinase activity <br> 1.109e-02","GO:0019199 (145) <br> transmembrane receptor protein kinase activity <br> 1.160e-02","GO:0017111 (656) <br> nucleoside-triphosphatase activity <br> 1.383e-02","GO:0008186 (78) <br> ATP-dependent activity, acting on RNA <br> 1.965e-02","GO:0005178 (147) <br> integrin binding <br> 1.985e-02","GO:0017154 (12) <br> semaphorin receptor activity <br> 2.000e-02","GO:0016462 (712) <br> pyrophosphatase activity <br> 2.616e-02","GO:0016817 (715) <br> hydrolase activity, acting on acid anhydrides <br> 2.703e-02","GO:0016818 (715) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 2.703e-02","GO:0032553 (1934) <br> ribonucleotide binding <br> 2.714e-02","GO:0043168 (2410) <br> anion binding <br> 3.045e-02","GO:0016772 (954) <br> transferase activity, transferring phosphorus-containing groups <br> 3.143e-02","GO:0035639 (1848) <br> purine ribonucleoside triphosphate binding <br> 4.021e-02","GO:0004222 (110) <br> metalloendopeptidase activity <br> 4.158e-02","GO:0016404 (2) <br> 15-hydroxyprostaglandin dehydrogenase (NAD+) activity <br> 4.678e-02","GO:0015132 (2) <br> prostaglandin transmembrane transporter activity <br> 4.678e-02"],"key":["GO:0005201","GO:0030020","GO:0019838","GO:0005515","GO:0097367","GO:0140657","GO:0016887","GO:0048407","GO:0004386","GO:0008094","GO:0043167","GO:0005524","GO:0005520","GO:0008201","GO:0032559","GO:0005198","GO:0030554","GO:0005539","GO:0004714","GO:1901681","GO:0140640","GO:0004713","GO:0019199","GO:0017111","GO:0008186","GO:0005178","GO:0017154","GO:0016462","GO:0016817","GO:0016818","GO:0032553","GO:0043168","GO:0016772","GO:0035639","GO:0004222","GO:0016404","GO:0015132"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[17.0157952573456,14.671965682599,16.6649501000042,22.48676457789,20.4116592941492,18.6332205816173,17.8814290290267,11.9626902987181,16.8993690490748,16.2971610434966,21.5246314898742,19.906995199969,14.0285832754376,16.9898127358403,19.9569562634249,19.3783169237459,19.9654575586114,17.5048066811551,16.5367347658025,17.6341184370492,18.7630422549985,16.7406840604815,16.751166604946,18.8644595290412,15.7757036870604,16.7718916003282,12.1689282166855,18.9707406860299,18.9761758161221,18.9761758161221,20.2119698514182,20.4728915150577,19.3444242069535,20.1575130581732,16.3259182730869,3.77952755905512,3.77952755905512],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData939b13f9","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[199.008342484717,199.359714675084],"y":[4.44099243564838,3.05487656207781],"text":["KEGG:04512 (88) <br> ECM-receptor interaction <br> 3.622e-05","KEGG:04974 (103) <br> Protein digestion and absorption <br> 8.813e-04"],"key":["KEGG:04512","KEGG:04974"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":[15.9715998038261,16.222563454708],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedData939b13f9","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,49.9528814359943],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.8499055266887,71.2657061880019],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.1627302786963,193.967406707605],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.864430798299,200.003897024091],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[25.9764407179972,62.5578058573453,134.565068493151,198.934163911195],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"23e86913b1f":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"23e8d035cd":{"x":{},"xend":{},"y":{},"yend":{}},"23e85b551d61":{"x":{},"xend":{},"y":{},"yend":{}},"23e86054134":{"x":{},"xend":{},"y":{},"yend":{}},"23e814b640eb":{"x":{},"xend":{},"y":{},"yend":{}},"23e821772a51":{"x":{},"y":{}},"23e87a481352":{"yintercept":{}}},"cur_data":"23e86913b1f","visdat":{"23e86913b1f":["function (y) ","x"],"23e8d035cd":["function (y) ","x"],"23e85b551d61":["function (y) ","x"],"23e86054134":["function (y) ","x"],"23e814b640eb":["function (y) ","x"],"23e821772a51":["function (y) ","x"],"23e87a481352":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData939b13f9"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-7827ab1876f7c57926b2" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7827ab1876f7c57926b2">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178"],["GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","KEGG","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","GO:BP","KEGG","GO:MF","GO:MF","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP"],[267,15160,8814,3168,15045,1136,1693,2675,2363,5674,10020,4718,4583,9855,2716,1444,2130,11644,4023,1197,1093,9946,15339,431,8813,7877,206,11658,1875,244,7744,12131,13211,10299,3590,3278,3127,1462,294,2739,671,1138,4067,2715,2127,1451,10264,3057,14961,7608,9854,463,277,7550,20235,14849,15340,10312,15013,11910,14858,7745,3845,8389,1002,311,29257,384,1957,6761,5822,816,3569,2125,2129,2123,15613,8389,14850,1884,384,3361,10811,1880,11177,2035,4209,426,5402,1690,4210,4134,4874,1895,15352,19777,1280,2128,1450,16045,15173,2136,14854,17112,15712,509,11995,7722,16236,240,539,11907,11178,2124,15351,1776,15347,1279,15172,4511,2126,15609,3975,24970,15043,16239,25663,24867,4793,272,2140,2139,7653,3160,2020,15336,1673,3995,21112,300,507,14819,11561,250,3128,14859,3068,3663,3664,7736,3540,17329,3777,3778,5396,3571,25155,1850,283,6762,15586,3738,3588,3591,3296,3694,12138,2868,6332,3595,11470,868,12704,16241,18599,3505,2951,12136],["basement membrane","anatomical structure development","developmental process","multicellular organism development","system development","preribosome","extracellular matrix structural constituent","collagen-containing extracellular matrix","rRNA processing","rRNA metabolic process","ncRNA metabolic process","extracellular matrix structural constituent conferring tensile strength","growth factor binding","cell junction organization","intracellular organelle lumen","membrane-enclosed lumen","organelle lumen","ribosome biogenesis","anatomical structure morphogenesis","extracellular matrix","external encapsulating structure","ncRNA processing","regulation of developmental process","endoplasmic reticulum lumen","multicellular organismal process","maturation of SSU-rRNA","maturation of SSU-rRNA from tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)","ribosomal small subunit biogenesis","protein binding","collagen trimer","extracellular matrix organization","extracellular structure organization","external encapsulating structure organization","tube development","complex of collagen trimers","nervous system development","integrin-mediated signaling pathway","small-subunit processome","ECM-receptor interaction","extracellular exosome","vasculature development","90S preribosome","extracellular vesicle","extracellular membrane-bounded organelle","extracellular organelle","vesicle","tube morphogenesis","cell adhesion","anatomical structure formation involved in morphogenesis","biological adhesion","cell junction assembly","blood vessel development","extracellular space","neurogenesis","circulatory system development","animal organ development","regulation of cellular process","carbohydrate derivative binding","generation of neurons","ATP-dependent activity","positive regulation of cellular process","collagen fibril organization","ATP hydrolysis activity","platelet-derived growth factor binding","helicase activity","nucleoplasm","regulation of multicellular organismal development","Protein digestion and absorption","ATP-dependent activity, acting on DNA","ion binding","cell migration","endomembrane system","cell population proliferation","non-membrane-bounded organelle","intracellular non-membrane-bounded organelle","organelle","regulation of cellular component organization","hemidesmosome assembly","blood vessel morphogenesis","ATP binding","cytoplasm","endoderm development","endodermal cell differentiation","insulin-like growth factor binding","collagen-activated tyrosine kinase receptor signaling pathway","heparin binding","animal organ morphogenesis","endoplasmic reticulum","adenyl ribonucleotide binding","structural molecule activity","tissue development","embryo development","adenyl nucleotide binding","glycosaminoglycan binding","synapse organization","cellular component organization or biogenesis","transmembrane receptor protein tyrosine kinase activity","intracellular membrane-bounded organelle","nuclear lumen","localization of cell","cell motility","laminin complex","positive regulation of biological process","head development","regulation of multicellular organismal process","endoderm formation","sulfur compound binding","cell differentiation","regulation of nervous system development","extracellular region","zonula adherens","catalytic activity, acting on a nucleic acid","collagen-activated signaling pathway","membrane-bounded organelle","regulation of synapse organization","cardiac chamber development","regulation of synapse structure or activity","protein tyrosine kinase activity","cellular developmental process","transmembrane receptor protein kinase activity","intracellular organelle","synaptic assembly at neuromuscular junction","nucleoside-triphosphatase activity","cellular response to oxygen-containing compound","tissue morphogenesis","regulation of synapse assembly","negative regulation of stem cell population maintenance","organic substance biosynthetic process","regulation of cell communication","laminin-5 complex","laminin-11 complex","laminin-10 complex","regulation of signaling","cell-cell signaling","ATP-dependent activity, acting on RNA","regulation of biological process","integrin binding","semaphorin receptor activity","positive regulation of cell cycle process","nuclear envelope lumen","formation of primary germ layer","cell development","regulation of cell population proliferation","collagen type IV trimer","cell periphery","negative regulation of cellular process","cell surface receptor signaling pathway","biosynthetic process","macromolecule biosynthetic process","neuron differentiation","pyrophosphatase activity","negative regulation of cell death","hydrolase activity, acting on acid anhydrides","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","ribonucleotide binding","negative regulation of cell population proliferation","regulation of cell junction assembly","cardiac septum development","intracellular anatomical structure","anion binding","positive regulation of developmental process","transferase activity, transferring phosphorus-containing groups","network-forming collagen trimer","collagen network","brain development","glutamatergic synapse","negative regulation of programmed cell death","movement of cell or subcellular component","purine ribonucleoside triphosphate binding","basement membrane collagen trimer","locomotion","metalloendopeptidase activity","cellular biosynthetic process","positive regulation of synapse assembly","cellular response to chemical stress","15-hydroxyprostaglandin dehydrogenase (NAD+) activity","prostaglandin transmembrane transporter activity","regulation of programmed cell death"],["3.8e-14","1.4e-10","2.5e-10","2.9e-10","1.6e-09","4.7e-09","6.0e-09","7.7e-09","2.2e-08","3.5e-08","4.9e-08","8.3e-08","2.5e-07","2.7e-07","3.5e-07","3.5e-07","3.5e-07","3.7e-07","4.2e-07","5.0e-07","5.1e-07","1.1e-06","1.5e-06","1.5e-06","2.0e-06","2.7e-06","4.8e-06","5.2e-06","6.4e-06","6.5e-06","1.5e-05","1.6e-05","1.7e-05","2.0e-05","2.1e-05","2.2e-05","2.7e-05","2.9e-05","3.6e-05","4.9e-05","4.9e-05","5.7e-05","6.0e-05","6.1e-05","6.1e-05","6.1e-05","7.2e-05","8.9e-05","9.1e-05","9.8e-05","1.3e-04","1.4e-04","1.9e-04","2.3e-04","2.8e-04","3.0e-04","3.0e-04","3.4e-04","3.5e-04","3.8e-04","4.5e-04","4.6e-04","4.8e-04","5.6e-04","5.8e-04","7.4e-04","7.6e-04","8.8e-04","8.8e-04","9.5e-04","9.7e-04","1.0e-03","1.1e-03","1.3e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.5e-03","1.6e-03","2.2e-03","2.3e-03","2.4e-03","2.5e-03","2.6e-03","2.6e-03","2.7e-03","2.8e-03","3.1e-03","3.1e-03","3.3e-03","3.3e-03","3.5e-03","4.4e-03","4.5e-03","4.7e-03","4.7e-03","4.7e-03","4.7e-03","4.9e-03","4.9e-03","5.0e-03","5.2e-03","5.3e-03","5.4e-03","6.0e-03","6.3e-03","6.3e-03","6.7e-03","7.4e-03","7.4e-03","7.4e-03","7.9e-03","7.9e-03","8.8e-03","9.4e-03","9.9e-03","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.3e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.3e-02","2.4e-02","2.4e-02","2.4e-02","2.4e-02","2.5e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.9e-02","3.0e-02","3.0e-02","3.0e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.4e-02","3.6e-02","3.6e-02","3.6e-02","4.0e-02","4.1e-02","4.1e-02","4.2e-02","4.4e-02","4.4e-02","4.6e-02","4.7e-02","4.7e-02","4.7e-02"],[97,5718,6326,5114,4784,79,173,421,225,263,536,41,137,687,6569,6570,6570,302,2685,560,561,399,2421,311,7474,50,37,73,14799,93,323,324,326,1047,21,2428,112,39,88,2110,724,27,2132,2133,2133,3942,849,1480,1123,1487,420,693,3353,1639,1089,3494,11067,2288,1519,550,5561,71,316,11,160,4161,1315,103,108,6060,1454,4674,1940,6362,6362,16674,2246,5,608,1502,12193,81,46,29,12,170,1066,2007,1565,980,1912,1088,1576,242,422,7994,126,14514,5442,1645,1645,12,6094,812,2623,55,265,4128,438,4303,9,607,17,15557,209,167,215,144,4210,145,15870,7,656,1164,638,103,7,5869,3313,3,3,3,3327,1646,78,12142,147,12,235,10,121,2079,1648,7,6181,4818,2783,5956,4880,1364,712,971,715,715,1934,707,200,106,17758,2410,1271,954,8,8,765,332,883,2074,1848,9,1821,110,5794,63,322,2,2,1439],[14,63,66,89,84,12,12,16,18,19,26,6,11,24,106,106,106,19,55,16,16,21,34,9,71,9,8,10,188,6,9,9,9,29,4,33,9,7,9,27,19,6,27,27,27,37,25,27,29,27,16,18,34,36,23,58,96,33,34,10,79,5,8,3,7,69,30,6,6,53,32,40,38,69,69,189,42,3,19,22,150,6,5,4,3,9,18,23,22,19,36,26,22,4,5,112,7,170,81,33,33,3,81,18,30,5,4,54,15,36,3,16,3,178,4,9,4,7,48,7,180,2,9,13,15,5,3,66,50,2,2,2,50,18,4,86,6,3,9,3,6,37,20,2,44,67,27,66,57,28,9,16,9,9,23,12,9,7,150,29,18,19,2,2,18,11,15,23,22,2,33,2,64,4,9,1,1,24]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4 class="tabset tabset-fade">Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/WCE_CDDP_vs_WCE_NT.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each
protein-protein interaction is annotated with one or more ‘scores’.
Importantly, these scores do <strong>not</strong> indicate the strength
or the specificity of the interaction. Instead, they are indicators of
<strong>confidence</strong>, i.e. how likely STRING judges an
interaction to be true, given the available evidence. All scores rank
from 0 to 1, with 1 being the highest possible confidence. A score of
0.5 would indicate that roughly every second interaction might be
erroneous (i.e., a false positive).</p>
<pre class="r"><code># if(leng_dep != 0){
#   
#   string_db &lt;- STRINGdb$new(version=&quot;11&quot;, species=9606, score_threshold=400)
# 
#   string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)
# 
#   hits &lt;- string_mapped$STRING_id[1:leng_dep]
# 
#   string_db$plot_network(hits, add_link = T)
# }else{
#   print(&quot;NOT enough data.&quot;)
# }</code></pre>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex
networks and integrating these with any type of attribute data. In this
presentation all proteins showing at least one confidence level from the
STRING database are shown. The <strong>color</strong> code is relative
to <strong>fold change</strong> and the <strong>size</strong> is
directly proportional to the <strong>significance
(p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(&quot;assets/network_FC_pvalue.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network.
Instead of nodes representing genes, nodes represent pathways or
functions. Edges between these pathways or functions represent shared
genes or pathway crosstalk. An enrichment map is a way to visualize your
enrichment results to help reduce redundancy and uncover main
themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(&quot;assets/network_enrichmentmap.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
</div>
<div id="section" class="section level2 unnumbered">
<h2 class="unnumbered"></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.8                 purrr_0.3.4                
 [5] readr_2.1.2                 tidyr_1.2.0                
 [7] tibble_3.1.6                tidyverse_1.3.1            
 [9] openxlsx_4.2.5              here_1.0.1                 
[11] RColorBrewer_1.1-2          RCy3_2.10.2                
[13] STRINGdb_2.2.2              naniar_0.6.1               
[15] visdat_0.5.3                ggrepel_0.9.1              
[17] ggplot2_3.3.5               UpSetR_1.4.0               
[19] ggstatsplot_0.9.1           datapasta_3.1.0            
[21] patchwork_1.1.1             DT_0.21                    
[23] rrvgo_1.2.0                 org.Hs.eg.db_3.12.0        
[25] AnnotationDbi_1.52.0        gprofiler2_0.2.1           
[27] enrichR_3.0                 preprocessCore_1.52.1      
[29] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[31] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[33] IRanges_2.24.1              S4Vectors_0.28.1           
[35] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[37] matrixStats_0.61.0          DEP_1.12.0                 
[39] MBQN_2.2.0                  workflowr_1.7.0            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3         R.methodsS3_1.8.1      bit64_4.0.5           
  [4] knitr_1.37             DelayedArray_0.16.2    R.utils_2.11.0        
  [7] data.table_1.14.2      RCurl_1.98-1.6         doParallel_1.0.17     
 [10] generics_0.1.2         callr_3.7.0            RSQLite_2.2.10        
 [13] correlation_0.8.0      chron_2.3-56           bit_4.0.4             
 [16] tzdb_0.2.0             lubridate_1.8.0        xml2_1.3.3            
 [19] httpuv_1.6.5           assertthat_0.2.1       WRS2_1.1-3            
 [22] xfun_0.30              hms_1.1.1              jquerylib_0.1.4       
 [25] evaluate_0.15          promises_1.2.0.1       fansi_1.0.2           
 [28] readxl_1.3.1           caTools_1.18.2         dbplyr_2.1.1          
 [31] igraph_1.2.11          DBI_1.1.2              htmlwidgets_1.5.4     
 [34] reshape_0.8.8          paletteer_1.4.0        hash_2.2.6.1          
 [37] ellipsis_0.3.2         crosstalk_1.2.0        backports_1.4.1       
 [40] insight_0.16.0         gridBase_0.4-7         vctrs_0.3.8           
 [43] imputeLCMD_2.0         Cairo_1.5-14           cachem_1.0.6          
 [46] withr_2.5.0            treemap_2.4-3          cluster_2.1.0         
 [49] lazyeval_0.2.2         crayon_1.5.1           labeling_0.4.2        
 [52] pkgconfig_2.0.3        slam_0.1-50            statsExpressions_1.3.0
 [55] ProtGenerics_1.22.0    wordcloud_2.6          rlang_1.0.2           
 [58] RJSONIO_1.3-1.6        lifecycle_1.0.1        sandwich_3.0-1        
 [61] affyio_1.60.0          BiocFileCache_1.14.0   modelr_0.1.8          
 [64] cellranger_1.1.0       rprojroot_2.0.2        graph_1.68.0          
 [67] datawizard_0.3.0       Matrix_1.2-18          mc2d_0.1-21           
 [70] zoo_1.8-9              reprex_2.0.1           whisker_0.4           
 [73] GlobalOptions_0.1.2    processx_3.5.2         pheatmap_1.0.12       
 [76] png_0.1-7              viridisLite_0.4.0      rjson_0.2.21          
 [79] mzR_2.24.1             parameters_0.16.0      bitops_1.0-7          
 [82] shinydashboard_0.7.2   getPass_0.2-2          R.oo_1.24.0           
 [85] KernSmooth_2.23-17     blob_1.2.2             shape_1.4.6           
 [88] tmvtnorm_1.4-10        scales_1.1.1           memoise_2.0.1         
 [91] magrittr_2.0.3         plyr_1.8.6             gplots_3.1.1          
 [94] zlibbioc_1.36.0        compiler_4.0.3         plotrix_3.8-2         
 [97] pcaMethods_1.82.0      clue_0.3-60            cli_3.2.0             
[100] affy_1.68.0            XVector_0.30.0         ps_1.6.0              
[103] MASS_7.3-53            tidyselect_1.1.2       vsn_3.58.0            
[106] stringi_1.6.2          highr_0.9              yaml_2.2.1            
[109] GOSemSim_2.16.1        norm_1.0-9.5           MALDIquant_1.21       
[112] grid_4.0.3             sass_0.4.1             tools_4.0.3           
[115] circlize_0.4.14        rstudioapi_0.13        foreach_1.5.2         
[118] git2r_0.29.0           gridExtra_2.3          farver_2.1.0          
[121] mzID_1.28.0            digest_0.6.29          BiocManager_1.30.16   
[124] shiny_1.7.1            proto_1.0.0            Rcpp_1.0.8.3          
[127] broom_0.7.12           performance_0.8.0      later_1.3.0           
[130] ncdf4_1.19             httr_1.4.2             MSnbase_2.15.7        
[133] ComplexHeatmap_2.6.2   colorspace_2.0-3       rvest_1.0.2           
[136] XML_3.99-0.9           fs_1.5.2               rematch2_2.1.2        
[139] gmm_1.6-6              plotly_4.10.0          xtable_1.8-4          
[142] jsonlite_1.8.0         zeallot_0.1.0          NLP_0.2-1             
[145] R6_2.5.1               tm_0.7-8               gsubfn_0.7            
[148] pillar_1.7.0           htmltools_0.5.2        mime_0.12             
[151] glue_1.6.2             fastmap_1.1.0          BiocParallel_1.24.1   
[154] codetools_0.2-16       mvtnorm_1.1-3          utf8_1.2.2            
[157] lattice_0.20-41        bslib_0.3.1            sqldf_0.4-11          
[160] curl_4.3.2             gtools_3.9.2           zip_2.2.0             
[163] GO.db_3.12.1           limma_3.46.0           rmarkdown_2.12        
[166] munsell_0.5.0          GetoptLong_1.0.5       GenomeInfoDbData_1.2.4
[169] iterators_1.0.14       impute_1.64.0          haven_2.4.3           
[172] gtable_0.3.0           bayestestR_0.11.5     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
